---
output: github_document
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  fig.path = "man/figures/README-",
  out.width = "100%"
)
```

# TimeSeriesAnalysis

<!-- badges: start -->
<!-- badges: end -->

TimeSeriesAnalysis is a transcriptomic analysis tool for both RNA sequencing and microarray data

## Overview
TimeSeriesAnalysis (TSA) is an analysis and visualization package for RNAseq and
microarray data. TS extracts significant genes from time course
transcriptomic data by performing differential gene expression on both
the conditional and temporal axes. It then employs partitioning
algorithm based on recursive thresholding (PART) clustering to identify
small genomic clusters of relevance, followed by running the clusters
through gprofiler to reveal the biological relevance of each cluster.


MAKE INTO LIST
TS performs: data normalization and processing PCA plots Differential
gene expression (conditional and temporal) PART clustering Heatmaps
Trajectory of identified clusters Gprofiler (functional enrichment)
analysis of clusters Dotplots and MDS plots of Gprofiler results Nearest
ancestor clustering of GOs GO ancestor queries


## Installation

You can install the development version of TimeSeriesAnalysis like so:

``` r
install.packages("devtools")
devtools::install_github("Ylefol/TimeSeriesAnalysis")
```

Certain bioconductor packages may have to be installed before
installation of TimeSeriesAnalysis. This can be done with the following
code snippet:

``` r
if (!require("BiocManager", quietly = TRUE))
  install.packages("BiocManager")
bio_pkgs <- c(PACKAGES HERE)
BiocManager::install(bio_pkgs)
```

## Rmarkdown format

TimeSeriesAnalysis was developed with user-friendliness in mind, the
core script of the package is a Rmarkdown file with explanations of what
each code block performs. Computationally intensive code blocks save
their work in order to avoid loss of computation time.
TimeSeriesAnalysis comes with a test dataset. Users are recommended to
first run the package with the test data to then modify the Rmarkdown
script with the necessary information for their purposes (input files,
organism, genes of interest etc…). The rmarkdown script and the expected
result from an example run can be found and downloaded from this
repository, they are located in the ‘rmarkdown_method’ folder. If users
prefer a script approach, the equivalent of the rmarkdown report is
provided in two scripts, one for the computation tasks and a second for
the analyses. These two scripts are found in the ‘script_method’ folder.

## Data

Three different datasets have been used as example datasets for this pipeline,
all while this pipeline serves as a platform to publish and share these datasets.

The PBMC dataset is a time series experiment with three time points that explores 
and compares three AICDA/AID (Activation Induced Cytidine DeAminase) activation cocktails.
The experiment seeked to identify which activation cocktail properly activated AID
through both the expression of the AID gene and the activation of class switch recombination.

The MURINE dataset is another AID related time series experiment, where mice were generated and
one was given an AID activation cocktail. The experiment contains one replicate per condition and
10 timepoints. Due to only having one replicate per condition per timepoint, subsequent timepoints
were combined to create an intermediary timepoint with two replicates in each condition.

The Celegans dataset is an ageing related experiment. Two celegans models were used: BY372 and N2.
The first being a parkinsons model and the second being a wild type. Each model was split into
two groups, one that would be fed krill oil and the other not (control). Each experiment
has three timepoints: Days 1, 3, and 6.

## Tutorial

A tutorial can be found within the [pipelines website](https://ylefol.github.io/TimeSeriesAnalysis/). 
Tutorials using all three data sets can be found in the examples tab of the website. Additionally the 
documentation can be found in the references tab.

## Publication

An manuscript detailing the TimeSeriesAnalysis pipeline has been submitted for publication. Both the ReadMe and
the website will be updated when updates are available.
