<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>TimeSeries Analysis – PBMC example run • TimeSeriesAnalysis</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/spacelab/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="TimeSeries Analysis – PBMC example run">
<meta property="og:description" content="TimeSeriesAnalysis">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TimeSeriesAnalysis</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">01.01.01</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Examples
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/TS_analysis_PBMC.html">PBMC analysis</a>
    </li>
  </ul>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/Ylefol/TimeSeriesAnalysis/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="TS_analysis_PBMC_files/htmlwidgets-1.6.2/htmlwidgets.js"></script><script src="TS_analysis_PBMC_files/plotly-binding-4.10.2/plotly.js"></script><script src="TS_analysis_PBMC_files/typedarray-0.1/typedarray.min.js"></script><link href="TS_analysis_PBMC_files/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="TS_analysis_PBMC_files/crosstalk-1.2.0/js/crosstalk.min.js"></script><link href="TS_analysis_PBMC_files/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet">
<script src="TS_analysis_PBMC_files/plotly-main-2.11.1/plotly-latest.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>TimeSeries Analysis – PBMC example run</h1>
                        <h4 data-toc-skip class="author">Yohan
Lefol</h4>
            
            <h4 data-toc-skip class="date">28. June 2023</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/Ylefol/TimeSeriesAnalysis/blob/HEAD/vignettes/TS_analysis_PBMC.Rmd" class="external-link"><code>vignettes/TS_analysis_PBMC.Rmd</code></a></small>
      <div class="hidden name"><code>TS_analysis_PBMC.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="timeseries-analysis">TimeSeries analysis<a class="anchor" aria-label="anchor" href="#timeseries-analysis"></a>
</h2>
<div class="section level3">
<h3 id="parameter-set-up">Parameter set up<a class="anchor" aria-label="anchor" href="#parameter-set-up"></a>
</h3>
<p>Set the below parameters for the downstream analysis</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Give names to saved object and name of results repository</span></span>
<span><span class="va">name_result_folder</span><span class="op">&lt;-</span><span class="st">'TS_results_PBMC_example/'</span></span>
<span><span class="va">obj_name</span><span class="op">&lt;-</span><span class="st">'timeSeries_obj_PBMC_example.Rdata'</span></span>
<span></span>
<span><span class="co">#Set-up time series object parameters</span></span>
<span><span class="va">diff_exp_type</span><span class="op">&lt;-</span><span class="st">'DESeq2'</span> <span class="co">#package used for DE – can also be 'limma'</span></span>
<span><span class="va">p_val_filter_type</span><span class="op">&lt;-</span><span class="st">'padj'</span> <span class="co">#Either padj or pvalue, used to filter for significance</span></span>
<span><span class="va">p_thresh</span><span class="op">&lt;-</span><span class="fl">0.05</span> <span class="co">#pvalue or padj value threshold for significance</span></span>
<span><span class="va">l2fc_thresh</span><span class="op">&lt;-</span><span class="fl">1</span> <span class="co">#log(2)foldChange threshold for significance</span></span>
<span><span class="va">name_control</span><span class="op">&lt;-</span><span class="st">'LPS'</span> <span class="co">#Name of experiment as seen in the sample file</span></span>
<span><span class="va">name_experiment</span><span class="op">&lt;-</span><span class="st">'IgM'</span> <span class="co">#Name of control as seen in the sample file</span></span>
<span><span class="va">graphic_vector</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"#e31a1c"</span>,<span class="st">"#1f78b4"</span><span class="op">)</span> <span class="co">#Pre-set colors for the groups</span></span>
<span></span>
<span><span class="co">#Declare organism and load library</span></span>
<span><span class="va">org_sem_sim</span><span class="op">=</span><span class="st">'org.Hs.eg.db'</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st">'org.Hs.eg.db'</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Loading required package: AnnotationDbi</span></span></code></pre>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Define specimen and ontology parameters</span></span>
<span><span class="va">my_ont_gpro</span><span class="op">=</span><span class="st">'GO:BP'</span></span>
<span><span class="va">my_ont_sem_sim</span><span class="op">=</span><span class="st">'BP'</span></span>
<span></span>
<span><span class="va">my_org_gpro</span><span class="op">=</span><span class="st">'hsapiens'</span> <span class="co">#Set the species for the gprofiler analysis</span></span>
<span></span>
<span><span class="co"># The seed serves to create reproducible results with PART.</span></span>
<span><span class="co"># A seed will ensure that the random components of PART clustering will be the same</span></span>
<span><span class="co"># as long as the same seed is used. For more information on seeds, please consult</span></span>
<span><span class="co"># this link: https://www.r-bloggers.com/2018/07/%F0%9F%8C%B1-setting-a-seed-in-r-when-using-parallel-simulation/</span></span>
<span><span class="va">PART_seed</span><span class="op">=</span><span class="fl">123456</span></span>
<span></span>
<span><span class="va">PART_l2fc</span><span class="op">&lt;-</span><span class="fl">2</span> <span class="co">#log(2)foldChange threshold for PART clustering</span></span>
<span><span class="va">PART_min_clust</span><span class="op">&lt;-</span><span class="fl">50</span> <span class="co">#Minimum cluster size for PART</span></span>
<span><span class="va">PART_recursion</span><span class="op">&lt;-</span><span class="fl">100</span> <span class="co">#Number of recursions, default is 100, using 10 for example</span></span>
<span></span>
<span><span class="va">log_tp_traj</span><span class="op">&lt;-</span><span class="cn">FALSE</span> <span class="co">#Defines if timepoints should be log transformed for illustration purposes</span></span>
<span></span>
<span><span class="co"># Allows for all temporal combinations to be done instead</span></span>
<span><span class="co"># of just sequential comparison. ex: do TP2vsTP1, TP3vsTP2, AND TP3vsTP1. In a normal instance</span></span>
<span><span class="co"># only the first two comparison of the example would be run.</span></span>
<span><span class="va">do_all_temporal_comparisons</span><span class="op">=</span><span class="cn">FALSE</span></span>
<span></span>
<span><span class="co">#Used to highlight specific genes regardless of differential gene expression significance</span></span>
<span><span class="va">genes_of_interest</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'AICDA'</span>,<span class="st">'APOBEC3H'</span>,<span class="st">'APOBEC3F'</span>,<span class="st">'APOBEC3D'</span>,<span class="st">'APOBEC3C'</span>,<span class="st">'APOBEC3G'</span>,<span class="st">'APOBEC3B'</span>,<span class="st">'APOBEC3A'</span>,<span class="st">'SMUG1'</span>,<span class="st">'UNG'</span>,<span class="st">'EGFR'</span><span class="op">)</span></span>
<span></span>
<span><span class="co">#The ancestors that will be queried, the ontology must be specified (BP,MF,or CC)</span></span>
<span><span class="co">#Set to an empty vector c() if not required by the analysis</span></span>
<span><span class="va">target_ancestors</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'GO:0002253'</span>,<span class="st">'GO:0019882'</span>,<span class="st">'GO:0002404'</span>,<span class="st">'GO:0002339'</span>,<span class="st">'GO:0042386'</span>,</span>
<span>                    <span class="st">'GO:0035172'</span>,<span class="st">'GO:0002252'</span>,<span class="st">'GO:0006955'</span>,<span class="st">'GO:0002520'</span>,<span class="st">'GO:0090713'</span>,</span>
<span>                    <span class="st">'GO:0045321'</span>,<span class="st">'GO:0001776'</span>,<span class="st">'GO:0050900'</span>,<span class="st">'GO:0031294'</span>,<span class="st">'GO:0002262'</span>,</span>
<span>                    <span class="st">'GO:0002683'</span>,<span class="st">'GO:0002684'</span>,<span class="st">'GO:0002440'</span>,<span class="st">'GO:0002682'</span>,<span class="st">'GO:0002200'</span>,</span>
<span>                    <span class="st">'GO:0045058'</span>,<span class="st">'GO:0002507'</span><span class="op">)</span></span>
<span><span class="va">ancestor_ontology</span><span class="op">&lt;-</span><span class="st">'BP'</span></span>
<span></span>
<span><span class="co">#Some extra set-up</span></span>
<span><span class="va">name_save_obj</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="va">obj_name</span><span class="op">)</span><span class="co">#The object will be saved in result folder</span></span>
<span><span class="co">#Create main directory for results, removed for vignettes</span></span>
<span><span class="co"># dir.create(name_result_folder)</span></span>
<span></span>
<span><span class="va">my_group_names</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">name_experiment</span>,<span class="va">name_control</span><span class="op">)</span> </span>
<span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">graphic_vector</span><span class="op">)</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">name_experiment</span>,<span class="va">name_control</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="create-timeseries-object">Create TimeSeries object<a class="anchor" aria-label="anchor" href="#create-timeseries-object"></a>
</h3>
<p>Using the parameter above, generate a TimeSeries_Object and load the
example data as well as the semantic similarity data</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/methods/new.html" class="external-link">new</a></span><span class="op">(</span><span class="st">'TimeSeries_Object'</span>,</span>
<span>                 group_names<span class="op">=</span><span class="va">my_group_names</span>,group_colors<span class="op">=</span><span class="va">graphic_vector</span>,DE_method<span class="op">=</span><span class="va">diff_exp_type</span>,</span>
<span>                 DE_p_filter<span class="op">=</span><span class="va">p_val_filter_type</span>,DE_p_thresh<span class="op">=</span><span class="va">p_thresh</span>,DE_l2fc_thresh<span class="op">=</span><span class="va">l2fc_thresh</span>,</span>
<span>                 PART_l2fc_thresh<span class="op">=</span><span class="va">PART_l2fc</span>,sem_sim_org<span class="op">=</span><span class="va">org_sem_sim</span>,Gpro_org<span class="op">=</span><span class="va">my_org_gpro</span><span class="op">)</span></span>
<span><span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TS_load_example_data.html">TS_load_example_data</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span></span>
<span><span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/add_semantic_similarity_data.html">add_semantic_similarity_data</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">my_ont_sem_sim</span>,vignette_run<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="differental-gene-expression-analysis">Differental gene expression analysis<a class="anchor" aria-label="anchor" href="#differental-gene-expression-analysis"></a>
</h3>
<p>Loads the necessary functions to perform differential gene expression
analysis If the tool used is DESeq2<span class="citation">(Love, Huber,
and Anders 2014)</span>, the data needs to be normalized using it’s
method. If the tool used is limma<span class="citation">(Smyth
2005)</span>, the normalized matrix should have been inputed, and thus
no normalization is needed.</p>
<p>This code chunk performs both the conditional and temporal
differential gene expression and saves the results within the Time
Series object.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Perform normalization if the DESeq2 tool is being used and if normalized matrix doesn't exist</span></span>
<span><span class="kw">if</span> <span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'DE_method'</span><span class="op">)</span><span class="op">==</span><span class="st">'DESeq2'</span> <span class="op">&amp;</span> <span class="st">'norm'</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="fu">assays</span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'exp_data'</span><span class="op">)</span><span class="op">)</span><span class="op">)</span><span class="op">!=</span><span class="cn">TRUE</span><span class="op">)</span><span class="op">{</span></span>
<span>  <span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/normalize_timeSeries_with_deseq2.html">normalize_timeSeries_with_deseq2</a></span><span class="op">(</span>time_object<span class="op">=</span><span class="va">TS_object</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span></span>
<span><span class="co">#Perform conditional differential gene expression analysis</span></span>
<span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/conditional_DE_wrapper.html">conditional_DE_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span></span>
<span><span class="co">#Perform temporal differential gene expression analysis</span></span>
<span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/temporal_DE_wrapper.html">temporal_DE_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span>,do_all_combinations<span class="op">=</span><span class="va">do_all_temporal_comparisons</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="part-clustering">PART clustering<a class="anchor" aria-label="anchor" href="#part-clustering"></a>
</h3>
<p>The code chunk below prepares and initiates PART clustering<span class="citation">(Nilsen et al. 2013)</span>. It first retrieves the
number of significant genes for PART clustering in the ‘signi_genes’
variable This chunk can be quite lengthy depending on the number of
genes included for clustering as well as the number of recursions set
for the clustering</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Extract genes for PART clustering based on defined log(2)foldChange threshold</span></span>
<span><span class="va">signi_genes</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/select_genes_with_l2fc.html">select_genes_with_l2fc</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span></span>
<span></span>
<span><span class="co">#Use all samples, but implement a custom order. In this case it is reversed</span></span>
<span><span class="va">samp_dta</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/exp_sample_data.html">exp_sample_data</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span></span>
<span><span class="va">TS_groups</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'group_names'</span><span class="op">)</span></span>
<span><span class="va">samps_2</span><span class="op">&lt;-</span><span class="va">samp_dta</span><span class="op">$</span><span class="va">sample</span><span class="op">[</span><span class="va">samp_dta</span><span class="op">$</span><span class="va">group</span><span class="op">==</span><span class="va">TS_groups</span><span class="op">[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="va">samps_1</span><span class="op">&lt;-</span><span class="va">samp_dta</span><span class="op">$</span><span class="va">sample</span><span class="op">[</span><span class="va">samp_dta</span><span class="op">$</span><span class="va">group</span><span class="op">==</span><span class="va">TS_groups</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span></span>
<span></span>
<span><span class="co">#Create the matrix that will be used for PART clustering</span></span>
<span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/prep_counts_for_PART.html">prep_counts_for_PART</a></span><span class="op">(</span>object<span class="op">=</span><span class="va">TS_object</span>,target_genes<span class="op">=</span><span class="va">signi_genes</span>,</span>
<span>                                scale<span class="op">=</span><span class="cn">TRUE</span>,target_samples<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">samps_2</span>,<span class="va">samps_1</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="co">#Sets a seed for reproducibility</span></span>
<span><span class="kw">if</span> <span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/NULL.html" class="external-link">is.null</a></span><span class="op">(</span><span class="va">PART_seed</span><span class="op">)</span><span class="op">==</span><span class="cn">FALSE</span><span class="op">)</span><span class="op">{</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/base/Random.html" class="external-link">set.seed</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/character.html" class="external-link">as.character</a></span><span class="op">(</span><span class="va">PART_seed</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/compute_PART.html">compute_PART</a></span><span class="op">(</span><span class="va">TS_object</span>,part_recursion<span class="op">=</span><span class="va">PART_recursion</span>,part_min_clust<span class="op">=</span><span class="va">PART_min_clust</span>,</span>
<span>                        custom_seed<span class="op">=</span><span class="va">PART_seed</span>,dist_param<span class="op">=</span><span class="st">"euclidean"</span>, hclust_param<span class="op">=</span><span class="st">"average"</span>,</span>
<span>                        vignette_run<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="gprofiler-analysis">Gprofiler analysis<a class="anchor" aria-label="anchor" href="#gprofiler-analysis"></a>
</h3>
<p>The code chunk below runs a gprofiler analysis<span class="citation">(Kolberg et al. 2020; Raudvere et al. 2019)</span>.
IMPORTANT: This function requires that there be a stable internet
connection lack of connection or intermittent drops in the connection
will result in an error and the termination (stop) of the code chunk</p>
<p>If an error has occured, this code chunk can be re-run separately
from the above chunks by uncommenting (removing the ‘#’ in front) the
“load(‘timeseries_obj_res.Rdata’)” line.</p>
<p>This will load the results saved after the PART clustering code
chunk</p>
<p>This code chunk will overwrite the saved object if it is completed.
The overwritten object will then contain the gprofiler analysis results
and can be used to generate plots with the downstream code chunks.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/run_gprofiler_PART_clusters.html">run_gprofiler_PART_clusters</a></span><span class="op">(</span><span class="va">TS_object</span>,vignette_run <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="co">#Run the gprofiler analysis</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="timeseries-analysis-results">TimeSeries analysis results<a class="anchor" aria-label="anchor" href="#timeseries-analysis-results"></a>
</h2>
<p>Most plots are created in SVG format for ease of editing with SVG
editing software such as InkScape (open source software). Some plots are
created in html format as they are interactive plots. These require a
web browser to open them</p>
<p>To convert SVG files to PNG/JPG/PDF, this website is available: <a href="https://svgtopng.com/" class="external-link uri">https://svgtopng.com/</a> HTML
files can be opened and then saved as PNG (or other) using the camera
icon in the top right of each respective interactive plot</p>
<div class="section level3">
<h3 id="timeseries-summary-of-experiment">TimeSeries summary of experiment<a class="anchor" aria-label="anchor" href="#timeseries-summary-of-experiment"></a>
</h3>
<p><strong>Name of groups beings compared:</strong> IgM vs LPS
<strong>Number of genes analyzed:</strong> 26485</p>
<p><strong>Differential expression parameters used:</strong>
<strong>Method:</strong> DESeq2 <strong>P statistic filter
used:</strong> padj with a 0.05 threshold <strong>Log(2)FoldChange
significance threshold used:</strong> 1</p>
<p><strong>PART parameters used:</strong> <strong>PART log(2)FoldChange
significance threshold:</strong> 2 <strong>number of
recursions:</strong> 100 <strong>minimum cluster size:</strong> 50
<strong>distance parameter:</strong> euclidean <strong>hclust
parameter:</strong> average <strong>custom seed used:</strong> 123456
<strong>PART computation time:</strong> 686.819 sec elapsed</p>
<p><strong>PCA of sample data:</strong>
<img src="TS_analysis_PBMC_files/figure-html/show_PCA_clust%20-1.png" width="100%"></p>
</div>
<div class="section level3">
<h3 id="differential-gene-expression-results-as-per-plot_wrapper-functions">Differential Gene Expression results (as per plot_wrapper
functions)<a class="anchor" aria-label="anchor" href="#differential-gene-expression-results-as-per-plot_wrapper-functions"></a>
</h3>
<p><strong>DEGs found per analysis:</strong></p>
<table class="table">
<thead><tr class="header">
<th align="left">experiment.name</th>
<th align="left">number.of.DEGs</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">IgM_vs_LPS_TP_1 (conditional)</td>
<td align="left">1763</td>
</tr>
<tr class="even">
<td align="left">IgM_vs_LPS_TP_3 (conditional)</td>
<td align="left">1369</td>
</tr>
<tr class="odd">
<td align="left">IgM_vs_LPS_TP_9 (conditional)</td>
<td align="left">1568</td>
</tr>
<tr class="even">
<td align="left">TP_3_vs_TP_1 (temporal)</td>
<td align="left">957</td>
</tr>
<tr class="odd">
<td align="left">TP_9_vs_TP_3 (temporal)</td>
<td align="left">86</td>
</tr>
<tr class="even">
<td align="left"><strong>total unique genes</strong></td>
<td align="left"><strong>3884</strong></td>
</tr>
</tbody>
</table>
<p>Each experiment listed in the above table has separate results which
can be viewed in the ‘TS_results’ folder. In the main results folder,
the differential gene expression results are located in the
‘DE_results_conditional’ and ‘DE_results_temporal’ plot. Each individual
experiment creates the following: <strong>DE_raw_data.csv</strong>: A
csv file containing all the genes and their associated differential
expression results, regardless of significance.
<strong>DE_sig_data</strong>: A csv file containing only the significant
differential expression results. <strong>MA_plot.png</strong>: A MA
plot, these are often used to evaluate the quality of the normalization.
<strong>volcano_plot.png</strong>: A volcano plot which splits
significantly up-regulated and down-regulated genes as well as
non-significant genes. <strong>PCA_plot</strong>: A PCA plot using only
the samples implicated in the differential gene expression analysis.</p>
<p>The pipeline also creates two large heatmaps which summarize
conditional and temporal differential expression results. The heatmaps
were created using the ComplexHeatmaps package<span class="citation">(Gu, Eils, and Schlesner 2016)</span>. These heatmaps
illustrate each experiment as different columns (distinguished by color)
while the rows distinguish the groups. Plotted in the heatmaps is the
log transformed counts, while the log transformed log(2)fold change is
seen in a histogram at the bottom of the heatmap. The number of genes in
each experiment (colored columns) is indicated in the legend at the
bottom of the heatmap. This heatmap is shown below. In the case of the
temporal heatmap, the rows are labelled ‘experiment’ and ‘control’. The
‘experiment’ represents the time points being compared while the control
represents the timepoint which the ‘experiment’ is compared against. For
example, TP2_vs_TP1 would have the ‘experiment’ be TP2 and the control
be TP1, while TP3_vs_TP2 would have TP3 as the ‘experiment’ and TP2 as
the ‘control’. Note that TP=time point.</p>
<p><strong>The conditional heatmap</strong>
<img src="TS_analysis_PBMC_files/figure-html/show_cond_heat_clust%20-1.png" width="100%">
## Genes of interest results</p>
<p><img src="TS_analysis_PBMC_files/figure-html/genes_interest-1.png" width="65%" style="float:right; padding:10px">
If genes of interest have been specified, the pipeline will extract the
differential gene expression results, both conditional and temporal, for
each gene requested, provided that they are found in the count matrix.
These data files, along with individual trajectory plots are stored in
the ‘TS_results/genes_of_interest’ folder. Below the trajectory of the
most variable gene is shown. If no genes of interest were submitted, a
random gene is taken from the differential expression results.
Illustrated in the trajectory plots is the timepoints on the x axis and
the number of reads (RNAseq) or expression (microarray) on the y axis.
Note that data files and trajectory plots are created for all genes of
interest found, regardless of their significance based on the
differential gene expression analyses.</p>
</div>
<div class="section level3">
<h3 id="cluster-results">Cluster results<a class="anchor" aria-label="anchor" href="#cluster-results"></a>
</h3>
<p>The code chunk below create the trajectory plots for the clusters
found by PART Trajectories are first calculated by calculating the mean
of each gene for each time point using the available replicates. The
mean value of each gene is calculated once for each group
(control/experiment). Afterwards, the genes are scaled in order to be
able to represent them on the same axis of values. Each gene is plotted
as an individual colored line’, a mean line (gray color) is also plotted
to show the overall trajectory of the cluster. These plots were inspired
by <span class="citation">(Nguyen 2021)</span></p>
<p>The log_tp_traj parameter defines if the time points should be log
transformed. A log transformation of the time points is useful if the
time points are not evenly sampled through time. For example, two time
points at 0 and 15 minutes, with another time point at 24 hours (1440
minutes) will result in the first two time points being very compressed
and possibly indistinguishable from each other.</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># dir.create(paste0(name_result_folder,'PART_results')) #create the directory to store results</span></span>
<span><span class="va">PART_heat</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/PART_heat_map.html">PART_heat_map</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="cn">NULL</span><span class="op">)</span> <span class="co">#Create a summary heatmap, returns plot if save location is NULL</span></span>
<span><span class="va">ts_data</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/calculate_cluster_traj_data.html">calculate_cluster_traj_data</a></span><span class="op">(</span><span class="va">TS_object</span>,scale_feat<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span> <span class="co">#Calculate scaled gene values for genes of clusters</span></span>
<span><span class="va">mean_ts_data</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/calculate_mean_cluster_traj.html">calculate_mean_cluster_traj</a></span><span class="op">(</span><span class="va">ts_data</span><span class="op">)</span> <span class="co">#Calculate the mean scaled values for each cluster</span></span>
<span></span>
<span><span class="co">#Function which determines the number of SVG files to plot all cluster trajectories</span></span>
<span><span class="co">#Function commented out for vignettes</span></span>
<span><span class="co"># wrapper_cluster_trajectory(TS_object,ts_data,mean_ts_data,log_TP=log_tp_traj,plot_name=paste0(name_result_folder,'/PART_results/Ctraj'))</span></span>
<span></span>
<span><span class="co">#find the most variable cluster, subset necessary data and plot it on a single column</span></span>
<span><span class="co">#The function will also filter for a cluster which has gprofiler results</span></span>
<span><span class="va">target_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_most_variable_cluster.html">find_most_variable_cluster</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">mean_ts_data</span><span class="op">)</span></span>
<span><span class="va">sub_ts</span><span class="op">&lt;-</span><span class="va">ts_data</span><span class="op">[</span><span class="va">ts_data</span><span class="op">$</span><span class="va">cluster</span><span class="op">==</span><span class="va">target_clust</span>,<span class="op">]</span></span>
<span><span class="va">sub_mean</span><span class="op">&lt;-</span><span class="va">mean_ts_data</span><span class="op">[</span><span class="va">mean_ts_data</span><span class="op">$</span><span class="va">cluster</span><span class="op">==</span><span class="va">target_clust</span>,<span class="op">]</span></span>
<span><span class="va">plt_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/plot_cluster_traj.html">plot_cluster_traj</a></span><span class="op">(</span>object <span class="op">=</span> <span class="va">TS_object</span>,ts_data <span class="op">=</span> <span class="va">sub_ts</span>,ts_mean_data <span class="op">=</span> <span class="va">sub_mean</span>, num_col<span class="op">=</span><span class="fl">1</span>,rem_legend_axis<span class="op">=</span><span class="cn">TRUE</span>,log_TP<span class="op">=</span><span class="va">log_tp_traj</span><span class="op">)</span></span>
<span><span class="va">cluster_map_sum</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'PART_results'</span><span class="op">)</span><span class="op">$</span><span class="va">cluster_map</span><span class="op">[</span>,<span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">1</span><span class="op">)</span><span class="op">]</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/row.names.html" class="external-link">row.names</a></span><span class="op">(</span><span class="va">cluster_map_sum</span><span class="op">)</span><span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'Cluster name'</span>,<span class="st">'Number of genes'</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/BiocGenerics/man/row_colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">cluster_map_sum</span><span class="op">)</span><span class="op">=</span><span class="cn">NULL</span></span></code></pre></div>
<p><strong>PART result summary:</strong></p>
<table class="table">
<colgroup>
<col width="23%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="4%">
<col width="5%">
<col width="5%">
<col width="4%">
<col width="5%">
<col width="4%">
</colgroup>
<tbody>
<tr class="odd">
<td align="left">Cluster name</td>
<td align="left">C1</td>
<td align="left">C10</td>
<td align="left">C11</td>
<td align="left">C12</td>
<td align="left">C13</td>
<td align="left">C14</td>
<td align="left">C2</td>
<td align="left">C3</td>
<td align="left">C4</td>
<td align="left">C5</td>
<td align="left">C6</td>
<td align="left">C7</td>
<td align="left">C8</td>
<td align="left">C9</td>
</tr>
<tr class="even">
<td align="left">Number of genes</td>
<td align="left">165</td>
<td align="left">56</td>
<td align="left">101</td>
<td align="left">50</td>
<td align="left">126</td>
<td align="left">178</td>
<td align="left">263</td>
<td align="left">162</td>
<td align="left">80</td>
<td align="left">150</td>
<td align="left">115</td>
<td align="left">67</td>
<td align="left">119</td>
<td align="left">59</td>
</tr>
</tbody>
</table>
<p>All cluster results are found in the ‘PART_results’ folder, within
this folder you will find the following, Note that this folder exists if
the a save_location is given to the function and that the function is
used.:</p>
<p><strong>PART_heat.svg</strong>: A svg heatmap of the PART result –
detailed below. <strong>PART_heat_with_names.svg</strong>: The same plot
as above but with sample and gene names included. These may not be
visible depending on the number of genes included as well as the length
of the sample names. <strong>PART_heat_cmap.csv</strong>: A csv file
representing a ‘cluster_map’ which provides the cluster in which each
gene was sorted in as well as the color (in HEX code) given to that
cluster. <strong>PART_heat_data.csv</strong>: The z-score values used in
the ‘PART_heat’ heatmaps. <strong>Ctraj.svg</strong>: At least one, and
possibly several of these files will be in the folder. This/these SVG
plots show the trajectory of each cluster individually while being split
by group. Meaning the trajectory of each cluster is shown twice, once
per group. Information concerning the cluster and group is shown in the
title of each subplot. One example of cluster trajectory is shown
below.</p>
<p><strong>PART heatmap and cluster trajectory</strong> In this heatmap
we illustrate all the genes which were submitted for PART clustering.
Each gene is illustrated using a Z-score, which measures the number of
standard deviations that a gene has from it’s mean. In essence it
measures the variability. Clusters are shown as the colored bar on the
left hand side of the plot while groups and timepoints are shown as the
colored bars on the top of the heatmap. PART heatmaps were created with
ComplexHeatmaps<span class="citation">(Gu, Eils, and Schlesner
2016)</span>. The cluster trajectory of the most variable cluster (C13)
is illustrated, with the trajectory of the experiment group at the top
and the control group at the bottom. The x axis represents timepoints
and the y axis represents the scaled mean expression of the genes. The
expression of each gene is obtained by calculating the mean between all
replicates for each time point in each group. The values are then scaled
in order to illustrate their trajectory. The overall mean of the cluster
is shown as a thicker gray line.</p>
<p><img src="TS_analysis_PBMC_files/figure-html/PART_results,figures-side-1.png" width="49%" height="49%"><img src="TS_analysis_PBMC_files/figure-html/PART_results,figures-side-2.png" width="49%" height="49%"></p>
</div>
<div class="section level3">
<h3 id="gprofiler-results-standard">Gprofiler results – standard<a class="anchor" aria-label="anchor" href="#gprofiler-results-standard"></a>
</h3>
<p>The below chunk creates the gprofiler result. Some parameters
relating to the species and ontology need to be defined the
Multi-Dimensional Scaling (MDS) plots that will be generated by the last
function of this code chunk. Both versions of the MDS plots seen below
were inspired by <span class="citation">(Brionne, Juanchich, and
Hennequet-Antier 2019)</span>.</p>
<p>The ontology needs to be defined for the MDS plots as well. Note that
MDS plots utilise the semantic similarity metric which currently only
supports GO ontology (MF – MolecularFunctions, CC – Cellular Components,
and BP – Biological processes). These have been defined at the start.
The first function plots the standard gprofiler plots (gost plots) to
the directory, and it retrieves the information of the requested
ontology into the GO_clusters dataframe. This dataframe is then used
with the semantic distance to create two versions of the MDS plot.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Create standard gprofiler results</span></span>
<span><span class="va">gpro_res</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/gprofiler_cluster_analysis.html">gprofiler_cluster_analysis</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">my_ont_gpro</span>,save_path <span class="op">=</span> <span class="cn">NULL</span>,return_specific_cluster<span class="op">=</span><span class="va">target_clust</span>,return_interactive<span class="op">=</span><span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="va">GO_clusters</span><span class="op">&lt;-</span><span class="va">gpro_res</span><span class="op">[[</span><span class="st">'GO_df'</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="va">sem_dta</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'sem_list'</span><span class="op">)</span></span>
<span><span class="co">#Plot and save MDS and clustered MDS</span></span>
<span><span class="va">MDS_plots</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/wrapper_MDS_and_MDS_clusters.html">wrapper_MDS_and_MDS_clusters</a></span><span class="op">(</span><span class="va">GO_clusters</span>,<span class="va">sem_dta</span>,<span class="va">my_ont_sem_sim</span>,target_dir<span class="op">=</span><span class="cn">NULL</span>,return_plot<span class="op">=</span><span class="cn">TRUE</span>,term_type_gg<span class="op">=</span><span class="cn">FALSE</span><span class="op">)</span></span></code></pre></div>
<p>Each cluster has it’s own individual gprofiler plot, these are found
in ‘TS_results/gprofiler_results/figures’. The gost plot for the most
variable cluster (C13) is shown below. These interactive plots show the
results of various databases when queried with a list of genes. More
information concerning these plots and their various datasources can be
found at the following link: <a href="https://biit.cs.ut.ee/gprofiler/gost" class="external-link uri">https://biit.cs.ut.ee/gprofiler/gost</a> Below is a
non-interactive version of the plot, interactive versions can be
obtained by setting the return_interactive’ parameter to the above
wrapper function to `TRUE’. In the interactive version you can hover
over each dot to obtain the relevan information of that specific
dot.</p>
<p><img src="TS_analysis_PBMC_files/figure-html/show_gpro_clust-1.png" width="100%"></p>
<p>In addition to the standard gprofiler plot, the TimeSeries pipeline
creates Multi-Dimensional Scaling (MDS) plots. These plots illustrate a
custom selected ontology defined by the ‘my_ont_gpro’ parameter. The
first MDS plots shows all the found terms of the specified ontology and
distinguishes them based on the cluster(s) in which they are found. In a
MDS plot, the distance between each dot represents the semantic
similarity between terms<span class="citation">(Yu et al. 2010; Wang et
al. 2007)</span>. This can be loosely defined as the ‘closeness’ between
terms. Semantic similarity takes into account many factors, such as the
content of the terms (genes which belong to it), it’s associations to
other terms, the similarity or lack there-of in function etc… In an MDS
plot, the dimensions will be adjusted to fill the space with all the
terms, therefor there is no set metric to measure the similarity. The
purpose of the MDS plot is moreso to help visualize different groupings
(or clusters) of relevant terms. This plot can be seen by entering .
<!-- # ```{r show_MDS_terms ,message=FALSE,warnings=FALSE,echo=FALSE,out.width='100%'} -->
<!-- # MDS_plots[['MDS_terms']] --> <!-- # ``` --> Due to the large
number of possible hits in a standard term based MDS, an alternate
version was created. The second version of the MDS plot provides a
nearest ancestor clustering approach. All terms found are brought up to
their nearest common ancestor. Since these databases, GO ontology in
particular, have a tree-like hierarchy, we are able to find common
ancestors for each term which may provide a broader view if too many
individual terms exist. This approach does not distinguish based on
clusters in which terms were found, meaning that terms from different
clusters may be paired together. The size of each dot in this approach
is relative to the number of terms sorted in that common ancestor (dot)
while the color is representative of the dominant cluster in that common
ancestor. For example, if an ancestor has 4 terms within it, and 3 of
those terms are found in cluster 1 (C1), the dot will carry the color
associated to C1. An interactive version can also be obtained by setting
the ‘term_type_gg’ parameter to FALSE</p>
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-2f03197fd14ac36b17bb" style="width:100%;height:432.632880098888px;"></div>
<script type="application/json" data-for="htmlwidget-2f03197fd14ac36b17bb">{"x":{"visdat":{"949017bd71e85":["function () ","plotlyVisDat"],"949017a3f47bb":["function () ","data"]},"cur_data":"949017a3f47bb","attrs":{"949017a3f47bb":{"alpha_stroke":1,"sizes":[20,50],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","text":{},"showlegend":false,"size":{},"fill":{},"marker":{"sizemode":"diameter","opacity":0.40000000000000002,"line":{"color":["#CAEABC","#FAF9B6","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#FAF9B6","#FAF9B6","#FAF9B6","#8DD3C7","#8DD3C7","#FAF9B6","#8DD3C7","#E89191","#C7B1C9","#E89191","#E89191","#E89191","#C7B1C9","#E89191","#E89191","#E89191","#FAF9B6","#E89191","#8DD3C7","#FAF9B6","#FAF9B6","#8DD3C7","#E89191","#FAF9B6","#FAF9B6","#CAEABC","#CAEABC","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#C7B1C9","#FAF9B6","#E89191","#D4D6A2","#D4D6A2","#D7D4CB","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#DE8B88","#E0C464","#DE8B88","#8DD3C7","#8DD3C7","#D4D6A2","#D4D6A2","#E89191","#A6B1B0","#8DD3C7","#FAF9B6","#8DD3C7","#FAF9B6","#CAEABC"]},"color":["#CAEABC","#FAF9B6","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#FAF9B6","#FAF9B6","#FAF9B6","#8DD3C7","#8DD3C7","#FAF9B6","#8DD3C7","#E89191","#C7B1C9","#E89191","#E89191","#E89191","#C7B1C9","#E89191","#E89191","#E89191","#FAF9B6","#E89191","#8DD3C7","#FAF9B6","#FAF9B6","#8DD3C7","#E89191","#FAF9B6","#FAF9B6","#CAEABC","#CAEABC","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#C7B1C9","#FAF9B6","#E89191","#D4D6A2","#D4D6A2","#D7D4CB","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#DE8B88","#E0C464","#DE8B88","#8DD3C7","#8DD3C7","#D4D6A2","#D4D6A2","#E89191","#A6B1B0","#8DD3C7","#FAF9B6","#8DD3C7","#FAF9B6","#CAEABC"]},"visible":true,"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"fillcolor":"rgba(31,119,180,0.5)","x":[0.19523384260125323,-0.28424018004274121,-0.38564503578743958,-0.26489384755977852,-0.28942193633553864,-0.10848357240089941,-0.25391525227006995,-0.16289114190310092,0.041475829327286828,0.2408524548089517,-0.027961559526752833,-0.11122797999921664,0.21755645948672278,0.035806870330111547,0.10772725926196984,-0.027595147219821901,0.19228583778769381,0.3332137730942642,0.28568837382877144,0.27614947827429515,0.22917031714917555,0.13739093306091757,0.032431315585878077,0.21881867820456116,0.11509899286995054,0.052961696551926889,-0.10112395176080097,0.32357563431180508,0.33566673161168081,0.14559776781949124,-0.047393713414921067,-0.18835102176688823,-0.45195086935318479,-0.42081359128417323,-0.43651277670226057,-0.35959662337076215,-0.20711813071376695,0.024526650556767043,-0.17576582768448376,0.37371624398325709,0.18902184846450842,0.34954311605522492,-0.065309259628603039,-0.35104765242408786,-0.27789104968575079,-0.32819316347408795,-0.095604714314681288,0.12816455558788681,0.053436638559618908,0.12065462702847177,0.063197793413472661,0.095148167889765989,0.088430664258522765,0.23995947946805796,-0.052036323313440658,0.24935958599582347,0.25550800609700902,-0.18015940450114026,0.1242086848391996,0.1418261986305136,-0.3582607803564079],"y":[0.13732698415322667,0.30040646031267632,0.28045781232856665,-0.19799288443118898,-0.14666666600116396,-0.17083534433790226,0.23325837489247667,0.33281720171058315,0.04890790725350435,0.042544183920957844,-0.051262711683962091,0.24184264066616817,-0.0050170847427233904,0.029667289208482672,-0.29822295112093572,-0.090087732993452144,-0.20498069549241921,0.063282628940713959,0.079441513341423103,0.050101741145262974,-0.28143316395115214,-0.34663608668356766,-0.15110084559172515,-0.29463771213889867,0.011131454445578132,0.12606683456669265,0.2250414142649316,0.11043310023910344,0.10052518981987518,0.20857839539789905,0.084185918931568557,0.25064857495305365,-0.028002109608031251,-0.04852006541962773,-0.052147070784052001,-0.13733215011424829,0.25619016408386563,0.20226830772891338,-0.23675736166092262,0.13246477284439823,0.20901373070809323,0.1502811096284874,-0.11923039023711343,-0.15336237023060131,-0.079462699356185121,-0.17930387548012491,0.31454664615246575,0.1479657678498954,-0.25593832438710418,0.030623034558369431,0.094799473210780152,-0.35839205386249473,-0.36078904321871036,0.098837101132542571,-0.17524764383003658,0.10255010860792368,0.10181578597833295,0.25558061959719941,-0.28130224839763479,-0.18221247449347081,-0.16672848232455972],"type":"scatter","mode":"markers","text":["GO.cluster: 14 <br>GO.count: 2  -  C10: 1, C11: 1 <br>GO.ID: GO:0000003 <br>GO.name: reproduction","GO.cluster: 33 <br>GO.count: 15  -  C11: 13, C10: 2 <br>GO.ID: GO:0006996 <br>GO.name: organelle organization","GO.cluster: 29 <br>GO.count: 8  -  C11: 5, C10: 3 <br>GO.ID: GO:0022402 <br>GO.name: cell cycle process","GO.cluster: 23 <br>GO.count: 10  -  C10: 7, C11: 3 <br>GO.ID: GO:0045786 <br>GO.name: negative regulation of cell cycle","GO.cluster: 24 <br>GO.count: 8  -  C11: 6, C10: 2 <br>GO.ID: GO:0051726 <br>GO.name: regulation of cell cycle","GO.cluster: 3 <br>GO.count: 7  -  C11: 3, C10: 2, C12: 2 <br>GO.ID: GO:0019538 <br>GO.name: protein metabolic process","GO.cluster: 10 <br>GO.count: 11  -  C11: 9, C10: 2 <br>GO.ID: GO:0007049 <br>GO.name: cell cycle","GO.cluster: 36 <br>GO.count: 11  -  C11: 7, C10: 4 <br>GO.ID: GO:0016043 <br>GO.name: cellular component organization","GO.cluster: 1 <br>GO.count: 11  -  C11: 11 <br>GO.ID: GO:0006259 <br>GO.name: DNA metabolic process","GO.cluster: 39 <br>GO.count: 6  -  C1: 3, C13: 1, C4: 1, C6: 1 <br>GO.ID: GO:0032502 <br>GO.name: developmental process","GO.cluster: 38 <br>GO.count: 7  -  C1: 3, C6: 2, C8: 2 <br>GO.ID: GO:0048869 <br>GO.name: cellular developmental process","GO.cluster: 9 <br>GO.count: 3  -  C11: 3 <br>GO.ID: GO:0051301 <br>GO.name: cell division","GO.cluster: 40 <br>GO.count: 11  -  C1: 6, C8: 3, C2: 1, C4: 1 <br>GO.ID: GO:0048856 <br>GO.name: anatomical structure development","GO.cluster: 17 <br>GO.count: 8  -  C14: 6, C1: 2 <br>GO.ID: GO:0001775 <br>GO.name: cell activation","GO.cluster: 53 <br>GO.count: 5  -  C13: 2, C1: 1, C4: 1, C8: 1 <br>GO.ID: GO:0032501 <br>GO.name: multicellular organismal process","GO.cluster: 21 <br>GO.count: 7  -  C14: 7 <br>GO.ID: GO:0001906 <br>GO.name: cell killing","GO.cluster: 56 <br>GO.count: 10  -  C14: 10 <br>GO.ID: GO:0045088 <br>GO.name: regulation of innate immune response","GO.cluster: 57 <br>GO.count: 9  -  C14: 9 <br>GO.ID: GO:0043207 <br>GO.name: response to external biotic stimulus","GO.cluster: 59 <br>GO.count: 5  -  C13: 3, C14: 2 <br>GO.ID: GO:0006955 <br>GO.name: immune response","GO.cluster: 60 <br>GO.count: 4  -  C14: 4 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","GO.cluster: 58 <br>GO.count: 5  -  C14: 3, C8: 2 <br>GO.ID: GO:0050776 <br>GO.name: regulation of immune response","GO.cluster: 61 <br>GO.count: 9  -  C14: 9 <br>GO.ID: GO:0002682 <br>GO.name: regulation of immune system process","GO.cluster: 45 <br>GO.count: 8  -  C11: 2, C14: 2, C13: 1, C3: 1, C6: 1, C8: 1 <br>GO.ID: GO:0007165 <br>GO.name: signal transduction","GO.cluster: 49 <br>GO.count: 10  -  C14: 7, C1: 2, C8: 1 <br>GO.ID: GO:0048583 <br>GO.name: regulation of response to stimulus","GO.cluster: 6 <br>GO.count: 3  -  C1: 3 <br>GO.ID: GO:1901071 <br>GO.name: glucosamine-containing compound metabolic process","GO.cluster: 2 <br>GO.count: 7  -  C11: 3, C14: 2, C10: 1, C7: 1 <br>GO.ID: GO:0008150 <br>GO.name: biological <br>GO.name: process","GO.cluster: 16 <br>GO.count: 13  -  C11: 8, C10: 2, C13: 1, C3: 1, C4: 1 <br>GO.ID: GO:0008150 <br>GO.name: biological <br>GO.name: process","GO.cluster: 43 <br>GO.count: 2  -  C1: 1, C2: 1 <br>GO.ID: GO:0042330 <br>GO.name: taxis","GO.cluster: 42 <br>GO.count: 4  -  C14: 2, C1: 1, C2: 1 <br>GO.ID: GO:0006950 <br>GO.name: response to stress","GO.cluster: 46 <br>GO.count: 3  -  C11: 2, C1: 1 <br>GO.ID: GO:0051716 <br>GO.name: cellular response to stimulus","GO.cluster: 19 <br>GO.count: 6  -  C11: 2, C1: 1, C14: 1, C2: 1, C8: 1 <br>GO.ID: GO:0009987 <br>GO.name: cellular process","GO.cluster: 8 <br>GO.count: 4  -  C10: 2, C11: 2 <br>GO.ID: GO:0007059 <br>GO.name: chromosome segregation","GO.cluster: 30 <br>GO.count: 10  -  C10: 6, C11: 4 <br>GO.ID: GO:0010564 <br>GO.name: regulation of cell cycle process","GO.cluster: 31 <br>GO.count: 2  -  C11: 2 <br>GO.ID: GO:0007088 <br>GO.name: regulation of mitotic nuclear division","GO.cluster: 32 <br>GO.count: 7  -  C10: 4, C11: 3 <br>GO.ID: GO:0033046 <br>GO.name: negative regulation of sister chromatid segregation","GO.cluster: 25 <br>GO.count: 8  -  C11: 7, C10: 1 <br>GO.ID: GO:1901990 <br>GO.name: regulation of mitotic cell cycle phase transition","GO.cluster: 7 <br>GO.count: 14  -  C11: 14 <br>GO.ID: GO:0022414 <br>GO.name: reproductive process","GO.cluster: 18 <br>GO.count: 2  -  C13: 1, C3: 1 <br>GO.ID: GO:0007154 <br>GO.name: cell communication","GO.cluster: 22 <br>GO.count: 13  -  C11: 5, C8: 3, C10: 2, C14: 1, C2: 1, C3: 1 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process","GO.cluster: 44 <br>GO.count: 6  -  C14: 3, C8: 2, C3: 1 <br>GO.ID: GO:0050896 <br>GO.name: response to stimulus","GO.cluster: 48 <br>GO.count: 7  -  C8: 3, C14: 2, C6: 2 <br>GO.ID: GO:0042221 <br>GO.name: response to chemical","GO.cluster: 41 <br>GO.count: 10  -  C8: 4, C2: 2, C6: 2, C1: 1, C14: 1 <br>GO.ID: GO:0042221 <br>GO.name: response to chemical","GO.cluster: 4 <br>GO.count: 4  -  C12: 3, C11: 1 <br>GO.ID: GO:0006793 <br>GO.name: phosphorus metabolic process","GO.cluster: 28 <br>GO.count: 12  -  C11: 8, C10: 4 <br>GO.ID: GO:0051726 <br>GO.name: regulation of cell cycle","GO.cluster: 20 <br>GO.count: 8  -  C10: 5, C11: 3 <br>GO.ID: GO:0051128 <br>GO.name: regulation of cellular component organization","GO.cluster: 26 <br>GO.count: 7  -  C11: 4, C10: 3 <br>GO.ID: GO:0045787 <br>GO.name: positive regulation of cell cycle","GO.cluster: 35 <br>GO.count: 4  -  C11: 2, C10: 1, C2: 1 <br>GO.ID: GO:0071840 <br>GO.name: cellular component organization or biogenesis","GO.cluster: 12 <br>GO.count: 6  -  C2: 4, C14: 1, C8: 1 <br>GO.ID: GO:0048870 <br>GO.name: cell motility","GO.cluster: 5 <br>GO.count: 4  -  C6: 4 <br>GO.ID: GO:0019216 <br>GO.name: regulation of lipid metabolic process","GO.cluster: 47 <br>GO.count: 5  -  C2: 3, C1: 1, C14: 1 <br>GO.ID: GO:0071345 <br>GO.name: cellular response to cytokine stimulus","GO.cluster: 37 <br>GO.count: 10  -  C1: 10 <br>GO.ID: GO:0022008 <br>GO.name: neurogenesis","GO.cluster: 54 <br>GO.count: 4  -  C1: 2, C4: 1, C6: 1 <br>GO.ID: GO:0050793 <br>GO.name: regulation of developmental process","GO.cluster: 51 <br>GO.count: 5  -  C8: 2, C1: 1, C3: 1, C6: 1 <br>GO.ID: GO:0050789 <br>GO.name: regulation of biological process","GO.cluster: 13 <br>GO.count: 1  -  C8: 1 <br>GO.ID: GO:0023052 <br>GO.name: signaling","GO.cluster: 11 <br>GO.count: 2  -  C14: 1, C8: 1 <br>GO.ID: GO:2000145 <br>GO.name: regulation of cell motility","GO.cluster: 52 <br>GO.count: 1  -  C4: 1 <br>GO.ID: GO:0032501 <br>GO.name: multicellular organismal process","GO.cluster: 15 <br>GO.count: 3  -  C1: 2, C14: 1 <br>GO.ID: GO:0008150 <br>GO.name: biological <br>GO.name: process","GO.cluster: 34 <br>GO.count: 4  -  C11: 3, C10: 1 <br>GO.ID: GO:0065004 <br>GO.name: protein-DNA complex assembly","GO.cluster: 55 <br>GO.count: 5  -  C1: 5 <br>GO.ID: GO:2000026 <br>GO.name: regulation of multicellular organismal development","GO.cluster: 50 <br>GO.count: 2  -  C11: 1, C3: 1 <br>GO.ID: GO:0065007 <br>GO.name: biological regulation","GO.cluster: 27 <br>GO.count: 3  -  C10: 2, C11: 1 <br>GO.ID: GO:1905818 <br>GO.name: regulation of chromosome separation"],"showlegend":false,"fill":"","marker":{"color":["#CAEABC","#FAF9B6","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#FAF9B6","#FAF9B6","#FAF9B6","#8DD3C7","#8DD3C7","#FAF9B6","#8DD3C7","#E89191","#C7B1C9","#E89191","#E89191","#E89191","#C7B1C9","#E89191","#E89191","#E89191","#FAF9B6","#E89191","#8DD3C7","#FAF9B6","#FAF9B6","#8DD3C7","#E89191","#FAF9B6","#FAF9B6","#CAEABC","#CAEABC","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#C7B1C9","#FAF9B6","#E89191","#D4D6A2","#D4D6A2","#D7D4CB","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#DE8B88","#E0C464","#DE8B88","#8DD3C7","#8DD3C7","#D4D6A2","#D4D6A2","#E89191","#A6B1B0","#8DD3C7","#FAF9B6","#8DD3C7","#FAF9B6","#CAEABC"],"size":[22.142857142857142,50,35,39.285714285714292,35,32.857142857142854,41.428571428571431,41.428571428571431,41.428571428571431,30.714285714285715,32.857142857142854,24.285714285714285,41.428571428571431,35,28.571428571428569,32.857142857142854,39.285714285714292,37.142857142857139,28.571428571428569,26.428571428571427,28.571428571428569,37.142857142857139,35,39.285714285714292,24.285714285714285,32.857142857142854,45.714285714285708,22.142857142857142,26.428571428571427,24.285714285714285,30.714285714285715,26.428571428571427,39.285714285714292,22.142857142857142,32.857142857142854,35,47.857142857142861,22.142857142857142,45.714285714285708,30.714285714285715,32.857142857142854,39.285714285714292,26.428571428571427,43.571428571428569,35,32.857142857142854,26.428571428571427,30.714285714285715,26.428571428571427,28.571428571428569,39.285714285714292,26.428571428571427,28.571428571428569,20,22.142857142857142,20,24.285714285714285,26.428571428571427,28.571428571428569,22.142857142857142,24.285714285714285],"sizemode":"diameter","opacity":0.40000000000000002,"line":{"color":["#CAEABC","#FAF9B6","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#FAF9B6","#FAF9B6","#FAF9B6","#8DD3C7","#8DD3C7","#FAF9B6","#8DD3C7","#E89191","#C7B1C9","#E89191","#E89191","#E89191","#C7B1C9","#E89191","#E89191","#E89191","#FAF9B6","#E89191","#8DD3C7","#FAF9B6","#FAF9B6","#8DD3C7","#E89191","#FAF9B6","#FAF9B6","#CAEABC","#CAEABC","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#C7B1C9","#FAF9B6","#E89191","#D4D6A2","#D4D6A2","#D7D4CB","#FAF9B6","#CAEABC","#FAF9B6","#FAF9B6","#DE8B88","#E0C464","#DE8B88","#8DD3C7","#8DD3C7","#D4D6A2","#D4D6A2","#E89191","#A6B1B0","#8DD3C7","#FAF9B6","#8DD3C7","#FAF9B6","#CAEABC"]}},"visible":true,"textfont":{"size":[22.142857142857142,50,35,39.285714285714292,35,32.857142857142854,41.428571428571431,41.428571428571431,41.428571428571431,30.714285714285715,32.857142857142854,24.285714285714285,41.428571428571431,35,28.571428571428569,32.857142857142854,39.285714285714292,37.142857142857139,28.571428571428569,26.428571428571427,28.571428571428569,37.142857142857139,35,39.285714285714292,24.285714285714285,32.857142857142854,45.714285714285708,22.142857142857142,26.428571428571427,24.285714285714285,30.714285714285715,26.428571428571427,39.285714285714292,22.142857142857142,32.857142857142854,35,47.857142857142861,22.142857142857142,45.714285714285708,30.714285714285715,32.857142857142854,39.285714285714292,26.428571428571427,43.571428571428569,35,32.857142857142854,26.428571428571427,30.714285714285715,26.428571428571427,28.571428571428569,39.285714285714292,26.428571428571427,28.571428571428569,20,22.142857142857142,20,24.285714285714285,26.428571428571427,28.571428571428569,22.142857142857142,24.285714285714285]},"error_y":{"color":"rgba(31,119,180,1)","width":[]},"error_x":{"color":"rgba(31,119,180,1)","width":[]},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 id="gprofiler-results-custom">Gprofiler results – custom<a class="anchor" aria-label="anchor" href="#gprofiler-results-custom"></a>
</h3>
<p>Dotplots were designed to illustrate the top n term results for any
single ontology. In the below code chunk three different ontologies are
demonstrated, GO:BP (biological processes), REAC (REACTOME) <span class="citation">(Jassal et al. 2020)</span> and KEGG<span class="citation">(Kanehisa et al. 2017)</span>. The ‘target_top’
parameter defines the number of terms (from most significant to least)
per cluster that will be shown on the dotplot.</p>
<p>All terms shown are considered to be significant, the ‘top’ aspect
reflects significance based on the adjusted p value. The adjusted p
value is negatively log transformed in order to offer a better
visualization, with most significant being in red and less significant
(but still statistically significant) in blue.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">target_top</span><span class="op">=</span><span class="fl">5</span></span>
<span><span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'REAC'</span></span>
<span><span class="va">gpro_REAC_dotplot</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span>,file_loc<span class="op">=</span><span class="cn">NULL</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span>,return_plot <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<p>Illustrated below is the dotplot for the REACTOME pathway.
<img src="TS_analysis_PBMC_files/figure-html/show_dotplot%20-1.png" width="100%"></p>
</div>
<div class="section level3">
<h3 id="ancestor-query-plots">Ancestor query plots<a class="anchor" aria-label="anchor" href="#ancestor-query-plots"></a>
</h3>
<p>Ancestor query plots were designed to query results relating to
specific GOs.</p>
<p>This plot searches for children of the queried term IDs. Pathways,
such as GO pathways are hierarchically organized, so a single ID will be
associated to other IDs. For example, the ‘immune system process
(GO:0002376)’ contains several ‘children’, each of these children is
likely to have children of their own and so on. If we use the ID for
‘immune system process’ this plot will find all children grand-children
great-grand children etc… of ‘immune system process’. It will then split
the children based on the PART cluster in which they were found, these
results are illustrated in dotplot format. This is accomplished via the
GO.db package<span class="citation">(Carlson 2021)</span></p>
<p>Several websites exist to search for GO ancestors of interest,
‘AMIGO2’ is one and ‘quickGO’ is another.</p>
<p>A word of caution with this type of plot. It will always link terms
to the most upstream ancestor, so if the query contains two term IDs
which are related to each other, the plot will illustrate all results as
belonging to the term ID which is the most upstream (oldest/ancestor).
Each ancestor queried for will be illustrated in different colors on the
dotplot.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Dotplot for terms relating to specific ancestors</span></span>
<span><span class="co">#Check if analysis is required, if not, set to null or empty</span></span>
<span><span class="kw">if</span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="va">target_ancestors</span><span class="op">)</span><span class="op">==</span><span class="fl">0</span><span class="op">)</span><span class="op">{</span></span>
<span>  <span class="va">ancestor_plots</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span></span>
<span>  <span class="va">ancestor_plots</span><span class="op">[[</span><span class="st">'MDS'</span><span class="op">]</span><span class="op">]</span><span class="op">&lt;-</span><span class="cn">NULL</span></span>
<span>  <span class="va">GOs_ancestors_clust</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="cn">NULL</span><span class="op">)</span></span>
<span><span class="op">}</span><span class="kw">else</span><span class="op">{</span></span>
<span>  <span class="va">GOs_ancestors_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_relation_to_ancestors.html">find_relation_to_ancestors</a></span><span class="op">(</span><span class="va">target_ancestors</span>,<span class="va">GO_clusters</span>,ontology <span class="op">=</span> <span class="va">ancestor_ontology</span><span class="op">)</span></span>
<span>  <span class="va">ancestor_plots</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/wrapper_ancestor_curation_plots.html">wrapper_ancestor_curation_plots</a></span><span class="op">(</span><span class="va">GOs_ancestors_clust</span>,<span class="va">sem_dta</span>,return_plot<span class="op">=</span><span class="cn">TRUE</span>,target_dir<span class="op">=</span><span class="cn">NULL</span>,term_type_gg<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="op">}</span></span></code></pre></div>
<p>Two formats are generated for the ancestor plot. The first is a MDS
plot. The second is a dotplot version. If more than one term was found
to be associated to ancestors, the MDS plot will be created and can be
seen below, otherwise only the dotplot will be created. The dotplot will
not be created if no terms were found. The files for both of these plots
can be found in ‘TS_results/ancestor_plots’. In the MDS and dotplot
alike, the color of each term represents the ancestor to which they have
been associated and the size is proportional to the -log transformed
adjusted p-value. In the MDS plot, the PART clusters in which these
terms are found can be seen by hovering over each dot, in the dotplot
version, this information is made available on the x axis while the y
axis represents the terms. For the MDS plot, an interactive version can
also be obtained by setting the `term_type_gg’ parameter to FALSE.</p>
<p><img src="TS_analysis_PBMC_files/figure-html/show_ancestor_MDS%20-1.png" width="100%"></p>
</div>
<div class="section level3">
<h3 class="tabset unlisted unnumbered" id="section"></h3>
<p>A interactive table is also generated in order to better navigate the
results in the event of a large number of terms found for many different
ancestors. The csv version of this data can be found in
‘TS_results/ancestor_plots’.</p>
<div class="section level4">
<h4 id="go0002252">GO:0002252<a class="anchor" aria-label="anchor" href="#go0002252"></a>
</h4>
<p>Ancestor name: immune effector process</p>
<table class="table">
<colgroup>
<col width="13%">
<col width="72%">
<col width="14%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0001909</td>
<td align="left">leukocyte mediated cytotoxicity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0001910</td>
<td align="left">regulation of leukocyte mediated cytotoxicity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002228</td>
<td align="left">natural killer cell mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002252</td>
<td align="left">immune effector process</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002263</td>
<td align="left">cell activation involved in immune response</td>
<td align="left">C13</td>
</tr>
<tr class="even">
<td align="left">GO:0002366</td>
<td align="left">leukocyte activation involved in immune response</td>
<td align="left">C13</td>
</tr>
<tr class="odd">
<td align="left">GO:0002443</td>
<td align="left">leukocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002449</td>
<td align="left">lymphocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002697</td>
<td align="left">regulation of immune effector process</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002699</td>
<td align="left">positive regulation of immune effector process</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002703</td>
<td align="left">regulation of leukocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002705</td>
<td align="left">positive regulation of leukocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002706</td>
<td align="left">regulation of lymphocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002708</td>
<td align="left">positive regulation of lymphocyte mediated
immunity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002715</td>
<td align="left">regulation of natural killer cell mediated
immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002717</td>
<td align="left">positive regulation of natural killer cell mediated
immunity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0042267</td>
<td align="left">natural killer cell mediated cytotoxicity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0042269</td>
<td align="left">regulation of natural killer cell mediated
cytotoxicity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0045954</td>
<td align="left">positive regulation of natural killer cell mediated
cytotoxicity</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002253">GO:0002253<a class="anchor" aria-label="anchor" href="#go0002253"></a>
</h4>
<p>Ancestor name: activation of immune response</p>
<table class="table">
<colgroup>
<col width="12%">
<col width="74%">
<col width="13%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002218</td>
<td align="left">activation of innate immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002220</td>
<td align="left">innate immune response activating cell surface receptor
signaling pathway</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002223</td>
<td align="left">stimulatory C-type lectin receptor signaling
pathway</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002758</td>
<td align="left">innate immune response-activating signal
transduction</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002682">GO:0002682<a class="anchor" aria-label="anchor" href="#go0002682"></a>
</h4>
<p>Ancestor name: regulation of immune system process</p>
<table class="table">
<colgroup>
<col width="7%">
<col width="84%">
<col width="8%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002253</td>
<td align="left">activation of immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002682</td>
<td align="left">regulation of immune system process</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0002682</td>
<td align="left">regulation of immune system process</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002683</td>
<td align="left">negative regulation of immune system process</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002684</td>
<td align="left">positive regulation of immune system process</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002694</td>
<td align="left">regulation of leukocyte activation</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002768</td>
<td align="left">immune response-regulating cell surface receptor
signaling pathway</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002819</td>
<td align="left">regulation of adaptive immune response</td>
<td align="left">C8</td>
</tr>
<tr class="odd">
<td align="left">GO:0002822</td>
<td align="left">regulation of adaptive immune response based on somatic
recombination of immune receptors built from immunoglobulin superfamily
domains</td>
<td align="left">C8</td>
</tr>
<tr class="even">
<td align="left">GO:0032814</td>
<td align="left">regulation of natural killer cell activation</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0045088</td>
<td align="left">regulation of innate immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0045089</td>
<td align="left">positive regulation of innate immune response</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0050776</td>
<td align="left">regulation of immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0050778</td>
<td align="left">positive regulation of immune response</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0051249</td>
<td align="left">regulation of lymphocyte activation</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0006955">GO:0006955<a class="anchor" aria-label="anchor" href="#go0006955"></a>
</h4>
<p>Ancestor name: immune response</p>
<table class="table">
<colgroup>
<col width="8%">
<col width="82%">
<col width="8%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002250</td>
<td align="left">adaptive immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002460</td>
<td align="left">adaptive immune response based on somatic recombination
of immune receptors built from immunoglobulin superfamily domains</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0006955</td>
<td align="left">immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0006955</td>
<td align="left">immune response</td>
<td align="left">C13</td>
</tr>
<tr class="odd">
<td align="left">GO:0045087</td>
<td align="left">innate immune response</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0045321">GO:0045321<a class="anchor" aria-label="anchor" href="#go0045321"></a>
</h4>
<p>Ancestor name: leukocyte activation</p>
<table class="table">
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0030101</td>
<td align="left">natural killer cell activation</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0042110</td>
<td align="left">T cell activation</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0045321</td>
<td align="left">leukocyte activation</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0045321</td>
<td align="left">leukocyte activation</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0046649</td>
<td align="left">lymphocyte activation</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0050900">GO:0050900<a class="anchor" aria-label="anchor" href="#go0050900"></a>
</h4>
<p>Ancestor name: leukocyte migration</p>
<table class="table">
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0030593</td>
<td align="left">neutrophil chemotaxis</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0050900</td>
<td align="left">leukocyte migration</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:1990266</td>
<td align="left">neutrophil migration</td>
<td align="left">C2</td>
</tr>
</tbody>
</table>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## R version 4.3.0 (2023-04-21)</span></span>
<span><span class="co">## Platform: x86_64-pc-linux-gnu (64-bit)</span></span>
<span><span class="co">## Running under: Pop!_OS 22.04 LTS</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Matrix products: default</span></span>
<span><span class="co">## BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0 </span></span>
<span><span class="co">## LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## locale:</span></span>
<span><span class="co">##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span></span>
<span><span class="co">##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></span>
<span><span class="co">##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></span>
<span><span class="co">##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></span>
<span><span class="co">##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span></span>
<span><span class="co">## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## time zone: Europe/Oslo</span></span>
<span><span class="co">## tzcode source: system (glibc)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## attached base packages:</span></span>
<span><span class="co">## [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">## [8] base     </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## other attached packages:</span></span>
<span><span class="co">##  [1] org.Hs.eg.db_3.17.0         AnnotationDbi_1.62.1       </span></span>
<span><span class="co">##  [3] ggplot2_3.4.2               TimeSeriesAnalysis_01.01.01</span></span>
<span><span class="co">##  [5] BiocFileCache_2.8.0         dbplyr_2.3.2               </span></span>
<span><span class="co">##  [7] SummarizedExperiment_1.30.2 Biobase_2.60.0             </span></span>
<span><span class="co">##  [9] GenomicRanges_1.52.0        GenomeInfoDb_1.36.1        </span></span>
<span><span class="co">## [11] IRanges_2.34.1              S4Vectors_0.38.1           </span></span>
<span><span class="co">## [13] BiocGenerics_0.46.0         MatrixGenerics_1.12.2      </span></span>
<span><span class="co">## [15] matrixStats_1.0.0           knitr_1.43                 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## loaded via a namespace (and not attached):</span></span>
<span><span class="co">##   [1] RColorBrewer_1.1-3      rstudioapi_0.14         jsonlite_1.8.5         </span></span>
<span><span class="co">##   [4] shape_1.4.6             magrittr_2.0.3          farver_2.1.1           </span></span>
<span><span class="co">##   [7] rmarkdown_2.22          GlobalOptions_0.1.2     fs_1.6.2               </span></span>
<span><span class="co">##  [10] zlibbioc_1.46.0         ragg_1.2.5              vctrs_0.6.3            </span></span>
<span><span class="co">##  [13] memoise_2.0.1           RCurl_1.98-1.12         htmltools_0.5.5        </span></span>
<span><span class="co">##  [16] S4Arrays_1.0.4          dynamicTreeCut_1.63-1   curl_5.0.1             </span></span>
<span><span class="co">##  [19] tictoc_1.2              sass_0.4.6              bslib_0.5.0            </span></span>
<span><span class="co">##  [22] htmlwidgets_1.6.2       desc_1.4.2              plyr_1.8.8             </span></span>
<span><span class="co">##  [25] plotly_4.10.2           cachem_1.0.8            mime_0.12              </span></span>
<span><span class="co">##  [28] lifecycle_1.0.3         iterators_1.0.14        pkgconfig_2.0.3        </span></span>
<span><span class="co">##  [31] Matrix_1.5-4.1          R6_2.5.1                fastmap_1.1.1          </span></span>
<span><span class="co">##  [34] GenomeInfoDbData_1.2.10 shiny_1.7.4             clue_0.3-64            </span></span>
<span><span class="co">##  [37] digest_0.6.32           colorspace_2.1-0        DESeq2_1.40.2          </span></span>
<span><span class="co">##  [40] rprojroot_2.0.3         textshaping_0.3.6       crosstalk_1.2.0        </span></span>
<span><span class="co">##  [43] RSQLite_2.3.1           filelock_1.0.2          labeling_0.4.2         </span></span>
<span><span class="co">##  [46] org.Mm.eg.db_3.17.0     fansi_1.0.4             httr_1.4.6             </span></span>
<span><span class="co">##  [49] mgcv_1.8-42             compiler_4.3.0          bit64_4.0.5            </span></span>
<span><span class="co">##  [52] withr_2.5.0             doParallel_1.0.17       BiocParallel_1.34.2    </span></span>
<span><span class="co">##  [55] DBI_1.1.3               highr_0.10              DelayedArray_0.26.3    </span></span>
<span><span class="co">##  [58] rjson_0.2.21            tools_4.3.0             httpuv_1.6.11          </span></span>
<span><span class="co">##  [61] glue_1.6.2              promises_1.2.0.1        nlme_3.1-162           </span></span>
<span><span class="co">##  [64] GOSemSim_2.26.0         grid_4.3.0              cluster_2.1.4          </span></span>
<span><span class="co">##  [67] reshape2_1.4.4          generics_0.1.3          gtable_0.3.3           </span></span>
<span><span class="co">##  [70] tidyr_1.3.0             data.table_1.14.8       utf8_1.2.3             </span></span>
<span><span class="co">##  [73] XVector_0.40.0          ggrepel_0.9.3           foreach_1.5.2          </span></span>
<span><span class="co">##  [76] pillar_1.9.0            stringr_1.5.0           limma_3.56.2           </span></span>
<span><span class="co">##  [79] later_1.3.1             circlize_0.4.15         splines_4.3.0          </span></span>
<span><span class="co">##  [82] dplyr_1.1.2             lattice_0.21-8          bit_4.0.5              </span></span>
<span><span class="co">##  [85] tidyselect_1.2.0        GO.db_3.17.0            ComplexHeatmap_2.16.0  </span></span>
<span><span class="co">##  [88] locfit_1.5-9.8          Biostrings_2.68.1       xfun_0.39              </span></span>
<span><span class="co">##  [91] stringi_1.7.12          lazyeval_0.2.2          yaml_2.3.7             </span></span>
<span><span class="co">##  [94] evaluate_0.21           codetools_0.2-19        tibble_3.2.1           </span></span>
<span><span class="co">##  [97] BiocManager_1.30.21     cli_3.6.1               org.Ce.eg.db_3.17.0    </span></span>
<span><span class="co">## [100] xtable_1.8-4            systemfonts_1.0.4       munsell_0.5.0          </span></span>
<span><span class="co">## [103] jquerylib_0.1.4         Rcpp_1.0.10             gprofiler2_0.2.2       </span></span>
<span><span class="co">## [106] png_0.1-8               parallel_4.3.0          ellipsis_0.3.2         </span></span>
<span><span class="co">## [109] pkgdown_2.0.7           blob_1.2.4              bitops_1.0-7           </span></span>
<span><span class="co">## [112] viridisLite_0.4.2       scales_1.2.1            purrr_1.0.1            </span></span>
<span><span class="co">## [115] crayon_1.5.2            GetoptLong_1.0.5        rlang_1.1.1            </span></span>
<span><span class="co">## [118] KEGGREST_1.40.0</span></span></code></pre>
</div>
</div>
<div class="section level3">
<h3 class="unlisted unnumbered" id="section-1"></h3>
</div>
</div>
<div class="section level2">
<h2 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-brionne2019viseago" class="csl-entry">
Brionne, Aurélien, Amélie Juanchich, and Christelle Hennequet-Antier.
2019. <span>“ViSEAGO: A Bioconductor Package for Clustering Biological
Functions Using Gene Ontology and Semantic Similarity.”</span>
<em>BioData Mining</em> 12 (1): 1–13.
</div>
<div id="ref-GOdb_ref" class="csl-entry">
Carlson, Marc. 2021. <em>GO.db: A Set of Annotation Maps Describing the
Entire Gene Ontology</em>.
</div>
<div id="ref-gu2016complex" class="csl-entry">
Gu, Zuguang, Roland Eils, and Matthias Schlesner. 2016. <span>“Complex
Heatmaps Reveal Patterns and Correlations in Multidimensional Genomic
Data.”</span> <em>Bioinformatics</em> 32 (18): 2847–49.
</div>
<div id="ref-jassal2020reactome" class="csl-entry">
Jassal, Bijay, Lisa Matthews, Guilherme Viteri, Chuqiao Gong, Pascual
Lorente, Antonio Fabregat, Konstantinos Sidiropoulos, et al. 2020.
<span>“The Reactome Pathway Knowledgebase.”</span> <em>Nucleic Acids
Research</em> 48 (D1): D498–503.
</div>
<div id="ref-kanehisa2017kegg" class="csl-entry">
Kanehisa, Minoru, Miho Furumichi, Mao Tanabe, Yoko Sato, and Kanae
Morishima. 2017. <span>“KEGG: New Perspectives on Genomes, Pathways,
Diseases and Drugs.”</span> <em>Nucleic Acids Research</em> 45 (D1):
D353–61.
</div>
<div id="ref-kolberg2020gprofiler2" class="csl-entry">
Kolberg, Liis, Uku Raudvere, Ivan Kuzmin, Jaak Vilo, and Hedi Peterson.
2020. <span>“Gprofiler2–an r Package for Gene List Functional Enrichment
Analysis and Namespace Conversion Toolset g: Profiler.”</span>
<em>F1000Research</em> 9.
</div>
<div id="ref-love2014moderated" class="csl-entry">
Love, Michael I, Wolfgang Huber, and Simon Anders. 2014.
<span>“Moderated Estimation of Fold Change and Dispersion for RNA-Seq
Data with DESeq2.”</span> <em>Genome Biology</em> 15 (12): 1–21.
</div>
<div id="ref-timeseriesexperiment" class="csl-entry">
Nguyen, Lan Huong. 2021. <em>TimeSeriesExperiment: Analysis for Short
Time-Series Data</em>. <a href="https://github.com/nlhuong/TimeSeriesExperiment" class="external-link">https://github.com/nlhuong/TimeSeriesExperiment</a>.
</div>
<div id="ref-nilsen2013identifying" class="csl-entry">
Nilsen, Gro, Ørnulf Borgan, Knut LiestØl, and Ole Christian Lingjærde.
2013. <span>“Identifying Clusters in Genomics Data by Recursive
Partitioning.”</span> <em>Statistical Applications in Genetics and
Molecular Biology</em> 12 (5): 637–52.
</div>
<div id="ref-raudvere2019g" class="csl-entry">
Raudvere, Uku, Liis Kolberg, Ivan Kuzmin, Tambet Arak, Priit Adler, Hedi
Peterson, and Jaak Vilo. 2019. <span>“G: Profiler: A Web Server for
Functional Enrichment Analysis and Conversions of Gene Lists (2019
Update).”</span> <em>Nucleic Acids Research</em> 47 (W1): W191–98.
</div>
<div id="ref-smyth2005limma" class="csl-entry">
Smyth, Gordon K. 2005. <span>“Limma: Linear Models for Microarray
Data.”</span> In <em>Bioinformatics and Computational Biology Solutions
Using r and Bioconductor</em>, 397–420. Springer.
</div>
<div id="ref-wang2007new" class="csl-entry">
Wang, James Z, Zhidian Du, Rapeeporn Payattakool, Philip S Yu, and
Chin-Fu Chen. 2007. <span>“A New Method to Measure the Semantic
Similarity of GO Terms.”</span> <em>Bioinformatics</em> 23 (10):
1274–81.
</div>
<div id="ref-yu2010gosemsim" class="csl-entry">
Yu, Guangchuang, Fei Li, Yide Qin, Xiaochen Bo, Yibo Wu, and Shengqi
Wang. 2010. <span>“GOSemSim: An r Package for Measuring Semantic
Similarity Among GO Terms and Gene Products.”</span>
<em>Bioinformatics</em> 26 (7): 976–78.
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Yohan Lefol.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
