<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>TimeSeries Analysis – PBMC example run • TimeSeriesAnalysis</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/spacelab/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="TimeSeries Analysis – PBMC example run">
<meta property="og:description" content="TimeSeriesAnalysis">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TimeSeriesAnalysis</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Examples
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/TS_analysis_PBMC.html">PBMC analysis</a>
    </li>
    <li>
      <a href="../articles/TS_analysis_murine.html">Murine analysis</a>
    </li>
    <li>
      <a href="../articles/TS_analysis_celegans.html">Celegans analysis</a>
    </li>
  </ul>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/Ylefol/TimeSeriesAnalysis/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="TS_analysis_PBMC_files/htmlwidgets-1.5.4/htmlwidgets.js"></script><script src="TS_analysis_PBMC_files/plotly-binding-4.10.0/plotly.js"></script><script src="TS_analysis_PBMC_files/typedarray-0.1/typedarray.min.js"></script><link href="TS_analysis_PBMC_files/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="TS_analysis_PBMC_files/crosstalk-1.2.0/js/crosstalk.min.js"></script><link href="TS_analysis_PBMC_files/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet">
<script src="TS_analysis_PBMC_files/plotly-main-2.5.1/plotly-latest.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>TimeSeries Analysis – PBMC example run</h1>
                        <h4 data-toc-skip class="author">Yohan
Lefol</h4>
            
            <h4 data-toc-skip class="date">08. August 2022</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/Ylefol/TimeSeriesAnalysis/blob/HEAD/vignettes/TS_analysis_PBMC.Rmd" class="external-link"><code>vignettes/TS_analysis_PBMC.Rmd</code></a></small>
      <div class="hidden name"><code>TS_analysis_PBMC.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="timeseries-analysis">TimeSeries analysis<a class="anchor" aria-label="anchor" href="#timeseries-analysis"></a>
</h2>
<div class="section level3">
<h3 id="parameter-set-up">Parameter set up<a class="anchor" aria-label="anchor" href="#parameter-set-up"></a>
</h3>
<p>Set the below parameters for the downstream analysis</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Give names to saved object and name of results repository</span>
<span class="va">name_result_folder</span><span class="op">&lt;-</span><span class="st">'TS_results_PBMC_example/'</span>
<span class="va">obj_name</span><span class="op">&lt;-</span><span class="st">'timeSeries_obj_PBMC_example.Rdata'</span>

<span class="co">#Path to count data and sample data respectively</span>
<span class="va">my_path_data</span><span class="op">&lt;-</span><span class="st">'data/PBMC/raw_counts_TS'</span>
<span class="va">my_path_sample_dta</span><span class="op">&lt;-</span><span class="st">'data/PBMC/sample_file.csv'</span>

<span class="co">#Set-up time series object parameters</span>
<span class="va">diff_exp_type</span><span class="op">&lt;-</span><span class="st">'DESeq2'</span> <span class="co">#package used for DE – can also be 'limma'</span>
<span class="va">p_val_filter_type</span><span class="op">&lt;-</span><span class="st">'padj'</span> <span class="co">#Either padj or pvalue, used to filter for significance</span>
<span class="va">p_thresh</span><span class="op">&lt;-</span><span class="fl">0.05</span> <span class="co">#pvalue or padj value threshold for significance</span>
<span class="va">l2fc_thresh</span><span class="op">&lt;-</span><span class="fl">1</span> <span class="co">#log(2)foldChange threshold for significance</span>
<span class="va">name_control</span><span class="op">&lt;-</span><span class="st">'LPS'</span> <span class="co">#Name of experiment as seen in the sample file</span>
<span class="va">name_experiment</span><span class="op">&lt;-</span><span class="st">'IgM'</span> <span class="co">#Name of control as seen in the sample file</span>
<span class="va">graphic_vector</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"#e31a1c"</span>,<span class="st">"#1f78b4"</span><span class="op">)</span> <span class="co">#Pre-set colors for the groups</span>

<span class="co">#Declare organism and load library</span>
<span class="va">org_sem_sim</span><span class="op">=</span><span class="st">'org.Hs.eg.db'</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st">'org.Hs.eg.db'</span><span class="op">)</span></code></pre></div>
<pre><code><span class="co">## </span></code></pre>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#mmusculus</span>
<span class="co">#hsapiens</span>
<span class="co">#celegans</span>
<span class="va">my_org_gpro</span><span class="op">=</span><span class="st">'hsapiens'</span> <span class="co">#Set the species for the gprofiler analysis</span>

<span class="va">PART_l2fc</span><span class="op">&lt;-</span><span class="fl">2</span> <span class="co">#log(2)foldChange threshold for PART clustering</span>
<span class="va">PART_min_clust</span><span class="op">&lt;-</span><span class="fl">50</span> <span class="co">#Minimum cluster size for PART</span>
<span class="va">PART_recursion</span><span class="op">&lt;-</span><span class="fl">100</span> <span class="co">#Number of recursions, default is 100, using 10 for example</span>

<span class="va">log_tp_traj</span><span class="op">&lt;-</span><span class="cn">F</span> <span class="co">#Defines if timepoints should be log transformed for illustration purposes</span>

<span class="co">#Used to highlight specific genes regardless of differential gene expression significance</span>
<span class="va">genes_of_interest</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="op">)</span>
<span class="va">genes_of_interest</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'AICDA'</span>,<span class="st">'APOBEC3H'</span>,<span class="st">'APOBEC3F'</span>,<span class="st">'APOBEC3D'</span>,<span class="st">'APOBEC3C'</span>,<span class="st">'APOBEC3G'</span>,<span class="st">'APOBEC3B'</span>,<span class="st">'APOBEC3A'</span>,<span class="st">'SMUG1'</span>,<span class="st">'UNG'</span>,<span class="st">'EGFR'</span><span class="op">)</span>

<span class="co">#Some extra set-up</span>
<span class="va">name_save_obj</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="va">obj_name</span><span class="op">)</span><span class="co">#The object will be saved in result folder</span>
<span class="co">#Create main directory for results</span>
<span class="fu"><a href="https://rdrr.io/r/base/files2.html" class="external-link">dir.create</a></span><span class="op">(</span><span class="va">name_result_folder</span><span class="op">)</span></code></pre></div>
<pre><code><span class="co">## Warning in dir.create(name_result_folder): 'TS_results_PBMC_example' already</span>
<span class="co">## exists</span></code></pre>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">my_group_names</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">name_experiment</span>,<span class="va">name_control</span><span class="op">)</span> 
<span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">graphic_vector</span><span class="op">)</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">name_experiment</span>,<span class="va">name_control</span><span class="op">)</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="create-timeseries-object">Create TimeSeries object<a class="anchor" aria-label="anchor" href="#create-timeseries-object"></a>
</h3>
<p>This code chunk first checks if a timeseries object has been saved,
if that is the case, the object is loaded instead of being created
again. To reset this analysis, the saved timeseries object should be
deleted or moved from the repository.<br>
If no timeseries object was found, the code chunk creates the timeseries
object using the parameters defined above. The raw matrix is also
created using the second function</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw">if</span><span class="op">(</span><span class="va">obj_name</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/list.files.html" class="external-link">list.files</a></span><span class="op">(</span><span class="va">name_result_folder</span><span class="op">)</span><span class="op">==</span><span class="cn">F</span><span class="op">)</span><span class="op">{</span>
  <span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/methods/new.html" class="external-link">new</a></span><span class="op">(</span><span class="st">'TimeSeries_Object'</span>,sample_data<span class="op">=</span><span class="fu"><a href="../reference/prep_sample_data.html">prep_sample_data</a></span><span class="op">(</span><span class="va">my_path_sample_dta</span>,<span class="va">my_group_names</span><span class="op">)</span>,
                 group_names<span class="op">=</span><span class="va">my_group_names</span>,group_colors<span class="op">=</span><span class="va">graphic_vector</span>,DE_method<span class="op">=</span><span class="va">diff_exp_type</span>,
                 DE_p_filter<span class="op">=</span><span class="va">p_val_filter_type</span>,DE_p_thresh<span class="op">=</span><span class="va">p_thresh</span>,DE_l2fc_thresh<span class="op">=</span><span class="va">l2fc_thresh</span>,
                 PART_l2fc_thresh<span class="op">=</span><span class="va">PART_l2fc</span>,sem_sim_org<span class="op">=</span><span class="va">org_sem_sim</span>,Gpro_org<span class="op">=</span><span class="va">my_org_gpro</span><span class="op">)</span>
<span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/create_raw_count_matrix.html">create_raw_count_matrix</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">my_path_data</span><span class="op">)</span>
<span class="op">}</span><span class="kw">else</span><span class="op">{</span>
  <span class="fu"><a href="https://rdrr.io/r/base/load.html" class="external-link">load</a></span><span class="op">(</span><span class="va">name_save_obj</span><span class="op">)</span>
<span class="op">}</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="differental-gene-expression-analysis">Differental gene expression analysis<a class="anchor" aria-label="anchor" href="#differental-gene-expression-analysis"></a>
</h3>
<p>Loads the necessary functions to perform differential gene expression
analysis If the tool used is DESeq2<span class="citation">(Love, Huber,
and Anders 2014)</span>, the data needs to be normalized using it’s
method. If the tool used is limma<span class="citation">(Smyth
2005)</span>, the normalized matrix should have been inputed, and thus
no normalization is needed.</p>
<p>This code chunk performs both the conditional and temporal
differential gene expression and saves the results within the Time
Series object.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Perform normalization if the DESeq2 tool is being used and if normalized matrix doesn't exist</span>
<span class="kw">if</span> <span class="op">(</span><span class="va">TS_object</span><span class="op">@</span><span class="va">DE_method</span><span class="op">==</span><span class="st">'DESeq2'</span> <span class="op">&amp;</span> <span class="st">'norm'</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">@</span><span class="va">count_matrix</span><span class="op">)</span><span class="op">!=</span><span class="cn">T</span><span class="op">)</span><span class="op">{</span>
  <span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/normalize_timeSeries_with_deseq2.html">normalize_timeSeries_with_deseq2</a></span><span class="op">(</span>time_object<span class="op">=</span><span class="va">TS_object</span><span class="op">)</span>
<span class="op">}</span>

<span class="co">#Perform conditional differential gene expression analysis</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/conditional_DE_wrapper.html">conditional_DE_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span>
<span class="co">#Perform temporal differential gene expression analysis</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/temporal_DE_wrapper.html">temporal_DE_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span>

<span class="co">#save the timeseries object</span>
<span class="fu"><a href="https://rdrr.io/r/base/save.html" class="external-link">save</a></span><span class="op">(</span><span class="va">TS_object</span>,file<span class="op">=</span><span class="va">name_save_obj</span><span class="op">)</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="part-clustering">PART clustering<a class="anchor" aria-label="anchor" href="#part-clustering"></a>
</h3>
<p>The code chunk below prepares and initiates PART clustering<span class="citation">(Nilsen et al. 2013)</span>. It first retrieves the
number of significant genes for PART clustering in the ‘signi_genes’
variable This chunk can be quite lengthy depending on the number of
genes included for clustering as well as the number of recursions set
for the clustering</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Extract genes for PART clustering based on defined log(2)foldChange threshold</span>
<span class="va">signi_genes</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/select_genes_with_l2fc.html">select_genes_with_l2fc</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span>

<span class="co">#Use all samples, but implement a custom order. In this case it is reversed</span>
<span class="va">samps_2</span><span class="op">&lt;-</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sample_data</span><span class="op">$</span><span class="va">sample</span><span class="op">[</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sample_data</span><span class="op">$</span><span class="va">group</span><span class="op">==</span><span class="va">TS_object</span><span class="op">@</span><span class="va">group_names</span><span class="op">[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span>
<span class="va">samps_1</span><span class="op">&lt;-</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sample_data</span><span class="op">$</span><span class="va">sample</span><span class="op">[</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sample_data</span><span class="op">$</span><span class="va">group</span><span class="op">==</span><span class="va">TS_object</span><span class="op">@</span><span class="va">group_names</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span>

<span class="co">#Create the matrix that will be used for PART clustering</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/prep_counts_for_PART.html">prep_counts_for_PART</a></span><span class="op">(</span>object<span class="op">=</span><span class="va">TS_object</span>,target_genes<span class="op">=</span><span class="va">signi_genes</span>,
                                     scale<span class="op">=</span><span class="cn">T</span>,target_samples<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">samps_2</span>,<span class="va">samps_1</span><span class="op">)</span><span class="op">)</span>

<span class="co">#Initiate PART clustering with a pre-defined seed – 123456</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/compute_PART.html">compute_PART</a></span><span class="op">(</span><span class="va">TS_object</span>,part_recursion<span class="op">=</span><span class="va">PART_recursion</span>,part_min_clust<span class="op">=</span><span class="va">PART_min_clust</span>,
                        dist_param<span class="op">=</span><span class="st">"euclidean"</span>, hclust_param<span class="op">=</span><span class="st">"average"</span>,
                        custom_seed<span class="op">=</span><span class="fl">123456</span><span class="op">)</span>
<span class="co">#Save the TimeSeries object to the directory to prevent loss of data in the event</span>
<span class="co">#of a downstream error</span>
<span class="fu"><a href="https://rdrr.io/r/base/save.html" class="external-link">save</a></span><span class="op">(</span><span class="va">TS_object</span>,file<span class="op">=</span><span class="va">name_save_obj</span><span class="op">)</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="gprofiler-analysis">Gprofiler analysis<a class="anchor" aria-label="anchor" href="#gprofiler-analysis"></a>
</h3>
<p>The code chunk below runs a gprofiler analysis<span class="citation">(Kolberg et al. 2020; Raudvere et al. 2019)</span>.
IMPORTANT: This function requires that there be a stable internet
connection lack of connection or intermittent drops in the connection
will result in an error and the termination (stop) of the code chunk</p>
<p>If an error has occured, this code chunk can be re-run separately
from the above chunks by uncommenting (removing the ‘#’ in front) the
“load(‘timeseries_obj_res.Rdata’)” line.</p>
<p>This will load the results saved after the PART clustering code
chunk</p>
<p>This code chunk will overwrite the saved object if it is completed.
The overwritten object will then contain the gprofiler analysis results
and can be used to generate plots with the downstream code chunks.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># load('timeseries_obj_res.Rdata')</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/run_gprofiler_PART_clusters.html">run_gprofiler_PART_clusters</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span> <span class="co">#Run the gprofiler analysis</span>
<span class="co">#Save the results in a Rdata obbject</span>
<span class="fu"><a href="https://rdrr.io/r/base/save.html" class="external-link">save</a></span><span class="op">(</span><span class="va">TS_object</span>,file<span class="op">=</span><span class="va">name_save_obj</span><span class="op">)</span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="timeseries-analysis-results">TimeSeries analysis results<a class="anchor" aria-label="anchor" href="#timeseries-analysis-results"></a>
</h2>
<p>Most plots are created in SVG format for ease of editing with SVG
editing software such as InkScape (open source software). Some plots are
created in html format as they are interactive plots. These require a
web browser to open them</p>
<p>To convert SVG files to PNG/JPG/PDF, this website is available: <a href="https://svgtopng.com/" class="external-link uri">https://svgtopng.com/</a><br>
HTML files can be opened and then saved as PNG (or other) using the
camera icon in the top right of each respective interactive plot</p>
<div class="section level3">
<h3 id="timeseries-summary-of-experiment">TimeSeries summary of experiment<a class="anchor" aria-label="anchor" href="#timeseries-summary-of-experiment"></a>
</h3>
<p><strong>Name of groups beings compared:</strong> IgM vs LPS<br><strong>Number of genes analyzed:</strong> 26485</p>
<p><strong>Differential expression parameters used:</strong><br><strong>Method:</strong> DESeq2<br><strong>P statistic filter used:</strong> padj with a 0.05
threshold<br><strong>Log(2)FoldChange significance threshold used:</strong> 1</p>
<p><strong>PART parameters used:</strong><br><strong>PART log(2)FoldChange significance threshold:</strong> 2<br><strong>number of recursions:</strong> 100<br><strong>minimum cluster size:</strong> 50<br><strong>distance parameter:</strong> euclidean<br><strong>hclust parameter:</strong> average<br><strong>custom seed used:</strong> 123456<br><strong>PART computation time:</strong> 683.532 sec elapsed</p>
<p><strong>PCA of sample data:</strong><br><img src="TS_results_PBMC_example/PCA_plot.png" width="2480"></p>
</div>
<div class="section level3">
<h3 id="differential-gene-expression-results">Differential Gene Expression results<a class="anchor" aria-label="anchor" href="#differential-gene-expression-results"></a>
</h3>
<p><strong>DEGs found per analysis:</strong></p>
<table class="table">
<thead><tr class="header">
<th align="left">experiment.name</th>
<th align="left">number.of.DEGs</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">IgM_vs_LPS_TP_1 (conditional)</td>
<td align="left">1763</td>
</tr>
<tr class="even">
<td align="left">IgM_vs_LPS_TP_3 (conditional)</td>
<td align="left">1369</td>
</tr>
<tr class="odd">
<td align="left">IgM_vs_LPS_TP_9 (conditional)</td>
<td align="left">1568</td>
</tr>
<tr class="even">
<td align="left">TP_3_vs_TP_1 (temporal)</td>
<td align="left">957</td>
</tr>
<tr class="odd">
<td align="left">TP_9_vs_TP_3 (temporal)</td>
<td align="left">86</td>
</tr>
<tr class="even">
<td align="left"><strong>total unique genes</strong></td>
<td align="left"><strong>3884</strong></td>
</tr>
</tbody>
</table>
<p>Each experiment listed in the above table has separate results which
can be viewed in the ‘TS_results’ folder. In the main results folder,
the differential gene expression results are located in the
‘DE_results_conditional’ and ‘DE_results_temporal’ plot.<br>
Each individual experiment creates the following:<br><strong>DE_raw_data.csv</strong>: A csv file containing all the genes
and their associated differential expression results, regardless of
significance.<br><strong>DE_sig_data</strong>: A csv file containing only the significant
differential expression results.<br><strong>MA_plot.png</strong>: A MA plot, these are often used to
evaluate the quality of the normalization.<br><strong>volcano_plot.png</strong>: A volcano plot which splits
significantly up-regulated and down-regulated genes as well as
non-significant genes.<br><strong>PCA_plot</strong>: A PCA plot using only the samples implicated
in the differential gene expression analysis.</p>
<p>The pipeline also creates two large heatmaps which summarize
conditional and temporal differential expression results. The heatmaps
were created using the ComplexHeatmaps package<span class="citation">(Gu, Eils, and Schlesner 2016)</span>.<br>
These heatmaps illustrate each experiment as different columns
(distinguished by color) while the rows distinguish the groups. Plotted
in the heatmaps is the log transformed counts, while the log transformed
log(2)fold change is seen in a histogram at the bottom of the heatmap.
The number of genes in each experiment (colored columns) is indicated in
the legend at the bottom of the heatmap.<br>
In the case of the temporal heatmap, the rows are labelled ‘experiment’
and ‘control’. The ‘experiment’ represents the time points being
compared while the control represents the timepoint which the
‘experiment’ is compared against. For example, TP2_vs_TP1 would have the
‘experiment’ be TP2 and the control be TP1, while TP3_vs_TP2 would have
TP3 as the ‘experiment’ and TP2 as the ‘control’. Note that TP=time
point.</p>
<p><strong>The conditional heatmap</strong><br><img src="TS_results_PBMC_example/DE_results_conditional/custom_heat_conditional.svg"><!-- --></p>
<p><strong>The temporal heatmap</strong><br><img src="TS_results_PBMC_example/DE_results_temporal/custom_heat_temporal.svg"><!-- --></p>
</div>
<div class="section level3">
<h3 id="genes-of-interest-results">Genes of interest results<a class="anchor" aria-label="anchor" href="#genes-of-interest-results"></a>
</h3>
<p><img src="TS_results_PBMC_example/genes_of_interest/APOBEC3G_trajectory.png" width="65%" style="float:right; padding:10px">
If genes of interest have been specified, the pipeline will extract the
differential gene expression results, both conditional and temporal, for
each gene requested, provided that they are found in the count matrix.
These data files, along with individual trajectory plots are stored in
the ‘TS_results/genes_of_interest’ folder. Below the trajectory of the
most variable gene is shown. If no genes of interest were submitted, a
random gene is taken from the differential expression results.
Illustrated in the trajectory plots is the timepoints on the x axis and
the number of reads (RNAseq) or expression (microarray) on the y
axis.<br>
Note that data files and trajectory plots are created for all genes of
interest found, regardless of their significance based on the
differential gene expression analyses.</p>
</div>
<div class="section level3">
<h3 id="cluster-results">Cluster results<a class="anchor" aria-label="anchor" href="#cluster-results"></a>
</h3>
<p>The code chunk below create the trajectory plots for the clusters
found by PART Trajectories are first calculated by calculating the mean
of each gene for each time point using the available replicates. The
mean value of each gene is calculated once for each group
(control/experiment). Afterwards, the genes are scaled in order to be
able to represent them on the same axis of values. Each gene is plotted
as an individual colored line’, a mean line (gray color) is also plotted
to show the overall trajectory of the cluster. These plots were inspired
by <span class="citation">(Nguyen 2021)</span></p>
<p>The log_tp_traj parameter defines if the time points should be log
transformed. A log transformation of the time points is useful if the
time points are not evenly sampled through time. For example, two time
points at 0 and 15 minutes, with another time point at 24 hours (1440
minutes) will result in the first two time points being very compressed
and possibly indistinguishable from each other.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/files2.html" class="external-link">dir.create</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="st">'PART_results'</span><span class="op">)</span><span class="op">)</span> <span class="co">#create the directory to store results</span>
<span class="fu"><a href="../reference/PART_heat_map.html">PART_heat_map</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="st">'PART_results/PART_heat'</span><span class="op">)</span><span class="op">)</span> <span class="co">#Create a summary heatmap</span>
<span class="va">ts_data</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/calculate_cluster_traj_data.html">calculate_cluster_traj_data</a></span><span class="op">(</span><span class="va">TS_object</span>,scale_feat<span class="op">=</span><span class="cn">T</span><span class="op">)</span> <span class="co">#Calculate scaled gene values for genes of clusters</span>
<span class="va">mean_ts_data</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/calculate_mean_cluster_traj.html">calculate_mean_cluster_traj</a></span><span class="op">(</span><span class="va">ts_data</span><span class="op">)</span> <span class="co">#Calculate the mean scaled values for each cluster</span>

<span class="co">#Function which determines the number of SVG files to plot all cluster trajectories</span>
<span class="fu"><a href="../reference/wrapper_cluster_trajectory.html">wrapper_cluster_trajectory</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">ts_data</span>,<span class="va">mean_ts_data</span>,log_TP<span class="op">=</span><span class="va">log_tp_traj</span>,plot_name<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="st">'/PART_results/Ctraj'</span><span class="op">)</span><span class="op">)</span>

<span class="co">#find the most variable cluster, subset necessary data and plot it on a single column</span>
<span class="co">#The function will also filter for a cluster which has gprofiler results</span>
<span class="va">target_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_most_variable_cluster.html">find_most_variable_cluster</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">mean_ts_data</span><span class="op">)</span>
<span class="va">sub_ts</span><span class="op">&lt;-</span><span class="va">ts_data</span><span class="op">[</span><span class="va">ts_data</span><span class="op">$</span><span class="va">cluster</span><span class="op">==</span><span class="va">target_clust</span>,<span class="op">]</span>
<span class="va">sub_mean</span><span class="op">&lt;-</span><span class="va">mean_ts_data</span><span class="op">[</span><span class="va">mean_ts_data</span><span class="op">$</span><span class="va">cluster</span><span class="op">==</span><span class="va">target_clust</span>,<span class="op">]</span>
<span class="va">plt_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/plot_cluster_traj.html">plot_cluster_traj</a></span><span class="op">(</span>object <span class="op">=</span> <span class="va">TS_object</span>,ts_data <span class="op">=</span> <span class="va">sub_ts</span>,ts_mean_data <span class="op">=</span> <span class="va">sub_mean</span>, num_col<span class="op">=</span><span class="fl">1</span>,rem_legend_axis<span class="op">=</span><span class="cn">T</span>,log_TP<span class="op">=</span><span class="va">log_tp_traj</span><span class="op">)</span>
<span class="va">cluster_map_sum</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">@</span><span class="va">PART_results</span><span class="op">$</span><span class="va">cluster_map</span><span class="op">[</span>,<span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">1</span><span class="op">)</span><span class="op">]</span><span class="op">)</span><span class="op">)</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/base/row.names.html" class="external-link">row.names</a></span><span class="op">(</span><span class="va">cluster_map_sum</span><span class="op">)</span><span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'Cluster name'</span>,<span class="st">'Number of genes'</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/pkg/AnnotationDbi/man/Bimap-toTable.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">cluster_map_sum</span><span class="op">)</span><span class="op">=</span><span class="cn">NULL</span></code></pre></div>
<p><strong>PART result summary:</strong></p>
<table class="table">
<colgroup>
<col width="23%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="5%">
<col width="4%">
<col width="5%">
<col width="5%">
<col width="4%">
<col width="5%">
<col width="4%">
</colgroup>
<tbody>
<tr class="odd">
<td align="left">Cluster name</td>
<td align="left">C1</td>
<td align="left">C10</td>
<td align="left">C11</td>
<td align="left">C12</td>
<td align="left">C13</td>
<td align="left">C14</td>
<td align="left">C2</td>
<td align="left">C3</td>
<td align="left">C4</td>
<td align="left">C5</td>
<td align="left">C6</td>
<td align="left">C7</td>
<td align="left">C8</td>
<td align="left">C9</td>
</tr>
<tr class="even">
<td align="left">Number of genes</td>
<td align="left">165</td>
<td align="left">56</td>
<td align="left">101</td>
<td align="left">50</td>
<td align="left">126</td>
<td align="left">178</td>
<td align="left">263</td>
<td align="left">162</td>
<td align="left">80</td>
<td align="left">150</td>
<td align="left">115</td>
<td align="left">67</td>
<td align="left">119</td>
<td align="left">59</td>
</tr>
</tbody>
</table>
<p>All cluster results are found in the ‘PART_results’ folder, within
this folder you will find the following:</p>
<p><strong>PART_heat.svg</strong>: A svg heatmap of the PART result –
detailed below.<br><strong>PART_heat_with_names.svg</strong>: The same plot as above but
with sample and gene names included. These may not be visible depending
on the number of genes included as well as the length of the sample
names.<br><strong>PART_heat_cmap.csv</strong>: A csv file representing a
‘cluster_map’ which provides the cluster in which each gene was sorted
in as well as the color (in HEX code) given to that cluster.<br><strong>PART_heat_data.csv</strong>: The z-score values used in the
‘PART_heat’ heatmaps.<br><strong>Ctraj.svg</strong>: At least one, and possibly several of these
files will be in the folder. This/these SVG plots show the trajectory of
each cluster individually while being split by group. Meaning the
trajectory of each cluster is shown twice, once per group. Information
concerning the cluster and group is shown in the title of each subplot.
One example of cluster trajectory is shown below.</p>
<p><strong>PART heatmap and cluster trajectory</strong><br>
In this heatmap we illustrate all the genes which were submitted for
PART clustering. Each gene is illustrated using a Z-score, which
measures the number of standard deviations that a gene has from it’s
mean. In essence it measures the variability.<br>
Clusters are shown as the colored bar on the left hand side of the plot
while groups and timepoints are shown as the colored bars on the top of
the heatmap. PART heatmaps were created with ComplexHeatmaps<span class="citation">(Gu, Eils, and Schlesner 2016)</span>.<br>
The cluster trajectory of the most variable cluster (C4) is illustrated,
with the trajectory of the experiment group at the top and the control
group at the bottom. The x axis represents timepoints and the y axis
represents the scaled mean expression of the genes. The expression of
each gene is obtained by calculating the mean between all replicates for
each time point in each group. The values are then scaled in order to
illustrate their trajectory. The overall mean of the cluster is shown as
a thicker gray line.</p>
<p><img src="TS_results_PBMC_example/PART_results/PART_heat.svg" width="49%" height="49%"><img src="TS_analysis_PBMC_files/figure-html/test,figures-side-2.png" width="49%" height="49%"></p>
</div>
<div class="section level3">
<h3 id="gprofiler-results-standard">Gprofiler results – standard<a class="anchor" aria-label="anchor" href="#gprofiler-results-standard"></a>
</h3>
<p>The below chunk creates the gprofiler result. Some parameters
relating to the species and ontology need to be defined The species
parameter are used to calculate the semantic distance which impacts the
Multi-Dimensional Scaling (MDS) plots that will be generated by the last
function of this code chunk. Both versions of the MDS plots seen below
were inspired by <span class="citation">(Brionne, Juanchich, and
Hennequet-Antier 2019)</span>.</p>
<p>The ontology needs to be defined for the MDS plots as well. Note that
MDS plots utilise the semantic similarity metric which currently only
supports GO ontology (MF – MolecularFunctions, CC – Cellular Components,
and BP – Biological processes).<br>
The first function plots the standard gprofiler plots (gost plots) to
the directory, and it retrieves the information of the requested
ontology into the GO_clusters dataframe. This dataframe is then used
with the semantic distance to create two versions of the MDS plot.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Define specimen and ontology parameters</span>
<span class="va">my_ont_gpro</span><span class="op">=</span><span class="st">'GO:BP'</span>
<span class="va">my_ont_sem_sim</span><span class="op">=</span><span class="st">'BP'</span>

<span class="co">#Create standard gprofiler results</span>
<span class="va">gpro_res</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/gprofiler_cluster_analysis.html">gprofiler_cluster_analysis</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">my_ont_gpro</span>,save_path <span class="op">=</span> <span class="va">name_result_folder</span>,return_specific_cluster<span class="op">=</span><span class="va">target_clust</span><span class="op">)</span>
<span class="va">GO_clusters</span><span class="op">&lt;-</span><span class="va">gpro_res</span><span class="op">[[</span><span class="st">'GO_df'</span><span class="op">]</span><span class="op">]</span>
<span class="co">#Create semantic data</span>
<span class="va">sem_data</span> <span class="op">&lt;-</span> <span class="fu">godata</span><span class="op">(</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sem_sim_org</span>, ont<span class="op">=</span><span class="va">my_ont_sem_sim</span>, computeIC<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span>


<span class="va">found_clusters</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_clusters_from_termdist.html">find_clusters_from_termdist</a></span><span class="op">(</span><span class="va">GO_clusters</span>,<span class="va">sem_data</span><span class="op">)</span>
<span class="va">clusters</span><span class="op">&lt;-</span><span class="va">found_clusters</span><span class="op">[</span>,<span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'GO.ID'</span>,<span class="st">'GO.cluster'</span><span class="op">)</span><span class="op">]</span>
<span class="va">clusters</span><span class="op">&lt;-</span><span class="fu">as.data.table</span><span class="op">(</span><span class="va">clusters</span><span class="op">)</span>
<span class="va">clusters</span><span class="op">[</span>,<span class="st">"GO.cluster"</span><span class="op">:=</span><span class="fu"><a href="https://rdrr.io/r/base/factor.html" class="external-link">factor</a></span><span class="op">(</span><span class="va">clusters</span><span class="op">$</span><span class="va">GO.cluster</span>,levels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">clusters</span><span class="op">$</span><span class="va">GO.cluster</span><span class="op">)</span><span class="op">)</span><span class="op">]</span>

<span class="co">#Plot and save MDS and clustered MDS</span>
<span class="va">MDS_plots</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/wrapper_MDS_and_MDS_clusters.html">wrapper_MDS_and_MDS_clusters</a></span><span class="op">(</span><span class="va">GO_clusters</span>,<span class="va">sem_data</span>,<span class="va">my_ont_sem_sim</span>,target_dir<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="st">'gprofiler_results/'</span><span class="op">)</span>,return_plot<span class="op">=</span><span class="cn">T</span><span class="op">)</span></code></pre></div>
<p>Each cluster has it’s own individual gprofiler plot, these are found
in ‘TS_results/gprofiler_results/figures’. The gost plot for the most
variable cluster (C4) is shown below.<br>
These interactive plots show the results of various databases when
queried with a list of genes. More information concerning these plots
and their various datasources can be found at the following link: <a href="https://biit.cs.ut.ee/gprofiler/gost" class="external-link uri">https://biit.cs.ut.ee/gprofiler/gost</a></p>
<div id="htmlwidget-cf8d7f708f9f918cde8b" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-cf8d7f708f9f918cde8b">{"x":{"data":[{"x":[157.656457444648],"y":[1.30253438890085],"text":"CORUM:5691 (1) <br> TALL1 homo-oligomer complex <br> 4.983e-02","key":["CORUM:5691"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(102,170,0,1)","opacity":0.8,"size":3.77952755905512,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(102,170,0,1)"}},"hoveron":"points","set":"SharedData675e75f2","name":"CORUM","legendgroup":"CORUM","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[31.67617005686],"y":[1.7803482700674],"text":"GO:0030054 (2086) <br> cell junction <br> 1.658e-02","key":["GO:0030054"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":22.6771653543307,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData675e75f2","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[109.963818739485],"y":[1.41709254642366],"text":"REAC:R-HSA-177135 (6) <br> Conjugation of benzoate with glycine <br> 3.827e-02","key":["REAC:R-HSA-177135"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":12.9294063682454,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedData675e75f2","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[2,27.4892034473753],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[29.5576286236615,38.8014207364847],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[40.869845912771,103.915445285975],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[105.983870462262,107.125641159572],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[109.194066335858,114.532671715853],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[116.601096892139,140.679634369287],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(85,116,166,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[142.748059545573,148.125965003917],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(34,170,153,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[150.194390180204,151.919456777226],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(102,51,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[153.987881953513,160.02147819274],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(102,170,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[162.089903369026,196.386461217028],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(153,0,153,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[198.454886393314,200.002068425176],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(0,153,198,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":61.6829876535158,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG","REAC","TF","MIRNA","HPA","CORUM","HP","WP"],"tickvals":[14.7446017236876,34.1795246800731,72.3926455993732,106.554755810917,111.863369025855,128.640365630713,145.437012274745,151.056923478715,157.004680073126,179.238182293027,199.228477409245],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG","REAC","TF","MIRNA","HPA","CORUM","HP","WP"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-37.3931091739311},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"15346100f2a6d":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"153462cfd92":{"x":{},"xend":{},"y":{},"yend":{}},"153463945141f":{"x":{},"xend":{},"y":{},"yend":{}},"15346335cab4f":{"x":{},"xend":{},"y":{},"yend":{}},"153464cbe00a2":{"x":{},"xend":{},"y":{},"yend":{}},"153463ee089c9":{"x":{},"xend":{},"y":{},"yend":{}},"15346ecec54b":{"x":{},"xend":{},"y":{},"yend":{}},"153467d138c06":{"x":{},"xend":{},"y":{},"yend":{}},"153467e3df2b":{"x":{},"xend":{},"y":{},"yend":{}},"15346553a3e3b":{"x":{},"xend":{},"y":{},"yend":{}},"1534622472030":{"x":{},"xend":{},"y":{},"yend":{}},"15346785831ca":{"x":{},"xend":{},"y":{},"yend":{}},"153464fbd2a48":{"x":{},"y":{}},"1534661ba569f":{"yintercept":{}}},"cur_data":"15346100f2a6d","visdat":{"15346100f2a6d":["function (y) ","x"],"153462cfd92":["function (y) ","x"],"153463945141f":["function (y) ","x"],"15346335cab4f":["function (y) ","x"],"153464cbe00a2":["function (y) ","x"],"153463ee089c9":["function (y) ","x"],"15346ecec54b":["function (y) ","x"],"153467d138c06":["function (y) ","x"],"153467e3df2b":["function (y) ","x"],"15346553a3e3b":["function (y) ","x"],"1534622472030":["function (y) ","x"],"15346785831ca":["function (y) ","x"],"153464fbd2a48":["function (y) ","x"],"1534661ba569f":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData675e75f2"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>In addition to the standard gprofiler plot, the TimeSeries pipeline
creates Multi-Dimensional Scaling (MDS) plots. These plots illustrate a
custom selected ontology defined by the ‘my_ont_gpro’ parameter. The
first MDS plots shows all the found terms of the specified ontology and
distinguishes them based on the cluster(s) in which they are
found.<br>
In a MDS plot, the distance between each dot represents the semantic
similarity between terms<span class="citation">(Yu et al. 2010; Wang et
al. 2007)</span>. This can be loosely defined as the ‘closeness’ between
terms. Semantic similarity takes into account many factors, such as the
content of the terms (genes which belong to it), it’s associations to
other terms, the similarity or lack there-of in function etc…<br>
In an MDS plot, the dimensions will be adjusted to fill the space with
all the terms, therefor there is no set metric to measure the
similarity. The purpose of the MDS plot is moreso to help visualize
different groupings (or clusters) of relevant terms. Hovering the cursor
over the dots will provide specific information for that dot (term)</p>
<div id="htmlwidget-4b56424ae910ad4a97b0" style="width:100%;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4b56424ae910ad4a97b0">{"x":{"visdat":{"153461e57c5f5":["function () ","plotlyVisDat"],"153464fa07b75":["function () ","data"]},"cur_data":"153464fa07b75","attrs":{"153464fa07b75":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","color":{},"text":{},"showlegend":true,"colors":["#DE8B88","#C4D38D","#C5CA84","#CBB278","#ECC29F","#C8D883","#D2CE83","#D1CA78","#C3D58A","#C8D086","#C4B3AD","#C7CEB4","#C1D88E","#BFD391","#BEC591","#D6A29F","#C7BE83","#EAA998","#D1C180","#E39F7D","#BAB984","#E38E8C","#E0C464","#D4D6A2","#C6D884","#D7D4CB","#CAEABC","#C1E292","#B8DA68","#FAF9B6","#D9E98F","#D4D77E","#CBCFB9","#F1C4A3","#D4CF85","#BFCBB4","#E9B373","#9CA5BC","#E89191","#99C2BB","#BCC38F","#C8CC98","#A6B1B0","#C3BA8A","#CBC895"],"marker":{"size":20,"opacity":0.4,"showlegend":true},"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":true},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"x":[0.302394852152548,0.2166506913993,0.30107937273155,0.254661915683609,0.222565793154643,0.291042542806205,0.199864225794277,0.350762493192941,0.22271035246039,0.254287997349565,0.15405814492222,0.184216115625165,-0.0989218971137406,0.205963880368187,0.21075634370242,0.115389017590221,0.0952552229650151,0.208905800654267,0.216504686220069,0.233275514383421,0.196092037608803,0.297586862437082,0.0935868813251129,0.0745363914769651,0.117283296144537],"y":[-0.159498206249343,-0.230161126987755,-0.151363614069294,0.152969459027752,-0.226959008700088,-0.145840568254577,-0.222690230274192,-0.175568091423454,0.0754478231175654,0.10469835382261,-0.202090398449043,-0.0743052527539749,0.258203332602715,-0.193046572992229,0.183185292077881,-0.0392216688919525,-0.0322339564588516,-0.219655673856159,0.0720188033313314,0.108810475790565,-0.200135938353854,-0.15769608091774,-0.0924179297035599,0.284357233926079,-0.0383246604933612],"type":"scatter","mode":"markers","text":["modules/clusters: C1 <br>GO.ID: GO:0048646 <br>GO.name: anatomical structure formation involved in morphogenesis","modules/clusters: C1 <br>GO.ID: GO:0022008 <br>GO.name: neurogenesis","modules/clusters: C1 <br>GO.ID: GO:0048514 <br>GO.name: blood vessel morphogenesis","modules/clusters: C1 <br>GO.ID: GO:2000026 <br>GO.name: regulation of multicellular organismal development","modules/clusters: C1 <br>GO.ID: GO:0048699 <br>GO.name: generation of neurons","modules/clusters: C1 <br>GO.ID: GO:0001525 <br>GO.name: angiogenesis","modules/clusters: C1 <br>GO.ID: GO:0030182 <br>GO.name: neuron differentiation","modules/clusters: C1 <br>GO.ID: GO:0072359 <br>GO.name: circulatory system development","modules/clusters: C1 <br>GO.ID: GO:0045765 <br>GO.name: regulation of angiogenesis","modules/clusters: C1 <br>GO.ID: GO:1901342 <br>GO.name: regulation of vasculature development","modules/clusters: C1 <br>GO.ID: GO:0000904 <br>GO.name: cell morphogenesis involved in differentiation","modules/clusters: C1 <br>GO.ID: GO:0032963 <br>GO.name: collagen metabolic process","modules/clusters: C1 <br>GO.ID: GO:0030155 <br>GO.name: regulation of cell adhesion","modules/clusters: C1 <br>GO.ID: GO:0060113 <br>GO.name: inner ear receptor cell differentiation","modules/clusters: C1 <br>GO.ID: GO:0022603 <br>GO.name: regulation of anatomical structure morphogenesis","modules/clusters: C1 <br>GO.ID: GO:0006030 <br>GO.name: chitin metabolic process","modules/clusters: C1 <br>GO.ID: GO:0006032 <br>GO.name: chitin catabolic process","modules/clusters: C1 <br>GO.ID: GO:0042490 <br>GO.name: mechanoreceptor differentiation","modules/clusters: C1 <br>GO.ID: GO:0045766 <br>GO.name: positive regulation of angiogenesis","modules/clusters: C1 <br>GO.ID: GO:1904018 <br>GO.name: positive regulation of vasculature development","modules/clusters: C1 <br>GO.ID: GO:0048667 <br>GO.name: cell morphogenesis involved in neuron differentiation","modules/clusters: C1 <br>GO.ID: GO:0000902 <br>GO.name: cell morphogenesis","modules/clusters: C1 <br>GO.ID: GO:1905517 <br>GO.name: macrophage migration","modules/clusters: C1 <br>GO.ID: GO:0040013 <br>GO.name: negative regulation of locomotion","modules/clusters: C1 <br>GO.ID: GO:1901072 <br>GO.name: glucosamine-containing compound catabolic process"],"showlegend":true,"marker":{"color":"rgba(222,139,136,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(222,139,136,1)"}},"name":"C1","textfont":{"color":"rgba(222,139,136,1)"},"error_y":{"color":"rgba(222,139,136,1)"},"error_x":{"color":"rgba(222,139,136,1)"},"line":{"color":"rgba(222,139,136,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.253821836409357],"y":[-0.0628614771437084],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C12/C13/C14 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","showlegend":true,"marker":{"color":"rgba(196,211,141,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(196,211,141,1)"}},"name":"C1/C12/C13/C14","textfont":{"color":"rgba(196,211,141,1)"},"error_y":{"color":"rgba(196,211,141,1)"},"error_x":{"color":"rgba(196,211,141,1)"},"line":{"color":"rgba(196,211,141,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.311925399355197],"y":[-0.026937072169365],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C13/C14 <br>GO.ID: GO:0009605 <br>GO.name: response to external stimulus","showlegend":true,"marker":{"color":"rgba(197,202,132,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(197,202,132,1)"}},"name":"C1/C13/C14","textfont":{"color":"rgba(197,202,132,1)"},"error_y":{"color":"rgba(197,202,132,1)"},"error_x":{"color":"rgba(197,202,132,1)"},"line":{"color":"rgba(197,202,132,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.030284955759422,0.0919537946532616,-0.0934296291563106,-0.0506982228119052,0.0123403894975279,0.254090441307748,0.190845571508676,0.0626355059547488],"y":[0.197395134392619,0.33594710446833,0.252884800159881,0.223278948623899,-0.185923470686256,-0.0160309775268336,0.326165488228099,-0.117687655143861],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C14 <br>GO.ID: GO:0030334 <br>GO.name: regulation of cell migration","modules/clusters: C1/C14 <br>GO.ID: GO:0040012 <br>GO.name: regulation of locomotion","modules/clusters: C1/C14 <br>GO.ID: GO:0051270 <br>GO.name: regulation of cellular component movement","modules/clusters: C1/C14 <br>GO.ID: GO:2000145 <br>GO.name: regulation of cell motility","modules/clusters: C1/C14 <br>GO.ID: GO:0001775 <br>GO.name: cell activation","modules/clusters: C1/C14 <br>GO.ID: GO:0006954 <br>GO.name: inflammatory response","modules/clusters: C1/C14 <br>GO.ID: GO:0032101 <br>GO.name: regulation of response to external stimulus","modules/clusters: C1/C14 <br>GO.ID: GO:0045321 <br>GO.name: leukocyte activation"],"showlegend":true,"marker":{"color":"rgba(203,178,120,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(203,178,120,1)"}},"name":"C1/C14","textfont":{"color":"rgba(203,178,120,1)"},"error_y":{"color":"rgba(203,178,120,1)"},"error_x":{"color":"rgba(203,178,120,1)"},"line":{"color":"rgba(203,178,120,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.325036224492603,0.196446964503217,0.155044164178345,0.0808022198229966,0.122047323701056,0.346105739657733,0.350164625229108,0.174321183583798],"y":[-0.174228556367302,-0.216689064149494,-0.231011195202548,0.333435127083553,-0.238794819583076,-0.178032118887212,-0.181467086725604,0.246523723151311],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2 <br>GO.ID: GO:0009653 <br>GO.name: anatomical structure morphogenesis","modules/clusters: C1/C2 <br>GO.ID: GO:0048666 <br>GO.name: neuron development","modules/clusters: C1/C2 <br>GO.ID: GO:0048468 <br>GO.name: cell development","modules/clusters: C1/C2 <br>GO.ID: GO:0032879 <br>GO.name: regulation of localization","modules/clusters: C1/C2 <br>GO.ID: GO:0031175 <br>GO.name: neuron projection development","modules/clusters: C1/C2 <br>GO.ID: GO:0048513 <br>GO.name: animal organ development","modules/clusters: C1/C2 <br>GO.ID: GO:0007399 <br>GO.name: nervous system development","modules/clusters: C1/C2 <br>GO.ID: GO:0051094 <br>GO.name: positive regulation of developmental process"],"showlegend":true,"marker":{"color":"rgba(236,194,159,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(236,194,159,1)"}},"name":"C1/C2","textfont":{"color":"rgba(236,194,159,1)"},"error_y":{"color":"rgba(236,194,159,1)"},"error_x":{"color":"rgba(236,194,159,1)"},"line":{"color":"rgba(236,194,159,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.132471379361078],"y":[0.304813310738183],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C10/C13/C14 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process","showlegend":true,"marker":{"color":"rgba(200,216,131,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(200,216,131,1)"}},"name":"C1/C2/C10/C13/C14","textfont":{"color":"rgba(200,216,131,1)"},"error_y":{"color":"rgba(200,216,131,1)"},"error_x":{"color":"rgba(200,216,131,1)"},"line":{"color":"rgba(200,216,131,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.118743935343567,0.184261226249327,0.352180319679365,0.0113521737202379,0.226065176002919,0.0353009151141648,0.00643049694647317],"y":[0.372656958101919,0.378400502369157,-0.176039329023015,-0.188631847839211,-0.0973813056800484,-0.132112405359959,-0.174044329032375],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C14 <br>GO.ID: GO:0002682 <br>GO.name: regulation of immune system process","modules/clusters: C1/C2/C14 <br>GO.ID: GO:0048583 <br>GO.name: regulation of response to stimulus","modules/clusters: C1/C2/C14 <br>GO.ID: GO:0035295 <br>GO.name: tube development","modules/clusters: C1/C2/C14 <br>GO.ID: GO:0006928 <br>GO.name: movement of cell or subcellular component","modules/clusters: C1/C2/C14 <br>GO.ID: GO:0040011 <br>GO.name: locomotion","modules/clusters: C1/C2/C14 <br>GO.ID: GO:0048870 <br>GO.name: cell motility","modules/clusters: C1/C2/C14 <br>GO.ID: GO:0051674 <br>GO.name: localization of cell"],"showlegend":true,"marker":{"color":"rgba(210,206,131,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(210,206,131,1)"}},"name":"C1/C2/C14","textfont":{"color":"rgba(210,206,131,1)"},"error_y":{"color":"rgba(210,206,131,1)"},"error_x":{"color":"rgba(210,206,131,1)"},"line":{"color":"rgba(210,206,131,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.000744018820506567,0.0468486204422975],"y":[-0.169790434020489,-0.125580323265076],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C3/C14 <br>GO.ID: GO:0007155 <br>GO.name: cell adhesion","modules/clusters: C1/C2/C3/C14 <br>GO.ID: GO:0016477 <br>GO.name: cell migration"],"showlegend":true,"marker":{"color":"rgba(209,202,120,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(209,202,120,1)"}},"name":"C1/C2/C3/C14","textfont":{"color":"rgba(209,202,120,1)"},"error_y":{"color":"rgba(209,202,120,1)"},"error_x":{"color":"rgba(209,202,120,1)"},"line":{"color":"rgba(209,202,120,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.333670304882377],"y":[-0.0465538685247625],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C3/C5/C7/C11/C13/C14 <br>GO.ID: GO:0050896 <br>GO.name: response to stimulus","showlegend":true,"marker":{"color":"rgba(195,213,138,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(195,213,138,1)"}},"name":"C1/C2/C3/C5/C7/C11/C13/C14","textfont":{"color":"rgba(195,213,138,1)"},"error_y":{"color":"rgba(195,213,138,1)"},"error_x":{"color":"rgba(195,213,138,1)"},"line":{"color":"rgba(195,213,138,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.301431601294705],"y":[-0.128261356634197],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C3/C7/C13/C14 <br>GO.ID: GO:0032501 <br>GO.name: multicellular organismal process","showlegend":true,"marker":{"color":"rgba(200,208,134,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(200,208,134,1)"}},"name":"C1/C2/C3/C7/C13/C14","textfont":{"color":"rgba(200,208,134,1)"},"error_y":{"color":"rgba(200,208,134,1)"},"error_x":{"color":"rgba(200,208,134,1)"},"line":{"color":"rgba(200,208,134,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.326244158832243],"y":[-0.162320422710284],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C5 <br>GO.ID: GO:0035239 <br>GO.name: tube morphogenesis","showlegend":true,"marker":{"color":"rgba(196,179,173,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(196,179,173,1)"}},"name":"C1/C2/C5","textfont":{"color":"rgba(196,179,173,1)"},"error_y":{"color":"rgba(196,179,173,1)"},"error_x":{"color":"rgba(196,179,173,1)"},"line":{"color":"rgba(196,179,173,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.170726211065628],"y":[0.283916443388506],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C5/C13 <br>GO.ID: GO:0050793 <br>GO.name: regulation of developmental process","showlegend":true,"marker":{"color":"rgba(199,206,180,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(199,206,180,1)"}},"name":"C1/C2/C5/C13","textfont":{"color":"rgba(199,206,180,1)"},"error_y":{"color":"rgba(199,206,180,1)"},"error_x":{"color":"rgba(199,206,180,1)"},"line":{"color":"rgba(199,206,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.110333804276303],"y":[-0.11388717314099],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C5/C6/C11/C13/C14 <br>GO.ID: GO:0051716 <br>GO.name: cellular response to stimulus","showlegend":true,"marker":{"color":"rgba(193,216,142,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(193,216,142,1)"}},"name":"C1/C2/C5/C6/C11/C13/C14","textfont":{"color":"rgba(193,216,142,1)"},"error_y":{"color":"rgba(193,216,142,1)"},"error_x":{"color":"rgba(193,216,142,1)"},"line":{"color":"rgba(193,216,142,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.000795002534706379,0.0105143357654529,0.220147093973533],"y":[0.244729180704555,-0.158452317560171,-0.0623795520187809],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C5/C6/C13/C14 <br>GO.ID: GO:0007166 <br>GO.name: cell surface receptor signaling pathway","modules/clusters: C1/C2/C5/C6/C13/C14 <br>GO.ID: GO:0007154 <br>GO.name: cell communication","modules/clusters: C1/C2/C5/C6/C13/C14 <br>GO.ID: GO:0023052 <br>GO.name: signaling"],"showlegend":true,"marker":{"color":"rgba(191,211,145,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(191,211,145,1)"}},"name":"C1/C2/C5/C6/C13/C14","textfont":{"color":"rgba(191,211,145,1)"},"error_y":{"color":"rgba(191,211,145,1)"},"error_x":{"color":"rgba(191,211,145,1)"},"line":{"color":"rgba(191,211,145,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0258259340409079],"y":[0.254336155691216],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C5/C6/C14 <br>GO.ID: GO:0007165 <br>GO.name: signal transduction","showlegend":true,"marker":{"color":"rgba(190,197,145,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(190,197,145,1)"}},"name":"C1/C2/C5/C6/C14","textfont":{"color":"rgba(190,197,145,1)"},"error_y":{"color":"rgba(190,197,145,1)"},"error_x":{"color":"rgba(190,197,145,1)"},"line":{"color":"rgba(190,197,145,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.113451676002905,0.10253068692881],"y":[-0.218289413877798,-0.233595362208343],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C5/C7 <br>GO.ID: GO:0030154 <br>GO.name: cell differentiation","modules/clusters: C1/C2/C5/C7 <br>GO.ID: GO:0048869 <br>GO.name: cellular developmental process"],"showlegend":true,"marker":{"color":"rgba(214,162,159,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(214,162,159,1)"}},"name":"C1/C2/C5/C7","textfont":{"color":"rgba(214,162,159,1)"},"error_y":{"color":"rgba(214,162,159,1)"},"error_x":{"color":"rgba(214,162,159,1)"},"line":{"color":"rgba(214,162,159,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.152011770121533,0.363971224509931],"y":[0.312862877558013,-0.186389122786849],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C5/C7/C14 <br>GO.ID: GO:0051239 <br>GO.name: regulation of multicellular organismal process","modules/clusters: C1/C2/C5/C7/C14 <br>GO.ID: GO:0048731 <br>GO.name: system development"],"showlegend":true,"marker":{"color":"rgba(199,190,131,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(199,190,131,1)"}},"name":"C1/C2/C5/C7/C14","textfont":{"color":"rgba(199,190,131,1)"},"error_y":{"color":"rgba(199,190,131,1)"},"error_x":{"color":"rgba(199,190,131,1)"},"line":{"color":"rgba(199,190,131,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.312894232658785],"y":[-0.0477506558625528],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C7 <br>GO.ID: GO:0042221 <br>GO.name: response to chemical","showlegend":true,"marker":{"color":"rgba(234,169,152,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(234,169,152,1)"}},"name":"C1/C2/C7","textfont":{"color":"rgba(234,169,152,1)"},"error_y":{"color":"rgba(234,169,152,1)"},"error_x":{"color":"rgba(234,169,152,1)"},"line":{"color":"rgba(234,169,152,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.36893506892952,0.350072365522052,0.326342428872749],"y":[-0.183625268566452,-0.18953482332529,-0.167117204273505],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C7/C14 <br>GO.ID: GO:0007275 <br>GO.name: multicellular organism development","modules/clusters: C1/C2/C7/C14 <br>GO.ID: GO:0048856 <br>GO.name: anatomical structure development","modules/clusters: C1/C2/C7/C14 <br>GO.ID: GO:0032502 <br>GO.name: developmental process"],"showlegend":true,"marker":{"color":"rgba(209,193,128,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(209,193,128,1)"}},"name":"C1/C2/C7/C14","textfont":{"color":"rgba(209,193,128,1)"},"error_y":{"color":"rgba(209,193,128,1)"},"error_x":{"color":"rgba(209,193,128,1)"},"line":{"color":"rgba(209,193,128,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.329162620308671,0.335597981126436,0.259417467073515,0.268022836185909],"y":[-0.166905796897392,-0.167586679554156,-0.0266179268414551,-0.0269874331683461],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C3 <br>GO.ID: GO:0001568 <br>GO.name: blood vessel development","modules/clusters: C1/C3 <br>GO.ID: GO:0001944 <br>GO.name: vasculature development","modules/clusters: C1/C3 <br>GO.ID: GO:0006935 <br>GO.name: chemotaxis","modules/clusters: C1/C3 <br>GO.ID: GO:0042330 <br>GO.name: taxis"],"showlegend":true,"marker":{"color":"rgba(227,159,125,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(227,159,125,1)"}},"name":"C1/C3","textfont":{"color":"rgba(227,159,125,1)"},"error_y":{"color":"rgba(227,159,125,1)"},"error_x":{"color":"rgba(227,159,125,1)"},"line":{"color":"rgba(227,159,125,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.157560574654396],"y":[0.273043440482969],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C5/C14 <br>GO.ID: GO:0051240 <br>GO.name: positive regulation of multicellular organismal process","showlegend":true,"marker":{"color":"rgba(186,185,132,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(186,185,132,1)"}},"name":"C1/C5/C14","textfont":{"color":"rgba(186,185,132,1)"},"error_y":{"color":"rgba(186,185,132,1)"},"error_x":{"color":"rgba(186,185,132,1)"},"line":{"color":"rgba(186,185,132,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0083688085494416,-0.0995254346002089],"y":[-0.182700782235417,0.268779257002956],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C7 <br>GO.ID: GO:0008283 <br>GO.name: cell population proliferation","modules/clusters: C1/C7 <br>GO.ID: GO:0042127 <br>GO.name: regulation of cell population proliferation"],"showlegend":true,"marker":{"color":"rgba(227,142,140,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(227,142,140,1)"}},"name":"C1/C7","textfont":{"color":"rgba(227,142,140,1)"},"error_y":{"color":"rgba(227,142,140,1)"},"error_x":{"color":"rgba(227,142,140,1)"},"line":{"color":"rgba(227,142,140,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.249203679101174,-0.206725626941199,-0.297124491707514,-0.237881493519284,-0.115870363975171,-0.167848993659553,-0.312866425007646,-0.284661205947731,0.120331345453554,-0.0803931967467086,0.0242844651318132,-0.221584595929993,-0.160173143230973,-0.000890818657890795,-0.195806305168712,0.0908929247908155,-0.294825493180344,0.0140371752689749,-0.280615534081981,-0.301096900337711,-0.0992030254705486,-0.352125791269029,-0.13181781768335,0.0235467180281417,-0.128610526974064,-0.104141844708279,0.117319601111567,-0.0958291602910484,-0.204237624222213,-0.319661390110842,-0.234553245127911,0.0347104991913952,-0.347259513044033,-0.309476503374948,-0.294135552289617,-0.0454064752994603,-0.279769394448644,-0.273550360357319,-0.06511948598802,0.037659813201871,-0.128371318026232,-0.279539990753032,-0.150911382090011,-0.190771309654752,0.0482156803345295,-0.11454043141334,0.0423354017373589],"y":[-0.174070502854996,-0.193239385223789,-0.245689902319898,-0.191290429883421,-0.18965044034947,-0.192272586438212,0.117379062982402,0.146076702150471,-0.0758477909692843,-0.197690915355292,-0.136070311161006,0.108270545981885,-0.197025691629695,-0.170067488941262,0.121118479169091,-0.0466184847172816,0.107358342535387,-0.162324846295775,0.172464030297881,0.0996227562830427,-0.168297506047518,-0.232960761515494,-0.230535750763363,0.217865031844486,-0.229437278270123,0.26464102126683,-0.0811560765029505,0.181518300375692,0.172805344988383,-0.231004893610432,-0.239702036189691,-0.0879225703166103,-0.241743859887077,-0.259386115396969,0.11296860684546,-0.164055220681076,0.144601824138608,-0.248107099886239,-0.206660790155003,-0.064601059037789,0.165863404076566,0.146057250923071,0.174891294955783,-0.195480262422613,-0.0900460041769967,-0.238589274475692,-0.088917213073012],"type":"scatter","mode":"markers","text":["modules/clusters: C10 <br>GO.ID: GO:0000086 <br>GO.name: G2/M transition of mitotic cell cycle","modules/clusters: C10 <br>GO.ID: GO:0044839 <br>GO.name: cell cycle G2/M phase transition","modules/clusters: C10 <br>GO.ID: GO:0090307 <br>GO.name: mitotic spindle assembly","modules/clusters: C10 <br>GO.ID: GO:0000281 <br>GO.name: mitotic cytokinesis","modules/clusters: C10 <br>GO.ID: GO:0034508 <br>GO.name: centromere complex assembly","modules/clusters: C10 <br>GO.ID: GO:0061640 <br>GO.name: cytoskeleton-dependent cytokinesis","modules/clusters: C10 <br>GO.ID: GO:0010389 <br>GO.name: regulation of G2/M transition of mitotic cell cycle","modules/clusters: C10 <br>GO.ID: GO:1902749 <br>GO.name: regulation of cell cycle G2/M phase transition","modules/clusters: C10 <br>GO.ID: GO:0006974 <br>GO.name: cellular response to DNA damage stimulus","modules/clusters: C10 <br>GO.ID: GO:0030261 <br>GO.name: chromosome condensation","modules/clusters: C10 <br>GO.ID: GO:0051656 <br>GO.name: establishment of organelle localization","modules/clusters: C10 <br>GO.ID: GO:0007095 <br>GO.name: mitotic G2 DNA damage checkpoint signaling","modules/clusters: C10 <br>GO.ID: GO:0061982 <br>GO.name: meiosis I cell cycle process","modules/clusters: C10 <br>GO.ID: GO:0007018 <br>GO.name: microtubule-based movement","modules/clusters: C10 <br>GO.ID: GO:0044773 <br>GO.name: mitotic DNA damage checkpoint signaling","modules/clusters: C10 <br>GO.ID: GO:0006281 <br>GO.name: DNA repair","modules/clusters: C10 <br>GO.ID: GO:0044774 <br>GO.name: mitotic DNA integrity checkpoint signaling","modules/clusters: C10 <br>GO.ID: GO:0051640 <br>GO.name: organelle localization","modules/clusters: C10 <br>GO.ID: GO:0032465 <br>GO.name: regulation of cytokinesis","modules/clusters: C10 <br>GO.ID: GO:0044818 <br>GO.name: mitotic G2/M transition checkpoint","modules/clusters: C10 <br>GO.ID: GO:0051382 <br>GO.name: kinetochore assembly","modules/clusters: C10 <br>GO.ID: GO:0007076 <br>GO.name: mitotic chromosome condensation","modules/clusters: C10 <br>GO.ID: GO:0070925 <br>GO.name: organelle assembly","modules/clusters: C10 <br>GO.ID: GO:0042770 <br>GO.name: signal transduction in response to DNA damage","modules/clusters: C10 <br>GO.ID: GO:0051383 <br>GO.name: kinetochore organization","modules/clusters: C10 <br>GO.ID: GO:0051302 <br>GO.name: regulation of cell division","modules/clusters: C10 <br>GO.ID: GO:0033554 <br>GO.name: cellular response to stress","modules/clusters: C10 <br>GO.ID: GO:0000077 <br>GO.name: DNA damage checkpoint signaling","modules/clusters: C10 <br>GO.ID: GO:0031570 <br>GO.name: DNA integrity checkpoint signaling","modules/clusters: C10 <br>GO.ID: GO:0051256 <br>GO.name: mitotic spindle midzone assembly","modules/clusters: C10 <br>GO.ID: GO:0007127 <br>GO.name: meiosis I","modules/clusters: C10 <br>GO.ID: GO:0006260 <br>GO.name: DNA replication","modules/clusters: C10 <br>GO.ID: GO:0000022 <br>GO.name: mitotic spindle elongation","modules/clusters: C10 <br>GO.ID: GO:0051231 <br>GO.name: spindle elongation","modules/clusters: C10 <br>GO.ID: GO:0010972 <br>GO.name: negative regulation of G2/M transition of mitotic cell cycle","modules/clusters: C10 <br>GO.ID: GO:0065004 <br>GO.name: protein-DNA complex assembly","modules/clusters: C10 <br>GO.ID: GO:1902750 <br>GO.name: negative regulation of cell cycle G2/M phase transition","modules/clusters: C10 <br>GO.ID: GO:0051255 <br>GO.name: spindle midzone assembly","modules/clusters: C10 <br>GO.ID: GO:0071824 <br>GO.name: protein-DNA complex subunit organization","modules/clusters: C10 <br>GO.ID: GO:0006468 <br>GO.name: protein phosphorylation","modules/clusters: C10 <br>GO.ID: GO:0000079 <br>GO.name: regulation of cyclin-dependent protein serine/threonine kinase activity","modules/clusters: C10 <br>GO.ID: GO:1901989 <br>GO.name: positive regulation of cell cycle phase transition","modules/clusters: C10 <br>GO.ID: GO:1904029 <br>GO.name: regulation of cyclin-dependent protein kinase activity","modules/clusters: C10 <br>GO.ID: GO:0007131 <br>GO.name: reciprocal meiotic recombination","modules/clusters: C10 <br>GO.ID: GO:0140527 <br>GO.name: reciprocal homologous recombination","modules/clusters: C10 <br>GO.ID: GO:0031109 <br>GO.name: microtubule polymerization or depolymerization","modules/clusters: C10 <br>GO.ID: GO:0035825 <br>GO.name: homologous recombination"],"showlegend":true,"marker":{"color":"rgba(224,196,100,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(224,196,100,1)"}},"name":"C10","textfont":{"color":"rgba(224,196,100,1)"},"error_y":{"color":"rgba(224,196,100,1)"},"error_x":{"color":"rgba(224,196,100,1)"},"line":{"color":"rgba(224,196,100,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.286662352877187,0.345849111015291],"y":[-0.170334619247054,-0.174706147606843],"type":"scatter","mode":"markers","text":["modules/clusters: C11 <br>GO.ID: GO:0048608 <br>GO.name: reproductive structure development","modules/clusters: C11 <br>GO.ID: GO:0061458 <br>GO.name: reproductive system development"],"showlegend":true,"marker":{"color":"rgba(212,214,162,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(212,214,162,1)"}},"name":"C11","textfont":{"color":"rgba(212,214,162,1)"},"error_y":{"color":"rgba(212,214,162,1)"},"error_x":{"color":"rgba(212,214,162,1)"},"line":{"color":"rgba(212,214,162,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.304589233370369],"y":[-0.0252005327793606],"type":"scatter","mode":"markers","text":"modules/clusters: C11/C14 <br>GO.ID: GO:0006950 <br>GO.name: response to stress","showlegend":true,"marker":{"color":"rgba(198,216,132,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(198,216,132,1)"}},"name":"C11/C14","textfont":{"color":"rgba(198,216,132,1)"},"error_y":{"color":"rgba(198,216,132,1)"},"error_x":{"color":"rgba(198,216,132,1)"},"line":{"color":"rgba(198,216,132,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0143744015965572,0.0603782716365913,-0.0354192042296988,-0.0504401126138096,-0.0702005213073864,-0.0449614523388262],"y":[0.122583753398071,0.160394616411366,0.172604574170754,0.186627407082385,0.205737692446876,0.156270598547733],"type":"scatter","mode":"markers","text":["modules/clusters: C12 <br>GO.ID: GO:0001933 <br>GO.name: negative regulation of protein phosphorylation","modules/clusters: C12 <br>GO.ID: GO:0051248 <br>GO.name: negative regulation of protein metabolic process","modules/clusters: C12 <br>GO.ID: GO:0042326 <br>GO.name: negative regulation of phosphorylation","modules/clusters: C12 <br>GO.ID: GO:0045936 <br>GO.name: negative regulation of phosphate metabolic process","modules/clusters: C12 <br>GO.ID: GO:0010563 <br>GO.name: negative regulation of phosphorus metabolic process","modules/clusters: C12 <br>GO.ID: GO:0032269 <br>GO.name: negative regulation of cellular protein metabolic process"],"showlegend":true,"marker":{"color":"rgba(215,212,203,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(215,212,203,1)"}},"name":"C12","textfont":{"color":"rgba(215,212,203,1)"},"error_y":{"color":"rgba(215,212,203,1)"},"error_x":{"color":"rgba(215,212,203,1)"},"line":{"color":"rgba(215,212,203,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.159248091475046,0.115668549601716,0.212314151840329,0.0369023023643935],"y":[0.326810802188094,0.220434103163857,-0.0755759925720478,-0.0839826666714949],"type":"scatter","mode":"markers","text":["modules/clusters: C13 <br>GO.ID: GO:0048585 <br>GO.name: negative regulation of response to stimulus","modules/clusters: C13 <br>GO.ID: GO:0001817 <br>GO.name: regulation of cytokine production","modules/clusters: C13 <br>GO.ID: GO:0001816 <br>GO.name: cytokine production","modules/clusters: C13 <br>GO.ID: GO:0035725 <br>GO.name: sodium ion transmembrane transport"],"showlegend":true,"marker":{"color":"rgba(202,234,188,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(202,234,188,1)"}},"name":"C13","textfont":{"color":"rgba(202,234,188,1)"},"error_y":{"color":"rgba(202,234,188,1)"},"error_x":{"color":"rgba(202,234,188,1)"},"line":{"color":"rgba(202,234,188,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.130508948653742,0.128302050949846,0.308261526088773,0.227984179563011],"y":[-0.0530544171988055,-0.0585508196174049,0.0032521310030988,-0.0397665534041008],"type":"scatter","mode":"markers","text":["modules/clusters: C13/C14 <br>GO.ID: GO:0002366 <br>GO.name: leukocyte activation involved in immune response","modules/clusters: C13/C14 <br>GO.ID: GO:0002263 <br>GO.name: cell activation involved in immune response","modules/clusters: C13/C14 <br>GO.ID: GO:0006955 <br>GO.name: immune response","modules/clusters: C13/C14 <br>GO.ID: GO:0002252 <br>GO.name: immune effector process"],"showlegend":true,"marker":{"color":"rgba(193,226,146,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(193,226,146,1)"}},"name":"C13/C14","textfont":{"color":"rgba(193,226,146,1)"},"error_y":{"color":"rgba(193,226,146,1)"},"error_x":{"color":"rgba(193,226,146,1)"},"line":{"color":"rgba(193,226,146,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.24941944090313,0.274124922636892,0.3054338437379,0.279993519554634,0.223348964019376,0.207485018169641,0.128352319543828,0.184383161863245,0.191385034437492,0.279973923828604,0.00922959109046861,0.175585887021266,0.177157228759842,0.239906169193853,-0.0989108286715768,0.240419039937208,0.179505278888844,0.186408040471239,0.257125042561494,0.127223731942332,0.184335580574936,0.170777661005384,0.0499448670456936,0.165275887860154,0.124973606576227,0.188501632641916,-0.0733921771150591,0.180323815170612,0.127331350117856,0.184928857791313,0.1927777898832,0.0801787007516989,0.128117755641872,0.125209734597482,0.184597239227682,0.141360744452118,0.123258663754582,-0.026762457005467,0.0679913760537949,-0.0964758758194558,0.099103345530077,0.0717135165190117,0.239806171600002,0.130160115971275,0.181205814466535,0.0850854383180517,-0.0355711954575903,0.253525713698851,0.216898744662245,0.232504495991026,0.262307952051251,0.257841855920478,-0.0292577618549986,0.0382355266979693,0.119681152800814,0.270178431126127,0.244218623309124,0.0284740583136749,0.164475618197086,-0.0167097888752336,0.106732263306417,0.0990470928422758,-0.0541324265502634,0.194246889508733,0.051573915986417],"y":[-0.0080321998747823,-0.00704468929102471,-0.0279838580091285,-0.013921730292051,-0.0889050765770172,-0.0262443924425931,0.331323807094918,0.285921935796789,0.333318212403153,-0.00147836324197203,-0.178203788020409,0.281228956431445,0.213579281109104,0.0115805291803765,0.275288299540693,0.0380208280171392,0.237574752432833,0.29269623213071,0.0347797000120115,0.275500599208242,0.294025158166815,0.210171359796707,-0.0913292603723592,0.189851917903712,0.318148505508503,0.338912644750022,0.244450486904293,0.291744654250139,0.282326690047498,0.327452056991186,-0.0204110185137936,0.215320760355257,0.24297489588907,0.24632102490441,0.325875170060312,-0.0114831274784719,0.219631422402094,0.246824901014781,-0.108076107207865,0.267078805413613,0.319004607700133,-0.100799762520878,-0.0114009085029903,-0.0823882135645356,0.207369876462329,0.190139790336812,0.273564513912972,-0.015983568710119,-0.0138758752417093,-0.00957057715477672,-0.0436626633417948,-0.00555550249050577,0.171910612267142,0.196348895107104,0.249989153559616,-0.0260625804751267,-0.0259422202892787,-0.110626834382985,-0.0648264203444326,0.239272803472981,0.235703286205482,0.232904801071151,0.220014276938839,0.261477586971795,0.261095871848918],"type":"scatter","mode":"markers","text":["modules/clusters: C14 <br>GO.ID: GO:0051707 <br>GO.name: response to other organism","modules/clusters: C14 <br>GO.ID: GO:0043207 <br>GO.name: response to external biotic stimulus","modules/clusters: C14 <br>GO.ID: GO:0009607 <br>GO.name: response to biotic stimulus","modules/clusters: C14 <br>GO.ID: GO:0006952 <br>GO.name: defense response","modules/clusters: C14 <br>GO.ID: GO:0044419 <br>GO.name: biological process involved in interspecies interaction between organisms","modules/clusters: C14 <br>GO.ID: GO:0002443 <br>GO.name: leukocyte mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0002684 <br>GO.name: positive regulation of immune system process","modules/clusters: C14 <br>GO.ID: GO:0031347 <br>GO.name: regulation of defense response","modules/clusters: C14 <br>GO.ID: GO:0050776 <br>GO.name: regulation of immune response","modules/clusters: C14 <br>GO.ID: GO:0002250 <br>GO.name: adaptive immune response","modules/clusters: C14 <br>GO.ID: GO:0001906 <br>GO.name: cell killing","modules/clusters: C14 <br>GO.ID: GO:0031349 <br>GO.name: positive regulation of defense response","modules/clusters: C14 <br>GO.ID: GO:0002715 <br>GO.name: regulation of natural killer cell mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0098542 <br>GO.name: defense response to other organism","modules/clusters: C14 <br>GO.ID: GO:0031341 <br>GO.name: regulation of cell killing","modules/clusters: C14 <br>GO.ID: GO:0002228 <br>GO.name: natural killer cell mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0045088 <br>GO.name: regulation of innate immune response","modules/clusters: C14 <br>GO.ID: GO:0032103 <br>GO.name: positive regulation of response to external stimulus","modules/clusters: C14 <br>GO.ID: GO:0045087 <br>GO.name: innate immune response","modules/clusters: C14 <br>GO.ID: GO:0002703 <br>GO.name: regulation of leukocyte mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0050778 <br>GO.name: positive regulation of immune response","modules/clusters: C14 <br>GO.ID: GO:0045089 <br>GO.name: positive regulation of innate immune response","modules/clusters: C14 <br>GO.ID: GO:0001909 <br>GO.name: leukocyte mediated cytotoxicity","modules/clusters: C14 <br>GO.ID: GO:0002717 <br>GO.name: positive regulation of natural killer cell mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0002697 <br>GO.name: regulation of immune effector process","modules/clusters: C14 <br>GO.ID: GO:0048584 <br>GO.name: positive regulation of response to stimulus","modules/clusters: C14 <br>GO.ID: GO:0031343 <br>GO.name: positive regulation of cell killing","modules/clusters: C14 <br>GO.ID: GO:0002833 <br>GO.name: positive regulation of response to biotic stimulus","modules/clusters: C14 <br>GO.ID: GO:0002699 <br>GO.name: positive regulation of immune effector process","modules/clusters: C14 <br>GO.ID: GO:0080134 <br>GO.name: regulation of response to stress","modules/clusters: C14 <br>GO.ID: GO:0002449 <br>GO.name: lymphocyte mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0042269 <br>GO.name: regulation of natural killer cell mediated cytotoxicity","modules/clusters: C14 <br>GO.ID: GO:0002706 <br>GO.name: regulation of lymphocyte mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0002705 <br>GO.name: positive regulation of leukocyte mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0002831 <br>GO.name: regulation of response to biotic stimulus","modules/clusters: C14 <br>GO.ID: GO:0042267 <br>GO.name: natural killer cell mediated cytotoxicity","modules/clusters: C14 <br>GO.ID: GO:0002708 <br>GO.name: positive regulation of lymphocyte mediated immunity","modules/clusters: C14 <br>GO.ID: GO:0001910 <br>GO.name: regulation of leukocyte mediated cytotoxicity","modules/clusters: C14 <br>GO.ID: GO:0046649 <br>GO.name: lymphocyte activation","modules/clusters: C14 <br>GO.ID: GO:0050865 <br>GO.name: regulation of cell activation","modules/clusters: C14 <br>GO.ID: GO:0002683 <br>GO.name: negative regulation of immune system process","modules/clusters: C14 <br>GO.ID: GO:0042110 <br>GO.name: T cell activation","modules/clusters: C14 <br>GO.ID: GO:0009617 <br>GO.name: response to bacterium","modules/clusters: C14 <br>GO.ID: GO:0071345 <br>GO.name: cellular response to cytokine stimulus","modules/clusters: C14 <br>GO.ID: GO:0002218 <br>GO.name: activation of innate immune response","modules/clusters: C14 <br>GO.ID: GO:0045954 <br>GO.name: positive regulation of natural killer cell mediated cytotoxicity","modules/clusters: C14 <br>GO.ID: GO:0002694 <br>GO.name: regulation of leukocyte activation","modules/clusters: C14 <br>GO.ID: GO:0006968 <br>GO.name: cellular defense response","modules/clusters: C14 <br>GO.ID: GO:0032496 <br>GO.name: response to lipopolysaccharide","modules/clusters: C14 <br>GO.ID: GO:0002237 <br>GO.name: response to molecule of bacterial origin","modules/clusters: C14 <br>GO.ID: GO:0034097 <br>GO.name: response to cytokine","modules/clusters: C14 <br>GO.ID: GO:0002460 <br>GO.name: adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains","modules/clusters: C14 <br>GO.ID: GO:0051712 <br>GO.name: positive regulation of killing of cells of another organism","modules/clusters: C14 <br>GO.ID: GO:0019221 <br>GO.name: cytokine-mediated signaling pathway","modules/clusters: C14 <br>GO.ID: GO:0002223 <br>GO.name: stimulatory C-type lectin receptor signaling pathway","modules/clusters: C14 <br>GO.ID: GO:1990840 <br>GO.name: response to lectin","modules/clusters: C14 <br>GO.ID: GO:1990858 <br>GO.name: cellular response to lectin","modules/clusters: C14 <br>GO.ID: GO:0046209 <br>GO.name: nitric oxide metabolic process","modules/clusters: C14 <br>GO.ID: GO:2001057 <br>GO.name: reactive nitrogen species metabolic process","modules/clusters: C14 <br>GO.ID: GO:0051249 <br>GO.name: regulation of lymphocyte activation","modules/clusters: C14 <br>GO.ID: GO:0002220 <br>GO.name: innate immune response activating cell surface receptor signaling pathway","modules/clusters: C14 <br>GO.ID: GO:0002758 <br>GO.name: innate immune response-activating signal transduction","modules/clusters: C14 <br>GO.ID: GO:0051709 <br>GO.name: regulation of killing of cells of another organism","modules/clusters: C14 <br>GO.ID: GO:0002253 <br>GO.name: activation of immune response","modules/clusters: C14 <br>GO.ID: GO:0002768 <br>GO.name: immune response-regulating cell surface receptor signaling pathway"],"showlegend":true,"marker":{"color":"rgba(184,218,104,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(184,218,104,1)"}},"name":"C14","textfont":{"color":"rgba(184,218,104,1)"},"error_y":{"color":"rgba(184,218,104,1)"},"error_x":{"color":"rgba(184,218,104,1)"},"line":{"color":"rgba(184,218,104,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.143105302501986,-0.0980359115450342,0.0894529345694761,0.213208492360508,0.132742869589665,0.130254199926407,-0.145788894802184,0.131246651822419,0.120584438028634,0.0368089267698266,0.130016439142903,0.131246651822419,0.000199017359478611,0.0207563222249228,0.256525749795581,0.00902463322039971,0.0751415381860228,0.137573137235279,0.134590914060383,0.00360141808748199,0.0418311996365656,0.047663870890492,-0.00946184641845706,0.256946218465355,0.0313121269870443,0.130779888665496,0.023510604852135,0.0943263058454702,0.0727568040060972],"y":[0.186031584311215,0.279255099023533,0.353783089963104,-0.10660921729359,0.103858379563987,0.266311768318298,0.274648931272253,0.0887293760515927,0.072452045692518,-0.106199714804233,-0.049489461785284,0.0887293760515927,0.241049724475646,-0.139663716782584,-0.0362852940169786,0.00209900789561909,0.263230813425903,0.151166441445298,0.123786014392629,0.273822520062707,-0.0626932249161784,-0.0803812381925355,-0.00336783012680233,-0.10366408288387,-0.122816693220322,0.145790459159375,0.00458885107137462,0.298402970382596,0.355362356925577],"type":"scatter","mode":"markers","text":["modules/clusters: C2 <br>GO.ID: GO:0065008 <br>GO.name: regulation of biological quality","modules/clusters: C2 <br>GO.ID: GO:0010646 <br>GO.name: regulation of cell communication","modules/clusters: C2 <br>GO.ID: GO:0023051 <br>GO.name: regulation of signaling","modules/clusters: C2 <br>GO.ID: GO:0051179 <br>GO.name: localization","modules/clusters: C2 <br>GO.ID: GO:0050801 <br>GO.name: ion homeostasis","modules/clusters: C2 <br>GO.ID: GO:0051241 <br>GO.name: negative regulation of multicellular organismal process","modules/clusters: C2 <br>GO.ID: GO:0048523 <br>GO.name: negative regulation of cellular process","modules/clusters: C2 <br>GO.ID: GO:0098771 <br>GO.name: inorganic ion homeostasis","modules/clusters: C2 <br>GO.ID: GO:0055065 <br>GO.name: metal ion homeostasis","modules/clusters: C2 <br>GO.ID: GO:0007267 <br>GO.name: cell-cell signaling","modules/clusters: C2 <br>GO.ID: GO:0006816 <br>GO.name: calcium ion transport","modules/clusters: C2 <br>GO.ID: GO:0055080 <br>GO.name: cation homeostasis","modules/clusters: C2 <br>GO.ID: GO:0035556 <br>GO.name: intracellular signal transduction","modules/clusters: C2 <br>GO.ID: GO:0006793 <br>GO.name: phosphorus metabolic process","modules/clusters: C2 <br>GO.ID: GO:0071495 <br>GO.name: cellular response to endogenous stimulus","modules/clusters: C2 <br>GO.ID: GO:0055082 <br>GO.name: cellular chemical homeostasis","modules/clusters: C2 <br>GO.ID: GO:0051049 <br>GO.name: regulation of transport","modules/clusters: C2 <br>GO.ID: GO:0042592 <br>GO.name: homeostatic process","modules/clusters: C2 <br>GO.ID: GO:0048878 <br>GO.name: chemical homeostasis","modules/clusters: C2 <br>GO.ID: GO:0009966 <br>GO.name: regulation of signal transduction","modules/clusters: C2 <br>GO.ID: GO:0018108 <br>GO.name: peptidyl-tyrosine phosphorylation","modules/clusters: C2 <br>GO.ID: GO:0018212 <br>GO.name: peptidyl-tyrosine modification","modules/clusters: C2 <br>GO.ID: GO:0019725 <br>GO.name: cellular homeostasis","modules/clusters: C2 <br>GO.ID: GO:0003008 <br>GO.name: system process","modules/clusters: C2 <br>GO.ID: GO:0006796 <br>GO.name: phosphate-containing compound metabolic process","modules/clusters: C2 <br>GO.ID: GO:0044093 <br>GO.name: positive regulation of molecular function","modules/clusters: C2 <br>GO.ID: GO:0006873 <br>GO.name: cellular ion homeostasis","modules/clusters: C2 <br>GO.ID: GO:0009893 <br>GO.name: positive regulation of metabolic process","modules/clusters: C2 <br>GO.ID: GO:0048519 <br>GO.name: negative regulation of biological process"],"showlegend":true,"marker":{"color":"rgba(250,249,182,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(250,249,182,1)"}},"name":"C2","textfont":{"color":"rgba(250,249,182,1)"},"error_y":{"color":"rgba(250,249,182,1)"},"error_x":{"color":"rgba(250,249,182,1)"},"line":{"color":"rgba(250,249,182,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0853219804143484],"y":[0.406247393786354],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C14 <br>GO.ID: GO:0050789 <br>GO.name: regulation of biological process","showlegend":true,"marker":{"color":"rgba(217,233,143,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(217,233,143,1)"}},"name":"C2/C14","textfont":{"color":"rgba(217,233,143,1)"},"error_y":{"color":"rgba(217,233,143,1)"},"error_x":{"color":"rgba(217,233,143,1)"},"line":{"color":"rgba(217,233,143,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.15325521051369],"y":[0.224530643794687],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C3/C14 <br>GO.ID: GO:0065007 <br>GO.name: biological regulation","showlegend":true,"marker":{"color":"rgba(212,215,126,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(212,215,126,1)"}},"name":"C2/C3/C14","textfont":{"color":"rgba(212,215,126,1)"},"error_y":{"color":"rgba(212,215,126,1)"},"error_x":{"color":"rgba(212,215,126,1)"},"line":{"color":"rgba(212,215,126,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0219283800199191,0.0383986019807482],"y":[0.233593975849046,0.223510737420617],"type":"scatter","mode":"markers","text":["modules/clusters: C2/C5 <br>GO.ID: GO:0007167 <br>GO.name: enzyme-linked receptor protein signaling pathway","modules/clusters: C2/C5 <br>GO.ID: GO:0007169 <br>GO.name: transmembrane receptor protein tyrosine kinase signaling pathway"],"showlegend":true,"marker":{"color":"rgba(203,207,185,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(203,207,185,1)"}},"name":"C2/C5","textfont":{"color":"rgba(203,207,185,1)"},"error_y":{"color":"rgba(203,207,185,1)"},"error_x":{"color":"rgba(203,207,185,1)"},"line":{"color":"rgba(203,207,185,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.280753564621095],"y":[-0.0442962855657386],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C7 <br>GO.ID: GO:1901700 <br>GO.name: response to oxygen-containing compound","showlegend":true,"marker":{"color":"rgba(241,196,163,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(241,196,163,1)"}},"name":"C2/C7","textfont":{"color":"rgba(241,196,163,1)"},"error_y":{"color":"rgba(241,196,163,1)"},"error_x":{"color":"rgba(241,196,163,1)"},"line":{"color":"rgba(241,196,163,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.106823908224572,0.128533539954031,0.133304436839549,0.28719725422837],"y":[0.359980364448935,-0.101033194891677,-0.0908262283697411,-0.0463940573287116],"type":"scatter","mode":"markers","text":["modules/clusters: C2/C7/C14 <br>GO.ID: GO:0048518 <br>GO.name: positive regulation of biological process","modules/clusters: C2/C7/C14 <br>GO.ID: GO:0070887 <br>GO.name: cellular response to chemical stimulus","modules/clusters: C2/C7/C14 <br>GO.ID: GO:0071310 <br>GO.name: cellular response to organic substance","modules/clusters: C2/C7/C14 <br>GO.ID: GO:0010033 <br>GO.name: response to organic substance"],"showlegend":true,"marker":{"color":"rgba(212,207,133,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(212,207,133,1)"}},"name":"C2/C7/C14","textfont":{"color":"rgba(212,207,133,1)"},"error_y":{"color":"rgba(212,207,133,1)"},"error_x":{"color":"rgba(212,207,133,1)"},"line":{"color":"rgba(212,207,133,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.104500818543013],"y":[0.2727416075766],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C7/C8 <br>GO.ID: GO:0048522 <br>GO.name: positive regulation of cellular process","showlegend":true,"marker":{"color":"rgba(191,203,180,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(191,203,180,1)"}},"name":"C2/C7/C8","textfont":{"color":"rgba(191,203,180,1)"},"error_y":{"color":"rgba(191,203,180,1)"},"error_x":{"color":"rgba(191,203,180,1)"},"line":{"color":"rgba(191,203,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0940370515245658,0.143142932249131,0.0938051921958742,0.123437767789959],"y":[-0.0927794374031315,-0.0564972287358432,-0.0928100076320545,-0.0704615208507822],"type":"scatter","mode":"markers","text":["modules/clusters: C3 <br>GO.ID: GO:0097530 <br>GO.name: granulocyte migration","modules/clusters: C3 <br>GO.ID: GO:0071621 <br>GO.name: granulocyte chemotaxis","modules/clusters: C3 <br>GO.ID: GO:1990266 <br>GO.name: neutrophil migration","modules/clusters: C3 <br>GO.ID: GO:0060326 <br>GO.name: cell chemotaxis"],"showlegend":true,"marker":{"color":"rgba(233,179,115,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(233,179,115,1)"}},"name":"C3","textfont":{"color":"rgba(233,179,115,1)"},"error_y":{"color":"rgba(233,179,115,1)"},"error_x":{"color":"rgba(233,179,115,1)"},"line":{"color":"rgba(233,179,115,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0717806415718385,0.0843809725570513,0.126937123319796,0.0613041582496399,0.31118092662006,-0.0914916784631184,-0.0284935369763505,0.0248283991244578,0.0620024931224499,0.0798656564658279,0.146848281175731,-0.0148790586034748,0.105963286574419,-0.0736462701740348],"y":[0.195272749678756,0.233132423793715,-0.0722049043883886,0.164767008688706,-0.165238180105943,0.24434774458254,0.16841083400155,0.150917917285124,0.220486629446042,0.222114699089124,0.239459530670135,0.19601477475264,-0.0573099588285795,0.238863978914578],"type":"scatter","mode":"markers","text":["modules/clusters: C5 <br>GO.ID: GO:0050994 <br>GO.name: regulation of lipid catabolic process","modules/clusters: C5 <br>GO.ID: GO:0019216 <br>GO.name: regulation of lipid metabolic process","modules/clusters: C5 <br>GO.ID: GO:0032870 <br>GO.name: cellular response to hormone stimulus","modules/clusters: C5 <br>GO.ID: GO:0050995 <br>GO.name: negative regulation of lipid catabolic process","modules/clusters: C5 <br>GO.ID: GO:0009888 <br>GO.name: tissue development","modules/clusters: C5 <br>GO.ID: GO:0034762 <br>GO.name: regulation of transmembrane transport","modules/clusters: C5 <br>GO.ID: GO:0045596 <br>GO.name: negative regulation of cell differentiation","modules/clusters: C5 <br>GO.ID: GO:0043407 <br>GO.name: negative regulation of MAP kinase activity","modules/clusters: C5 <br>GO.ID: GO:0045833 <br>GO.name: negative regulation of lipid metabolic process","modules/clusters: C5 <br>GO.ID: GO:0043269 <br>GO.name: regulation of ion transport","modules/clusters: C5 <br>GO.ID: GO:0051093 <br>GO.name: negative regulation of developmental process","modules/clusters: C5 <br>GO.ID: GO:0045595 <br>GO.name: regulation of cell differentiation","modules/clusters: C5 <br>GO.ID: GO:0071375 <br>GO.name: cellular response to peptide hormone stimulus","modules/clusters: C5 <br>GO.ID: GO:0060355 <br>GO.name: positive regulation of cell adhesion molecule production"],"showlegend":true,"marker":{"color":"rgba(156,165,188,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(156,165,188,1)"}},"name":"C5","textfont":{"color":"rgba(156,165,188,1)"},"error_y":{"color":"rgba(156,165,188,1)"},"error_x":{"color":"rgba(156,165,188,1)"},"line":{"color":"rgba(156,165,188,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.130657265746936,0.126882356277088,0.262926007086018,-0.00411249324700661,0.00430297779727496,-0.0201132872156881],"y":[-0.0833373025879246,-0.0891686011892018,-0.0445734311737561,0.119262707142805,0.168473894179503,0.139550116729303],"type":"scatter","mode":"markers","text":["modules/clusters: C7 <br>GO.ID: GO:0071396 <br>GO.name: cellular response to lipid","modules/clusters: C7 <br>GO.ID: GO:1901701 <br>GO.name: cellular response to oxygen-containing compound","modules/clusters: C7 <br>GO.ID: GO:0033993 <br>GO.name: response to lipid","modules/clusters: C7 <br>GO.ID: GO:0010867 <br>GO.name: positive regulation of triglyceride biosynthetic process","modules/clusters: C7 <br>GO.ID: GO:0045597 <br>GO.name: positive regulation of cell differentiation","modules/clusters: C7 <br>GO.ID: GO:0010866 <br>GO.name: regulation of triglyceride biosynthetic process"],"showlegend":true,"marker":{"color":"rgba(232,145,145,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(232,145,145,1)"}},"name":"C7","textfont":{"color":"rgba(232,145,145,1)"},"error_y":{"color":"rgba(232,145,145,1)"},"error_x":{"color":"rgba(232,145,145,1)"},"line":{"color":"rgba(232,145,145,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.377843590725143,-0.401967042744656],"y":[-0.237670908357366,0.0822788861697641],"type":"scatter","mode":"markers","text":["modules/clusters: C8/C9 <br>GO.ID: GO:0051306 <br>GO.name: mitotic sister chromatid separation","modules/clusters: C8/C9 <br>GO.ID: GO:0033045 <br>GO.name: regulation of sister chromatid segregation"],"showlegend":true,"marker":{"color":"rgba(153,194,187,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(153,194,187,1)"}},"name":"C8/C9","textfont":{"color":"rgba(153,194,187,1)"},"error_y":{"color":"rgba(153,194,187,1)"},"error_x":{"color":"rgba(153,194,187,1)"},"line":{"color":"rgba(153,194,187,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.415902920893404,-0.235374779356899,-0.287618759354263,-0.242015732057358,-0.275954299972298,-0.401812268208432,-0.418023882890418,-0.386313942008875,-0.418261226703912,-0.402713685927545,-0.358155535442974,-0.322713975595465,-0.0508543643944782,-0.294437714635582,0.00325120726271274,-0.329214226785702],"y":[0.0388120334098903,0.0860832283004147,-0.194793517186077,-0.213458494136338,0.207209647818272,0.0277602417755853,0.012688520317082,0.0747037172005966,0.0306066489634777,0.0592819175299342,0.133308035700353,0.156002626541794,-0.200634195213492,-0.226960475679829,-0.188127894323738,0.170536909412449],"type":"scatter","mode":"markers","text":["modules/clusters: C8/C9/C10 <br>GO.ID: GO:0007088 <br>GO.name: regulation of mitotic nuclear division","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0051783 <br>GO.name: regulation of nuclear division","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0044772 <br>GO.name: mitotic cell cycle phase transition","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0044770 <br>GO.name: cell cycle phase transition","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0007346 <br>GO.name: regulation of mitotic cell cycle","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0030071 <br>GO.name: regulation of mitotic metaphase/anaphase transition","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0007091 <br>GO.name: metaphase/anaphase transition of mitotic cell cycle","modules/clusters: C8/C9/C10 <br>GO.ID: GO:1902099 <br>GO.name: regulation of metaphase/anaphase transition of cell cycle","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0010965 <br>GO.name: regulation of mitotic sister chromatid separation","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0044784 <br>GO.name: metaphase/anaphase transition of cell cycle","modules/clusters: C8/C9/C10 <br>GO.ID: GO:1901990 <br>GO.name: regulation of mitotic cell cycle phase transition","modules/clusters: C8/C9/C10 <br>GO.ID: GO:1901987 <br>GO.name: regulation of cell cycle phase transition","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0007059 <br>GO.name: chromosome segregation","modules/clusters: C8/C9/C10 <br>GO.ID: GO:1903047 <br>GO.name: mitotic cell cycle process","modules/clusters: C8/C9/C10 <br>GO.ID: GO:0051301 <br>GO.name: cell division","modules/clusters: C8/C9/C10 <br>GO.ID: GO:1905818 <br>GO.name: regulation of chromosome separation"],"showlegend":true,"marker":{"color":"rgba(188,195,143,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(188,195,143,1)"}},"name":"C8/C9/C10","textfont":{"color":"rgba(188,195,143,1)"},"error_y":{"color":"rgba(188,195,143,1)"},"error_x":{"color":"rgba(188,195,143,1)"},"line":{"color":"rgba(188,195,143,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.188230505318201],"y":[-0.21187760444548],"type":"scatter","mode":"markers","text":"modules/clusters: C8/C9/C10/C11 <br>GO.ID: GO:0000278 <br>GO.name: mitotic cell cycle","showlegend":true,"marker":{"color":"rgba(200,204,152,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(200,204,152,1)"}},"name":"C8/C9/C10/C11","textfont":{"color":"rgba(200,204,152,1)"},"error_y":{"color":"rgba(200,204,152,1)"},"error_x":{"color":"rgba(200,204,152,1)"},"line":{"color":"rgba(200,204,152,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.294909965974548,-0.347489175360834,-0.178011750339989,-0.204063627236519,-0.345335556712141,-0.378454145989858,-0.397593802784535,-0.370786288552941,-0.380723814952501,-0.313697565932446,-0.167917640538239,-0.426121841328193,-0.384583479627252,-0.228825537407289,-0.234100831298871,-0.22352500837716,0.0476942169373777,0.0516174260364582,0.047654344146288,-0.280807002204344,0.0419318255749895,-0.186521390407643,-0.351298491710566,-0.217347135756544,-0.113185633254096,-0.293366361780194,-0.325609329147658,-0.39343585112695,-0.158389211777591,-0.116161259223508,-0.150010740173718,-0.0372059699205231,-0.0365018194758001,-0.346051749079679],"y":[0.106806133676701,0.0267656115822837,0.174016536403696,0.172162167895756,0.0226348704482321,0.0340133051136698,0.0308594483997305,0.0737982309938017,0.0841446352767741,0.153814317280874,0.229918859525335,0.0290525221476258,0.0410899466524705,0.0801393985971429,0.115646374189954,0.0999665192062212,-0.106355768122249,-0.102448098638376,-0.105762364119409,-0.258186206629338,-0.109889297417696,-0.183073150999097,0.0680021444118767,0.163680195308758,0.235053213327171,-0.264995911382334,-0.225480852830163,0.0433894247185023,-0.247237105182686,0.246302538705379,-0.211679693998375,0.168901384781833,0.163292911478362,-0.240367987032048],"type":"scatter","mode":"markers","text":["modules/clusters: C9 <br>GO.ID: GO:0071174 <br>GO.name: mitotic spindle checkpoint signaling","modules/clusters: C9 <br>GO.ID: GO:0007094 <br>GO.name: mitotic spindle assembly checkpoint signaling","modules/clusters: C9 <br>GO.ID: GO:0071173 <br>GO.name: spindle assembly checkpoint signaling","modules/clusters: C9 <br>GO.ID: GO:0031577 <br>GO.name: spindle checkpoint signaling","modules/clusters: C9 <br>GO.ID: GO:0045841 <br>GO.name: negative regulation of mitotic metaphase/anaphase transition","modules/clusters: C9 <br>GO.ID: GO:2000816 <br>GO.name: negative regulation of mitotic sister chromatid separation","modules/clusters: C9 <br>GO.ID: GO:0033048 <br>GO.name: negative regulation of mitotic sister chromatid segregation","modules/clusters: C9 <br>GO.ID: GO:0033046 <br>GO.name: negative regulation of sister chromatid segregation","modules/clusters: C9 <br>GO.ID: GO:1902100 <br>GO.name: negative regulation of metaphase/anaphase transition of cell cycle","modules/clusters: C9 <br>GO.ID: GO:1905819 <br>GO.name: negative regulation of chromosome separation","modules/clusters: C9 <br>GO.ID: GO:0051985 <br>GO.name: negative regulation of chromosome segregation","modules/clusters: C9 <br>GO.ID: GO:0033047 <br>GO.name: regulation of mitotic sister chromatid segregation","modules/clusters: C9 <br>GO.ID: GO:0045839 <br>GO.name: negative regulation of mitotic nuclear division","modules/clusters: C9 <br>GO.ID: GO:0051784 <br>GO.name: negative regulation of nuclear division","modules/clusters: C9 <br>GO.ID: GO:0010639 <br>GO.name: negative regulation of organelle organization","modules/clusters: C9 <br>GO.ID: GO:2001251 <br>GO.name: negative regulation of chromosome organization","modules/clusters: C9 <br>GO.ID: GO:1903083 <br>GO.name: protein localization to condensed chromosome","modules/clusters: C9 <br>GO.ID: GO:0034501 <br>GO.name: protein localization to kinetochore","modules/clusters: C9 <br>GO.ID: GO:0071459 <br>GO.name: protein localization to chromosome, centromeric region","modules/clusters: C9 <br>GO.ID: GO:0051298 <br>GO.name: centrosome duplication","modules/clusters: C9 <br>GO.ID: GO:0034502 <br>GO.name: protein localization to chromosome","modules/clusters: C9 <br>GO.ID: GO:0044771 <br>GO.name: meiotic cell cycle phase transition","modules/clusters: C9 <br>GO.ID: GO:0046605 <br>GO.name: regulation of centrosome cycle","modules/clusters: C9 <br>GO.ID: GO:0051129 <br>GO.name: negative regulation of cellular component organization","modules/clusters: C9 <br>GO.ID: GO:0051782 <br>GO.name: negative regulation of cell division","modules/clusters: C9 <br>GO.ID: GO:0007098 <br>GO.name: centrosome cycle","modules/clusters: C9 <br>GO.ID: GO:0051315 <br>GO.name: attachment of mitotic spindle microtubules to kinetochore","modules/clusters: C9 <br>GO.ID: GO:0007096 <br>GO.name: regulation of exit from mitosis","modules/clusters: C9 <br>GO.ID: GO:0031023 <br>GO.name: microtubule organizing center organization","modules/clusters: C9 <br>GO.ID: GO:0032886 <br>GO.name: regulation of microtubule-based process","modules/clusters: C9 <br>GO.ID: GO:0007057 <br>GO.name: spindle assembly involved in female meiosis I","modules/clusters: C9 <br>GO.ID: GO:1905342 <br>GO.name: positive regulation of protein localization to kinetochore","modules/clusters: C9 <br>GO.ID: GO:1905340 <br>GO.name: regulation of protein localization to kinetochore","modules/clusters: C9 <br>GO.ID: GO:0010458 <br>GO.name: exit from mitosis"],"showlegend":true,"marker":{"color":"rgba(166,177,176,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(166,177,176,1)"}},"name":"C9","textfont":{"color":"rgba(166,177,176,1)"},"error_y":{"color":"rgba(166,177,176,1)"},"error_x":{"color":"rgba(166,177,176,1)"},"line":{"color":"rgba(166,177,176,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.261770333986498,-0.334758132699157,-0.142424898866857,-0.167993376855944,-0.363763785598364,-0.387220991911225,-0.292807547643146,-0.302808967341477,-0.343607189206063,-0.165689266293397,-0.0126406948247285,-0.324588172893692,-0.162378322703121,-0.250509850082773,-0.143004217449774,-0.331371100861401,-0.230297202990876,-0.326546777780526,-0.238551437560368,0.0530094910456345,-0.264502958933542,-0.2345731969416,-0.340629513050964,-0.21690967063716,0.0262443353926153,-0.275042795350726,-0.291457481978318,-0.317488866707639,-0.251614338974324,-0.268098009538297,-0.130956766442007,-0.259912903942566,-0.330876403630409,-0.222568351080176,-0.0785423430393162,-0.216054164831532,-0.203802847417926,0.00906165907764992,0.0231605337439092],"y":[0.248293534734849,0.165573537559089,-0.253985116371616,-0.272923399102399,-0.268631225265582,-0.250046740761688,-0.274253370231966,-0.259572207739861,-0.2627820284574,-0.25295041065485,-0.212479211585791,-0.269136269097507,0.257930521845851,-0.211193001134354,-0.261985621335604,0.155960983672653,0.107788264421143,0.102522173588103,0.128747175340116,-0.0307525557085026,0.225481303483135,0.168996339995874,0.125388809254877,0.185985522375639,-0.0853948460313559,0.19203708465449,0.150897339951301,0.151525813500646,-0.211802640101811,-0.252159583175046,-0.263404674100065,-0.254571146500109,-0.225877872211529,0.221788113621852,-0.2475263073595,-0.181435177363375,-0.206966830300582,-0.123084567349267,-0.149697248879868],"type":"scatter","mode":"markers","text":["modules/clusters: C9/C10 <br>GO.ID: GO:0051726 <br>GO.name: regulation of cell cycle","modules/clusters: C9/C10 <br>GO.ID: GO:0010564 <br>GO.name: regulation of cell cycle process","modules/clusters: C9/C10 <br>GO.ID: GO:0000226 <br>GO.name: microtubule cytoskeleton organization","modules/clusters: C9/C10 <br>GO.ID: GO:0006996 <br>GO.name: organelle organization","modules/clusters: C9/C10 <br>GO.ID: GO:0140014 <br>GO.name: mitotic nuclear division","modules/clusters: C9/C10 <br>GO.ID: GO:0000070 <br>GO.name: mitotic sister chromatid segregation","modules/clusters: C9/C10 <br>GO.ID: GO:0007051 <br>GO.name: spindle organization","modules/clusters: C9/C10 <br>GO.ID: GO:0007052 <br>GO.name: mitotic spindle organization","modules/clusters: C9/C10 <br>GO.ID: GO:0000819 <br>GO.name: sister chromatid segregation","modules/clusters: C9/C10 <br>GO.ID: GO:0051276 <br>GO.name: chromosome organization","modules/clusters: C9/C10 <br>GO.ID: GO:0007017 <br>GO.name: microtubule-based process","modules/clusters: C9/C10 <br>GO.ID: GO:1902850 <br>GO.name: microtubule cytoskeleton organization involved in mitosis","modules/clusters: C9/C10 <br>GO.ID: GO:0051983 <br>GO.name: regulation of chromosome segregation","modules/clusters: C9/C10 <br>GO.ID: GO:0051304 <br>GO.name: chromosome separation","modules/clusters: C9/C10 <br>GO.ID: GO:0007010 <br>GO.name: cytoskeleton organization","modules/clusters: C9/C10 <br>GO.ID: GO:0010948 <br>GO.name: negative regulation of cell cycle process","modules/clusters: C9/C10 <br>GO.ID: GO:0033044 <br>GO.name: regulation of chromosome organization","modules/clusters: C9/C10 <br>GO.ID: GO:0007093 <br>GO.name: mitotic cell cycle checkpoint signaling","modules/clusters: C9/C10 <br>GO.ID: GO:0033043 <br>GO.name: regulation of organelle organization","modules/clusters: C9/C10 <br>GO.ID: GO:0140694 <br>GO.name: non-membrane-bounded organelle assembly","modules/clusters: C9/C10 <br>GO.ID: GO:0045786 <br>GO.name: negative regulation of cell cycle","modules/clusters: C9/C10 <br>GO.ID: GO:0000075 <br>GO.name: cell cycle checkpoint signaling","modules/clusters: C9/C10 <br>GO.ID: GO:1901991 <br>GO.name: negative regulation of mitotic cell cycle phase transition","modules/clusters: C9/C10 <br>GO.ID: GO:0051128 <br>GO.name: regulation of cellular component organization","modules/clusters: C9/C10 <br>GO.ID: GO:0006259 <br>GO.name: DNA metabolic process","modules/clusters: C9/C10 <br>GO.ID: GO:0045930 <br>GO.name: negative regulation of mitotic cell cycle","modules/clusters: C9/C10 <br>GO.ID: GO:0090068 <br>GO.name: positive regulation of cell cycle process","modules/clusters: C9/C10 <br>GO.ID: GO:1901988 <br>GO.name: negative regulation of cell cycle phase transition","modules/clusters: C9/C10 <br>GO.ID: GO:0008608 <br>GO.name: attachment of spindle microtubules to kinetochore","modules/clusters: C9/C10 <br>GO.ID: GO:0140013 <br>GO.name: meiotic nuclear division","modules/clusters: C9/C10 <br>GO.ID: GO:0016043 <br>GO.name: cellular component organization","modules/clusters: C9/C10 <br>GO.ID: GO:0051225 <br>GO.name: spindle assembly","modules/clusters: C9/C10 <br>GO.ID: GO:0007080 <br>GO.name: mitotic metaphase plate congression","modules/clusters: C9/C10 <br>GO.ID: GO:0045787 <br>GO.name: positive regulation of cell cycle","modules/clusters: C9/C10 <br>GO.ID: GO:0071840 <br>GO.name: cellular component organization or biogenesis","modules/clusters: C9/C10 <br>GO.ID: GO:0051310 <br>GO.name: metaphase plate congression","modules/clusters: C9/C10 <br>GO.ID: GO:0000910 <br>GO.name: cytokinesis","modules/clusters: C9/C10 <br>GO.ID: GO:0051303 <br>GO.name: establishment of chromosome localization","modules/clusters: C9/C10 <br>GO.ID: GO:0050000 <br>GO.name: chromosome localization"],"showlegend":true,"marker":{"color":"rgba(195,186,138,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(195,186,138,1)"}},"name":"C9/C10","textfont":{"color":"rgba(195,186,138,1)"},"error_y":{"color":"rgba(195,186,138,1)"},"error_x":{"color":"rgba(195,186,138,1)"},"line":{"color":"rgba(195,186,138,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.155485783066993,-0.246987161022543,-0.163577528182335,-0.164930661589128,-0.266763055375249,-0.188118031205983,-0.11579715462401],"y":[-0.225390601152579,-0.240971005550035,-0.250324033096186,-0.260691429757525,-0.221700461521683,-0.203936071623672,-0.191738118915755],"type":"scatter","mode":"markers","text":["modules/clusters: C9/C10/C11 <br>GO.ID: GO:0007049 <br>GO.name: cell cycle","modules/clusters: C9/C10/C11 <br>GO.ID: GO:0022402 <br>GO.name: cell cycle process","modules/clusters: C9/C10/C11 <br>GO.ID: GO:0000280 <br>GO.name: nuclear division","modules/clusters: C9/C10/C11 <br>GO.ID: GO:0048285 <br>GO.name: organelle fission","modules/clusters: C9/C10/C11 <br>GO.ID: GO:0098813 <br>GO.name: nuclear chromosome segregation","modules/clusters: C9/C10/C11 <br>GO.ID: GO:1903046 <br>GO.name: meiotic cell cycle process","modules/clusters: C9/C10/C11 <br>GO.ID: GO:0051321 <br>GO.name: meiotic cell cycle"],"showlegend":true,"marker":{"color":"rgba(203,200,149,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(203,200,149,1)"}},"name":"C9/C10/C11","textfont":{"color":"rgba(203,200,149,1)"},"error_y":{"color":"rgba(203,200,149,1)"},"error_x":{"color":"rgba(203,200,149,1)"},"line":{"color":"rgba(203,200,149,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>A second version of the MDS plot provides a nearest ancestor
clustering approach. All terms found are brought up to their nearest
common ancestor. Since these databases, GO ontology in particular, have
a tree-like hierarchy, we are able to find common ancestors for each
term which may provide a broader view if too many individual terms
exist.<br>
This approach does not distinguish based on clusters in which terms were
found, meaning that terms from different clusters may be paired
together.<br>
The size of each dot in this approach is relative to the number of terms
sorted in that common ancestor (dot) while the color is representative
of the dominant cluster in that common ancestor. For example, if an
ancestor has 4 terms within it, and 3 of those terms are found in
cluster 1 (C1), the dot will carry the color associated to C1.<br>
Hovering the cursor over the dots will provide more information on those
specific common ancestors.</p>
<div id="htmlwidget-466a64d082fca32223bf" style="width:100%;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-466a64d082fca32223bf">{"x":{"visdat":{"153461d7d787e":["function () ","plotlyVisDat"],"15346750e3c71":["function () ","data"]},"cur_data":"15346750e3c71","attrs":{"15346750e3c71":{"alpha_stroke":1,"sizes":[20,50],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","text":{},"showlegend":false,"size":{},"fill":{},"marker":{"sizemode":"diameter","opacity":0.4,"line":{"color":["#E0C464","#E0C464","#A6B1B0","#E0C464","#D7D4CB","#E0C464","#A6B1B0","#E0C464","#DE8B88","#DE8B88","#E0C464","#DE8B88","#DE8B88","#B8DA68","#CAEABC","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#FAF9B6","#B8DA68","#FAF9B6","#DE8B88","#E0C464","#FAF9B6","#E0C464","#FAF9B6","#DE8B88","#E9B373","#B8DA68","#E0C464","#E0C464","#A6B1B0","#A6B1B0","#A6B1B0","#E0C464","#FAF9B6","#B8DA68","#FAF9B6","#FAF9B6","#A6B1B0","#A6B1B0","#A6B1B0","#9CA5BC","#E0C464","#B8DA68","#DE8B88","#9CA5BC","#C7B1C9","#DE8B88","#B8DA68","#8DD3C7","#A6B1B0","#E0C464","#DE8B88","#DE8B88","#E0C464","#DE8B88"]},"color":["#E0C464","#E0C464","#A6B1B0","#E0C464","#D7D4CB","#E0C464","#A6B1B0","#E0C464","#DE8B88","#DE8B88","#E0C464","#DE8B88","#DE8B88","#B8DA68","#CAEABC","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#FAF9B6","#B8DA68","#FAF9B6","#DE8B88","#E0C464","#FAF9B6","#E0C464","#FAF9B6","#DE8B88","#E9B373","#B8DA68","#E0C464","#E0C464","#A6B1B0","#A6B1B0","#A6B1B0","#E0C464","#FAF9B6","#B8DA68","#FAF9B6","#FAF9B6","#A6B1B0","#A6B1B0","#A6B1B0","#9CA5BC","#E0C464","#B8DA68","#DE8B88","#9CA5BC","#C7B1C9","#DE8B88","#B8DA68","#8DD3C7","#A6B1B0","#E0C464","#DE8B88","#DE8B88","#E0C464","#DE8B88"]},"visible":true,"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"fillcolor":"rgba(31,119,180,0.5)","x":[-0.408571765984578,-0.46987427905579,-0.219995895654564,0.00435783241895117,-0.0144510295032564,-0.307115702738405,-0.328874070882052,-0.286011810096389,0.245499906782557,0.0440758367222869,-0.164287785214507,0.232214112812236,0.249327284481355,0.0286666711065434,0.198139517463036,0.0206336435477033,0.221804265900459,0.284413252957565,0.233820333573473,0.245552528776188,0.21010917774895,0.21797146750706,0.0804346199449367,0.268251861341853,0.219629021648073,0.121050511172137,0.0615311085740813,0.114238816593055,-0.0199548858304816,0.096336164281587,-0.0817158875660771,0.240339765003376,0.278685000541243,-0.287712308839891,-0.258219089279328,-0.372585504708125,-0.454745564588513,-0.429376655672195,-0.327285763574676,-0.0140184598082404,0.301159095338485,0.181070462072092,0.281201366173843,-0.270182193290357,-0.25434200213516,-0.076391293037793,0.107667933515487,-0.228466518257392,0.0636269478573249,0.0811736515398708,0.13207976292613,0.208576328549186,0.0655979023235572,0.117736751807028,-0.383557509104337,0.01168096097701,-0.197843691403538,0.185502770393349,0.22967427534325,-0.149102410380361,0.070898530289687],"y":[-0.154337918206022,-0.138181871392382,0.258316513332706,-0.0145430972266019,0.244301951312993,-0.112993742592589,-0.102554334263187,-0.205730202210511,-0.18334514111234,-0.204649165733176,-0.173014879107875,-0.140430233770489,-0.162500380698487,-0.15917995786671,0.279267381121563,0.149962332117026,0.190980454652812,-0.101454494890516,-0.123078343022866,-0.100039046670865,0.316469811051283,0.288719605136721,0.194341302000879,0.282906434210687,-0.164375322478826,-0.0495792090541213,-0.0908145028169913,-0.10289656925984,-0.171954616338794,0.0674212552088493,-0.20120917360152,-0.168021216853408,-0.148385493433963,-0.143933450140165,-0.141626018856473,-0.13765854474387,0.0409759874152376,0.174143271912024,0.215606597297216,-0.184088466839068,-0.174620587749303,0.336545051290639,-0.186725679498732,0.296980543620709,0.218127255770693,0.315187247546315,0.241035056623211,-0.220529248153176,-0.159882636926866,-0.184237982967402,0.280471169796705,-0.139686864159894,0.253012674673607,-0.183412472229385,0.18663774523882,0.037308596319396,-0.177085579991139,0.140831663628264,0.15773444791853,0.292761834079906,-0.202622148861773],"type":"scatter","mode":"markers","text":["GO.cluster: 7 <br>GO.count: 10  –  C10: 8, C9: 2 <br>GO.ID: GO:0006996 <br>GO.name: organelle organization","GO.cluster: 5 <br>GO.count: 4  –  C10: 3, C9: 1 <br>GO.ID: GO:0000819 <br>GO.name: sister chromatid segregation","GO.cluster: 15 <br>GO.count: 10  –  C9: 6, C10: 4 <br>GO.ID: GO:0051726 <br>GO.name: regulation of cell cycle","GO.cluster: 43 <br>GO.count: 6  –  C10: 5, C14: 1 <br>GO.ID: GO:0051716 <br>GO.name: cellular response to stimulus","GO.cluster: 25 <br>GO.count: 9  –  C12: 6, C10: 2, C5: 1 <br>GO.ID: GO:0019222 <br>GO.name: regulation of metabolic process","GO.cluster: 22 <br>GO.count: 5  –  C10: 3, C8: 1, C9: 1 <br>GO.ID: GO:0000278 <br>GO.name: mitotic cell cycle","GO.cluster: 6 <br>GO.count: 8  –  C9: 6, C10: 2 <br>GO.ID: GO:0006996 <br>GO.name: organelle organization","GO.cluster: 10 <br>GO.count: 2  –  C10: 1, C11: 1 <br>GO.ID: GO:0048285 <br>GO.name: organelle fission","GO.cluster: 38 <br>GO.count: 6  –  C1: 4, C5: 1, C7: 1 <br>GO.ID: GO:0032502 <br>GO.name: developmental process","GO.cluster: 37 <br>GO.count: 4  –  C1: 1, C2: 1, C5: 1, C7: 1 <br>GO.ID: GO:0048869 <br>GO.name: cellular developmental process","GO.cluster: 23 <br>GO.count: 3  –  C10: 2, C8: 1 <br>GO.ID: GO:0051301 <br>GO.name: cell division","GO.cluster: 40 <br>GO.count: 3  –  C1: 3 <br>GO.ID: GO:0035239 <br>GO.name: tube morphogenesis","GO.cluster: 41 <br>GO.count: 10  –  C1: 4, C11: 2, C14: 1, C2: 1, C3: 1, C7: 1 <br>GO.ID: GO:0007275 <br>GO.name: multicellular organism development","GO.cluster: 18 <br>GO.count: 4  –  C14: 4 <br>GO.ID: GO:0001775 <br>GO.name: cell activation","GO.cluster: 51 <br>GO.count: 5  –  C13: 2, C14: 1, C2: 1, C5: 1 <br>GO.ID: GO:0032501 <br>GO.name: multicellular organismal process","GO.cluster: 12 <br>GO.count: 7  –  C14: 7 <br>GO.ID: GO:0001906 <br>GO.name: cell killing","GO.cluster: 57 <br>GO.count: 10  –  C14: 10 <br>GO.ID: GO:0045088 <br>GO.name: regulation of innate immune response","GO.cluster: 58 <br>GO.count: 9  –  C14: 9 <br>GO.ID: GO:0043207 <br>GO.name: response to external biotic stimulus","GO.cluster: 62 <br>GO.count: 5  –  C14: 3, C13: 2 <br>GO.ID: GO:0006955 <br>GO.name: immune response","GO.cluster: 61 <br>GO.count: 4  –  C14: 3, C12: 1 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","GO.cluster: 60 <br>GO.count: 8  –  C14: 7, C1: 1 <br>GO.ID: GO:0002682 <br>GO.name: regulation of immune system process","GO.cluster: 59 <br>GO.count: 6  –  C14: 6 <br>GO.ID: GO:0002697 <br>GO.name: regulation of immune effector process","GO.cluster: 42 <br>GO.count: 8  –  C2: 4, C14: 2, C5: 1, C6: 1 <br>GO.ID: GO:0007165 <br>GO.name: signal transduction","GO.cluster: 53 <br>GO.count: 10  –  C14: 7, C1: 1, C13: 1, C2: 1 <br>GO.ID: GO:0048583 <br>GO.name: regulation of response to stimulus","GO.cluster: 50 <br>GO.count: 2  –  C2: 2 <br>GO.ID: GO:0032501 <br>GO.name: multicellular organismal process","GO.cluster: 28 <br>GO.count: 3  –  C1: 3 <br>GO.ID: GO:1901071 <br>GO.name: glucosamine-containing compound metabolic process","GO.cluster: 29 <br>GO.count: 4  –  C10: 4 <br>GO.ID: GO:0006259 <br>GO.name: DNA metabolic process","GO.cluster: 30 <br>GO.count: 10  –  C2: 4, C14: 3, C10: 2, C1: 1 <br>GO.ID: GO:0008150 <br>GO.name: biological <br>GO.name: process","GO.cluster: 32 <br>GO.count: 6  –  C10: 3, C13: 1, C2: 1, C9: 1 <br>GO.ID: GO:0051179 <br>GO.name: localization","GO.cluster: 52 <br>GO.count: 10  –  C2: 10 <br>GO.ID: GO:0065008 <br>GO.name: regulation of biological quality","GO.cluster: 20 <br>GO.count: 6  –  C1: 3, C9: 2, C10: 1 <br>GO.ID: GO:0009987 <br>GO.name: cellular process","GO.cluster: 44 <br>GO.count: 4  –  C3: 3, C1: 1 <br>GO.ID: GO:0042330 <br>GO.name: taxis","GO.cluster: 48 <br>GO.count: 4  –  C14: 3, C11: 1 <br>GO.ID: GO:0006950 <br>GO.name: response to stress","GO.cluster: 24 <br>GO.count: 7  –  C10: 4, C11: 2, C8: 1 <br>GO.ID: GO:0007049 <br>GO.name: cell cycle","GO.cluster: 21 <br>GO.count: 8  –  C10: 4, C9: 3, C11: 1 <br>GO.ID: GO:0051321 <br>GO.name: meiotic cell cycle","GO.cluster: 4 <br>GO.count: 3  –  C9: 3 <br>GO.ID: GO:0051310 <br>GO.name: metaphase plate congression","GO.cluster: 3 <br>GO.count: 4  –  C9: 4 <br>GO.ID: GO:0140014 <br>GO.name: mitotic nuclear division","GO.cluster: 2 <br>GO.count: 9  –  C9: 7, C10: 1, C8: 1 <br>GO.ID: GO:0051983 <br>GO.name: regulation of chromosome segregation","GO.cluster: 16 <br>GO.count: 7  –  C10: 5, C8: 1, C9: 1 <br>GO.ID: GO:1901990 <br>GO.name: regulation of mitotic cell cycle phase transition","GO.cluster: 19 <br>GO.count: 2  –  C2: 1, C5: 1 <br>GO.ID: GO:0007154 <br>GO.name: cell communication","GO.cluster: 49 <br>GO.count: 6  –  C14: 4, C1: 1, C2: 1 <br>GO.ID: GO:0050896 <br>GO.name: response to stimulus","GO.cluster: 56 <br>GO.count: 7  –  C2: 5, C14: 2 <br>GO.ID: GO:0065007 <br>GO.name: biological regulation","GO.cluster: 45 <br>GO.count: 5  –  C2: 2, C1: 1, C14: 1, C7: 1 <br>GO.ID: GO:0042221 <br>GO.name: response to chemical","GO.cluster: 17 <br>GO.count: 10  –  C9: 7, C10: 3 <br>GO.ID: GO:0051726 <br>GO.name: regulation of cell cycle","GO.cluster: 8 <br>GO.count: 8  –  C9: 7, C10: 1 <br>GO.ID: GO:0051128 <br>GO.name: regulation of cellular component organization","GO.cluster: 13 <br>GO.count: 11  –  C9: 4, C1: 2, C2: 2, C10: 1, C14: 1, C7: 1 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process","GO.cluster: 26 <br>GO.count: 6  –  C5: 4, C7: 2 <br>GO.ID: GO:0019216 <br>GO.name: regulation of lipid metabolic process","GO.cluster: 11 <br>GO.count: 6  –  C10: 5, C9: 1 <br>GO.ID: GO:0071840 <br>GO.name: cellular component organization or biogenesis","GO.cluster: 34 <br>GO.count: 5  –  C14: 2, C3: 2, C1: 1 <br>GO.ID: GO:0048870 <br>GO.name: cell motility","GO.cluster: 36 <br>GO.count: 8  –  C1: 7, C2: 1 <br>GO.ID: GO:0022008 <br>GO.name: neurogenesis","GO.cluster: 54 <br>GO.count: 7  –  C5: 3, C1: 2, C2: 1, C7: 1 <br>GO.ID: GO:0050793 <br>GO.name: regulation of developmental process","GO.cluster: 47 <br>GO.count: 1  –  C6: 1 <br>GO.ID: GO:0023052 <br>GO.name: signaling","GO.cluster: 35 <br>GO.count: 7  –  C1: 2, C14: 2, C5: 2, C2: 1 <br>GO.ID: GO:0032879 <br>GO.name: regulation of localization","GO.cluster: 46 <br>GO.count: 7  –  C14: 3, C5: 2, C7: 2 <br>GO.ID: GO:0070887 <br>GO.name: cellular response to chemical stimulus","GO.cluster: 1 <br>GO.count: 4  –  C8: 2, C9: 2 <br>GO.ID: GO:0033045 <br>GO.name: regulation of sister chromatid segregation","GO.cluster: 31 <br>GO.count: 6  –  C9: 6 <br>GO.ID: GO:0034502 <br>GO.name: protein localization to chromosome","GO.cluster: 9 <br>GO.count: 3  –  C10: 3 <br>GO.ID: GO:0065004 <br>GO.name: protein-DNA complex assembly","GO.cluster: 55 <br>GO.count: 3  –  C1: 2, C2: 1 <br>GO.ID: GO:0040011 <br>GO.name: locomotion","GO.cluster: 39 <br>GO.count: 5  –  C1: 5 <br>GO.ID: GO:2000026 <br>GO.name: regulation of multicellular organismal development","GO.cluster: 14 <br>GO.count: 5  –  C10: 3, C2: 1, C5: 1 <br>GO.ID: GO:0048522 <br>GO.name: positive regulation of cellular process","GO.cluster: 33 <br>GO.count: 2  –  C1: 1, C2: 1 <br>GO.ID: GO:0051179 <br>GO.name: localization"],"showlegend":false,"fill":"","marker":{"color":["#E0C464","#E0C464","#A6B1B0","#E0C464","#D7D4CB","#E0C464","#A6B1B0","#E0C464","#DE8B88","#DE8B88","#E0C464","#DE8B88","#DE8B88","#B8DA68","#CAEABC","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#FAF9B6","#B8DA68","#FAF9B6","#DE8B88","#E0C464","#FAF9B6","#E0C464","#FAF9B6","#DE8B88","#E9B373","#B8DA68","#E0C464","#E0C464","#A6B1B0","#A6B1B0","#A6B1B0","#E0C464","#FAF9B6","#B8DA68","#FAF9B6","#FAF9B6","#A6B1B0","#A6B1B0","#A6B1B0","#9CA5BC","#E0C464","#B8DA68","#DE8B88","#9CA5BC","#C7B1C9","#DE8B88","#B8DA68","#8DD3C7","#A6B1B0","#E0C464","#DE8B88","#DE8B88","#E0C464","#DE8B88"],"size":[47,29,47,35,44,32,41,23,35,29,26,26,47,29,32,38,47,44,32,29,41,35,41,47,23,26,29,47,35,47,35,29,29,38,41,26,29,44,38,23,35,38,32,47,41,50,35,35,32,41,38,20,38,38,29,35,26,26,32,32,23],"sizemode":"diameter","opacity":0.4,"line":{"color":["#E0C464","#E0C464","#A6B1B0","#E0C464","#D7D4CB","#E0C464","#A6B1B0","#E0C464","#DE8B88","#DE8B88","#E0C464","#DE8B88","#DE8B88","#B8DA68","#CAEABC","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#B8DA68","#FAF9B6","#B8DA68","#FAF9B6","#DE8B88","#E0C464","#FAF9B6","#E0C464","#FAF9B6","#DE8B88","#E9B373","#B8DA68","#E0C464","#E0C464","#A6B1B0","#A6B1B0","#A6B1B0","#E0C464","#FAF9B6","#B8DA68","#FAF9B6","#FAF9B6","#A6B1B0","#A6B1B0","#A6B1B0","#9CA5BC","#E0C464","#B8DA68","#DE8B88","#9CA5BC","#C7B1C9","#DE8B88","#B8DA68","#8DD3C7","#A6B1B0","#E0C464","#DE8B88","#DE8B88","#E0C464","#DE8B88"]}},"visible":true,"textfont":{"size":[47,29,47,35,44,32,41,23,35,29,26,26,47,29,32,38,47,44,32,29,41,35,41,47,23,26,29,47,35,47,35,29,29,38,41,26,29,44,38,23,35,38,32,47,41,50,35,35,32,41,38,20,38,38,29,35,26,26,32,32,23]},"error_y":{"color":"rgba(31,119,180,1)","width":[]},"error_x":{"color":"rgba(31,119,180,1)","width":[]},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 id="gprofiler-results-custom">Gprofiler results – custom<a class="anchor" aria-label="anchor" href="#gprofiler-results-custom"></a>
</h3>
<p>Dotplots were designed to illustrate the top n term results for any
single ontology. In the below code chunk three different ontologies are
demonstrated, GO:BP (biological processes), REAC (REACTOME) <span class="citation">(Jassal et al. 2020)</span> and KEGG<span class="citation">(Kanehisa et al. 2017)</span>. The ‘target_top’
parameter defines the number of terms (from most significant to least)
per cluster that will be shown on the dotplot.</p>
<p>All terms shown are considered to be significant, the ‘top’ aspect
reflects significance based on the adjusted p value. The adjusted p
value is negatively log transformed in order to offer a better
visualization, with most significant being in red and less significant
(but still statistically significant) in blue.</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">target_top</span><span class="op">=</span><span class="fl">10</span>

<span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'GO:BP'</span>
<span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span>gpro_file_location<span class="op">=</span><span class="va">name_result_folder</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span><span class="op">)</span>

<span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'REAC'</span>
<span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span>gpro_file_location<span class="op">=</span><span class="va">name_result_folder</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span><span class="op">)</span>

<span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'KEGG'</span>
<span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span>gpro_file_location<span class="op">=</span><span class="va">name_result_folder</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span><span class="op">)</span>

<span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'HP'</span>
<span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span>gpro_file_location<span class="op">=</span><span class="va">name_result_folder</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span><span class="op">)</span></code></pre></div>
<p>Illustrated below is the dotplot for the REACTOME pathway if it
exists, otherwise it will show <a href="GO:BP" class="uri">GO:BP</a> or
KEGG dotplots if those have been created. All dotplots created above can
be found in the ‘TS_results/gprofiler_results’ folder.<br><img src="TS_results_PBMC_example/gprofiler_results/REAC_dotplot.svg"><!-- --></p>
</div>
<div class="section level3">
<h3 id="ancestor-query-plots">Ancestor query plots<a class="anchor" aria-label="anchor" href="#ancestor-query-plots"></a>
</h3>
<p>Ancestor query plots were designed to query results relating to
specific GOs.</p>
<p>This plot searches for children of the queried term IDs. Pathways,
such as GO pathways are hierarchically organized, so a single ID will be
associated to other IDs. For example, the ‘immune system process
(GO:0002376)’ contains several ‘children’, each of these children is
likely to have children of their own and so on. If we use the ID for
‘immune system process’ this plot will find all children grand-children
great-grand children etc… of ‘immune system process’. It will then split
the children based on the PART cluster in which they were found, these
results are illustrated in dotplot format. This is accomplished via the
GO.db package<span class="citation">(Carlson 2021)</span></p>
<p>Several websites exist to search for GO ancestors of interest,
‘AMIGO2’ is one and ‘quickGO’ is another.</p>
<p>A word of caution with this type of plot. It will always link terms
to the most upstream ancestor, so if the query contains two term IDs
which are related to each other, the plot will illustrate all results as
belonging to the term ID which is the most upstream (oldest/ancestor).
Each ancestor queried for will be illustrated in different colors on the
dotplot.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Dotplot for terms relating to specific ancestors</span>
<span class="va">target_ancestors</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'GO:0002253'</span>,<span class="st">'GO:0019882'</span>,<span class="st">'GO:0002404'</span>,<span class="st">'GO:0002339'</span>,<span class="st">'GO:0042386'</span>,
                    <span class="st">'GO:0035172'</span>,<span class="st">'GO:0002252'</span>,<span class="st">'GO:0006955'</span>,<span class="st">'GO:0002520'</span>,<span class="st">'GO:0090713'</span>,
                    <span class="st">'GO:0045321'</span>,<span class="st">'GO:0001776'</span>,<span class="st">'GO:0050900'</span>,<span class="st">'GO:0031294'</span>,<span class="st">'GO:0002262'</span>,
                    <span class="st">'GO:0002683'</span>,<span class="st">'GO:0002684'</span>,<span class="st">'GO:0002440'</span>,<span class="st">'GO:0002682'</span>,<span class="st">'GO:0002200'</span>,
                    <span class="st">'GO:0045058'</span>,<span class="st">'GO:0002507'</span><span class="op">)</span>
<span class="va">ancestor_ontology</span><span class="op">&lt;-</span><span class="st">'BP'</span>

<span class="va">GOs_ancestors_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_relation_to_ancestors.html">find_relation_to_ancestors</a></span><span class="op">(</span><span class="va">target_ancestors</span>,<span class="va">GO_clusters</span>,ontology <span class="op">=</span> <span class="va">ancestor_ontology</span><span class="op">)</span>
<span class="va">ancestor_plots</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/wrapper_ancestor_curation_plots.html">wrapper_ancestor_curation_plots</a></span><span class="op">(</span><span class="va">GOs_ancestors_clust</span>,<span class="va">sem_data</span>,return_plot<span class="op">=</span><span class="cn">T</span>,target_dir<span class="op">=</span><span class="va">name_result_folder</span><span class="op">)</span></code></pre></div>
<p>Two formats are generated for the ancestor plot. The first is a MDS
plot. The second is a dotplot version. If more than one term was found
to be associated to ancestors, the MDS plot will be created and can be
seen below, otherwise only the dotplot will be created. The dotplot will
not be created if no terms were found. The files for both of these plots
can be found in ‘TS_results/ancestor_plots’. In the MDS and dotplot
alike, the color of each term represents the ancestor to which they have
been associated and the size is proportional to the -log transformed
adjusted p-value. In the MDS plot, the PART clusters in which these
terms are found can be seen by hovering over each dot, in the dotplot
version, this information is made available on the x axis while the y
axis represents the terms.</p>
<div id="htmlwidget-a1bfdb7fa73bfd3e0215" style="width:700px;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-a1bfdb7fa73bfd3e0215">{"x":{"visdat":{"153464f3447f9":["function () ","plotlyVisDat"],"15346172232ab":["function () ","data"]},"cur_data":"15346172232ab","attrs":{"15346172232ab":{"alpha_stroke":1,"sizes":[20,50],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","color":{},"text":{},"showlegend":true,"colors":["#E31A1C","#1F78B4","#33A02C","#FB9A99","#A6CEE3","#B2DF8A"],"size":{},"fill":{},"marker":{"sizemode":"diameter","opacity":0.4,"showlegend":true},"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":true},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"fillcolor":"rgba(227,26,28,0.5)","x":[0.446212680698696,0.411780024075835,0.403819018553647,0.417657709163879],"y":[-0.121628600837781,-0.159815150115054,-0.160398318866941,-0.150894650234641],"type":"scatter","mode":"markers","text":["Ancestor: activation of immune response <br>cluster(s): C14 <br>GO.ID: GO:0002218 <br>GO.name: activation of innate immune response <br>-log10(padj): 0.335659104339437","Ancestor: activation of immune response <br>cluster(s): C14 <br>GO.ID: GO:0002220 <br>GO.name: innate immune response activating cell surface receptor signaling pathway <br>-log10(padj): 0","Ancestor: activation of immune response <br>cluster(s): C14 <br>GO.ID: GO:0002223 <br>GO.name: stimulatory C-type lectin receptor signaling pathway <br>-log10(padj): 0","Ancestor: activation of immune response <br>cluster(s): C14 <br>GO.ID: GO:0002758 <br>GO.name: innate immune response-activating signal transduction <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(227,26,28,1)","size":[21.7583293316659,20,20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(227,26,28,1)"}},"name":"activation of immune response","textfont":{"color":"rgba(227,26,28,1)","size":[21.7583293316659,20,20,20]},"error_y":{"color":"rgba(227,26,28,1)","width":[]},"error_x":{"color":"rgba(227,26,28,1)","width":[]},"line":{"color":"rgba(227,26,28,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(31,120,180,0.5)","x":[-0.285123368051011,-0.118394776533293,0.188474878140028,-0.224271123178166,-0.224271123178166,-0.260541029223181,-0.260541029223181,-0.297078856806281,-0.246234468058166,-0.136358090613685,-0.0598349202266665,-0.13331211771572,-0.0615605603768059,-0.104753031704308,-0.0372733623794609,0.359394992916841,0.417767164866844,0.117379942518648,0.292301616626709,0.352699022527743],"y":[0.136836032536581,0.346002741638373,-0.0332327327970454,-0.189136326467037,-0.189136326467037,-0.206292381592251,-0.206292381592251,0.213974995114546,0.249329358207955,0.314198802817053,0.31878795040736,0.427008318190218,0.432835016161776,0.457254807564421,0.463700091841607,0.126311599768543,0.128493018682601,-0.0218079479269039,0.14059252704232,0.141951824933855],"type":"scatter","mode":"markers","text":["Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0001909 <br>GO.name: leukocyte mediated cytotoxicity <br>-log10(padj): 3.12085517534493","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0001910 <br>GO.name: regulation of leukocyte mediated cytotoxicity <br>-log10(padj): 1.30702244919577","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002228 <br>GO.name: natural killer cell mediated immunity <br>-log10(padj): 4.18770006300083","Ancestor: immune effector process <br>cluster(s): C13/C14 <br>GO.ID: GO:0002263 <br>GO.name: cell activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C13/C14 <br>GO.ID: GO:0002263 <br>GO.name: cell activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C13/C14 <br>GO.ID: GO:0002366 <br>GO.name: leukocyte activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C13/C14 <br>GO.ID: GO:0002366 <br>GO.name: leukocyte activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002443 <br>GO.name: leukocyte mediated immunity <br>-log10(padj): 5.72689822596692","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002449 <br>GO.name: lymphocyte mediated immunity <br>-log10(padj): 2.03295730601035","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002697 <br>GO.name: regulation of immune effector process <br>-log10(padj): 2.82942296065364","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002699 <br>GO.name: positive regulation of immune effector process <br>-log10(padj): 2.13479329355302","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002703 <br>GO.name: regulation of leukocyte mediated immunity <br>-log10(padj): 3.37167742375511","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002705 <br>GO.name: positive regulation of leukocyte mediated immunity <br>-log10(padj): 1.7721306764478","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002706 <br>GO.name: regulation of lymphocyte mediated immunity <br>-log10(padj): 1.82648732421199","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002708 <br>GO.name: positive regulation of lymphocyte mediated immunity <br>-log10(padj): 1.34412228649847","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002715 <br>GO.name: regulation of natural killer cell mediated immunity <br>-log10(padj): 4.5477761477913","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0002717 <br>GO.name: positive regulation of natural killer cell mediated immunity <br>-log10(padj): 2.8905546958325","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0042267 <br>GO.name: natural killer cell mediated cytotoxicity <br>-log10(padj): 1.7295670661888","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0042269 <br>GO.name: regulation of natural killer cell mediated cytotoxicity <br>-log10(padj): 1.90365476040898","Ancestor: immune effector process <br>cluster(s): C14 <br>GO.ID: GO:0045954 <br>GO.name: positive regulation of natural killer cell mediated cytotoxicity <br>-log10(padj): 0.317572823301805"],"showlegend":true,"fill":"","marker":{"color":"rgba(31,120,180,1)","size":[36.3484056405665,26.8467557705293,41.9370062000384,20,20,20,20,50,30.6495203465945,34.8217561183004,31.1829818305838,37.6623223814275,29.2831962775902,29.5679401945559,27.041100959699,43.8232423644483,35.1419909091075,29.0602294537727,29.9721770073236,21.6635854738707],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(31,120,180,1)"}},"name":"immune effector process","textfont":{"color":"rgba(31,120,180,1)","size":[36.3484056405665,26.8467557705293,41.9370062000384,20,20,20,20,50,30.6495203465945,34.8217561183004,31.1829818305838,37.6623223814275,29.2831962775902,29.5679401945559,27.041100959699,43.8232423644483,35.1419909091075,29.0602294537727,29.9721770073236,21.6635854738707]},"error_y":{"color":"rgba(31,120,180,1)","width":[]},"error_x":{"color":"rgba(31,120,180,1)","width":[]},"line":{"color":"rgba(31,120,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(51,160,44,0.5)","x":[-0.116051897944606,-0.120885272145307,0.266476785019981],"y":[-0.263396572744385,-0.254938302712031,-0.288011024000582],"type":"scatter","mode":"markers","text":["Ancestor: immune response <br>cluster(s): C14 <br>GO.ID: GO:0002250 <br>GO.name: adaptive immune response <br>-log10(padj): 4.87841391660418","Ancestor: immune response <br>cluster(s): C14 <br>GO.ID: GO:0002460 <br>GO.name: adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains <br>-log10(padj): 0","Ancestor: immune response <br>cluster(s): C14 <br>GO.ID: GO:0045087 <br>GO.name: innate immune response <br>-log10(padj): 3.52123386826647"],"showlegend":true,"fill":"","marker":{"color":"rgba(51,160,44,1)","size":[45.5552677424114,20,38.4457645098379],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(51,160,44,1)"}},"name":"immune response","textfont":{"color":"rgba(51,160,44,1)","size":[45.5552677424114,20,38.4457645098379]},"error_y":{"color":"rgba(51,160,44,1)","width":[]},"error_x":{"color":"rgba(51,160,44,1)","width":[]},"line":{"color":"rgba(51,160,44,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(251,154,153,0.5)","x":[-0.35160351966227,-0.369939973417336],"y":[-0.205620563499742,-0.209196265843175],"type":"scatter","mode":"markers","text":["Ancestor: leukocyte activation <br>cluster(s): C14 <br>GO.ID: GO:0042110 <br>GO.name: T cell activation <br>-log10(padj): 0.740730970711441","Ancestor: leukocyte activation <br>cluster(s): C14 <br>GO.ID: GO:0046649 <br>GO.name: lymphocyte activation <br>-log10(padj): 1.25629365129009"],"showlegend":true,"fill":"","marker":{"color":"rgba(251,154,153,1)","size":[23.8802730980244,26.5810161193042],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(251,154,153,1)"}},"name":"leukocyte activation","textfont":{"color":"rgba(251,154,153,1)","size":[23.8802730980244,26.5810161193042]},"error_y":{"color":"rgba(251,154,153,1)","width":[]},"error_x":{"color":"rgba(251,154,153,1)","width":[]},"line":{"color":"rgba(251,154,153,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(166,206,227,0.5)","x":[-0.222175865118415,-0.334713829402304,-0.325231942742099,-0.327125211366372],"y":[-0.300577544231484,-0.259503709818835,-0.245645411231639,-0.257768113719659],"type":"scatter","mode":"markers","text":["Ancestor: leukocyte migration <br>cluster(s): C3 <br>GO.ID: GO:0071621 <br>GO.name: granulocyte chemotaxis <br>-log10(padj): 0","Ancestor: leukocyte migration <br>cluster(s): C3 <br>GO.ID: GO:0097530 <br>GO.name: granulocyte migration <br>-log10(padj): 0","Ancestor: leukocyte migration <br>cluster(s): C1 <br>GO.ID: GO:1905517 <br>GO.name: macrophage migration <br>-log10(padj): 0","Ancestor: leukocyte migration <br>cluster(s): C3 <br>GO.ID: GO:1990266 <br>GO.name: neutrophil migration <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(166,206,227,1)","size":[20,20,20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(166,206,227,1)"}},"name":"leukocyte migration","textfont":{"color":"rgba(166,206,227,1)","size":20},"error_y":{"color":"rgba(166,206,227,1)","width":20},"error_x":{"color":"rgba(166,206,227,1)","width":20},"line":{"color":"rgba(166,206,227,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(178,223,138,0.5)","x":[0.146157505481104,-0.113393096075384,-0.0204554723975484,-0.183378170582667,0.0898586837972419,0.438935264003927,0.486435431206144,0.110900349320724,0.198454329149902,-0.210203289945493],"y":[-0.0469134631000676,0.134828690994714,0.175600474146815,0.01748568204005,-0.13062033348335,-0.128838180627108,-0.11761698005194,-0.0363845762587575,-0.0310400179500888,-0.0104860559190031],"type":"scatter","mode":"markers","text":["Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0002253 <br>GO.name: activation of immune response <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0002683 <br>GO.name: negative regulation of immune system process <br>-log10(padj): 0.995986272583971","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0002684 <br>GO.name: positive regulation of immune system process <br>-log10(padj): 5.36597405380246","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0002694 <br>GO.name: regulation of leukocyte activation <br>-log10(padj): 0.303872331442666","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0002768 <br>GO.name: immune response-regulating cell surface receptor signaling pathway <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0045088 <br>GO.name: regulation of innate immune response <br>-log10(padj): 3.98325595604189","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0045089 <br>GO.name: positive regulation of innate immune response <br>-log10(padj): 3.25286558788717","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0050776 <br>GO.name: regulation of immune response <br>-log10(padj): 4.98344701995072","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0050778 <br>GO.name: positive regulation of immune response <br>-log10(padj): 3.27237719572365","Ancestor: regulation of immune system process <br>cluster(s): C14 <br>GO.ID: GO:0051249 <br>GO.name: regulation of lymphocyte activation <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(178,223,138,1)","size":[20,25.2174121136009,48.1093211826537,21.5918162997808,20,40.8660384672858,37.0399339723099,46.1054771185985,37.1421443158499,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(178,223,138,1)"}},"name":"regulation of immune system process","textfont":{"color":"rgba(178,223,138,1)","size":[20,25.2174121136009,48.1093211826537,21.5918162997808,20,40.8660384672858,37.0399339723099,46.1054771185985,37.1421443158499,20]},"error_y":{"color":"rgba(178,223,138,1)","width":[]},"error_x":{"color":"rgba(178,223,138,1)","width":[]},"line":{"color":"rgba(178,223,138,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 class="tabset unlisted unnumbered" id="section"></h3>
<p>A interactive table is also generated in order to better navigate the
results in the event of a large number of terms found for many different
ancestors. The csv version of this data can be found in
‘TS_results/ancestor_plots’.</p>
<div class="section level4">
<h4 id="go0002252">GO:0002252<a class="anchor" aria-label="anchor" href="#go0002252"></a>
</h4>
<p>Ancestor name: immune effector process</p>
<table class="table">
<colgroup>
<col width="13%">
<col width="72%">
<col width="14%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0001909</td>
<td align="left">leukocyte mediated cytotoxicity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0001910</td>
<td align="left">regulation of leukocyte mediated cytotoxicity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002228</td>
<td align="left">natural killer cell mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002263</td>
<td align="left">cell activation involved in immune response</td>
<td align="left">C13</td>
</tr>
<tr class="odd">
<td align="left">GO:0002263</td>
<td align="left">cell activation involved in immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002366</td>
<td align="left">leukocyte activation involved in immune response</td>
<td align="left">C13</td>
</tr>
<tr class="odd">
<td align="left">GO:0002366</td>
<td align="left">leukocyte activation involved in immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002443</td>
<td align="left">leukocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002449</td>
<td align="left">lymphocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002697</td>
<td align="left">regulation of immune effector process</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002699</td>
<td align="left">positive regulation of immune effector process</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002703</td>
<td align="left">regulation of leukocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002705</td>
<td align="left">positive regulation of leukocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002706</td>
<td align="left">regulation of lymphocyte mediated immunity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002708</td>
<td align="left">positive regulation of lymphocyte mediated
immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002715</td>
<td align="left">regulation of natural killer cell mediated
immunity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002717</td>
<td align="left">positive regulation of natural killer cell mediated
immunity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0042267</td>
<td align="left">natural killer cell mediated cytotoxicity</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0042269</td>
<td align="left">regulation of natural killer cell mediated
cytotoxicity</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0045954</td>
<td align="left">positive regulation of natural killer cell mediated
cytotoxicity</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002253">GO:0002253<a class="anchor" aria-label="anchor" href="#go0002253"></a>
</h4>
<p>Ancestor name: activation of immune response</p>
<table class="table">
<colgroup>
<col width="12%">
<col width="74%">
<col width="13%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002218</td>
<td align="left">activation of innate immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002220</td>
<td align="left">innate immune response activating cell surface receptor
signaling pathway</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002223</td>
<td align="left">stimulatory C-type lectin receptor signaling
pathway</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002758</td>
<td align="left">innate immune response-activating signal
transduction</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002682">GO:0002682<a class="anchor" aria-label="anchor" href="#go0002682"></a>
</h4>
<p>Ancestor name: regulation of immune system process</p>
<table class="table">
<colgroup>
<col width="13%">
<col width="72%">
<col width="14%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002253</td>
<td align="left">activation of immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002683</td>
<td align="left">negative regulation of immune system process</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002684</td>
<td align="left">positive regulation of immune system process</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002694</td>
<td align="left">regulation of leukocyte activation</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0002768</td>
<td align="left">immune response-regulating cell surface receptor
signaling pathway</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0045088</td>
<td align="left">regulation of innate immune response</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0045089</td>
<td align="left">positive regulation of innate immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0050776</td>
<td align="left">regulation of immune response</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0050778</td>
<td align="left">positive regulation of immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0051249</td>
<td align="left">regulation of lymphocyte activation</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0006955">GO:0006955<a class="anchor" aria-label="anchor" href="#go0006955"></a>
</h4>
<p>Ancestor name: immune response</p>
<table class="table">
<colgroup>
<col width="8%">
<col width="82%">
<col width="8%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002250</td>
<td align="left">adaptive immune response</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0002460</td>
<td align="left">adaptive immune response based on somatic recombination
of immune receptors built from immunoglobulin superfamily domains</td>
<td align="left">C14</td>
</tr>
<tr class="odd">
<td align="left">GO:0045087</td>
<td align="left">innate immune response</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0045321">GO:0045321<a class="anchor" aria-label="anchor" href="#go0045321"></a>
</h4>
<p>Ancestor name: leukocyte activation</p>
<table class="table">
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0042110</td>
<td align="left">T cell activation</td>
<td align="left">C14</td>
</tr>
<tr class="even">
<td align="left">GO:0046649</td>
<td align="left">lymphocyte activation</td>
<td align="left">C14</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0050900">GO:0050900<a class="anchor" aria-label="anchor" href="#go0050900"></a>
</h4>
<p>Ancestor name: leukocyte migration</p>
<table class="table">
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0071621</td>
<td align="left">granulocyte chemotaxis</td>
<td align="left">C3</td>
</tr>
<tr class="even">
<td align="left">GO:0097530</td>
<td align="left">granulocyte migration</td>
<td align="left">C3</td>
</tr>
<tr class="odd">
<td align="left">GO:1905517</td>
<td align="left">macrophage migration</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:1990266</td>
<td align="left">neutrophil migration</td>
<td align="left">C3</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level3">
<h3 class="unlisted unnumbered" id="section-1"></h3>
</div>
</div>
<div class="section level2">
<h2 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-brionne2019viseago" class="csl-entry">
Brionne, Aurélien, Amélie Juanchich, and Christelle Hennequet-Antier.
2019. <span>“ViSEAGO: A Bioconductor Package for Clustering Biological
Functions Using Gene Ontology and Semantic Similarity.”</span>
<em>BioData Mining</em> 12 (1): 1–13.
</div>
<div id="ref-GOdb_ref" class="csl-entry">
Carlson, Marc. 2021. <em>GO.db: A Set of Annotation Maps Describing the
Entire Gene Ontology</em>.
</div>
<div id="ref-gu2016complex" class="csl-entry">
Gu, Zuguang, Roland Eils, and Matthias Schlesner. 2016. <span>“Complex
Heatmaps Reveal Patterns and Correlations in Multidimensional Genomic
Data.”</span> <em>Bioinformatics</em> 32 (18): 2847–49.
</div>
<div id="ref-jassal2020reactome" class="csl-entry">
Jassal, Bijay, Lisa Matthews, Guilherme Viteri, Chuqiao Gong, Pascual
Lorente, Antonio Fabregat, Konstantinos Sidiropoulos, et al. 2020.
<span>“The Reactome Pathway Knowledgebase.”</span> <em>Nucleic Acids
Research</em> 48 (D1): D498–503.
</div>
<div id="ref-kanehisa2017kegg" class="csl-entry">
Kanehisa, Minoru, Miho Furumichi, Mao Tanabe, Yoko Sato, and Kanae
Morishima. 2017. <span>“KEGG: New Perspectives on Genomes, Pathways,
Diseases and Drugs.”</span> <em>Nucleic Acids Research</em> 45 (D1):
D353–61.
</div>
<div id="ref-kolberg2020gprofiler2" class="csl-entry">
Kolberg, Liis, Uku Raudvere, Ivan Kuzmin, Jaak Vilo, and Hedi Peterson.
2020. <span>“Gprofiler2–an r Package for Gene List Functional Enrichment
Analysis and Namespace Conversion Toolset g: Profiler.”</span>
<em>F1000Research</em> 9.
</div>
<div id="ref-love2014moderated" class="csl-entry">
Love, Michael I, Wolfgang Huber, and Simon Anders. 2014.
<span>“Moderated Estimation of Fold Change and Dispersion for RNA-Seq
Data with DESeq2.”</span> <em>Genome Biology</em> 15 (12): 1–21.
</div>
<div id="ref-timeseriesexperiment" class="csl-entry">
Nguyen, Lan Huong. 2021. <em>TimeSeriesExperiment: Analysis for Short
Time-Series Data</em>. <a href="https://github.com/nlhuong/TimeSeriesExperiment" class="external-link">https://github.com/nlhuong/TimeSeriesExperiment</a>.
</div>
<div id="ref-nilsen2013identifying" class="csl-entry">
Nilsen, Gro, Ørnulf Borgan, Knut LiestØl, and Ole Christian Lingjærde.
2013. <span>“Identifying Clusters in Genomics Data by Recursive
Partitioning.”</span> <em>Statistical Applications in Genetics and
Molecular Biology</em> 12 (5): 637–52.
</div>
<div id="ref-raudvere2019g" class="csl-entry">
Raudvere, Uku, Liis Kolberg, Ivan Kuzmin, Tambet Arak, Priit Adler, Hedi
Peterson, and Jaak Vilo. 2019. <span>“G: Profiler: A Web Server for
Functional Enrichment Analysis and Conversions of Gene Lists (2019
Update).”</span> <em>Nucleic Acids Research</em> 47 (W1): W191–98.
</div>
<div id="ref-smyth2005limma" class="csl-entry">
Smyth, Gordon K. 2005. <span>“Limma: Linear Models for Microarray
Data.”</span> In <em>Bioinformatics and Computational Biology Solutions
Using r and Bioconductor</em>, 397–420. Springer.
</div>
<div id="ref-wang2007new" class="csl-entry">
Wang, James Z, Zhidian Du, Rapeeporn Payattakool, Philip S Yu, and
Chin-Fu Chen. 2007. <span>“A New Method to Measure the Semantic
Similarity of GO Terms.”</span> <em>Bioinformatics</em> 23 (10):
1274–81.
</div>
<div id="ref-yu2010gosemsim" class="csl-entry">
Yu, Guangchuang, Fei Li, Yide Qin, Xiaochen Bo, Yibo Wu, and Shengqi
Wang. 2010. <span>“GOSemSim: An r Package for Measuring Semantic
Similarity Among GO Terms and Gene Products.”</span>
<em>Bioinformatics</em> 26 (7): 976–78.
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Yohan Lefol.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.4.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
