<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>TimeSeries Analysis – Murine example run • TimeSeriesAnalysis</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="TimeSeries Analysis – Murine example run">
<meta property="og:description" content="TimeSeriesAnalysis">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TimeSeriesAnalysis</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/TS_analysis_PBMC.html">TimeSeries Analysis – PBMC example run</a>
    </li>
    <li>
      <a href="../articles/TS_analysis_celegans.html">TimeSeries Analysis – C.elegans example run</a>
    </li>
    <li>
      <a href="../articles/TS_analysis_murine.html">TimeSeries Analysis – Murine example run</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/Ylefol/TimeSeriesAnalysis/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="TS_analysis_murine_files/htmlwidgets-1.6.1/htmlwidgets.js"></script><script src="TS_analysis_murine_files/plotly-binding-4.10.1/plotly.js"></script><script src="TS_analysis_murine_files/typedarray-0.1/typedarray.min.js"></script><link href="TS_analysis_murine_files/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="TS_analysis_murine_files/crosstalk-1.2.0/js/crosstalk.min.js"></script><link href="TS_analysis_murine_files/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet">
<script src="TS_analysis_murine_files/plotly-main-2.11.1/plotly-latest.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>TimeSeries Analysis – Murine example run</h1>
                        <h4 data-toc-skip class="author">Yohan
Lefol</h4>
            
            <h4 data-toc-skip class="date">07. March 2023</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/Ylefol/TimeSeriesAnalysis/blob/HEAD/vignettes/TS_analysis_murine.Rmd" class="external-link"><code>vignettes/TS_analysis_murine.Rmd</code></a></small>
      <div class="hidden name"><code>TS_analysis_murine.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="timeseries-analysis">TimeSeries analysis<a class="anchor" aria-label="anchor" href="#timeseries-analysis"></a>
</h2>
<div class="section level3">
<h3 id="parameter-set-up">Parameter set up<a class="anchor" aria-label="anchor" href="#parameter-set-up"></a>
</h3>
<p>Set the below parameters for the downstream analysis</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Give names to saved object and name of results repository</span></span>
<span><span class="va">name_result_folder</span><span class="op">&lt;-</span><span class="st">'TS_results_murine_example/'</span></span>
<span><span class="va">obj_name</span><span class="op">&lt;-</span><span class="st">'timeSeries_obj_murine_example.Rdata'</span></span>
<span></span>
<span><span class="co">#Set-up time series object parameters</span></span>
<span><span class="va">diff_exp_type</span><span class="op">&lt;-</span><span class="st">'DESeq2'</span> <span class="co">#package used for DE – can also be 'limma'</span></span>
<span><span class="va">p_val_filter_type</span><span class="op">&lt;-</span><span class="st">'padj'</span> <span class="co">#Either padj or pvalue, used to filter for significance</span></span>
<span><span class="va">p_thresh</span><span class="op">&lt;-</span><span class="fl">0.05</span> <span class="co">#pvalue or padj value threshold for significance</span></span>
<span><span class="va">l2fc_thresh</span><span class="op">&lt;-</span><span class="fl">1</span> <span class="co">#log(2)foldChange threshold for significance</span></span>
<span><span class="va">name_control</span><span class="op">&lt;-</span><span class="st">'Control'</span> <span class="co">#Name of experiment as seen in the sample file</span></span>
<span><span class="va">name_experiment</span><span class="op">&lt;-</span><span class="st">'Treated'</span> <span class="co">#Name of control as seen in the sample file</span></span>
<span><span class="va">graphic_vector</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"#e31a1c"</span>,<span class="st">"#1f78b4"</span><span class="op">)</span> <span class="co">#Pre-set colors for the groups</span></span>
<span></span>
<span><span class="co">#Declare organism and load library</span></span>
<span><span class="va">org_sem_sim</span><span class="op">=</span><span class="st">'org.Mm.eg.db'</span></span>
<span><span class="kw">if</span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span>package<span class="op">=</span><span class="va">org_sem_sim</span><span class="op">)</span> <span class="op">==</span><span class="st">''</span><span class="op">)</span><span class="op">{</span><span class="co">#Package not installed</span></span>
<span>  <span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">require</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span>  <span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocManager/man/install.html" class="external-link">install</a></span><span class="op">(</span><span class="va">org_sem_sim</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st">'org.Mm.eg.db'</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Loading required package: AnnotationDbi</span></span></code></pre>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Define specimen and ontology parameters</span></span>
<span><span class="va">my_ont_gpro</span><span class="op">=</span><span class="st">'GO:BP'</span></span>
<span><span class="va">my_ont_sem_sim</span><span class="op">=</span><span class="st">'BP'</span></span>
<span></span>
<span><span class="va">my_org_gpro</span><span class="op">=</span><span class="st">'mmusculus'</span> <span class="co">#Set the species for the gprofiler analysis</span></span>
<span></span>
<span><span class="co"># The seed serves to create reproducible results with PART.</span></span>
<span><span class="co"># A seed will ensure that the random components of PART clustering will be the same</span></span>
<span><span class="co"># as long as the same seed is used. For more information on seeds, please consult</span></span>
<span><span class="co"># this link: https://www.r-bloggers.com/2018/07/%F0%9F%8C%B1-setting-a-seed-in-r-when-using-parallel-simulation/</span></span>
<span><span class="va">PART_seed</span><span class="op">=</span><span class="fl">123456</span></span>
<span></span>
<span><span class="va">PART_l2fc</span><span class="op">&lt;-</span><span class="fl">1</span> <span class="co">#log(2)foldChange threshold for PART clustering</span></span>
<span><span class="va">PART_min_clust</span><span class="op">&lt;-</span><span class="fl">50</span> <span class="co">#Minimum cluster size for PART</span></span>
<span><span class="va">PART_recursion</span><span class="op">&lt;-</span><span class="fl">100</span> <span class="co">#Number of recursions, default is 100, using 10 for example</span></span>
<span></span>
<span><span class="va">log_tp_traj</span><span class="op">&lt;-</span><span class="cn">TRUE</span> <span class="co">#Defines if timepoints should be log transformed for illustration purposes</span></span>
<span></span>
<span><span class="co"># Allows for all temporal combinations to be done instead</span></span>
<span><span class="co"># of just sequential comparison. ex: do TP2vsTP1, TP3vsTP2, AND TP3vsTP1. In a normal instance</span></span>
<span><span class="co"># only the first two comparison of the example would be run.</span></span>
<span><span class="va">do_all_temporal_comparisons</span><span class="op">=</span><span class="cn">FALSE</span></span>
<span></span>
<span><span class="co">#Used to highlight specific genes regardless of differential gene expression significance</span></span>
<span><span class="va">genes_of_interest</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="co">#The ancestors that will be queried, the ontology must be specified (BP,MF,or CC)</span></span>
<span><span class="co">#Set to an empty vector c() if not required by the analysis</span></span>
<span><span class="va">target_ancestors</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'GO:0002253'</span>,<span class="st">'GO:0019882'</span>,<span class="st">'GO:0002404'</span>,<span class="st">'GO:0002339'</span>,<span class="st">'GO:0042386'</span>,</span>
<span>                    <span class="st">'GO:0035172'</span>,<span class="st">'GO:0002252'</span>,<span class="st">'GO:0006955'</span>,<span class="st">'GO:0002520'</span>,<span class="st">'GO:0090713'</span>,</span>
<span>                    <span class="st">'GO:0045321'</span>,<span class="st">'GO:0001776'</span>,<span class="st">'GO:0050900'</span>,<span class="st">'GO:0031294'</span>,<span class="st">'GO:0002262'</span>,</span>
<span>                    <span class="st">'GO:0002683'</span>,<span class="st">'GO:0002684'</span>,<span class="st">'GO:0002440'</span>,<span class="st">'GO:0002682'</span>,<span class="st">'GO:0002200'</span>,</span>
<span>                    <span class="st">'GO:0045058'</span>,<span class="st">'GO:0002507'</span><span class="op">)</span></span>
<span><span class="va">ancestor_ontology</span><span class="op">&lt;-</span><span class="st">'BP'</span></span>
<span></span>
<span><span class="co">#Some extra set-up</span></span>
<span><span class="va">name_save_obj</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="va">obj_name</span><span class="op">)</span><span class="co">#The object will be saved in result folder</span></span>
<span><span class="co">#Create main directory for results</span></span>
<span><span class="co"># dir.create(name_result_folder)</span></span>
<span></span>
<span><span class="va">my_group_names</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">name_experiment</span>,<span class="va">name_control</span><span class="op">)</span> </span>
<span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">graphic_vector</span><span class="op">)</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">name_experiment</span>,<span class="va">name_control</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="create-timeseries-object">Create TimeSeries object<a class="anchor" aria-label="anchor" href="#create-timeseries-object"></a>
</h3>
<p>Using the parameter above, generate a TimeSeries_Object and load the
example data as well as the semantic similarity data</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/methods/new.html" class="external-link">new</a></span><span class="op">(</span><span class="st">'TimeSeries_Object'</span>,</span>
<span>                 group_names<span class="op">=</span><span class="va">my_group_names</span>,group_colors<span class="op">=</span><span class="va">graphic_vector</span>,DE_method<span class="op">=</span><span class="va">diff_exp_type</span>,</span>
<span>                 DE_p_filter<span class="op">=</span><span class="va">p_val_filter_type</span>,DE_p_thresh<span class="op">=</span><span class="va">p_thresh</span>,DE_l2fc_thresh<span class="op">=</span><span class="va">l2fc_thresh</span>,</span>
<span>                 PART_l2fc_thresh<span class="op">=</span><span class="va">PART_l2fc</span>,sem_sim_org<span class="op">=</span><span class="va">org_sem_sim</span>,Gpro_org<span class="op">=</span><span class="va">my_org_gpro</span><span class="op">)</span></span>
<span><span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TS_load_example_data.html">TS_load_example_data</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'MURINE'</span><span class="op">)</span></span>
<span><span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/add_semantic_similarity_data.html">add_semantic_similarity_data</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">my_ont_sem_sim</span>,vignette_run<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="differental-gene-expression-analysis">Differental gene expression analysis<a class="anchor" aria-label="anchor" href="#differental-gene-expression-analysis"></a>
</h3>
<p>Loads the necessary functions to perform differential gene expression
analysis If the tool used is DESeq2<span class="citation">(Love, Huber,
and Anders 2014)</span>, the data needs to be normalized using it’s
method. If the tool used is limma<span class="citation">(Smyth
2005)</span>, the normalized matrix should have been inputed, and thus
no normalization is needed.</p>
<p>This code chunk performs both the conditional and temporal
differential gene expression and saves the results within the Time
Series object.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Perform normalization if the DESeq2 tool is being used and if normalized matrix doesn't exist</span></span>
<span><span class="kw">if</span> <span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'DE_method'</span><span class="op">)</span><span class="op">==</span><span class="st">'DESeq2'</span> <span class="op">&amp;</span> <span class="st">'norm'</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="fu">assays</span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'exp_data'</span><span class="op">)</span><span class="op">)</span><span class="op">)</span><span class="op">!=</span><span class="cn">TRUE</span><span class="op">)</span><span class="op">{</span></span>
<span>  <span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/normalize_timeSeries_with_deseq2.html">normalize_timeSeries_with_deseq2</a></span><span class="op">(</span>time_object<span class="op">=</span><span class="va">TS_object</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span></span>
<span><span class="co">#Perform conditional differential gene expression analysis</span></span>
<span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/conditional_DE_wrapper.html">conditional_DE_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span></span>
<span><span class="co">#Perform temporal differential gene expression analysis</span></span>
<span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/temporal_DE_wrapper.html">temporal_DE_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span>,do_all_combinations<span class="op">=</span><span class="va">do_all_temporal_comparisons</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="part-clustering">PART clustering<a class="anchor" aria-label="anchor" href="#part-clustering"></a>
</h3>
<p>The code chunk below prepares and initiates PART clustering<span class="citation">(Nilsen et al. 2013)</span>. It first retrieves the
number of significant genes for PART clustering in the ‘signi_genes’
variable This chunk can be quite lengthy depending on the number of
genes included for clustering as well as the number of recursions set
for the clustering</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Extract genes for PART clustering based on defined log(2)foldChange threshold</span></span>
<span><span class="va">signi_genes</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/select_genes_with_l2fc.html">select_genes_with_l2fc</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span></span>
<span></span>
<span><span class="co">#Use all samples, but implement a custom order. In this case it is reversed</span></span>
<span><span class="va">samp_dta</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/exp_sample_data.html">exp_sample_data</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span></span>
<span><span class="va">samps_2</span><span class="op">&lt;-</span><span class="va">samp_dta</span><span class="op">$</span><span class="va">sample</span><span class="op">[</span><span class="va">samp_dta</span><span class="op">$</span><span class="va">group</span><span class="op">==</span><span class="va">TS_object</span><span class="op">@</span><span class="va">group_names</span><span class="op">[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="va">samps_1</span><span class="op">&lt;-</span><span class="va">samp_dta</span><span class="op">$</span><span class="va">sample</span><span class="op">[</span><span class="va">samp_dta</span><span class="op">$</span><span class="va">group</span><span class="op">==</span><span class="va">TS_object</span><span class="op">@</span><span class="va">group_names</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span></span>
<span></span>
<span><span class="co">#Create the matrix that will be used for PART clustering</span></span>
<span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/prep_counts_for_PART.html">prep_counts_for_PART</a></span><span class="op">(</span>object<span class="op">=</span><span class="va">TS_object</span>,target_genes<span class="op">=</span><span class="va">signi_genes</span>,</span>
<span>                                scale<span class="op">=</span><span class="cn">TRUE</span>,target_samples<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">samps_2</span>,<span class="va">samps_1</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="co">#Sets a seed for reproducibility</span></span>
<span><span class="kw">if</span> <span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/NULL.html" class="external-link">is.null</a></span><span class="op">(</span><span class="va">PART_seed</span><span class="op">)</span><span class="op">==</span><span class="cn">FALSE</span><span class="op">)</span><span class="op">{</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/base/Random.html" class="external-link">set.seed</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/character.html" class="external-link">as.character</a></span><span class="op">(</span><span class="va">PART_seed</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/compute_PART.html">compute_PART</a></span><span class="op">(</span><span class="va">TS_object</span>,part_recursion<span class="op">=</span><span class="va">PART_recursion</span>,part_min_clust<span class="op">=</span><span class="va">PART_min_clust</span>,</span>
<span>                        custom_seed<span class="op">=</span><span class="va">PART_seed</span>,dist_param<span class="op">=</span><span class="st">"euclidean"</span>, hclust_param<span class="op">=</span><span class="st">"average"</span>,</span>
<span>                        vignette_run<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="gprofiler-analysis">Gprofiler analysis<a class="anchor" aria-label="anchor" href="#gprofiler-analysis"></a>
</h3>
<p>The code chunk below runs a gprofiler analysis<span class="citation">(Kolberg et al. 2020; Raudvere et al. 2019)</span>.
IMPORTANT: This function requires that there be a stable internet
connection lack of connection or intermittent drops in the connection
will result in an error and the termination (stop) of the code chunk</p>
<p>If an error has occured, this code chunk can be re-run separately
from the above chunks by uncommenting (removing the ‘#’ in front) the
“load(‘timeseries_obj_res.Rdata’)” line.</p>
<p>This will load the results saved after the PART clustering code
chunk</p>
<p>This code chunk will overwrite the saved object if it is completed.
The overwritten object will then contain the gprofiler analysis results
and can be used to generate plots with the downstream code chunks.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/run_gprofiler_PART_clusters.html">run_gprofiler_PART_clusters</a></span><span class="op">(</span><span class="va">TS_object</span>,vignette_run <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="co">#Run the gprofiler analysis</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="timeseries-analysis-results">TimeSeries analysis results<a class="anchor" aria-label="anchor" href="#timeseries-analysis-results"></a>
</h2>
<p>Most plots are created in SVG format for ease of editing with SVG
editing software such as InkScape (open source software). Some plots are
created in html format as they are interactive plots. These require a
web browser to open them</p>
<p>To convert SVG files to PNG/JPG/PDF, this website is available: <a href="https://svgtopng.com/" class="external-link uri">https://svgtopng.com/</a> HTML
files can be opened and then saved as PNG (or other) using the camera
icon in the top right of each respective interactive plot</p>
<div class="section level3">
<h3 id="timeseries-summary-of-experiment">TimeSeries summary of experiment<a class="anchor" aria-label="anchor" href="#timeseries-summary-of-experiment"></a>
</h3>
<p><strong>Name of groups beings compared:</strong> Treated vs Control
<strong>Number of genes analyzed:</strong> 41115</p>
<p><strong>Differential expression parameters used:</strong>
<strong>Method:</strong> DESeq2 <strong>P statistic filter
used:</strong> padj with a 0.05 threshold <strong>Log(2)FoldChange
significance threshold used:</strong> 1</p>
<p><strong>PART parameters used:</strong> <strong>PART log(2)FoldChange
significance threshold:</strong> 1 <strong>number of
recursions:</strong> 100 <strong>minimum cluster size:</strong> 50
<strong>distance parameter:</strong> euclidean <strong>hclust
parameter:</strong> average <strong>custom seed used:</strong> 123456
<strong>PART computation time:</strong> 3155.613 sec elapsed</p>
<p><strong>PCA of sample data:</strong>
<img src="TS_analysis_murine_files/figure-html/show_PCA_clust%20-1.png" width="100%"></p>
</div>
<div class="section level3">
<h3 id="differential-gene-expression-results">Differential Gene Expression results<a class="anchor" aria-label="anchor" href="#differential-gene-expression-results"></a>
</h3>
<p><strong>DEGs found per analysis:</strong></p>
<table class="table">
<thead><tr class="header">
<th align="left">experiment.name</th>
<th align="left">number.of.DEGs</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">Treated_vs_Control_TP_7.5 (conditional)</td>
<td align="left">7</td>
</tr>
<tr class="even">
<td align="left">Treated_vs_Control_TP_540 (conditional)</td>
<td align="left">429</td>
</tr>
<tr class="odd">
<td align="left">Treated_vs_Control_TP_45 (conditional)</td>
<td align="left">37</td>
</tr>
<tr class="even">
<td align="left">Treated_vs_Control_TP_2160 (conditional)</td>
<td align="left">702</td>
</tr>
<tr class="odd">
<td align="left">Treated_vs_Control_TP_150 (conditional)</td>
<td align="left">136</td>
</tr>
<tr class="even">
<td align="left">TP_45_vs_TP_7.5 (temporal)</td>
<td align="left">38</td>
</tr>
<tr class="odd">
<td align="left">TP_150_vs_TP_45 (temporal)</td>
<td align="left">1297</td>
</tr>
<tr class="even">
<td align="left">TP_540_vs_TP_150 (temporal)</td>
<td align="left">1074</td>
</tr>
<tr class="odd">
<td align="left">TP_2160_vs_TP_540 (temporal)</td>
<td align="left">59</td>
</tr>
<tr class="even">
<td align="left"><strong>total unique genes</strong></td>
<td align="left"><strong>2851</strong></td>
</tr>
</tbody>
</table>
<p>Each experiment listed in the above table has separate results which
can be viewed in the ‘TS_results’ folder. In the main results folder,
the differential gene expression results are located in the
‘DE_results_conditional’ and ‘DE_results_temporal’ plot. Each individual
experiment creates the following: <strong>DE_raw_data.csv</strong>: A
csv file containing all the genes and their associated differential
expression results, regardless of significance.
<strong>DE_sig_data</strong>: A csv file containing only the significant
differential expression results. <strong>MA_plot.png</strong>: A MA
plot, these are often used to evaluate the quality of the normalization.
<strong>volcano_plot.png</strong>: A volcano plot which splits
significantly up-regulated and down-regulated genes as well as
non-significant genes. <strong>PCA_plot</strong>: A PCA plot using only
the samples implicated in the differential gene expression analysis.</p>
<p>The pipeline also creates two large heatmaps which summarize
conditional and temporal differential expression results. The heatmaps
were created using the ComplexHeatmaps package<span class="citation">(Gu, Eils, and Schlesner 2016)</span>. These heatmaps
illustrate each experiment as different columns (distinguished by color)
while the rows distinguish the groups. Plotted in the heatmaps is the
log transformed counts, while the log transformed log(2)fold change is
seen in a histogram at the bottom of the heatmap. The number of genes in
each experiment (colored columns) is indicated in the legend at the
bottom of the heatmap. In the case of the temporal heatmap, the rows are
labelled ‘experiment’ and ‘control’. The ‘experiment’ represents the
time points being compared while the control represents the timepoint
which the ‘experiment’ is compared against. For example, TP2_vs_TP1
would have the ‘experiment’ be TP2 and the control be TP1, while
TP3_vs_TP2 would have TP3 as the ‘experiment’ and TP2 as the ‘control’.
Note that TP=time point.</p>
<p><strong>The conditional heatmap</strong>
<img src="TS_analysis_murine_files/figure-html/show_cond_heat_clust%20-1.png" width="100%"><strong>The temporal heatmap</strong>
<img src="TS_analysis_murine_files/figure-html/show_temp_heat_clust%20-1.png" width="100%">
## Genes of interest results</p>
<p><img src="TS_analysis_murine_files/figure-html/genes_interest-1.png" width="65%" style="float:right; padding:10px">
If genes of interest have been specified, the pipeline will extract the
differential gene expression results, both conditional and temporal, for
each gene requested, provided that they are found in the count matrix.
These data files, along with individual trajectory plots are stored in
the ‘TS_results/genes_of_interest’ folder. Below the trajectory of the
most variable gene is shown. If no genes of interest were submitted, a
random gene is taken from the differential expression results.
Illustrated in the trajectory plots is the timepoints on the x axis and
the number of reads (RNAseq) or expression (microarray) on the y axis.
Note that data files and trajectory plots are created for all genes of
interest found, regardless of their significance based on the
differential gene expression analyses.</p>
</div>
<div class="section level3">
<h3 id="cluster-results">Cluster results<a class="anchor" aria-label="anchor" href="#cluster-results"></a>
</h3>
<p>The code chunk below create the trajectory plots for the clusters
found by PART Trajectories are first calculated by calculating the mean
of each gene for each time point using the available replicates. The
mean value of each gene is calculated once for each group
(control/experiment). Afterwards, the genes are scaled in order to be
able to represent them on the same axis of values. Each gene is plotted
as an individual colored line’, a mean line (gray color) is also plotted
to show the overall trajectory of the cluster. These plots were inspired
by <span class="citation">(Nguyen 2021)</span></p>
<p>The log_tp_traj parameter defines if the time points should be log
transformed. A log transformation of the time points is useful if the
time points are not evenly sampled through time. For example, two time
points at 0 and 15 minutes, with another time point at 24 hours (1440
minutes) will result in the first two time points being very compressed
and possibly indistinguishable from each other.</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># dir.create(paste0(name_result_folder,'PART_results')) #create the directory to store results</span></span>
<span><span class="va">PART_heat</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/PART_heat_map.html">PART_heat_map</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="cn">NULL</span><span class="op">)</span> <span class="co">#Create a summary heatmap, returns plot if save location is NULL</span></span>
<span><span class="va">ts_data</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/calculate_cluster_traj_data.html">calculate_cluster_traj_data</a></span><span class="op">(</span><span class="va">TS_object</span>,scale_feat<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span> <span class="co">#Calculate scaled gene values for genes of clusters</span></span>
<span><span class="va">mean_ts_data</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/calculate_mean_cluster_traj.html">calculate_mean_cluster_traj</a></span><span class="op">(</span><span class="va">ts_data</span><span class="op">)</span> <span class="co">#Calculate the mean scaled values for each cluster</span></span>
<span></span>
<span><span class="co">#Function which determines the number of SVG files to plot all cluster trajectories</span></span>
<span><span class="co">#Function commented out for vignettes</span></span>
<span><span class="co"># wrapper_cluster_trajectory(TS_object,ts_data,mean_ts_data,log_TP=log_tp_traj,plot_name=paste0(name_result_folder,'/PART_results/Ctraj'))</span></span>
<span></span>
<span><span class="co">#find the most variable cluster, subset necessary data and plot it on a single column</span></span>
<span><span class="co">#The function will also filter for a cluster which has gprofiler results</span></span>
<span><span class="va">target_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_most_variable_cluster.html">find_most_variable_cluster</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">mean_ts_data</span><span class="op">)</span></span>
<span><span class="va">sub_ts</span><span class="op">&lt;-</span><span class="va">ts_data</span><span class="op">[</span><span class="va">ts_data</span><span class="op">$</span><span class="va">cluster</span><span class="op">==</span><span class="va">target_clust</span>,<span class="op">]</span></span>
<span><span class="va">sub_mean</span><span class="op">&lt;-</span><span class="va">mean_ts_data</span><span class="op">[</span><span class="va">mean_ts_data</span><span class="op">$</span><span class="va">cluster</span><span class="op">==</span><span class="va">target_clust</span>,<span class="op">]</span></span>
<span><span class="va">plt_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/plot_cluster_traj.html">plot_cluster_traj</a></span><span class="op">(</span>object <span class="op">=</span> <span class="va">TS_object</span>,ts_data <span class="op">=</span> <span class="va">sub_ts</span>,ts_mean_data <span class="op">=</span> <span class="va">sub_mean</span>, num_col<span class="op">=</span><span class="fl">1</span>,rem_legend_axis<span class="op">=</span><span class="cn">TRUE</span>,log_TP<span class="op">=</span><span class="va">log_tp_traj</span><span class="op">)</span></span>
<span><span class="va">cluster_map_sum</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'PART_results'</span><span class="op">)</span><span class="op">$</span><span class="va">cluster_map</span><span class="op">[</span>,<span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">1</span><span class="op">)</span><span class="op">]</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/row.names.html" class="external-link">row.names</a></span><span class="op">(</span><span class="va">cluster_map_sum</span><span class="op">)</span><span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'Cluster name'</span>,<span class="st">'Number of genes'</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/BiocGenerics/man/row_colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">cluster_map_sum</span><span class="op">)</span><span class="op">=</span><span class="cn">NULL</span></span></code></pre></div>
<p><strong>PART result summary:</strong></p>
<table class="table">
<colgroup>
<col width="15%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="2%">
<col width="2%">
<col width="2%">
<col width="2%">
<col width="3%">
<col width="3%">
</colgroup>
<tbody>
<tr class="odd">
<td align="left">Cluster name</td>
<td align="left">C1</td>
<td align="left">C10</td>
<td align="left">C11</td>
<td align="left">C12</td>
<td align="left">C13</td>
<td align="left">C14</td>
<td align="left">C15</td>
<td align="left">C16</td>
<td align="left">C17</td>
<td align="left">C18</td>
<td align="left">C19</td>
<td align="left">C2</td>
<td align="left">C20</td>
<td align="left">C21</td>
<td align="left">C22</td>
<td align="left">C3</td>
<td align="left">C4</td>
<td align="left">C5</td>
<td align="left">C6</td>
<td align="left">C7</td>
<td align="left">C8</td>
<td align="left">C9</td>
</tr>
<tr class="even">
<td align="left">Number of genes</td>
<td align="left">418</td>
<td align="left">72</td>
<td align="left">92</td>
<td align="left">103</td>
<td align="left">53</td>
<td align="left">59</td>
<td align="left">58</td>
<td align="left">104</td>
<td align="left">99</td>
<td align="left">86</td>
<td align="left">129</td>
<td align="left">181</td>
<td align="left">67</td>
<td align="left">67</td>
<td align="left">183</td>
<td align="left">121</td>
<td align="left">57</td>
<td align="left">63</td>
<td align="left">74</td>
<td align="left">51</td>
<td align="left">567</td>
<td align="left">147</td>
</tr>
</tbody>
</table>
<p>All cluster results are found in the ‘PART_results’ folder, within
this folder you will find the following:</p>
<p><strong>PART_heat.svg</strong>: A svg heatmap of the PART result –
detailed below. <strong>PART_heat_with_names.svg</strong>: The same plot
as above but with sample and gene names included. These may not be
visible depending on the number of genes included as well as the length
of the sample names. <strong>PART_heat_cmap.csv</strong>: A csv file
representing a ‘cluster_map’ which provides the cluster in which each
gene was sorted in as well as the color (in HEX code) given to that
cluster. <strong>PART_heat_data.csv</strong>: The z-score values used in
the ‘PART_heat’ heatmaps. <strong>Ctraj.svg</strong>: At least one, and
possibly several of these files will be in the folder. This/these SVG
plots show the trajectory of each cluster individually while being split
by group. Meaning the trajectory of each cluster is shown twice, once
per group. Information concerning the cluster and group is shown in the
title of each subplot. One example of cluster trajectory is shown
below.</p>
<p><strong>PART heatmap and cluster trajectory</strong> In this heatmap
we illustrate all the genes which were submitted for PART clustering.
Each gene is illustrated using a Z-score, which measures the number of
standard deviations that a gene has from it’s mean. In essence it
measures the variability. Clusters are shown as the colored bar on the
left hand side of the plot while groups and timepoints are shown as the
colored bars on the top of the heatmap. PART heatmaps were created with
ComplexHeatmaps<span class="citation">(Gu, Eils, and Schlesner
2016)</span>. The cluster trajectory of the most variable cluster (C21)
is illustrated, with the trajectory of the experiment group at the top
and the control group at the bottom. The x axis represents timepoints
and the y axis represents the scaled mean expression of the genes. The
expression of each gene is obtained by calculating the mean between all
replicates for each time point in each group. The values are then scaled
in order to illustrate their trajectory. The overall mean of the cluster
is shown as a thicker gray line.</p>
<p><img src="TS_analysis_murine_files/figure-html/PART_results,figures-side-1.png" width="49%" height="49%"><img src="TS_analysis_murine_files/figure-html/PART_results,figures-side-2.png" width="49%" height="49%"></p>
</div>
<div class="section level3">
<h3 id="gprofiler-results-standard">Gprofiler results – standard<a class="anchor" aria-label="anchor" href="#gprofiler-results-standard"></a>
</h3>
<p>The below chunk creates the gprofiler result. Some parameters
relating to the species and ontology need to be defined the
Multi-Dimensional Scaling (MDS) plots that will be generated by the last
function of this code chunk. Both versions of the MDS plots seen below
were inspired by <span class="citation">(Brionne, Juanchich, and
Hennequet-Antier 2019)</span>.</p>
<p>The ontology needs to be defined for the MDS plots as well. Note that
MDS plots utilise the semantic similarity metric which currently only
supports GO ontology (MF – MolecularFunctions, CC – Cellular Components,
and BP – Biological processes). These have been defined at the start.
The first function plots the standard gprofiler plots (gost plots) to
the directory, and it retrieves the information of the requested
ontology into the GO_clusters dataframe. This dataframe is then used
with the semantic distance to create two versions of the MDS plot.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Create standard gprofiler results</span></span>
<span><span class="va">gpro_res</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/gprofiler_cluster_analysis.html">gprofiler_cluster_analysis</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">my_ont_gpro</span>,save_path <span class="op">=</span> <span class="cn">NULL</span>,return_specific_cluster<span class="op">=</span><span class="va">target_clust</span><span class="op">)</span></span>
<span><span class="va">GO_clusters</span><span class="op">&lt;-</span><span class="va">gpro_res</span><span class="op">[[</span><span class="st">'GO_df'</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="va">sem_dta</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/methods/slot.html" class="external-link">slot</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="st">'sem_list'</span><span class="op">)</span></span>
<span><span class="va">found_clusters</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_clusters_from_termdist.html">find_clusters_from_termdist</a></span><span class="op">(</span><span class="va">GO_clusters</span>,<span class="va">sem_dta</span><span class="op">)</span></span>
<span></span>
<span><span class="co">#Plot and save MDS and clustered MDS</span></span>
<span><span class="va">MDS_plots</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/wrapper_MDS_and_MDS_clusters.html">wrapper_MDS_and_MDS_clusters</a></span><span class="op">(</span><span class="va">GO_clusters</span>,<span class="va">sem_dta</span>,<span class="va">my_ont_sem_sim</span>,target_dir<span class="op">=</span><span class="cn">NULL</span>,return_plot<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<p>Each cluster has it’s own individual gprofiler plot, these are found
in ‘TS_results/gprofiler_results/figures’. The gost plot for the most
variable cluster (C21) is shown below. These interactive plots show the
results of various databases when queried with a list of genes. More
information concerning these plots and their various datasources can be
found at the following link: <a href="https://biit.cs.ut.ee/gprofiler/gost" class="external-link uri">https://biit.cs.ut.ee/gprofiler/gost</a></p>
<p></p>
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-0cead9175b555b0b61d5" style="width:100%;height:400px;"></div> <script type="application/json" data-for="htmlwidget-0cead9175b555b0b61d5">{"x":{"data":[{"x":[153.031199317054,153.033640104777],"y":[1.30119673273353,1.30119673273353],"text":["CORUM:1997 (1) <br> Adiponectin homotrimer complex <br> 4.998e-02","CORUM:1998 (1) <br> Adiponectin homohexamer complex <br> 4.998e-02"],"key":["CORUM:1997","CORUM:1998"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(102,170,0,1)","opacity":0.8,"size":3.77952755905512,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(102,170,0,1)"}},"hoveron":"points","set":"SharedDataee453eaf","name":"CORUM","legendgroup":"CORUM","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[55.443014188627,55.9694651675505,101.173568436509,54.9994304934229,71.9457900140497],"y":[2.28023458460011,1.69018903547435,1.68024976670887,1.5037147311067,1.48079314781984],"text":["GO:0007169 (599) <br> transmembrane receptor protein tyrosine kinase signaling pathway <br> 5.245e-03","GO:0007411 (248) <br> axon guidance <br> 2.041e-02","GO:0097485 (249) <br> neuron projection guidance <br> 2.088e-02","GO:0006954 (748) <br> inflammatory response <br> 3.135e-02","GO:0034330 (753) <br> cell junction organization <br> 3.305e-02"],"key":["GO:0007169","GO:0007411","GO:0097485","GO:0006954","GO:0034330"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.499459028659,18.3754778138446,18.3808035001434,19.7701468168411,19.7781942865456],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedDataee453eaf","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[36.5720133425314,42.2665710966514,36.7158399296431],"y":[4.27745884035528,1.58644114212959,1.37961714025595],"text":["GO:0009986 (1124) <br> cell surface <br> 5.279e-05","GO:0071944 (6980) <br> cell periphery <br> 2.592e-02","GO:0016020 (10323) <br> membrane <br> 4.172e-02"],"key":["GO:0009986","GO:0071944","GO:0016020"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.2548455902694,22.2727612612714,22.6771653543307],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDataee453eaf","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[5.73651833984369,5.12717092630101,12.995064729964],"y":[2.70180484862529,2.43821885573395,1.99773878865289],"text":["GO:0005003 (16) <br> ephrin receptor activity <br> 1.987e-03","GO:0004714 (62) <br> transmembrane receptor protein tyrosine kinase activity <br> 3.646e-03","GO:0019199 (80) <br> transmembrane receptor protein kinase activity <br> 1.005e-02"],"key":["GO:0005003","GO:0004714","GO:0019199"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[14.1300963866771,16.4078539055121,16.7919752861138],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDataee453eaf","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,32.2138821531],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[34.6510751960217,45.5380165187528],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[47.9752095616745,122.219421228198],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[124.656614271119,126.004382023855],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[128.441575066777,132.611612363216],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[135.048805406137,145.282578993365],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(85,116,166,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[147.719772036287,150.061914550535],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(34,170,153,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[152.499107593457,154.1539616696],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(102,170,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[156.591154712522,197.082679927623],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(153,0,153,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[199.519872970544,200.002437193043],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(0,153,198,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":61.6829876535158,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG","REAC","TF","MIRNA","CORUM","HP","WP"],"tickvals":[17.10694107655,40.0945458573873,85.097315394936,125.330498147487,130.526593714996,140.165692199751,148.890843293411,153.326534631528,176.836917320072,199.761155081794],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG","REAC","TF","MIRNA","CORUM","HP","WP"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-37.3931091739311},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"1e8d4640fbe31a":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"1e8d4649be5bc9":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d4659d671e3":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d46585e59aa":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d4618c67b1f":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d4612c5e1a":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d4656e1fb37":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d4631f01987":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d461570a09f":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d46321fc08e":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d466d2966c2":{"x":{},"xend":{},"y":{},"yend":{}},"1e8d46117aecd1":{"x":{},"y":{}},"1e8d465ae1b08":{"yintercept":{}}},"cur_data":"1e8d4640fbe31a","visdat":{"1e8d4640fbe31a":["function (y) ","x"],"1e8d4649be5bc9":["function (y) ","x"],"1e8d4659d671e3":["function (y) ","x"],"1e8d46585e59aa":["function (y) ","x"],"1e8d4618c67b1f":["function (y) ","x"],"1e8d4612c5e1a":["function (y) ","x"],"1e8d4656e1fb37":["function (y) ","x"],"1e8d4631f01987":["function (y) ","x"],"1e8d461570a09f":["function (y) ","x"],"1e8d46321fc08e":["function (y) ","x"],"1e8d466d2966c2":["function (y) ","x"],"1e8d46117aecd1":["function (y) ","x"],"1e8d465ae1b08":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDataee453eaf"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>In addition to the standard gprofiler plot, the TimeSeries pipeline
creates Multi-Dimensional Scaling (MDS) plots. These plots illustrate a
custom selected ontology defined by the ‘my_ont_gpro’ parameter. The
first MDS plots shows all the found terms of the specified ontology and
distinguishes them based on the cluster(s) in which they are found. In a
MDS plot, the distance between each dot represents the semantic
similarity between terms<span class="citation">(Yu et al. 2010; Wang et
al. 2007)</span>. This can be loosely defined as the ‘closeness’ between
terms. Semantic similarity takes into account many factors, such as the
content of the terms (genes which belong to it), it’s associations to
other terms, the similarity or lack there-of in function etc… In an MDS
plot, the dimensions will be adjusted to fill the space with all the
terms, therefor there is no set metric to measure the similarity. The
purpose of the MDS plot is moreso to help visualize different groupings
(or clusters) of relevant terms. Hovering the cursor over the dots will
provide specific information for that dot (term)</p>
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-602b1935cf0ac3846819" style="width:100%;height:432.632880098888px;"></div>
<script type="application/json" data-for="htmlwidget-602b1935cf0ac3846819">{"x":{"visdat":{"1e8d46766f87ee":["function () ","plotlyVisDat"],"1e8d46ebcdf56":["function () ","data"]},"cur_data":"1e8d46ebcdf56","attrs":{"1e8d46ebcdf56":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","color":{},"text":{},"showlegend":true,"colors":["#8DD3C7","#C3B6AB","#CEB9B1","#BAB3A3","#DFC7D1","#E7C8CC","#B9B3AD","#D1B5AE","#DEC4CF","#D6BCA6","#D2B7AD","#D3BDA9","#D3B9AD","#B1A9AF","#E4C7CA","#DCC6D1","#CABDB4","#DDC7D1","#C2B29A","#C5B39A","#E7C7C7","#CEC2B3","#E5C7CC","#CEB8AF","#D3BDAB","#9BB9BC","#B6B5A1","#BFC5BB","#B4CAC3","#B0C3A7","#C9CAB1","#B8D2C1","#B3E1C0","#D9C3A9","#E9E8C0","#D2A6B7","#E69394","#C799A3","#CDA594","#D8BBA7","#EAC3C6","#D4B5AF","#C3AA9D","#DCA38D","#E4B2A7","#F0BFC6","#F1B0BC","#D29092","#A9A0B2","#D0BDAB","#C5B9B6","#A9B2AD","#D3B387","#DBC2A1","#E3D2BB","#FCCDE5"],"marker":{"size":20,"opacity":0.4,"showlegend":true},"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":true},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"x":[-0.147975897763311,-0.403246176028609,0.0102991215791855,0.00331539568920419,-0.145000050519207,-0.461466758350271,0.0481613078497505,-0.154973363690624,-0.0697878070748991,-0.47264571061034,-0.440288616824673,0.160999295715956,0.0947943575780536,-0.223932461300861,0.132174320664696,-0.0977581179213287,0.104303599001697,-0.10891796449669,0.152185851295341,-0.0167275655309348,0.140774209488657,-0.0550169445824064,0.0569059182501987,0.15000975278608,-0.170593643719788,-0.142584924508026,0.0430702848259639,0.141560500713256,-0.105641853195725,0.0948013399398753,-0.360166182842111,-0.0745434744148785,0.0331553970522842,-0.0416550130299099,-0.0864663673948472,0.0681411356176663,0.218184561001579,0.0577304351629805,-0.397850160903528,0.119456000391929,-0.0582934848927647,-0.136033862058813,-0.119979035509505,0.11817800814435,0.0275684870332282,-0.134873263570386,-0.0577063150189904,-0.154714296357481,-0.0775609677833365],"y":[-0.140164566961617,0.0828320746940951,-0.00959379111344723,0.00102745499245073,-0.15586776666288,0.151839398357349,0.261481791674021,0.145748536890515,-0.24899091189242,0.160921648213003,0.145390734200798,-0.229201535944169,-0.0223531428959064,-0.12096528967035,-0.0450644144140368,-0.222979121775002,-0.0188421678803452,-0.134872623054523,-0.0467698774336795,0.319435047483413,0.085763327127001,-0.189916620524129,-0.094496219092766,-0.0515115896518216,-0.0579141047337247,-0.155090868160366,-0.00500110032373144,-0.151371083079559,-0.129242812465893,-0.0687893614478415,0.0782098230134692,-0.243364893701998,0.249367525556758,0.265248967266631,0.0694473152765672,-0.0363013449521895,-0.0793295517516696,-0.0211279196369945,0.137470130458595,0.317585717092738,-0.162998739989116,0.132478875306056,0.0171076538212835,-0.0416020260324088,0.219395994371777,0.133809058817031,-0.323088543326697,0.0522476338200308,0.155777914571599],"type":"scatter","mode":"markers","text":["modules/clusters: C1 <br>GO.ID: GO:0042330 <br>GO.name: taxis","modules/clusters: C1 <br>GO.ID: GO:0009888 <br>GO.name: tissue development","modules/clusters: C1 <br>GO.ID: GO:0006793 <br>GO.name: phosphorus metabolic process","modules/clusters: C1 <br>GO.ID: GO:0006796 <br>GO.name: phosphate-containing compound metabolic process","modules/clusters: C1 <br>GO.ID: GO:0006935 <br>GO.name: chemotaxis","modules/clusters: C1 <br>GO.ID: GO:0001568 <br>GO.name: blood vessel development","modules/clusters: C1 <br>GO.ID: GO:0002718 <br>GO.name: regulation of cytokine production involved in immune response","modules/clusters: C1 <br>GO.ID: GO:0002367 <br>GO.name: cytokine production involved in immune response","modules/clusters: C1 <br>GO.ID: GO:0060326 <br>GO.name: cell chemotaxis","modules/clusters: C1 <br>GO.ID: GO:0001944 <br>GO.name: vasculature development","modules/clusters: C1 <br>GO.ID: GO:0035239 <br>GO.name: tube morphogenesis","modules/clusters: C1 <br>GO.ID: GO:0038066 <br>GO.name: p38MAPK cascade","modules/clusters: C1 <br>GO.ID: GO:0071677 <br>GO.name: positive regulation of mononuclear cell migration","modules/clusters: C1 <br>GO.ID: GO:0045444 <br>GO.name: fat cell differentiation","modules/clusters: C1 <br>GO.ID: GO:0030335 <br>GO.name: positive regulation of cell migration","modules/clusters: C1 <br>GO.ID: GO:0030595 <br>GO.name: leukocyte chemotaxis","modules/clusters: C1 <br>GO.ID: GO:0002687 <br>GO.name: positive regulation of leukocyte migration","modules/clusters: C1 <br>GO.ID: GO:0050900 <br>GO.name: leukocyte migration","modules/clusters: C1 <br>GO.ID: GO:2000147 <br>GO.name: positive regulation of cell motility","modules/clusters: C1 <br>GO.ID: GO:0019538 <br>GO.name: protein metabolic process","modules/clusters: C1 <br>GO.ID: GO:0040017 <br>GO.name: positive regulation of locomotion","modules/clusters: C1 <br>GO.ID: GO:0098609 <br>GO.name: cell-cell adhesion","modules/clusters: C1 <br>GO.ID: GO:0002690 <br>GO.name: positive regulation of leukocyte chemotaxis","modules/clusters: C1 <br>GO.ID: GO:1903039 <br>GO.name: positive regulation of leukocyte cell-cell adhesion","modules/clusters: C1 <br>GO.ID: GO:0051179 <br>GO.name: localization","modules/clusters: C1 <br>GO.ID: GO:0071774 <br>GO.name: response to fibroblast growth factor","modules/clusters: C1 <br>GO.ID: GO:0002822 <br>GO.name: regulation of adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains","modules/clusters: C1 <br>GO.ID: GO:1900745 <br>GO.name: positive regulation of p38MAPK cascade","modules/clusters: C1 <br>GO.ID: GO:0071674 <br>GO.name: mononuclear cell migration","modules/clusters: C1 <br>GO.ID: GO:0046634 <br>GO.name: regulation of alpha-beta T cell activation","modules/clusters: C1 <br>GO.ID: GO:0060429 <br>GO.name: epithelium development","modules/clusters: C1 <br>GO.ID: GO:0071219 <br>GO.name: cellular response to molecule of bacterial origin","modules/clusters: C1 <br>GO.ID: GO:0002720 <br>GO.name: positive regulation of cytokine production involved in immune response","modules/clusters: C1 <br>GO.ID: GO:1901564 <br>GO.name: organonitrogen compound metabolic process","modules/clusters: C1 <br>GO.ID: GO:0045604 <br>GO.name: regulation of epidermal cell differentiation","modules/clusters: C1 <br>GO.ID: GO:0050870 <br>GO.name: positive regulation of T cell activation","modules/clusters: C1 <br>GO.ID: GO:0045785 <br>GO.name: positive regulation of cell adhesion","modules/clusters: C1 <br>GO.ID: GO:0050921 <br>GO.name: positive regulation of chemotaxis","modules/clusters: C1 <br>GO.ID: GO:0001525 <br>GO.name: angiogenesis","modules/clusters: C1 <br>GO.ID: GO:0002700 <br>GO.name: regulation of production of molecular mediator of immune response","modules/clusters: C1 <br>GO.ID: GO:0007159 <br>GO.name: leukocyte cell-cell adhesion","modules/clusters: C1 <br>GO.ID: GO:0035743 <br>GO.name: CD4-positive, alpha-beta T cell cytokine production","modules/clusters: C1 <br>GO.ID: GO:2000319 <br>GO.name: regulation of T-helper 17 cell differentiation","modules/clusters: C1 <br>GO.ID: GO:0071675 <br>GO.name: regulation of mononuclear cell migration","modules/clusters: C1 <br>GO.ID: GO:0002724 <br>GO.name: regulation of T cell cytokine production","modules/clusters: C1 <br>GO.ID: GO:0002369 <br>GO.name: T cell cytokine production","modules/clusters: C1 <br>GO.ID: GO:0071216 <br>GO.name: cellular response to biotic stimulus","modules/clusters: C1 <br>GO.ID: GO:0045619 <br>GO.name: regulation of lymphocyte differentiation","modules/clusters: C1 <br>GO.ID: GO:0045682 <br>GO.name: regulation of epidermis development"],"showlegend":true,"marker":{"color":"rgba(141,211,199,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(141,211,199,1)"}},"name":"C1","textfont":{"color":"rgba(141,211,199,1)"},"error_y":{"color":"rgba(141,211,199,1)"},"error_x":{"color":"rgba(141,211,199,1)"},"line":{"color":"rgba(141,211,199,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.193907003172015],"y":[0.232357701894572],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C10/C17 <br>GO.ID: GO:2000026 <br>GO.name: regulation of multicellular organismal development","showlegend":true,"marker":{"color":"rgba(195,182,171,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(195,182,171,1)"}},"name":"C1/C10/C17","textfont":{"color":"rgba(195,182,171,1)"},"error_y":{"color":"rgba(195,182,171,1)"},"error_x":{"color":"rgba(195,182,171,1)"},"line":{"color":"rgba(195,182,171,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.289039379495034,0.201648546982475],"y":[-0.109226547695022,0.0895280797935379],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C10/C17/C19/C2/C8 <br>GO.ID: GO:0048523 <br>GO.name: negative regulation of cellular process","modules/clusters: C1/C10/C17/C19/C2/C8 <br>GO.ID: GO:0048519 <br>GO.name: negative regulation of biological process"],"showlegend":true,"marker":{"color":"rgba(206,185,177,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(206,185,177,1)"}},"name":"C1/C10/C17/C19/C2/C8","textfont":{"color":"rgba(206,185,177,1)"},"error_y":{"color":"rgba(206,185,177,1)"},"error_x":{"color":"rgba(206,185,177,1)"},"line":{"color":"rgba(206,185,177,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.164337950910988],"y":[-0.179871035313097],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C10/C17/C20/C22 <br>GO.ID: GO:0009605 <br>GO.name: response to external stimulus","showlegend":true,"marker":{"color":"rgba(186,179,163,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(186,179,163,1)"}},"name":"C1/C10/C17/C20/C22","textfont":{"color":"rgba(186,179,163,1)"},"error_y":{"color":"rgba(186,179,163,1)"},"error_x":{"color":"rgba(186,179,163,1)"},"line":{"color":"rgba(186,179,163,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0761243248845869],"y":[0.170350531265609],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C10/C17/C20/C8/C9 <br>GO.ID: GO:0051239 <br>GO.name: regulation of multicellular organismal process","showlegend":true,"marker":{"color":"rgba(223,199,209,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(223,199,209,1)"}},"name":"C1/C10/C17/C20/C8/C9","textfont":{"color":"rgba(223,199,209,1)"},"error_y":{"color":"rgba(223,199,209,1)"},"error_x":{"color":"rgba(223,199,209,1)"},"line":{"color":"rgba(223,199,209,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.177746579184316],"y":[-0.212961693708],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C10/C17/C8/C9 <br>GO.ID: GO:0050896 <br>GO.name: response to stimulus","showlegend":true,"marker":{"color":"rgba(231,200,204,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(231,200,204,1)"}},"name":"C1/C10/C17/C8/C9","textfont":{"color":"rgba(231,200,204,1)"},"error_y":{"color":"rgba(231,200,204,1)"},"error_x":{"color":"rgba(231,200,204,1)"},"line":{"color":"rgba(231,200,204,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0856412383356654,-0.0688011362524261,0.179415133810676,0.205888336500334,0.239368524684535,0.0651760830177665,0.108194859034877,0.121265062253813,0.170100489392213,0.158491498838864,0.224978610652909,0.0875144643290937],"y":[-0.046441676752128,-0.155162992595639,-0.0950528158248542,-0.100042385328297,-0.103769019134033,-0.0286947291580756,0.0921278376320693,-0.0373760577251693,-0.0763187299111254,-0.0659918385065597,-0.0780937936369437,-0.032489371867354],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C17 <br>GO.ID: GO:0045595 <br>GO.name: regulation of cell differentiation","modules/clusters: C1/C17 <br>GO.ID: GO:0016477 <br>GO.name: cell migration","modules/clusters: C1/C17 <br>GO.ID: GO:0043065 <br>GO.name: positive regulation of apoptotic process","modules/clusters: C1/C17 <br>GO.ID: GO:0043068 <br>GO.name: positive regulation of programmed cell death","modules/clusters: C1/C17 <br>GO.ID: GO:0010942 <br>GO.name: positive regulation of cell death","modules/clusters: C1/C17 <br>GO.ID: GO:0050727 <br>GO.name: regulation of inflammatory response","modules/clusters: C1/C17 <br>GO.ID: GO:0002684 <br>GO.name: positive regulation of immune system process","modules/clusters: C1/C17 <br>GO.ID: GO:0032101 <br>GO.name: regulation of response to external stimulus","modules/clusters: C1/C17 <br>GO.ID: GO:0002694 <br>GO.name: regulation of leukocyte activation","modules/clusters: C1/C17 <br>GO.ID: GO:0030334 <br>GO.name: regulation of cell migration","modules/clusters: C1/C17 <br>GO.ID: GO:0008284 <br>GO.name: positive regulation of cell population proliferation","modules/clusters: C1/C17 <br>GO.ID: GO:0031347 <br>GO.name: regulation of defense response"],"showlegend":true,"marker":{"color":"rgba(185,179,173,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(185,179,173,1)"}},"name":"C1/C17","textfont":{"color":"rgba(185,179,173,1)"},"error_y":{"color":"rgba(185,179,173,1)"},"error_x":{"color":"rgba(185,179,173,1)"},"line":{"color":"rgba(185,179,173,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.227194027978394],"y":[0.319765709878702],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C18/C19/C2/C8 <br>GO.ID: GO:0051171 <br>GO.name: regulation of nitrogen compound metabolic process","showlegend":true,"marker":{"color":"rgba(209,181,174,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(209,181,174,1)"}},"name":"C1/C17/C18/C19/C2/C8","textfont":{"color":"rgba(209,181,174,1)"},"error_y":{"color":"rgba(209,181,174,1)"},"error_x":{"color":"rgba(209,181,174,1)"},"line":{"color":"rgba(209,181,174,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.369534730066585],"y":[0.0215580226993402],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C18/C2/C20/C8/C9 <br>GO.ID: GO:0032502 <br>GO.name: developmental process","showlegend":true,"marker":{"color":"rgba(222,196,207,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(222,196,207,1)"}},"name":"C1/C17/C18/C2/C20/C8/C9","textfont":{"color":"rgba(222,196,207,1)"},"error_y":{"color":"rgba(222,196,207,1)"},"error_x":{"color":"rgba(222,196,207,1)"},"line":{"color":"rgba(222,196,207,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.226519488168505,0.309257784077134],"y":[0.0898134575504476,-0.123844721209991],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C17/C18/C2/C22/C8 <br>GO.ID: GO:0050789 <br>GO.name: regulation of biological process","modules/clusters: C1/C17/C18/C2/C22/C8 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process"],"showlegend":true,"marker":{"color":"rgba(214,188,166,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(214,188,166,1)"}},"name":"C1/C17/C18/C2/C22/C8","textfont":{"color":"rgba(214,188,166,1)"},"error_y":{"color":"rgba(214,188,166,1)"},"error_x":{"color":"rgba(214,188,166,1)"},"line":{"color":"rgba(214,188,166,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.290634907131913],"y":[-0.152344583216438],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C18/C2/C8 <br>GO.ID: GO:0010646 <br>GO.name: regulation of cell communication","showlegend":true,"marker":{"color":"rgba(210,183,173,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(210,183,173,1)"}},"name":"C1/C17/C18/C2/C8","textfont":{"color":"rgba(210,183,173,1)"},"error_y":{"color":"rgba(210,183,173,1)"},"error_x":{"color":"rgba(210,183,173,1)"},"line":{"color":"rgba(210,183,173,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.188681213528829],"y":[0.0973008869814167],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C19/C2/C22/C8 <br>GO.ID: GO:0048518 <br>GO.name: positive regulation of biological process","showlegend":true,"marker":{"color":"rgba(211,189,169,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(211,189,169,1)"}},"name":"C1/C17/C19/C2/C22/C8","textfont":{"color":"rgba(211,189,169,1)"},"error_y":{"color":"rgba(211,189,169,1)"},"error_x":{"color":"rgba(211,189,169,1)"},"line":{"color":"rgba(211,189,169,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.222508071595432],"y":[0.320170409022183],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C19/C8 <br>GO.ID: GO:0080090 <br>GO.name: regulation of primary metabolic process","showlegend":true,"marker":{"color":"rgba(211,185,173,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(211,185,173,1)"}},"name":"C1/C17/C19/C8","textfont":{"color":"rgba(211,185,173,1)"},"error_y":{"color":"rgba(211,185,173,1)"},"error_x":{"color":"rgba(211,185,173,1)"},"line":{"color":"rgba(211,185,173,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0512999286929691,0.0644945974980172],"y":[-0.225091382852517,-0.0250425882833157],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C17/C2 <br>GO.ID: GO:0007155 <br>GO.name: cell adhesion","modules/clusters: C1/C17/C2 <br>GO.ID: GO:0045597 <br>GO.name: positive regulation of cell differentiation"],"showlegend":true,"marker":{"color":"rgba(177,169,175,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(177,169,175,1)"}},"name":"C1/C17/C2","textfont":{"color":"rgba(177,169,175,1)"},"error_y":{"color":"rgba(177,169,175,1)"},"error_x":{"color":"rgba(177,169,175,1)"},"line":{"color":"rgba(177,169,175,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.517509322708343],"y":[0.167004100452871],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C2/C20/C22/C8/C9 <br>GO.ID: GO:0048731 <br>GO.name: system development","showlegend":true,"marker":{"color":"rgba(228,199,202,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(228,199,202,1)"}},"name":"C1/C17/C2/C20/C22/C8/C9","textfont":{"color":"rgba(228,199,202,1)"},"error_y":{"color":"rgba(228,199,202,1)"},"error_x":{"color":"rgba(228,199,202,1)"},"line":{"color":"rgba(228,199,202,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.454473454631783,-0.499358638805019],"y":[0.0833610699152679,0.15432735188966],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C17/C2/C20/C8/C9 <br>GO.ID: GO:0048856 <br>GO.name: anatomical structure development","modules/clusters: C1/C17/C2/C20/C8/C9 <br>GO.ID: GO:0007275 <br>GO.name: multicellular organism development"],"showlegend":true,"marker":{"color":"rgba(220,198,209,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(220,198,209,1)"}},"name":"C1/C17/C2/C20/C8/C9","textfont":{"color":"rgba(220,198,209,1)"},"error_y":{"color":"rgba(220,198,209,1)"},"error_x":{"color":"rgba(220,198,209,1)"},"line":{"color":"rgba(220,198,209,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.425669898938423,0.231463612680771,0.274663250481081,-0.066426612915372,0.214460658149964,0.227670225723627,0.269961338747659,0.190493863690084,-0.0744271714917334,-0.0197354043935912,0.149623474194459],"y":[0.0871557796268687,-0.264273318178501,-0.101702123358194,-0.358060630259218,0.0401467773305741,-0.20740791591715,-0.132820989267575,0.0399557605666504,-0.317321476453871,-0.304110233557863,-0.0345112820758649],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0009653 <br>GO.name: anatomical structure morphogenesis","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0035556 <br>GO.name: intracellular signal transduction","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0048522 <br>GO.name: positive regulation of cellular process","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0070887 <br>GO.name: cellular response to chemical stimulus","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0023051 <br>GO.name: regulation of signaling","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0009968 <br>GO.name: negative regulation of signal transduction","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0010648 <br>GO.name: negative regulation of cell communication","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0023057 <br>GO.name: negative regulation of signaling","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0071310 <br>GO.name: cellular response to organic substance","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0007154 <br>GO.name: cell communication","modules/clusters: C1/C17/C2/C8 <br>GO.ID: GO:0048585 <br>GO.name: negative regulation of response to stimulus"],"showlegend":true,"marker":{"color":"rgba(202,189,180,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(202,189,180,1)"}},"name":"C1/C17/C2/C8","textfont":{"color":"rgba(202,189,180,1)"},"error_y":{"color":"rgba(202,189,180,1)"},"error_x":{"color":"rgba(202,189,180,1)"},"line":{"color":"rgba(202,189,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.24102248405278,-0.239007427303427],"y":[-0.140485815143943,-0.166235002777048],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C17/C20/C8/C9 <br>GO.ID: GO:0030154 <br>GO.name: cell differentiation","modules/clusters: C1/C17/C20/C8/C9 <br>GO.ID: GO:0048869 <br>GO.name: cellular developmental process"],"showlegend":true,"marker":{"color":"rgba(221,199,209,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(221,199,209,1)"}},"name":"C1/C17/C20/C8/C9","textfont":{"color":"rgba(221,199,209,1)"},"error_y":{"color":"rgba(221,199,209,1)"},"error_x":{"color":"rgba(221,199,209,1)"},"line":{"color":"rgba(221,199,209,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.143011040607777],"y":[-0.128531109270682],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C21/C22 <br>GO.ID: GO:0006954 <br>GO.name: inflammatory response","showlegend":true,"marker":{"color":"rgba(194,178,154,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(194,178,154,1)"}},"name":"C1/C17/C21/C22","textfont":{"color":"rgba(194,178,154,1)"},"error_y":{"color":"rgba(194,178,154,1)"},"error_x":{"color":"rgba(194,178,154,1)"},"line":{"color":"rgba(194,178,154,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.139410673086924],"y":[0.0839159482649435],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C22 <br>GO.ID: GO:0002682 <br>GO.name: regulation of immune system process","showlegend":true,"marker":{"color":"rgba(197,179,154,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(197,179,154,1)"}},"name":"C1/C17/C22","textfont":{"color":"rgba(197,179,154,1)"},"error_y":{"color":"rgba(197,179,154,1)"},"error_x":{"color":"rgba(197,179,154,1)"},"line":{"color":"rgba(197,179,154,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.232956821005911],"y":[-0.266450126491254],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C17/C22/C8/C9 <br>GO.ID: GO:0007166 <br>GO.name: cell surface receptor signaling pathway","showlegend":true,"marker":{"color":"rgba(231,199,199,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(231,199,199,1)"}},"name":"C1/C17/C22/C8/C9","textfont":{"color":"rgba(231,199,199,1)"},"error_y":{"color":"rgba(231,199,199,1)"},"error_x":{"color":"rgba(231,199,199,1)"},"line":{"color":"rgba(231,199,199,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.512560283445747,-0.393636022664263,0.0508939254485448,-0.168721769053053,-0.0656684486080477,-0.0453909854524271,0.248106427435111,0.273838350658041,0.204769518309135,0.236118918785152,0.0618882680552219,-0.0378020269515732,-0.00179699081020707,-0.0363905131458688,0.252570346151596,0.169523823370038,0.215830391282814],"y":[0.163902606636906,0.0824369987802068,0.174299995933067,-0.0558352163741443,-0.168546444457685,-0.271873329675027,-0.236492048288882,-0.13204712948394,-0.12511217546687,-0.129894738859294,0.164193496165885,-0.243313126377684,0.145152649971306,-0.219280174007482,-0.27086116075304,-0.0421441534573215,-0.201289262816991],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C17/C8 <br>GO.ID: GO:0048513 <br>GO.name: animal organ development","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0048646 <br>GO.name: anatomical structure formation involved in morphogenesis","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0051240 <br>GO.name: positive regulation of multicellular organismal process","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0040011 <br>GO.name: locomotion","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0048870 <br>GO.name: cell motility","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0008219 <br>GO.name: cell death","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0009966 <br>GO.name: regulation of signal transduction","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0010941 <br>GO.name: regulation of cell death","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0042981 <br>GO.name: regulation of apoptotic process","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0043067 <br>GO.name: regulation of programmed cell death","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0051241 <br>GO.name: negative regulation of multicellular organismal process","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0012501 <br>GO.name: programmed cell death","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0051094 <br>GO.name: positive regulation of developmental process","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0006915 <br>GO.name: apoptotic process","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0007165 <br>GO.name: signal transduction","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0048583 <br>GO.name: regulation of response to stimulus","modules/clusters: C1/C17/C8 <br>GO.ID: GO:0009967 <br>GO.name: positive regulation of signal transduction"],"showlegend":true,"marker":{"color":"rgba(206,194,179,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(206,194,179,1)"}},"name":"C1/C17/C8","textfont":{"color":"rgba(206,194,179,1)"},"error_y":{"color":"rgba(206,194,179,1)"},"error_x":{"color":"rgba(206,194,179,1)"},"line":{"color":"rgba(206,194,179,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0188886144711318,-0.302208353970277],"y":[0.142017799462307,0.052022616271518],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C17/C8/C9 <br>GO.ID: GO:0050793 <br>GO.name: regulation of developmental process","modules/clusters: C1/C17/C8/C9 <br>GO.ID: GO:0032501 <br>GO.name: multicellular organismal process"],"showlegend":true,"marker":{"color":"rgba(229,199,204,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(229,199,204,1)"}},"name":"C1/C17/C8/C9","textfont":{"color":"rgba(229,199,204,1)"},"error_y":{"color":"rgba(229,199,204,1)"},"error_x":{"color":"rgba(229,199,204,1)"},"line":{"color":"rgba(229,199,204,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.257694116669967],"y":[0.297740834552753],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C18/C19/C2/C8 <br>GO.ID: GO:0019222 <br>GO.name: regulation of metabolic process","showlegend":true,"marker":{"color":"rgba(206,184,175,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(206,184,175,1)"}},"name":"C1/C18/C19/C2/C8","textfont":{"color":"rgba(206,184,175,1)"},"error_y":{"color":"rgba(206,184,175,1)"},"error_x":{"color":"rgba(206,184,175,1)"},"line":{"color":"rgba(206,184,175,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.327023090143929],"y":[0.117188072686083],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C18/C8 <br>GO.ID: GO:0031323 <br>GO.name: regulation of cellular metabolic process","showlegend":true,"marker":{"color":"rgba(211,189,171,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(211,189,171,1)"}},"name":"C1/C18/C8","textfont":{"color":"rgba(211,189,171,1)"},"error_y":{"color":"rgba(211,189,171,1)"},"error_x":{"color":"rgba(211,189,171,1)"},"line":{"color":"rgba(211,189,171,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.212308197240253,0.0174275824909824,-0.0236618436164026,-0.416804619442111,-0.0495377825903871,-0.0704580372861382,-0.00374365506087932,-0.477677953288525,-0.160045998321767,-0.210984619459719,-0.133305852875899],"y":[-0.255098495886303,0.162987790627217,0.284908196172111,0.149007789670583,0.265644228189459,-0.301648461256072,0.00571784448173394,0.154490704669181,-0.173791993054763,0.200384385431301,0.0804983557318934],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2 <br>GO.ID: GO:0000165 <br>GO.name: MAPK cascade","modules/clusters: C1/C2 <br>GO.ID: GO:0006468 <br>GO.name: protein phosphorylation","modules/clusters: C1/C2 <br>GO.ID: GO:0036211 <br>GO.name: protein modification process","modules/clusters: C1/C2 <br>GO.ID: GO:0048514 <br>GO.name: blood vessel morphogenesis","modules/clusters: C1/C2 <br>GO.ID: GO:0043412 <br>GO.name: macromolecule modification","modules/clusters: C1/C2 <br>GO.ID: GO:1901701 <br>GO.name: cellular response to oxygen-containing compound","modules/clusters: C1/C2 <br>GO.ID: GO:0016310 <br>GO.name: phosphorylation","modules/clusters: C1/C2 <br>GO.ID: GO:0035295 <br>GO.name: tube development","modules/clusters: C1/C2 <br>GO.ID: GO:0009719 <br>GO.name: response to endogenous stimulus","modules/clusters: C1/C2 <br>GO.ID: GO:1903708 <br>GO.name: positive regulation of hemopoiesis","modules/clusters: C1/C2 <br>GO.ID: GO:1902107 <br>GO.name: positive regulation of leukocyte differentiation"],"showlegend":true,"marker":{"color":"rgba(155,185,188,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(155,185,188,1)"}},"name":"C1/C2","textfont":{"color":"rgba(155,185,188,1)"},"error_y":{"color":"rgba(155,185,188,1)"},"error_x":{"color":"rgba(155,185,188,1)"},"line":{"color":"rgba(155,185,188,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.492551521498049],"y":[0.162892421920994],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C22 <br>GO.ID: GO:0072359 <br>GO.name: circulatory system development","showlegend":true,"marker":{"color":"rgba(182,181,161,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(182,181,161,1)"}},"name":"C1/C2/C22","textfont":{"color":"rgba(182,181,161,1)"},"error_y":{"color":"rgba(182,181,161,1)"},"error_x":{"color":"rgba(182,181,161,1)"},"line":{"color":"rgba(182,181,161,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.228932090906244,0.19434781822842],"y":[0.402631398078854,0.397580750477398],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C8 <br>GO.ID: GO:0060255 <br>GO.name: regulation of macromolecule metabolic process","modules/clusters: C1/C2/C8 <br>GO.ID: GO:0010468 <br>GO.name: regulation of gene expression"],"showlegend":true,"marker":{"color":"rgba(191,197,187,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(191,197,187,1)"}},"name":"C1/C2/C8","textfont":{"color":"rgba(191,197,187,1)"},"error_y":{"color":"rgba(191,197,187,1)"},"error_x":{"color":"rgba(191,197,187,1)"},"line":{"color":"rgba(191,197,187,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0902487562078964],"y":[0.364859803983074],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C20/C8 <br>GO.ID: GO:0001819 <br>GO.name: positive regulation of cytokine production","showlegend":true,"marker":{"color":"rgba(180,202,195,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(180,202,195,1)"}},"name":"C1/C20/C8","textfont":{"color":"rgba(180,202,195,1)"},"error_y":{"color":"rgba(180,202,195,1)"},"error_x":{"color":"rgba(180,202,195,1)"},"line":{"color":"rgba(180,202,195,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0618460401240388],"y":[-0.0121812263486821],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C22 <br>GO.ID: GO:0002819 <br>GO.name: regulation of adaptive immune response","showlegend":true,"marker":{"color":"rgba(176,195,167,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(176,195,167,1)"}},"name":"C1/C22","textfont":{"color":"rgba(176,195,167,1)"},"error_y":{"color":"rgba(176,195,167,1)"},"error_x":{"color":"rgba(176,195,167,1)"},"line":{"color":"rgba(176,195,167,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.116236640763071,-0.153005229093377],"y":[0.388222271060573,0.292695649973173],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C22/C8 <br>GO.ID: GO:0001817 <br>GO.name: regulation of cytokine production","modules/clusters: C1/C22/C8 <br>GO.ID: GO:0001816 <br>GO.name: cytokine production"],"showlegend":true,"marker":{"color":"rgba(201,202,177,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(201,202,177,1)"}},"name":"C1/C22/C8","textfont":{"color":"rgba(201,202,177,1)"},"error_y":{"color":"rgba(201,202,177,1)"},"error_x":{"color":"rgba(201,202,177,1)"},"line":{"color":"rgba(201,202,177,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.222331755243467,0.196965927016079,0.195364403902536,-0.485292087700422,0.182274255905182,0.28260330594951,0.025797508693197,-0.0767189602942301],"y":[0.298717403034201,0.390904898207422,0.313311452468858,0.158033521506885,0.313384829810222,0.129619973461834,0.236563779373911,0.188383265103695],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C8 <br>GO.ID: GO:0009893 <br>GO.name: positive regulation of metabolic process","modules/clusters: C1/C8 <br>GO.ID: GO:0010604 <br>GO.name: positive regulation of macromolecule metabolic process","modules/clusters: C1/C8 <br>GO.ID: GO:0051173 <br>GO.name: positive regulation of nitrogen compound metabolic process","modules/clusters: C1/C8 <br>GO.ID: GO:0007399 <br>GO.name: nervous system development","modules/clusters: C1/C8 <br>GO.ID: GO:0006357 <br>GO.name: regulation of transcription by RNA polymerase II","modules/clusters: C1/C8 <br>GO.ID: GO:0031325 <br>GO.name: positive regulation of cellular metabolic process","modules/clusters: C1/C8 <br>GO.ID: GO:0006366 <br>GO.name: transcription by RNA polymerase II","modules/clusters: C1/C8 <br>GO.ID: GO:0009058 <br>GO.name: biosynthetic process"],"showlegend":true,"marker":{"color":"rgba(184,210,193,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(184,210,193,1)"}},"name":"C1/C8","textfont":{"color":"rgba(184,210,193,1)"},"error_y":{"color":"rgba(184,210,193,1)"},"error_x":{"color":"rgba(184,210,193,1)"},"line":{"color":"rgba(184,210,193,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.177885591655691,0.257724199274391],"y":[-0.0799853139361017,-0.101074734028138],"type":"scatter","mode":"markers","text":["modules/clusters: C10 <br>GO.ID: GO:0032956 <br>GO.name: regulation of actin cytoskeleton organization","modules/clusters: C10 <br>GO.ID: GO:0032970 <br>GO.name: regulation of actin filament-based process"],"showlegend":true,"marker":{"color":"rgba(179,225,192,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(179,225,192,1)"}},"name":"C10","textfont":{"color":"rgba(179,225,192,1)"},"error_y":{"color":"rgba(179,225,192,1)"},"error_x":{"color":"rgba(179,225,192,1)"},"line":{"color":"rgba(179,225,192,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.216361986728941],"y":[-0.0554227150445693],"type":"scatter","mode":"markers","text":"modules/clusters: C10/C17/C22/C8 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","showlegend":true,"marker":{"color":"rgba(217,195,169,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(217,195,169,1)"}},"name":"C10/C17/C22/C8","textfont":{"color":"rgba(217,195,169,1)"},"error_y":{"color":"rgba(217,195,169,1)"},"error_x":{"color":"rgba(217,195,169,1)"},"line":{"color":"rgba(217,195,169,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0430305232446236,-0.0623117433472944,-0.0685481304655631],"y":[-0.141702908375178,-0.116130698903987,-0.161376474424481],"type":"scatter","mode":"markers","text":["modules/clusters: C13 <br>GO.ID: GO:0051310 <br>GO.name: metaphase plate congression","modules/clusters: C13 <br>GO.ID: GO:0051303 <br>GO.name: establishment of chromosome localization","modules/clusters: C13 <br>GO.ID: GO:0050000 <br>GO.name: chromosome localization"],"showlegend":true,"marker":{"color":"rgba(233,232,192,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(233,232,192,1)"}},"name":"C13","textfont":{"color":"rgba(233,232,192,1)"},"error_y":{"color":"rgba(233,232,192,1)"},"error_x":{"color":"rgba(233,232,192,1)"},"line":{"color":"rgba(233,232,192,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0372382698263899,-0.043141374256907,-0.0316095918623847,-0.0545141989734078,-0.0344090070863713,-0.0570799950957876,-0.0410200776522549,-0.0627855086352088,-0.0558805527834035,-0.0554778742091127,-0.0610720156065459,-0.048639014313475,-0.0581122482274498,-0.0449862569440071,-0.0577562217278674,0.217826461395726,-0.0528800524319524,-0.041640303163589,-0.0698999403696543,0.00859041431009597,-0.0461475446355403,-0.0544082744623164,-0.0428145188078844,-0.0456516126639694,-0.0563780580155176,-0.0521283474599629,-0.0454197222557223,-0.0452273398107099,0.138227559647782,0.0192579160520106,-0.0553583759879059,-0.039736210138499,0.005097450816604,-0.0439499489588277,0.266955889074828,-0.0655672677539203,0.263169234808965,-0.0446421197516087,-0.0651358146035865,-0.0492511379717516,0.175474526996575,0.21610330407783,-0.0508472335401371,0.0915345880960768,-0.0713073503933138,0.106955800358396,0.1389992621239,0.184384520575386,-0.0562705649978998,0.139942499775597,0.170542316768479,0.148516565084248,-0.0317808835969421,-0.0514998468423137,0.0666730375489224,0.181848141235529,0.151252986413119,0.193376948251795,0.192383032190313,0.0622414433285306,0.16588138202935,0.11895781073246,0.107313746985197,0.139776011803176,0.14096109939999,-0.0543530274449118,0.19766055694251,0.241680074322647,0.173411007169807,0.134290889604395,0.119691141260617,-0.00273655620644054,0.00211387020352752,-0.0418915243635548,0.162880170212272,-0.0687079834138014,0.221543068757607,-0.0445920167477327,-0.0405413894917559,-0.058177346130586,0.256201287891042,0.25724771364276,0.174367945697812,0.156301359727398,-0.0875772306638185,-0.00193329291086113,0.0093833992274374,0.171870370833929,-0.00706196936603458,0.162080208584395,0.198695570184487,0.011885467728394,-0.0742511570469788,-0.00504235142519633],"y":[-0.1684415669956,-0.20631927951025,-0.145729959753688,-0.224464651661698,-0.173016503977167,-0.158195280622524,-0.206591682563773,-0.244555944259409,-0.175070561710523,0.187235348470885,0.184789302971956,0.217741585094561,-0.250593969472825,-0.253894659309216,0.194180640691503,-0.136514280992211,0.222671854644801,-0.159383002909354,0.181694833017871,0.14014638099614,-0.200053437435142,-0.175332749811517,-0.151944228265388,-0.155269403535476,0.204167782468038,-0.176652363575809,0.259023109059646,0.233886579373682,-0.101655271737964,0.099978578037749,-0.265081819837911,-0.182441113737358,0.0878307803118756,-0.140423669019633,-0.125346292225234,0.203927096704988,-0.112380129725752,-0.11085473175296,-0.243491477836763,-0.179330353667383,-0.0952742766440304,-0.11095657048059,0.234387589316534,-0.0713857402193943,0.165825543593386,-0.0877161647690587,-0.0965012801750362,-0.0994165675845158,0.232485613346127,-0.0887188010051235,-0.0784786574973784,-0.105209836878339,-0.140829628530785,0.258706793249478,-0.0557251792417291,-0.106516645657721,-0.0958395549575048,-0.118515283387959,-0.0981518021230427,-0.0500091205029659,-0.0989606890518758,-0.0757374318874661,-0.0689268916867333,-0.0773717764413739,-0.0811063105232202,-0.198855400818324,-0.102486244012394,-0.0921987613214425,-0.0799162718049651,-0.0942389487566853,-0.0798528039871106,0.129240187065516,0.099194802158125,-0.203979744219247,-0.0756437888065733,0.225772849671321,-0.111018153271751,-0.187941331641169,-0.193157958211723,-0.173673882974797,-0.118062102793719,-0.10567914378247,-0.112641912400593,-0.0871058561652811,0.151148517690587,0.125818176144002,0.144135073494726,-0.109048448592464,0.14885669981445,-0.106782449161218,-0.089536372185743,0.112509960089287,0.0317686306125757,0.126888504760579],"type":"scatter","mode":"markers","text":["modules/clusters: C16 <br>GO.ID: GO:0140014 <br>GO.name: mitotic nuclear division","modules/clusters: C16 <br>GO.ID: GO:1903047 <br>GO.name: mitotic cell cycle process","modules/clusters: C16 <br>GO.ID: GO:0000070 <br>GO.name: mitotic sister chromatid segregation","modules/clusters: C16 <br>GO.ID: GO:0000278 <br>GO.name: mitotic cell cycle","modules/clusters: C16 <br>GO.ID: GO:0000819 <br>GO.name: sister chromatid segregation","modules/clusters: C16 <br>GO.ID: GO:0000280 <br>GO.name: nuclear division","modules/clusters: C16 <br>GO.ID: GO:0098813 <br>GO.name: nuclear chromosome segregation","modules/clusters: C16 <br>GO.ID: GO:0051301 <br>GO.name: cell division","modules/clusters: C16 <br>GO.ID: GO:0048285 <br>GO.name: organelle fission","modules/clusters: C16 <br>GO.ID: GO:0006695 <br>GO.name: cholesterol biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:1902653 <br>GO.name: secondary alcohol biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0016126 <br>GO.name: sterol biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0007059 <br>GO.name: chromosome segregation","modules/clusters: C16 <br>GO.ID: GO:0022402 <br>GO.name: cell cycle process","modules/clusters: C16 <br>GO.ID: GO:0008203 <br>GO.name: cholesterol metabolic process","modules/clusters: C16 <br>GO.ID: GO:0010564 <br>GO.name: regulation of cell cycle process","modules/clusters: C16 <br>GO.ID: GO:0016125 <br>GO.name: sterol metabolic process","modules/clusters: C16 <br>GO.ID: GO:1902850 <br>GO.name: microtubule cytoskeleton organization involved in mitosis","modules/clusters: C16 <br>GO.ID: GO:1902652 <br>GO.name: secondary alcohol metabolic process","modules/clusters: C16 <br>GO.ID: GO:0008299 <br>GO.name: isoprenoid biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0000910 <br>GO.name: cytokinesis","modules/clusters: C16 <br>GO.ID: GO:0051276 <br>GO.name: chromosome organization","modules/clusters: C16 <br>GO.ID: GO:0007052 <br>GO.name: mitotic spindle organization","modules/clusters: C16 <br>GO.ID: GO:0000281 <br>GO.name: mitotic cytokinesis","modules/clusters: C16 <br>GO.ID: GO:0046165 <br>GO.name: alcohol biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0061640 <br>GO.name: cytoskeleton-dependent cytokinesis","modules/clusters: C16 <br>GO.ID: GO:0008610 <br>GO.name: lipid biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0006694 <br>GO.name: steroid biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0007088 <br>GO.name: regulation of mitotic nuclear division","modules/clusters: C16 <br>GO.ID: GO:0044255 <br>GO.name: cellular lipid metabolic process","modules/clusters: C16 <br>GO.ID: GO:0007049 <br>GO.name: cell cycle","modules/clusters: C16 <br>GO.ID: GO:0007051 <br>GO.name: spindle organization","modules/clusters: C16 <br>GO.ID: GO:0006720 <br>GO.name: isoprenoid metabolic process","modules/clusters: C16 <br>GO.ID: GO:0051225 <br>GO.name: spindle assembly","modules/clusters: C16 <br>GO.ID: GO:0051726 <br>GO.name: regulation of cell cycle","modules/clusters: C16 <br>GO.ID: GO:0006066 <br>GO.name: alcohol metabolic process","modules/clusters: C16 <br>GO.ID: GO:0051983 <br>GO.name: regulation of chromosome segregation","modules/clusters: C16 <br>GO.ID: GO:0090307 <br>GO.name: mitotic spindle assembly","modules/clusters: C16 <br>GO.ID: GO:0007017 <br>GO.name: microtubule-based process","modules/clusters: C16 <br>GO.ID: GO:0000226 <br>GO.name: microtubule cytoskeleton organization","modules/clusters: C16 <br>GO.ID: GO:0051783 <br>GO.name: regulation of nuclear division","modules/clusters: C16 <br>GO.ID: GO:0033043 <br>GO.name: regulation of organelle organization","modules/clusters: C16 <br>GO.ID: GO:0008202 <br>GO.name: steroid metabolic process","modules/clusters: C16 <br>GO.ID: GO:0030071 <br>GO.name: regulation of mitotic metaphase/anaphase transition","modules/clusters: C16 <br>GO.ID: GO:0044283 <br>GO.name: small molecule biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0007091 <br>GO.name: metaphase/anaphase transition of mitotic cell cycle","modules/clusters: C16 <br>GO.ID: GO:0010965 <br>GO.name: regulation of mitotic sister chromatid separation","modules/clusters: C16 <br>GO.ID: GO:0033044 <br>GO.name: regulation of chromosome organization","modules/clusters: C16 <br>GO.ID: GO:1901617 <br>GO.name: organic hydroxy compound biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:1902099 <br>GO.name: regulation of metaphase/anaphase transition of cell cycle","modules/clusters: C16 <br>GO.ID: GO:2001251 <br>GO.name: negative regulation of chromosome organization","modules/clusters: C16 <br>GO.ID: GO:0044784 <br>GO.name: metaphase/anaphase transition of cell cycle","modules/clusters: C16 <br>GO.ID: GO:0051306 <br>GO.name: mitotic sister chromatid separation","modules/clusters: C16 <br>GO.ID: GO:0006629 <br>GO.name: lipid metabolic process","modules/clusters: C16 <br>GO.ID: GO:0007094 <br>GO.name: mitotic spindle assembly checkpoint signaling","modules/clusters: C16 <br>GO.ID: GO:0071173 <br>GO.name: spindle assembly checkpoint signaling","modules/clusters: C16 <br>GO.ID: GO:0071174 <br>GO.name: mitotic spindle checkpoint signaling","modules/clusters: C16 <br>GO.ID: GO:0031577 <br>GO.name: spindle checkpoint signaling","modules/clusters: C16 <br>GO.ID: GO:0032465 <br>GO.name: regulation of cytokinesis","modules/clusters: C16 <br>GO.ID: GO:0045841 <br>GO.name: negative regulation of mitotic metaphase/anaphase transition","modules/clusters: C16 <br>GO.ID: GO:0033045 <br>GO.name: regulation of sister chromatid segregation","modules/clusters: C16 <br>GO.ID: GO:0033048 <br>GO.name: negative regulation of mitotic sister chromatid segregation","modules/clusters: C16 <br>GO.ID: GO:2000816 <br>GO.name: negative regulation of mitotic sister chromatid separation","modules/clusters: C16 <br>GO.ID: GO:0033046 <br>GO.name: negative regulation of sister chromatid segregation","modules/clusters: C16 <br>GO.ID: GO:1902100 <br>GO.name: negative regulation of metaphase/anaphase transition of cell cycle","modules/clusters: C16 <br>GO.ID: GO:0006996 <br>GO.name: organelle organization","modules/clusters: C16 <br>GO.ID: GO:1905818 <br>GO.name: regulation of chromosome separation","modules/clusters: C16 <br>GO.ID: GO:0051985 <br>GO.name: negative regulation of chromosome segregation","modules/clusters: C16 <br>GO.ID: GO:1905819 <br>GO.name: negative regulation of chromosome separation","modules/clusters: C16 <br>GO.ID: GO:0033047 <br>GO.name: regulation of mitotic sister chromatid segregation","modules/clusters: C16 <br>GO.ID: GO:0045839 <br>GO.name: negative regulation of mitotic nuclear division","modules/clusters: C16 <br>GO.ID: GO:0009240 <br>GO.name: isopentenyl diphosphate biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0046490 <br>GO.name: isopentenyl diphosphate metabolic process","modules/clusters: C16 <br>GO.ID: GO:0051304 <br>GO.name: chromosome separation","modules/clusters: C16 <br>GO.ID: GO:0051784 <br>GO.name: negative regulation of nuclear division","modules/clusters: C16 <br>GO.ID: GO:1901615 <br>GO.name: organic hydroxy compound metabolic process","modules/clusters: C16 <br>GO.ID: GO:0007346 <br>GO.name: regulation of mitotic cell cycle","modules/clusters: C16 <br>GO.ID: GO:0032506 <br>GO.name: cytokinetic process","modules/clusters: C16 <br>GO.ID: GO:0008608 <br>GO.name: attachment of spindle microtubules to kinetochore","modules/clusters: C16 <br>GO.ID: GO:0007010 <br>GO.name: cytoskeleton organization","modules/clusters: C16 <br>GO.ID: GO:0051128 <br>GO.name: regulation of cellular component organization","modules/clusters: C16 <br>GO.ID: GO:0051302 <br>GO.name: regulation of cell division","modules/clusters: C16 <br>GO.ID: GO:0051988 <br>GO.name: regulation of attachment of spindle microtubules to kinetochore","modules/clusters: C16 <br>GO.ID: GO:1901990 <br>GO.name: regulation of mitotic cell cycle phase transition","modules/clusters: C16 <br>GO.ID: GO:0044281 <br>GO.name: small molecule metabolic process","modules/clusters: C16 <br>GO.ID: GO:0016114 <br>GO.name: terpenoid biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0008654 <br>GO.name: phospholipid biosynthetic process","modules/clusters: C16 <br>GO.ID: GO:0032954 <br>GO.name: regulation of cytokinetic process","modules/clusters: C16 <br>GO.ID: GO:0019287 <br>GO.name: isopentenyl diphosphate biosynthetic process, mevalonate pathway","modules/clusters: C16 <br>GO.ID: GO:0007093 <br>GO.name: mitotic cell cycle checkpoint signaling","modules/clusters: C16 <br>GO.ID: GO:0010639 <br>GO.name: negative regulation of organelle organization","modules/clusters: C16 <br>GO.ID: GO:0006644 <br>GO.name: phospholipid metabolic process","modules/clusters: C16 <br>GO.ID: GO:0140694 <br>GO.name: non-membrane-bounded organelle assembly","modules/clusters: C16 <br>GO.ID: GO:0045337 <br>GO.name: farnesyl diphosphate biosynthetic process"],"showlegend":true,"marker":{"color":"rgba(210,166,183,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(210,166,183,1)"}},"name":"C16","textfont":{"color":"rgba(210,166,183,1)"},"error_y":{"color":"rgba(210,166,183,1)"},"error_x":{"color":"rgba(210,166,183,1)"},"line":{"color":"rgba(210,166,183,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.253267579276476,0.204627304352107,0.209653546139302,0.228109795425328,0.224892922830831,0.164919580333637,-0.0469324318750351,0.218405929042495,0.238490349953197,0.190475791350421,-0.104847047635342,-0.396517624512316,-0.130699439744689,-0.0806652896443631,-0.142519418971846,0.175402982004436,0.176881502694096,0.0299095285008815,0.122344706076601,0.11467347769059,0.231166339930174,-0.365681387315271,0.158994497363575,-0.033265835321496,0.0776844220969752,-0.115030927733515,-0.384843621068119,0.198867277132316,0.00987540051100863,0.182348293888726,0.161036548035793,0.151490208945806,0.17981622625684,-0.380744539029966,-0.36771883025065,-0.371057556591291,0.189218413659445,-0.117295479820555,0.174870279694817,0.0870895239067075,-0.158025229274865,0.187736314982144,0.213959120713239,0.154611115943226,-0.11414600156379],"y":[-0.111498734166708,-0.259076158824515,-0.0909779473606296,-0.113267990099171,-0.270701573671721,-0.240337119645017,-0.22604833032412,-0.107441126091132,-0.0852822709619116,-0.102030548274946,-0.242461865163338,0.00902471847179908,0.0709370746961219,-0.234065394982925,-0.156274748587047,-0.102035529513965,-0.189500548574947,0.0492863056999522,-0.0405971040015433,-0.228970113400424,-0.0862155534339625,-0.0082794811071489,0.374792092903334,-0.186934503218523,-0.0328315831841413,-0.207668343939888,0.0163526002150915,-0.226587014326464,0.0470328534170022,-0.191752999105192,0.215289252242024,0.210924825857387,0.0473691834920096,0.0133744055332362,0.133822006757183,0.129787349114709,-0.224275574803889,-0.150113425663514,0.0795312860006282,-0.0206042758241582,0.0706689058023842,-0.22685171603119,-0.229678903476859,-0.0791113587938317,-0.140011461877201],"type":"scatter","mode":"markers","text":["modules/clusters: C17 <br>GO.ID: GO:0060548 <br>GO.name: negative regulation of cell death","modules/clusters: C17 <br>GO.ID: GO:0097191 <br>GO.name: extrinsic apoptotic signaling pathway","modules/clusters: C17 <br>GO.ID: GO:1901215 <br>GO.name: negative regulation of neuron death","modules/clusters: C17 <br>GO.ID: GO:1901214 <br>GO.name: regulation of neuron death","modules/clusters: C17 <br>GO.ID: GO:0097190 <br>GO.name: apoptotic signaling pathway","modules/clusters: C17 <br>GO.ID: GO:0008630 <br>GO.name: intrinsic apoptotic signaling pathway in response to DNA damage","modules/clusters: C17 <br>GO.ID: GO:0070997 <br>GO.name: neuron death","modules/clusters: C17 <br>GO.ID: GO:0043069 <br>GO.name: negative regulation of programmed cell death","modules/clusters: C17 <br>GO.ID: GO:0008285 <br>GO.name: negative regulation of cell population proliferation","modules/clusters: C17 <br>GO.ID: GO:0043066 <br>GO.name: negative regulation of apoptotic process","modules/clusters: C17 <br>GO.ID: GO:0002263 <br>GO.name: cell activation involved in immune response","modules/clusters: C17 <br>GO.ID: GO:0030099 <br>GO.name: myeloid cell differentiation","modules/clusters: C17 <br>GO.ID: GO:0045656 <br>GO.name: negative regulation of monocyte differentiation","modules/clusters: C17 <br>GO.ID: GO:0071560 <br>GO.name: cellular response to transforming growth factor beta stimulus","modules/clusters: C17 <br>GO.ID: GO:0071559 <br>GO.name: response to transforming growth factor beta","modules/clusters: C17 <br>GO.ID: GO:0043523 <br>GO.name: regulation of neuron apoptotic process","modules/clusters: C17 <br>GO.ID: GO:1902042 <br>GO.name: negative regulation of extrinsic apoptotic signaling pathway via death domain receptors","modules/clusters: C17 <br>GO.ID: GO:0050790 <br>GO.name: regulation of catalytic activity","modules/clusters: C17 <br>GO.ID: GO:0080134 <br>GO.name: regulation of response to stress","modules/clusters: C17 <br>GO.ID: GO:0007179 <br>GO.name: transforming growth factor beta receptor signaling pathway","modules/clusters: C17 <br>GO.ID: GO:0007162 <br>GO.name: negative regulation of cell adhesion","modules/clusters: C17 <br>GO.ID: GO:0030217 <br>GO.name: T cell differentiation","modules/clusters: C17 <br>GO.ID: GO:0030162 <br>GO.name: regulation of proteolysis","modules/clusters: C17 <br>GO.ID: GO:0051402 <br>GO.name: neuron apoptotic process","modules/clusters: C17 <br>GO.ID: GO:0045596 <br>GO.name: negative regulation of cell differentiation","modules/clusters: C17 <br>GO.ID: GO:0002366 <br>GO.name: leukocyte activation involved in immune response","modules/clusters: C17 <br>GO.ID: GO:0002573 <br>GO.name: myeloid leukocyte differentiation","modules/clusters: C17 <br>GO.ID: GO:2001236 <br>GO.name: regulation of extrinsic apoptotic signaling pathway","modules/clusters: C17 <br>GO.ID: GO:0051336 <br>GO.name: regulation of hydrolase activity","modules/clusters: C17 <br>GO.ID: GO:2001237 <br>GO.name: negative regulation of extrinsic apoptotic signaling pathway","modules/clusters: C17 <br>GO.ID: GO:0045981 <br>GO.name: positive regulation of nucleotide metabolic process","modules/clusters: C17 <br>GO.ID: GO:1900544 <br>GO.name: positive regulation of purine nucleotide metabolic process","modules/clusters: C17 <br>GO.ID: GO:0023056 <br>GO.name: positive regulation of signaling","modules/clusters: C17 <br>GO.ID: GO:0042063 <br>GO.name: gliogenesis","modules/clusters: C17 <br>GO.ID: GO:0008584 <br>GO.name: male gonad development","modules/clusters: C17 <br>GO.ID: GO:0046546 <br>GO.name: development of primary male sexual characteristics","modules/clusters: C17 <br>GO.ID: GO:1902041 <br>GO.name: regulation of extrinsic apoptotic signaling pathway via death domain receptors","modules/clusters: C17 <br>GO.ID: GO:0046631 <br>GO.name: alpha-beta T cell activation","modules/clusters: C17 <br>GO.ID: GO:0040012 <br>GO.name: regulation of locomotion","modules/clusters: C17 <br>GO.ID: GO:0050776 <br>GO.name: regulation of immune response","modules/clusters: C17 <br>GO.ID: GO:0045655 <br>GO.name: regulation of monocyte differentiation","modules/clusters: C17 <br>GO.ID: GO:0002764 <br>GO.name: immune response-regulating signaling pathway","modules/clusters: C17 <br>GO.ID: GO:2001233 <br>GO.name: regulation of apoptotic signaling pathway","modules/clusters: C17 <br>GO.ID: GO:0043524 <br>GO.name: negative regulation of neuron apoptotic process","modules/clusters: C17 <br>GO.ID: GO:0035710 <br>GO.name: CD4-positive, alpha-beta T cell activation"],"showlegend":true,"marker":{"color":"rgba(230,147,148,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(230,147,148,1)"}},"name":"C17","textfont":{"color":"rgba(230,147,148,1)"},"error_y":{"color":"rgba(230,147,148,1)"},"error_x":{"color":"rgba(230,147,148,1)"},"line":{"color":"rgba(230,147,148,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.485637602325157,-0.481583549383032,-0.404183503661917,-0.472482086576148,-0.369924412859229,-0.0777389143240655,-0.154378218509222,0.246937893075722,0.285826276110074,0.251936522307842],"y":[0.131317907183826,0.139863970526746,0.012380470175759,0.137028953471574,-0.0123816870483667,-0.280366538633831,-0.169472588845343,-0.117624001136724,0.116192194543125,0.11450820788722],"type":"scatter","mode":"markers","text":["modules/clusters: C17/C2 <br>GO.ID: GO:0002520 <br>GO.name: immune system development","modules/clusters: C17/C2 <br>GO.ID: GO:0048534 <br>GO.name: hematopoietic or lymphoid organ development","modules/clusters: C17/C2 <br>GO.ID: GO:1903131 <br>GO.name: mononuclear cell differentiation","modules/clusters: C17/C2 <br>GO.ID: GO:0030097 <br>GO.name: hemopoiesis","modules/clusters: C17/C2 <br>GO.ID: GO:0030098 <br>GO.name: lymphocyte differentiation","modules/clusters: C17/C2 <br>GO.ID: GO:0071363 <br>GO.name: cellular response to growth factor stimulus","modules/clusters: C17/C2 <br>GO.ID: GO:0070848 <br>GO.name: response to growth factor","modules/clusters: C17/C2 <br>GO.ID: GO:0050865 <br>GO.name: regulation of cell activation","modules/clusters: C17/C2 <br>GO.ID: GO:0051174 <br>GO.name: regulation of phosphorus metabolic process","modules/clusters: C17/C2 <br>GO.ID: GO:0019220 <br>GO.name: regulation of phosphate metabolic process"],"showlegend":true,"marker":{"color":"rgba(199,153,163,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(199,153,163,1)"}},"name":"C17/C2","textfont":{"color":"rgba(199,153,163,1)"},"error_y":{"color":"rgba(199,153,163,1)"},"error_x":{"color":"rgba(199,153,163,1)"},"line":{"color":"rgba(199,153,163,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.076117274617488,-0.118450523197693,-0.122534620221294],"y":[-0.25794322271076,-0.19873259024604,-0.179842358115429],"type":"scatter","mode":"markers","text":["modules/clusters: C17/C2/C22 <br>GO.ID: GO:0001775 <br>GO.name: cell activation","modules/clusters: C17/C2/C22 <br>GO.ID: GO:0045321 <br>GO.name: leukocyte activation","modules/clusters: C17/C2/C22 <br>GO.ID: GO:0046649 <br>GO.name: lymphocyte activation"],"showlegend":true,"marker":{"color":"rgba(205,165,148,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(205,165,148,1)"}},"name":"C17/C2/C22","textfont":{"color":"rgba(205,165,148,1)"},"error_y":{"color":"rgba(205,165,148,1)"},"error_x":{"color":"rgba(205,165,148,1)"},"line":{"color":"rgba(205,165,148,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.162873296712034],"y":[-0.183936097577401],"type":"scatter","mode":"markers","text":"modules/clusters: C17/C2/C22/C8 <br>GO.ID: GO:0006950 <br>GO.name: response to stress","showlegend":true,"marker":{"color":"rgba(216,187,167,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(216,187,167,1)"}},"name":"C17/C2/C22/C8","textfont":{"color":"rgba(216,187,167,1)"},"error_y":{"color":"rgba(216,187,167,1)"},"error_x":{"color":"rgba(216,187,167,1)"},"line":{"color":"rgba(216,187,167,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0781016610784861],"y":[0.0368187587547724],"type":"scatter","mode":"markers","text":"modules/clusters: C17/C2/C22/C8/C9 <br>GO.ID: GO:0065007 <br>GO.name: biological regulation","showlegend":true,"marker":{"color":"rgba(234,195,198,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(234,195,198,1)"}},"name":"C17/C2/C22/C8/C9","textfont":{"color":"rgba(234,195,198,1)"},"error_y":{"color":"rgba(234,195,198,1)"},"error_x":{"color":"rgba(234,195,198,1)"},"line":{"color":"rgba(234,195,198,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.401353715244431,-0.118319461332359,-0.0524461673521885,0.181375255145714,-0.158791102346064,-0.163194070402043,0.213981154291839],"y":[0.00959647621185204,-0.106245078280747,-0.398709671116359,0.405164250307181,-0.176340548295804,-0.192821484116354,-0.258033049942693],"type":"scatter","mode":"markers","text":["modules/clusters: C17/C2/C8 <br>GO.ID: GO:0002521 <br>GO.name: leukocyte differentiation","modules/clusters: C17/C2/C8 <br>GO.ID: GO:0023052 <br>GO.name: signaling","modules/clusters: C17/C2/C8 <br>GO.ID: GO:0051716 <br>GO.name: cellular response to stimulus","modules/clusters: C17/C2/C8 <br>GO.ID: GO:0051246 <br>GO.name: regulation of protein metabolic process","modules/clusters: C17/C2/C8 <br>GO.ID: GO:1901700 <br>GO.name: response to oxygen-containing compound","modules/clusters: C17/C2/C8 <br>GO.ID: GO:0010033 <br>GO.name: response to organic substance","modules/clusters: C17/C2/C8 <br>GO.ID: GO:0007167 <br>GO.name: enzyme-linked receptor protein signaling pathway"],"showlegend":true,"marker":{"color":"rgba(212,181,175,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(212,181,175,1)"}},"name":"C17/C2/C8","textfont":{"color":"rgba(212,181,175,1)"},"error_y":{"color":"rgba(212,181,175,1)"},"error_x":{"color":"rgba(212,181,175,1)"},"line":{"color":"rgba(212,181,175,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.187134167948747],"y":[-0.158969737027193],"type":"scatter","mode":"markers","text":"modules/clusters: C17/C20/C22 <br>GO.ID: GO:0006955 <br>GO.name: immune response","showlegend":true,"marker":{"color":"rgba(195,170,157,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(195,170,157,1)"}},"name":"C17/C20/C22","textfont":{"color":"rgba(195,170,157,1)"},"error_y":{"color":"rgba(195,170,157,1)"},"error_x":{"color":"rgba(195,170,157,1)"},"line":{"color":"rgba(195,170,157,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.151575979591179,-0.120590182111632,-0.188814333514743,-0.118620620442888,-0.173991724392043,-0.169768773260592,-0.140707267537769,-0.145268324875548,-0.133006314014523,-0.16223424673798],"y":[-0.152811377817636,-0.163588688809088,-0.0325225761218355,-0.188268170256914,-0.12975407338258,-0.0472828235093311,-0.109149996089513,-0.138255479443198,-0.106738257346734,-0.174930499987972],"type":"scatter","mode":"markers","text":["modules/clusters: C17/C22 <br>GO.ID: GO:0006952 <br>GO.name: defense response","modules/clusters: C17/C22 <br>GO.ID: GO:0042110 <br>GO.name: T cell activation","modules/clusters: C17/C22 <br>GO.ID: GO:0002252 <br>GO.name: immune effector process","modules/clusters: C17/C22 <br>GO.ID: GO:0002285 <br>GO.name: lymphocyte activation involved in immune response","modules/clusters: C17/C22 <br>GO.ID: GO:0002250 <br>GO.name: adaptive immune response","modules/clusters: C17/C22 <br>GO.ID: GO:0044419 <br>GO.name: biological process involved in interspecies interaction between organisms","modules/clusters: C17/C22 <br>GO.ID: GO:0051707 <br>GO.name: response to other organism","modules/clusters: C17/C22 <br>GO.ID: GO:0043207 <br>GO.name: response to external biotic stimulus","modules/clusters: C17/C22 <br>GO.ID: GO:0098542 <br>GO.name: defense response to other organism","modules/clusters: C17/C22 <br>GO.ID: GO:0009607 <br>GO.name: response to biotic stimulus"],"showlegend":true,"marker":{"color":"rgba(220,163,141,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(220,163,141,1)"}},"name":"C17/C22","textfont":{"color":"rgba(220,163,141,1)"},"error_y":{"color":"rgba(220,163,141,1)"},"error_x":{"color":"rgba(220,163,141,1)"},"line":{"color":"rgba(220,163,141,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.259598840651102,-0.0597860670987189,0.00946553175536763,0.0534307756500433,0.138942752411355,0.0287698390455959,0.257023935876664],"y":[-0.104819371551787,-0.242706418398936,0.137655258548337,0.0468841203696572,-0.027248247150325,0.0478929378078133,-0.125828661354243],"type":"scatter","mode":"markers","text":["modules/clusters: C17/C8 <br>GO.ID: GO:0042127 <br>GO.name: regulation of cell population proliferation","modules/clusters: C17/C8 <br>GO.ID: GO:0008283 <br>GO.name: cell population proliferation","modules/clusters: C17/C8 <br>GO.ID: GO:0051093 <br>GO.name: negative regulation of developmental process","modules/clusters: C17/C8 <br>GO.ID: GO:0065009 <br>GO.name: regulation of molecular function","modules/clusters: C17/C8 <br>GO.ID: GO:0048584 <br>GO.name: positive regulation of response to stimulus","modules/clusters: C17/C8 <br>GO.ID: GO:0044093 <br>GO.name: positive regulation of molecular function","modules/clusters: C17/C8 <br>GO.ID: GO:0010647 <br>GO.name: positive regulation of cell communication"],"showlegend":true,"marker":{"color":"rgba(228,178,167,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(228,178,167,1)"}},"name":"C17/C8","textfont":{"color":"rgba(228,178,167,1)"},"error_y":{"color":"rgba(228,178,167,1)"},"error_x":{"color":"rgba(228,178,167,1)"},"line":{"color":"rgba(228,178,167,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0517916109959345],"y":[0.0422524269294006],"type":"scatter","mode":"markers","text":"modules/clusters: C17/C8/C9 <br>GO.ID: GO:0065008 <br>GO.name: regulation of biological quality","showlegend":true,"marker":{"color":"rgba(240,191,198,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(240,191,198,1)"}},"name":"C17/C8/C9","textfont":{"color":"rgba(240,191,198,1)"},"error_y":{"color":"rgba(240,191,198,1)"},"error_x":{"color":"rgba(240,191,198,1)"},"line":{"color":"rgba(240,191,198,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.108020161830031],"y":[0.168856447375774],"type":"scatter","mode":"markers","text":"modules/clusters: C17/C9 <br>GO.ID: GO:0022603 <br>GO.name: regulation of anatomical structure morphogenesis","showlegend":true,"marker":{"color":"rgba(241,176,188,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(241,176,188,1)"}},"name":"C17/C9","textfont":{"color":"rgba(241,176,188,1)"},"error_y":{"color":"rgba(241,176,188,1)"},"error_x":{"color":"rgba(241,176,188,1)"},"line":{"color":"rgba(241,176,188,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0618772062183371,-0.127791849809394],"y":[-0.225535793418973,-0.0151037991864696],"type":"scatter","mode":"markers","text":["modules/clusters: C19 <br>GO.ID: GO:0051641 <br>GO.name: cellular localization","modules/clusters: C19 <br>GO.ID: GO:0072348 <br>GO.name: sulfur compound transport"],"showlegend":true,"marker":{"color":"rgba(210,144,146,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(210,144,146,1)"}},"name":"C19","textfont":{"color":"rgba(210,144,146,1)"},"error_y":{"color":"rgba(210,144,146,1)"},"error_x":{"color":"rgba(210,144,146,1)"},"line":{"color":"rgba(210,144,146,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.377056005654148,0.177878834788415,-0.382327665257674,0.159532838768592,0.161191066480982,0.17492469936528,0.174727827324528,0.21100458334202,-0.436288069598026,-0.0504163867479799,-0.374531572728663,-0.40445670017524,0.170277843603813,-0.419856085825273,0.225106829829762,0.203595256906795,0.261421131840825,0.23342388061549,-0.172252518887132,0.190254217997586,-0.41972327672951,-0.149613055643989,-0.341806038743146,-0.373626925532682,0.172501087507334,-0.45605929553667,0.204222582798747,0.203942079950573,-0.47387162686541,0.192315505450582,-0.145962797582612,0.208786124396417,0.24838259066897,-0.15764361209742,-0.466625926930028,-0.144364964914416,-0.00540264379803737,0.223046277631196,-0.00385739502968538,-0.0421822399693592,0.164749788838574,0.157536684840832,-0.167469133730522,-0.447173006941244,-0.446957435475466,-0.461861853859214,0.194947718451662,-0.0469981832109741,-0.377952365009878,0.176928126723055,-0.36476140232413,0.183134167441269,0.200763454169137,-0.0244296554161537,-0.028300493835307,-0.47813865694158,-0.0189455237955654,0.0115395347134119,0.115088653653802,-0.161137555819515,0.191123834403623,0.197096210052501],"y":[0.134028177458227,0.378137037251498,-0.00351779940680762,0.201678447990887,0.199249236105254,0.199788008661533,0.191158228577098,0.117350408984187,0.151348901239998,-0.1574442104872,-0.00452580109424284,0.139559378286842,0.226492419513425,0.14182111538153,0.110339010639173,0.243608411742964,0.122746301113011,0.12140501098779,-0.211216065254659,0.239329673257054,0.14435606198262,-0.148760390763744,0.0405933773206061,0.122841917915922,0.372080341218627,0.148515714876495,-0.0848891330647244,-0.219699823083902,0.159612795438942,-0.187991029990284,-0.141400505043069,-0.110530733573187,-0.121677509479795,-0.172553118269273,0.153106753528339,0.0713764415855006,0.0506795695379491,0.253670371192267,-0.260142863588784,-0.192423866845134,0.382381779816662,0.21601094140792,-0.214275512555855,0.148185507468577,0.145367894199669,0.146359249051668,-0.244176888887235,-0.17236575383425,0.018982731882517,0.113531668442981,-0.00551856127481751,0.231518723021735,0.109321678211898,-0.185424285862796,-0.171697823151286,0.156418442009212,-0.204329561473228,0.049249415697797,-0.227986541892073,0.0492169193543152,0.10633388651315,-0.248382343766181],"type":"scatter","mode":"markers","text":["modules/clusters: C2 <br>GO.ID: GO:0060412 <br>GO.name: ventricular septum morphogenesis","modules/clusters: C2 <br>GO.ID: GO:0010629 <br>GO.name: negative regulation of gene expression","modules/clusters: C2 <br>GO.ID: GO:0002329 <br>GO.name: pre-B cell differentiation","modules/clusters: C2 <br>GO.ID: GO:0035195 <br>GO.name: miRNA-mediated gene silencing","modules/clusters: C2 <br>GO.ID: GO:0035194 <br>GO.name: post-transcriptional gene silencing by RNA","modules/clusters: C2 <br>GO.ID: GO:0016441 <br>GO.name: post-transcriptional gene silencing","modules/clusters: C2 <br>GO.ID: GO:0031047 <br>GO.name: gene silencing by RNA","modules/clusters: C2 <br>GO.ID: GO:0042326 <br>GO.name: negative regulation of phosphorylation","modules/clusters: C2 <br>GO.ID: GO:0003007 <br>GO.name: heart morphogenesis","modules/clusters: C2 <br>GO.ID: GO:0001783 <br>GO.name: B cell apoptotic process","modules/clusters: C2 <br>GO.ID: GO:0002327 <br>GO.name: immature B cell differentiation","modules/clusters: C2 <br>GO.ID: GO:0060411 <br>GO.name: cardiac septum morphogenesis","modules/clusters: C2 <br>GO.ID: GO:0001933 <br>GO.name: negative regulation of protein phosphorylation","modules/clusters: C2 <br>GO.ID: GO:0003281 <br>GO.name: ventricular septum development","modules/clusters: C2 <br>GO.ID: GO:0042325 <br>GO.name: regulation of phosphorylation","modules/clusters: C2 <br>GO.ID: GO:0031400 <br>GO.name: negative regulation of protein modification process","modules/clusters: C2 <br>GO.ID: GO:0010563 <br>GO.name: negative regulation of phosphorus metabolic process","modules/clusters: C2 <br>GO.ID: GO:0045936 <br>GO.name: negative regulation of phosphate metabolic process","modules/clusters: C2 <br>GO.ID: GO:0042221 <br>GO.name: response to chemical","modules/clusters: C2 <br>GO.ID: GO:0001932 <br>GO.name: regulation of protein phosphorylation","modules/clusters: C2 <br>GO.ID: GO:0003206 <br>GO.name: cardiac chamber morphogenesis","modules/clusters: C2 <br>GO.ID: GO:0009611 <br>GO.name: response to wounding","modules/clusters: C2 <br>GO.ID: GO:0021522 <br>GO.name: spinal cord motor neuron differentiation","modules/clusters: C2 <br>GO.ID: GO:0001701 <br>GO.name: in utero embryonic development","modules/clusters: C2 <br>GO.ID: GO:0010608 <br>GO.name: post-transcriptional regulation of gene expression","modules/clusters: C2 <br>GO.ID: GO:0003279 <br>GO.name: cardiac septum development","modules/clusters: C2 <br>GO.ID: GO:0048167 <br>GO.name: regulation of synaptic plasticity","modules/clusters: C2 <br>GO.ID: GO:0043408 <br>GO.name: regulation of MAPK cascade","modules/clusters: C2 <br>GO.ID: GO:0007507 <br>GO.name: heart development","modules/clusters: C2 <br>GO.ID: GO:0043409 <br>GO.name: negative regulation of MAPK cascade","modules/clusters: C2 <br>GO.ID: GO:0071495 <br>GO.name: cellular response to endogenous stimulus","modules/clusters: C2 <br>GO.ID: GO:0050804 <br>GO.name: modulation of chemical synaptic transmission","modules/clusters: C2 <br>GO.ID: GO:0099177 <br>GO.name: regulation of trans-synaptic signaling","modules/clusters: C2 <br>GO.ID: GO:0010035 <br>GO.name: response to inorganic substance","modules/clusters: C2 <br>GO.ID: GO:0003205 <br>GO.name: cardiac chamber development","modules/clusters: C2 <br>GO.ID: GO:1902105 <br>GO.name: regulation of leukocyte differentiation","modules/clusters: C2 <br>GO.ID: GO:0051348 <br>GO.name: negative regulation of transferase activity","modules/clusters: C2 <br>GO.ID: GO:0031399 <br>GO.name: regulation of protein modification process","modules/clusters: C2 <br>GO.ID: GO:0007267 <br>GO.name: cell-cell signaling","modules/clusters: C2 <br>GO.ID: GO:0071887 <br>GO.name: leukocyte apoptotic process","modules/clusters: C2 <br>GO.ID: GO:0051248 <br>GO.name: negative regulation of protein metabolic process","modules/clusters: C2 <br>GO.ID: GO:0006469 <br>GO.name: negative regulation of protein kinase activity","modules/clusters: C2 <br>GO.ID: GO:0090398 <br>GO.name: cellular senescence","modules/clusters: C2 <br>GO.ID: GO:0003231 <br>GO.name: cardiac ventricle development","modules/clusters: C2 <br>GO.ID: GO:0021517 <br>GO.name: ventral spinal cord development","modules/clusters: C2 <br>GO.ID: GO:0009790 <br>GO.name: embryo development","modules/clusters: C2 <br>GO.ID: GO:0070371 <br>GO.name: ERK1 and ERK2 cascade","modules/clusters: C2 <br>GO.ID: GO:0070227 <br>GO.name: lymphocyte apoptotic process","modules/clusters: C2 <br>GO.ID: GO:0021515 <br>GO.name: cell differentiation in spinal cord","modules/clusters: C2 <br>GO.ID: GO:0033673 <br>GO.name: negative regulation of kinase activity","modules/clusters: C2 <br>GO.ID: GO:0030183 <br>GO.name: B cell differentiation","modules/clusters: C2 <br>GO.ID: GO:0045859 <br>GO.name: regulation of protein kinase activity","modules/clusters: C2 <br>GO.ID: GO:0043549 <br>GO.name: regulation of kinase activity","modules/clusters: C2 <br>GO.ID: GO:0098916 <br>GO.name: anterograde trans-synaptic signaling","modules/clusters: C2 <br>GO.ID: GO:0007268 <br>GO.name: chemical synaptic transmission","modules/clusters: C2 <br>GO.ID: GO:0009887 <br>GO.name: animal organ morphogenesis","modules/clusters: C2 <br>GO.ID: GO:0099537 <br>GO.name: trans-synaptic signaling","modules/clusters: C2 <br>GO.ID: GO:0051338 <br>GO.name: regulation of transferase activity","modules/clusters: C2 <br>GO.ID: GO:0030509 <br>GO.name: BMP signaling pathway","modules/clusters: C2 <br>GO.ID: GO:0046637 <br>GO.name: regulation of alpha-beta T cell differentiation","modules/clusters: C2 <br>GO.ID: GO:0042509 <br>GO.name: regulation of tyrosine phosphorylation of STAT protein","modules/clusters: C2 <br>GO.ID: GO:0007178 <br>GO.name: transmembrane receptor protein serine/threonine kinase signaling pathway"],"showlegend":true,"marker":{"color":"rgba(169,160,178,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(169,160,178,1)"}},"name":"C2","textfont":{"color":"rgba(169,160,178,1)"},"error_y":{"color":"rgba(169,160,178,1)"},"error_x":{"color":"rgba(169,160,178,1)"},"line":{"color":"rgba(169,160,178,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.153600087066442],"y":[-0.166230336040177],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C22/C8 <br>GO.ID: GO:0034097 <br>GO.name: response to cytokine","showlegend":true,"marker":{"color":"rgba(208,189,171,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(208,189,171,1)"}},"name":"C2/C22/C8","textfont":{"color":"rgba(208,189,171,1)"},"error_y":{"color":"rgba(208,189,171,1)"},"error_x":{"color":"rgba(208,189,171,1)"},"line":{"color":"rgba(208,189,171,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.235539743722846,0.208495990615711,0.297301700562608,0.252397559512923,0.195214803150606,0.206822013331908,0.242509629756513],"y":[0.292763692229504,0.386722858447544,0.123627630064675,-0.104408498040558,0.295974968214617,0.309161480048912,0.157243091042655],"type":"scatter","mode":"markers","text":["modules/clusters: C2/C8 <br>GO.ID: GO:0009892 <br>GO.name: negative regulation of metabolic process","modules/clusters: C2/C8 <br>GO.ID: GO:0010605 <br>GO.name: negative regulation of macromolecule metabolic process","modules/clusters: C2/C8 <br>GO.ID: GO:0031324 <br>GO.name: negative regulation of cellular metabolic process","modules/clusters: C2/C8 <br>GO.ID: GO:0030155 <br>GO.name: regulation of cell adhesion","modules/clusters: C2/C8 <br>GO.ID: GO:0009890 <br>GO.name: negative regulation of biosynthetic process","modules/clusters: C2/C8 <br>GO.ID: GO:0051172 <br>GO.name: negative regulation of nitrogen compound metabolic process","modules/clusters: C2/C8 <br>GO.ID: GO:0031327 <br>GO.name: negative regulation of cellular biosynthetic process"],"showlegend":true,"marker":{"color":"rgba(197,185,182,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(197,185,182,1)"}},"name":"C2/C8","textfont":{"color":"rgba(197,185,182,1)"},"error_y":{"color":"rgba(197,185,182,1)"},"error_x":{"color":"rgba(197,185,182,1)"},"line":{"color":"rgba(197,185,182,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.197182234188963,-0.247938200752224,-0.24757120341357,-0.05960743842311],"y":[-0.247965175066796,-0.0686776389263702,-0.0954272660318976,-0.18570747540964],"type":"scatter","mode":"markers","text":["modules/clusters: C21 <br>GO.ID: GO:0007169 <br>GO.name: transmembrane receptor protein tyrosine kinase signaling pathway","modules/clusters: C21 <br>GO.ID: GO:0007411 <br>GO.name: axon guidance","modules/clusters: C21 <br>GO.ID: GO:0097485 <br>GO.name: neuron projection guidance","modules/clusters: C21 <br>GO.ID: GO:0034330 <br>GO.name: cell junction organization"],"showlegend":true,"marker":{"color":"rgba(169,178,173,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(169,178,173,1)"}},"name":"C21","textfont":{"color":"rgba(169,178,173,1)"},"error_y":{"color":"rgba(169,178,173,1)"},"error_x":{"color":"rgba(169,178,173,1)"},"line":{"color":"rgba(169,178,173,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0505531047728503,-0.15528076157329,0.0711405502694089,0.0945212029302309,-0.161434915797099,-0.164091805602612,-0.360291357212207,-0.168531319742929,0.0374219662894837,-0.0832805436636675,-0.117769173708523,0.153735204461516,0.0999208969739213,0.0531739749819813,-0.0631128088225495,0.110438311313748,-0.0929437204134094,0.257106002295527,0.0346521033369087,-0.0903307648115194,0.0479561825157996,0.165651770051013],"y":[0.0792843254586736,-0.00614860800473893,0.081174349107049,-0.0683907950643851,0.0494948815500345,-0.107359877099587,-0.00361896424704294,-0.0167521088731086,0.0855222811855186,-0.155298685483274,-0.149790938553193,-0.0531550029440117,0.0836446386153665,0.0878124249445045,-0.241013839316722,-0.0725827758996087,0.114430086894712,-0.102966944043806,0.177632637883251,0.0922021004044537,0.186294973988543,-0.25777686778424],"type":"scatter","mode":"markers","text":["modules/clusters: C22 <br>GO.ID: GO:0002706 <br>GO.name: regulation of lymphocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0002449 <br>GO.name: lymphocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0002703 <br>GO.name: regulation of leukocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0046643 <br>GO.name: regulation of gamma-delta T cell activation","modules/clusters: C22 <br>GO.ID: GO:0045586 <br>GO.name: regulation of gamma-delta T cell differentiation","modules/clusters: C22 <br>GO.ID: GO:0002460 <br>GO.name: adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains","modules/clusters: C22 <br>GO.ID: GO:0042492 <br>GO.name: gamma-delta T cell differentiation","modules/clusters: C22 <br>GO.ID: GO:0002443 <br>GO.name: leukocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0002708 <br>GO.name: positive regulation of lymphocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0001909 <br>GO.name: leukocyte mediated cytotoxicity","modules/clusters: C22 <br>GO.ID: GO:0046629 <br>GO.name: gamma-delta T cell activation","modules/clusters: C22 <br>GO.ID: GO:0001910 <br>GO.name: regulation of leukocyte mediated cytotoxicity","modules/clusters: C22 <br>GO.ID: GO:0002697 <br>GO.name: regulation of immune effector process","modules/clusters: C22 <br>GO.ID: GO:0002705 <br>GO.name: positive regulation of leukocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0001906 <br>GO.name: cell killing","modules/clusters: C22 <br>GO.ID: GO:0050863 <br>GO.name: regulation of T cell activation","modules/clusters: C22 <br>GO.ID: GO:0002381 <br>GO.name: immunoglobulin production involved in immunoglobulin-mediated immune response","modules/clusters: C22 <br>GO.ID: GO:0031341 <br>GO.name: regulation of cell killing","modules/clusters: C22 <br>GO.ID: GO:2000558 <br>GO.name: positive regulation of immunoglobulin production in mucosal tissue","modules/clusters: C22 <br>GO.ID: GO:0002426 <br>GO.name: immunoglobulin production in mucosal tissue","modules/clusters: C22 <br>GO.ID: GO:2000557 <br>GO.name: regulation of immunoglobulin production in mucosal tissue","modules/clusters: C22 <br>GO.ID: GO:0019221 <br>GO.name: cytokine-mediated signaling pathway"],"showlegend":true,"marker":{"color":"rgba(211,179,135,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(211,179,135,1)"}},"name":"C22","textfont":{"color":"rgba(211,179,135,1)"},"error_y":{"color":"rgba(211,179,135,1)"},"error_x":{"color":"rgba(211,179,135,1)"},"line":{"color":"rgba(211,179,135,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.167792750335699],"y":[0.381956549101576],"type":"scatter","mode":"markers","text":"modules/clusters: C22/C8 <br>GO.ID: GO:0010628 <br>GO.name: positive regulation of gene expression","showlegend":true,"marker":{"color":"rgba(219,194,161,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(219,194,161,1)"}},"name":"C22/C8","textfont":{"color":"rgba(219,194,161,1)"},"error_y":{"color":"rgba(219,194,161,1)"},"error_x":{"color":"rgba(219,194,161,1)"},"line":{"color":"rgba(219,194,161,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.158592226236983,0.160110639161382,0.170261294469309,0.163745888452553,0.177758333146862,0.186425573491135,0.183881233393209,0.0286047015802735,0.0218570282854601,0.023862167633462,0.236413042119348,0.247481621247705,0.16767803475769,0.00841881581629929,-0.0470798951749637,0.00841881581629928,0.178392175010259,0.213843145127143,0.227735268394744,0.0216526623288513,0.202807726274158,0.271357877013936,0.215897524876756,-0.0342563410377825,0.166927840062179,0.216273496237742,0.160897737585494,0.16115925783346,0.154647863941483,0.0191876051725983,-0.0586109005437825,0.149245687707566,-0.0508910457295981,0.229644832614767,0.014958201778049,0.184086547802416,0.224533513630705,-0.425030939057862,-0.064828450913945,0.050896120756131,-0.153250722405853,-0.115470869881169,-0.166750153599745,-0.128308981984767,0.173377661256302,0.199282648690192,0.0269405256972702,-0.481945080278832,-0.0773085946619676,0.208098912183598,0.187740020909931,-0.320335502893369,-0.061364207263526,-0.14466519169782,-0.154447640820593],"y":[0.367586964312835,0.291808279107351,0.277586829653793,0.275212156726682,0.313814501282227,0.29243485700168,0.292541208275053,0.244548839224938,0.229564102231073,0.238104242887746,0.302489033479901,0.24352798025439,0.294679132100946,0.0597157571389563,0.252595615599963,0.0597157571389563,0.394157636796072,0.286068664817101,0.235155727138065,0.204339646937935,0.289680659438046,0.156667787335543,0.23825723998651,0.302322300853579,0.303066379306822,0.309123910235849,0.301805165579822,0.279761211040242,0.277245777428527,0.0918110720330258,-0.332887194838055,0.370616921365316,0.268630521512541,0.161594554414608,0.0391155652518521,0.301069544853322,-0.229167579645516,0.13190590493142,-0.295723755948753,0.363254959449147,-0.163384950386175,-0.00237239114895536,-0.0414066246507102,-0.0157012543285653,-0.174443493578293,0.307795903368027,0.0427928442475453,0.157984683999683,-0.277557796213044,-0.0890752635535033,-0.183962607599382,-0.0714543402107255,-0.153511825459499,-0.0209937239687092,-0.0288066091612479],"type":"scatter","mode":"markers","text":["modules/clusters: C8 <br>GO.ID: GO:0010558 <br>GO.name: negative regulation of macromolecule biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0045892 <br>GO.name: negative regulation of DNA-templated transcription","modules/clusters: C8 <br>GO.ID: GO:1903507 <br>GO.name: negative regulation of nucleic acid-templated transcription","modules/clusters: C8 <br>GO.ID: GO:1902679 <br>GO.name: negative regulation of RNA biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0006355 <br>GO.name: regulation of DNA-templated transcription","modules/clusters: C8 <br>GO.ID: GO:1903506 <br>GO.name: regulation of nucleic acid-templated transcription","modules/clusters: C8 <br>GO.ID: GO:2001141 <br>GO.name: regulation of RNA biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0006351 <br>GO.name: DNA-templated transcription","modules/clusters: C8 <br>GO.ID: GO:0097659 <br>GO.name: nucleic acid-templated transcription","modules/clusters: C8 <br>GO.ID: GO:0032774 <br>GO.name: RNA biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0051252 <br>GO.name: regulation of RNA metabolic process","modules/clusters: C8 <br>GO.ID: GO:0019219 <br>GO.name: regulation of nucleobase-containing compound metabolic process","modules/clusters: C8 <br>GO.ID: GO:0000122 <br>GO.name: negative regulation of transcription by RNA polymerase II","modules/clusters: C8 <br>GO.ID: GO:0018130 <br>GO.name: heterocycle biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:1901362 <br>GO.name: organic cyclic compound biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0019438 <br>GO.name: aromatic compound biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0010556 <br>GO.name: regulation of macromolecule biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0051253 <br>GO.name: negative regulation of RNA metabolic process","modules/clusters: C8 <br>GO.ID: GO:0045934 <br>GO.name: negative regulation of nucleobase-containing compound metabolic process","modules/clusters: C8 <br>GO.ID: GO:0034654 <br>GO.name: nucleobase-containing compound biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0051254 <br>GO.name: positive regulation of RNA metabolic process","modules/clusters: C8 <br>GO.ID: GO:0031326 <br>GO.name: regulation of cellular biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0045935 <br>GO.name: positive regulation of nucleobase-containing compound metabolic process","modules/clusters: C8 <br>GO.ID: GO:0009059 <br>GO.name: macromolecule biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0045944 <br>GO.name: positive regulation of transcription by RNA polymerase II","modules/clusters: C8 <br>GO.ID: GO:0009889 <br>GO.name: regulation of biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0045893 <br>GO.name: positive regulation of DNA-templated transcription","modules/clusters: C8 <br>GO.ID: GO:1903508 <br>GO.name: positive regulation of nucleic acid-templated transcription","modules/clusters: C8 <br>GO.ID: GO:1902680 <br>GO.name: positive regulation of RNA biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0044271 <br>GO.name: cellular nitrogen compound biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0033554 <br>GO.name: cellular response to stress","modules/clusters: C8 <br>GO.ID: GO:0010557 <br>GO.name: positive regulation of macromolecule biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:1901576 <br>GO.name: organic substance biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0031328 <br>GO.name: positive regulation of cellular biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0044249 <br>GO.name: cellular biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0009891 <br>GO.name: positive regulation of biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:1902531 <br>GO.name: regulation of intracellular signal transduction","modules/clusters: C8 <br>GO.ID: GO:0001763 <br>GO.name: morphogenesis of a branching structure","modules/clusters: C8 <br>GO.ID: GO:0034976 <br>GO.name: response to endoplasmic reticulum stress","modules/clusters: C8 <br>GO.ID: GO:0032922 <br>GO.name: circadian regulation of gene expression","modules/clusters: C8 <br>GO.ID: GO:0033993 <br>GO.name: response to lipid","modules/clusters: C8 <br>GO.ID: GO:0006816 <br>GO.name: calcium ion transport","modules/clusters: C8 <br>GO.ID: GO:0048511 <br>GO.name: rhythmic process","modules/clusters: C8 <br>GO.ID: GO:0006811 <br>GO.name: ion transport","modules/clusters: C8 <br>GO.ID: GO:0045742 <br>GO.name: positive regulation of epidermal growth factor receptor signaling pathway","modules/clusters: C8 <br>GO.ID: GO:0051091 <br>GO.name: positive regulation of DNA-binding transcription factor activity","modules/clusters: C8 <br>GO.ID: GO:0042592 <br>GO.name: homeostatic process","modules/clusters: C8 <br>GO.ID: GO:0061458 <br>GO.name: reproductive system development","modules/clusters: C8 <br>GO.ID: GO:0071345 <br>GO.name: cellular response to cytokine stimulus","modules/clusters: C8 <br>GO.ID: GO:0022407 <br>GO.name: regulation of cell-cell adhesion","modules/clusters: C8 <br>GO.ID: GO:1901186 <br>GO.name: positive regulation of ERBB signaling pathway","modules/clusters: C8 <br>GO.ID: GO:0048468 <br>GO.name: cell development","modules/clusters: C8 <br>GO.ID: GO:0034220 <br>GO.name: ion transmembrane transport","modules/clusters: C8 <br>GO.ID: GO:0007623 <br>GO.name: circadian rhythm","modules/clusters: C8 <br>GO.ID: GO:0022414 <br>GO.name: reproductive process"],"showlegend":true,"marker":{"color":"rgba(227,210,187,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(227,210,187,1)"}},"name":"C8","textfont":{"color":"rgba(227,210,187,1)"},"error_y":{"color":"rgba(227,210,187,1)"},"error_x":{"color":"rgba(227,210,187,1)"},"line":{"color":"rgba(227,210,187,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.208297576354836,0.0539087443756402,-0.136801263408778,0.121943273515027,-0.150248636430112,-0.109367341724991,0.0355002532487073],"y":[-0.248737684664601,0.154110602437833,-0.0240588054600769,0.0864647935886551,-0.0378214415648419,0.152812685430551,0.138958643010011],"type":"scatter","mode":"markers","text":["modules/clusters: C9 <br>GO.ID: GO:0007188 <br>GO.name: adenylate cyclase-modulating G protein-coupled receptor signaling pathway","modules/clusters: C9 <br>GO.ID: GO:0044057 <br>GO.name: regulation of system process","modules/clusters: C9 <br>GO.ID: GO:0006810 <br>GO.name: transport","modules/clusters: C9 <br>GO.ID: GO:0051050 <br>GO.name: positive regulation of transport","modules/clusters: C9 <br>GO.ID: GO:0051234 <br>GO.name: establishment of localization","modules/clusters: C9 <br>GO.ID: GO:0022604 <br>GO.name: regulation of cell morphogenesis","modules/clusters: C9 <br>GO.ID: GO:0031644 <br>GO.name: regulation of nervous system process"],"showlegend":true,"marker":{"color":"rgba(252,205,229,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(252,205,229,1)"}},"name":"C9","textfont":{"color":"rgba(252,205,229,1)"},"error_y":{"color":"rgba(252,205,229,1)"},"error_x":{"color":"rgba(252,205,229,1)"},"line":{"color":"rgba(252,205,229,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>A second version of the MDS plot provides a nearest ancestor
clustering approach. All terms found are brought up to their nearest
common ancestor. Since these databases, GO ontology in particular, have
a tree-like hierarchy, we are able to find common ancestors for each
term which may provide a broader view if too many individual terms
exist. This approach does not distinguish based on clusters in which
terms were found, meaning that terms from different clusters may be
paired together. The size of each dot in this approach is relative to
the number of terms sorted in that common ancestor (dot) while the color
is representative of the dominant cluster in that common ancestor. For
example, if an ancestor has 4 terms within it, and 3 of those terms are
found in cluster 1 (C1), the dot will carry the color associated to C1.
Hovering the cursor over the dots will provide more information on those
specific common ancestors.</p>
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-7aa6e25aded1bc874647" style="width:100%;height:432.632880098888px;"></div>
<script type="application/json" data-for="htmlwidget-7aa6e25aded1bc874647">{"x":{"visdat":{"1e8d465d3cad8a":["function () ","plotlyVisDat"],"1e8d464e4e633":["function () ","data"]},"cur_data":"1e8d464e4e633","attrs":{"1e8d464e4e633":{"alpha_stroke":1,"sizes":[20,50],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","text":{},"showlegend":false,"size":{},"fill":{},"marker":{"sizemode":"diameter","opacity":0.4,"line":{"color":["#D2A6B7","#E3D2BB","#E69394","#D2A6B7","#D2A6B7","#A9A0B2","#E3D2BB","#D2A6B7","#E69394","#8DD3C7","#D3B387","#A9A0B2","#8DD3C7","#D3B387","#E69394","#A9A0B2","#D3B387","#8DD3C7","#8DD3C7","#8DD3C7","#A9A0B2","#D2A6B7","#D2A6B7","#D2A6B7","#8DD3C7","#E9E8C0","#8DD3C7","#A9A0B2","#D2A6B7","#D2A6B7","#8DD3C7","#E69394","#A9A0B2","#D2A6B7","#E69394","#8DD3C7","#8DD3C7","#8DD3C7","#E3D2BB","#E3D2BB","#E3D2BB","#A9A0B2","#A9A0B2","#A9A0B2","#D2A6B7","#E69394","#E69394","#8DD3C7","#E3D2BB","#E3D2BB","#A9A0B2","#8DD3C7","#8DD3C7","#E3D2BB","#8DD3C7","#E69394","#FCCDE5","#A9A0B2","#E69394","#E3D2BB","#D3B387","#8DD3C7","#E69394","#E69394"]},"color":["#D2A6B7","#E3D2BB","#E69394","#D2A6B7","#D2A6B7","#A9A0B2","#E3D2BB","#D2A6B7","#E69394","#8DD3C7","#D3B387","#A9A0B2","#8DD3C7","#D3B387","#E69394","#A9A0B2","#D3B387","#8DD3C7","#8DD3C7","#8DD3C7","#A9A0B2","#D2A6B7","#D2A6B7","#D2A6B7","#8DD3C7","#E9E8C0","#8DD3C7","#A9A0B2","#D2A6B7","#D2A6B7","#8DD3C7","#E69394","#A9A0B2","#D2A6B7","#E69394","#8DD3C7","#8DD3C7","#8DD3C7","#E3D2BB","#E3D2BB","#E3D2BB","#A9A0B2","#A9A0B2","#A9A0B2","#D2A6B7","#E69394","#E69394","#8DD3C7","#E3D2BB","#E3D2BB","#A9A0B2","#8DD3C7","#8DD3C7","#E3D2BB","#8DD3C7","#E69394","#FCCDE5","#A9A0B2","#E69394","#E3D2BB","#D3B387","#8DD3C7","#E69394","#E69394"]},"visible":true,"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"fillcolor":"rgba(31,119,180,0.5)","x":[-0.0165991328826609,0.2536961223896,0.108864683333148,-0.06192271026098,-0.0474887690667474,-0.327656980833151,-0.323677108442179,-0.0792371388421792,-0.0513128541193871,-0.0102396740992095,0.0601913224268149,0.274397733894178,-0.183356613951747,-0.260526817665869,-0.175433522469863,-0.287739928138353,-0.0749564330011612,0.120499688406878,-0.195037154653505,0.179989648073478,-0.310402869795596,-0.0329907634347017,-0.00819163119233769,0.0796815658364182,0.0460657667005286,-0.142495113711442,-0.352101856864699,-0.331647490906044,0.181746166180799,0.270619470405034,0.155338765120955,-0.175954614173782,0.0702255480786612,0.256627926762407,-0.284256119593218,0.285272567572211,-0.362420295599015,-0.342466722452905,0.273177054603945,0.261400583395294,0.151330835844698,-0.337887144360435,0.226387304407028,0.372101492956927,0.237182007418654,0.265772216962835,0.269870603359071,-0.139714324447471,0.231700940220301,0.340537819781241,0.130832077182579,0.0131722500424296,0.160649581342686,-0.20535712838252,-0.193984293258209,0.0022928535092485,0.0958192978077605,-0.222918633524889,0.167198856114401,0.0196607638630726,-0.300412184511915,-0.0190705891953217,0.163258509169651,0.131894590668559],"y":[-0.177349826548485,0.362625480560635,-0.248872706593625,-0.152553260289354,-0.18172448886501,0.0745354704249439,0.0789365383627734,-0.191457580414046,-0.165218799356753,0.323297109753571,-0.0260748427525244,0.265498677560604,0.00925997594394174,0.0693041321762435,-0.0929603134174912,-0.00874171188186168,0.261595385928679,-0.126316856318098,-0.0775341084534919,-0.194099000234875,0.0938632010221824,0.430027344577162,0.445115163435224,0.310479141648173,0.146364126050102,-0.0495853297234567,0.0133532923105313,0.0192375025997536,-0.201857811926732,-0.229626229684326,-0.181826586784009,-0.141517498193392,-0.179352045889142,-0.236617837382912,0.0834019171811244,-0.238865988603704,0.0156717449800752,0.0562029928572568,0.271774714135532,0.271903984779649,-0.224477117360138,0.0147514797468502,0.353226056433883,0.0955871797328373,-0.227235844488764,-0.227562893868696,-0.217536138100411,-0.0827329297208341,0.187109459189353,0.285765497160655,0.461755481741177,-0.0164153257164942,-0.0711167407755491,0.0161203181753237,-0.118170869712623,-0.0566890403585957,0.00982984259286984,-0.156299791006737,-0.0472511183209605,0.0108563890656734,-0.00165597906287694,-0.0794355343452791,-0.182597804295528,-0.226119649680004],"type":"scatter","mode":"markers","text":["GO.cluster: 48 <br>GO.count: 9  -  C16: 9 <br>GO.ID: GO:0022402 <br>GO.name: cell cycle process","GO.cluster: 19 <br>GO.count: 18  -  C8: 18 <br>GO.ID: GO:0034654 <br>GO.name: nucleobase-containing compound biosynthetic process","GO.cluster: 37 <br>GO.count: 15  -  C17: 4, C2: 4, C1: 3, C21: 1, C22: 1, C8: 1, C9: 1 <br>GO.ID: GO:0007165 <br>GO.name: signal transduction","GO.cluster: 47 <br>GO.count: 7  -  C16: 6, C21: 1 <br>GO.ID: GO:0016043 <br>GO.name: cellular component organization","GO.cluster: 64 <br>GO.count: 12  -  C16: 12 <br>GO.ID: GO:0009987 <br>GO.name: cellular process","GO.cluster: 4 <br>GO.count: 19  -  C2: 8, C1: 6, C17: 2, C8: 2, C22: 1 <br>GO.ID: GO:0048856 <br>GO.name: anatomical structure development","GO.cluster: 1 <br>GO.count: 3  -  C8: 2, C2: 1 <br>GO.ID: GO:0009653 <br>GO.name: anatomical structure morphogenesis","GO.cluster: 63 <br>GO.count: 6  -  C16: 2, C17: 2, C22: 1, C8: 1 <br>GO.ID: GO:0009987 <br>GO.name: cellular process","GO.cluster: 62 <br>GO.count: 8  -  C17: 3, C2: 3, C8: 2 <br>GO.ID: GO:0008219 <br>GO.name: cell death","GO.cluster: 17 <br>GO.count: 10  -  C1: 8, C20: 1, C8: 1 <br>GO.ID: GO:0010467 <br>GO.name: gene expression","GO.cluster: 56 <br>GO.count: 9  -  C22: 8, C17: 1 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","GO.cluster: 14 <br>GO.count: 8  -  C2: 8 <br>GO.ID: GO:0036211 <br>GO.name: protein modification process","GO.cluster: 54 <br>GO.count: 6  -  C1: 2, C22: 2, C17: 1, C20: 1 <br>GO.ID: GO:0006955 <br>GO.name: immune response","GO.cluster: 55 <br>GO.count: 4  -  C22: 3, C10: 1 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","GO.cluster: 52 <br>GO.count: 9  -  C17: 5, C2: 2, C22: 2 <br>GO.ID: GO:0001775 <br>GO.name: cell activation","GO.cluster: 8 <br>GO.count: 13  -  C2: 6, C17: 5, C22: 1, C8: 1 <br>GO.ID: GO:0002520 <br>GO.name: immune system development","GO.cluster: 53 <br>GO.count: 4  -  C22: 4 <br>GO.ID: GO:0002381 <br>GO.name: immunoglobulin production involved in immunoglobulin-mediated immune response","GO.cluster: 59 <br>GO.count: 6  -  C1: 6 <br>GO.ID: GO:2000145 <br>GO.name: regulation of cell motility","GO.cluster: 57 <br>GO.count: 6  -  C1: 4, C21: 2 <br>GO.ID: GO:0006935 <br>GO.name: chemotaxis","GO.cluster: 51 <br>GO.count: 5  -  C1: 2, C22: 2, C17: 1 <br>GO.ID: GO:0050865 <br>GO.name: regulation of cell activation","GO.cluster: 3 <br>GO.count: 9  -  C2: 9 <br>GO.ID: GO:0007507 <br>GO.name: heart development","GO.cluster: 22 <br>GO.count: 12  -  C16: 8, C8: 3, C1: 1 <br>GO.ID: GO:0008152 <br>GO.name: metabolic process","GO.cluster: 21 <br>GO.count: 8  -  C16: 8 <br>GO.ID: GO:0006629 <br>GO.name: lipid metabolic process","GO.cluster: 20 <br>GO.count: 10  -  C16: 10 <br>GO.ID: GO:0044255 <br>GO.name: cellular lipid metabolic process","GO.cluster: 12 <br>GO.count: 3  -  C1: 2, C2: 1 <br>GO.ID: GO:0006793 <br>GO.name: phosphorus metabolic process","GO.cluster: 60 <br>GO.count: 11  -  C13: 3, C8: 3, C19: 2, C9: 2, C1: 1 <br>GO.ID: GO:0051179 <br>GO.name: localization","GO.cluster: 42 <br>GO.count: 3  -  C1: 2, C20: 1 <br>GO.ID: GO:0009605 <br>GO.name: response to external stimulus","GO.cluster: 40 <br>GO.count: 4  -  C2: 2, C22: 2 <br>GO.ID: GO:0006950 <br>GO.name: response to stress","GO.cluster: 50 <br>GO.count: 15  -  C16: 15 <br>GO.ID: GO:0010564 <br>GO.name: regulation of cell cycle process","GO.cluster: 32 <br>GO.count: 6  -  C16: 4, C1: 1, C2: 1 <br>GO.ID: GO:0048523 <br>GO.name: negative regulation of cellular process","GO.cluster: 30 <br>GO.count: 9  -  C1: 5, C8: 2, C17: 1, C2: 1 <br>GO.ID: GO:0007155 <br>GO.name: cell adhesion","GO.cluster: 46 <br>GO.count: 8  -  C17: 4, C8: 2, C1: 1, C2: 1 <br>GO.ID: GO:0070887 <br>GO.name: cellular response to chemical stimulus","GO.cluster: 34 <br>GO.count: 7  -  C2: 7 <br>GO.ID: GO:0007267 <br>GO.name: cell-cell signaling","GO.cluster: 31 <br>GO.count: 11  -  C16: 11 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process","GO.cluster: 61 <br>GO.count: 9  -  C17: 4, C8: 3, C16: 1, C9: 1 <br>GO.ID: GO:0008150 <br>GO.name: biological <br>GO.name: process","GO.cluster: 33 <br>GO.count: 9  -  C1: 2, C17: 2, C8: 2, C10: 1, C16: 1, C22: 1 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process","GO.cluster: 43 <br>GO.count: 4  -  C1: 1, C17: 1, C2: 1, C22: 1 <br>GO.ID: GO:0050896 <br>GO.name: response to stimulus","GO.cluster: 2 <br>GO.count: 4  -  C1: 2, C20: 1, C9: 1 <br>GO.ID: GO:0032502 <br>GO.name: developmental process","GO.cluster: 10 <br>GO.count: 6  -  C8: 3, C1: 1, C18: 1, C2: 1 <br>GO.ID: GO:0019222 <br>GO.name: regulation of metabolic process","GO.cluster: 9 <br>GO.count: 4  -  C8: 2, C1: 1, C2: 1 <br>GO.ID: GO:0019222 <br>GO.name: regulation of metabolic process","GO.cluster: 39 <br>GO.count: 9  -  C8: 5, C2: 2, C1: 1, C17: 1 <br>GO.ID: GO:0009966 <br>GO.name: regulation of signal transduction","GO.cluster: 44 <br>GO.count: 9  -  C2: 4, C8: 2, C1: 1, C17: 1, C22: 1 <br>GO.ID: GO:0042221 <br>GO.name: response to chemical","GO.cluster: 16 <br>GO.count: 15  -  C2: 8, C8: 5, C1: 1, C22: 1 <br>GO.ID: GO:0060255 <br>GO.name: regulation of macromolecule metabolic process","GO.cluster: 13 <br>GO.count: 11  -  C2: 7, C17: 2, C1: 1, C8: 1 <br>GO.ID: GO:0031323 <br>GO.name: regulation of cellular metabolic process","GO.cluster: 49 <br>GO.count: 8  -  C16: 7, C10: 1 <br>GO.ID: GO:0051128 <br>GO.name: regulation of cellular component organization","GO.cluster: 29 <br>GO.count: 10  -  C17: 7, C1: 3 <br>GO.ID: GO:0010941 <br>GO.name: regulation of cell death","GO.cluster: 28 <br>GO.count: 4  -  C17: 3, C2: 1 <br>GO.ID: GO:0048522 <br>GO.name: positive regulation of cellular process","GO.cluster: 58 <br>GO.count: 4  -  C1: 3, C8: 1 <br>GO.ID: GO:0048870 <br>GO.name: cell motility","GO.cluster: 11 <br>GO.count: 7  -  C8: 7 <br>GO.ID: GO:0044249 <br>GO.name: cellular biosynthetic process","GO.cluster: 18 <br>GO.count: 8  -  C8: 6, C17: 2 <br>GO.ID: GO:0019219 <br>GO.name: regulation of nucleobase-containing compound metabolic process","GO.cluster: 15 <br>GO.count: 7  -  C2: 3, C1: 2, C17: 1, C8: 1 <br>GO.ID: GO:0071704 <br>GO.name: organic substance metabolic process","GO.cluster: 25 <br>GO.count: 8  -  C1: 3, C9: 2, C10: 1, C17: 1, C8: 1 <br>GO.ID: GO:0050793 <br>GO.name: regulation of developmental process","GO.cluster: 36 <br>GO.count: 3  -  C1: 1, C17: 1, C8: 1 <br>GO.ID: GO:0023051 <br>GO.name: regulation of signaling","GO.cluster: 35 <br>GO.count: 1  -  C8: 1 <br>GO.ID: GO:0023052 <br>GO.name: signaling","GO.cluster: 6 <br>GO.count: 4  -  C1: 3, C8: 1 <br>GO.ID: GO:0048869 <br>GO.name: cellular developmental process","GO.cluster: 24 <br>GO.count: 7  -  C17: 4, C1: 2, C8: 1 <br>GO.ID: GO:0048583 <br>GO.name: regulation of response to stimulus","GO.cluster: 26 <br>GO.count: 5  -  C9: 2, C1: 1, C17: 1, C8: 1 <br>GO.ID: GO:0051239 <br>GO.name: regulation of multicellular organismal process","GO.cluster: 45 <br>GO.count: 5  -  C2: 2, C8: 2, C1: 1 <br>GO.ID: GO:0051716 <br>GO.name: cellular response to stimulus","GO.cluster: 27 <br>GO.count: 6  -  C17: 4, C1: 1, C9: 1 <br>GO.ID: GO:0050789 <br>GO.name: regulation of biological process","GO.cluster: 23 <br>GO.count: 9  -  C8: 4, C17: 3, C2: 2 <br>GO.ID: GO:0065007 <br>GO.name: biological regulation","GO.cluster: 41 <br>GO.count: 4  -  C22: 2, C1: 1, C17: 1 <br>GO.ID: GO:0043207 <br>GO.name: response to external biotic stimulus","GO.cluster: 7 <br>GO.count: 9  -  C1: 4, C17: 2, C2: 2, C22: 1 <br>GO.ID: GO:1903706 <br>GO.name: regulation of hemopoiesis","GO.cluster: 5 <br>GO.count: 3  -  C17: 2, C1: 1 <br>GO.ID: GO:0045595 <br>GO.name: regulation of cell differentiation","GO.cluster: 38 <br>GO.count: 5  -  C17: 5 <br>GO.ID: GO:2001233 <br>GO.name: regulation of apoptotic signaling pathway"],"showlegend":false,"fill":"","marker":{"color":["#D2A6B7","#E3D2BB","#E69394","#D2A6B7","#D2A6B7","#A9A0B2","#E3D2BB","#D2A6B7","#E69394","#8DD3C7","#D3B387","#A9A0B2","#8DD3C7","#D3B387","#E69394","#A9A0B2","#D3B387","#8DD3C7","#8DD3C7","#8DD3C7","#A9A0B2","#D2A6B7","#D2A6B7","#D2A6B7","#8DD3C7","#E9E8C0","#8DD3C7","#A9A0B2","#D2A6B7","#D2A6B7","#8DD3C7","#E69394","#A9A0B2","#D2A6B7","#E69394","#8DD3C7","#8DD3C7","#8DD3C7","#E3D2BB","#E3D2BB","#E3D2BB","#A9A0B2","#A9A0B2","#A9A0B2","#D2A6B7","#E69394","#E69394","#8DD3C7","#E3D2BB","#E3D2BB","#A9A0B2","#8DD3C7","#8DD3C7","#E3D2BB","#8DD3C7","#E69394","#FCCDE5","#A9A0B2","#E69394","#E3D2BB","#D3B387","#8DD3C7","#E69394","#E69394"],"size":[33.3333333333333,48.3333333333333,43.3333333333333,30,38.3333333333333,50,23.3333333333333,28.3333333333333,31.6666666666667,35,33.3333333333333,31.6666666666667,28.3333333333333,25,33.3333333333333,40,25,28.3333333333333,28.3333333333333,26.6666666666667,33.3333333333333,38.3333333333333,31.6666666666667,35,23.3333333333333,36.6666666666667,23.3333333333333,25,43.3333333333333,28.3333333333333,33.3333333333333,31.6666666666667,30,36.6666666666667,33.3333333333333,33.3333333333333,25,25,28.3333333333333,25,33.3333333333333,33.3333333333333,43.3333333333333,36.6666666666667,31.6666666666667,35,25,25,30,31.6666666666667,30,31.6666666666667,23.3333333333333,20,25,30,26.6666666666667,26.6666666666667,28.3333333333333,33.3333333333333,25,33.3333333333333,23.3333333333333,26.6666666666667],"sizemode":"diameter","opacity":0.4,"line":{"color":["#D2A6B7","#E3D2BB","#E69394","#D2A6B7","#D2A6B7","#A9A0B2","#E3D2BB","#D2A6B7","#E69394","#8DD3C7","#D3B387","#A9A0B2","#8DD3C7","#D3B387","#E69394","#A9A0B2","#D3B387","#8DD3C7","#8DD3C7","#8DD3C7","#A9A0B2","#D2A6B7","#D2A6B7","#D2A6B7","#8DD3C7","#E9E8C0","#8DD3C7","#A9A0B2","#D2A6B7","#D2A6B7","#8DD3C7","#E69394","#A9A0B2","#D2A6B7","#E69394","#8DD3C7","#8DD3C7","#8DD3C7","#E3D2BB","#E3D2BB","#E3D2BB","#A9A0B2","#A9A0B2","#A9A0B2","#D2A6B7","#E69394","#E69394","#8DD3C7","#E3D2BB","#E3D2BB","#A9A0B2","#8DD3C7","#8DD3C7","#E3D2BB","#8DD3C7","#E69394","#FCCDE5","#A9A0B2","#E69394","#E3D2BB","#D3B387","#8DD3C7","#E69394","#E69394"]}},"visible":true,"textfont":{"size":[33.3333333333333,48.3333333333333,43.3333333333333,30,38.3333333333333,50,23.3333333333333,28.3333333333333,31.6666666666667,35,33.3333333333333,31.6666666666667,28.3333333333333,25,33.3333333333333,40,25,28.3333333333333,28.3333333333333,26.6666666666667,33.3333333333333,38.3333333333333,31.6666666666667,35,23.3333333333333,36.6666666666667,23.3333333333333,25,43.3333333333333,28.3333333333333,33.3333333333333,31.6666666666667,30,36.6666666666667,33.3333333333333,33.3333333333333,25,25,28.3333333333333,25,33.3333333333333,33.3333333333333,43.3333333333333,36.6666666666667,31.6666666666667,35,25,25,30,31.6666666666667,30,31.6666666666667,23.3333333333333,20,25,30,26.6666666666667,26.6666666666667,28.3333333333333,33.3333333333333,25,33.3333333333333,23.3333333333333,26.6666666666667]},"error_y":{"color":"rgba(31,119,180,1)","width":[]},"error_x":{"color":"rgba(31,119,180,1)","width":[]},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 id="gprofiler-results-custom">Gprofiler results – custom<a class="anchor" aria-label="anchor" href="#gprofiler-results-custom"></a>
</h3>
<p>Dotplots were designed to illustrate the top n term results for any
single ontology. In the below code chunk three different ontologies are
demonstrated, GO:BP (biological processes), REAC (REACTOME) <span class="citation">(Jassal et al. 2020)</span> and KEGG<span class="citation">(Kanehisa et al. 2017)</span>. The ‘target_top’
parameter defines the number of terms (from most significant to least)
per cluster that will be shown on the dotplot.</p>
<p>All terms shown are considered to be significant, the ‘top’ aspect
reflects significance based on the adjusted p value. The adjusted p
value is negatively log transformed in order to offer a better
visualization, with most significant being in red and less significant
(but still statistically significant) in blue.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">target_top</span><span class="op">=</span><span class="fl">10</span></span>
<span><span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'REAC'</span></span>
<span><span class="va">gpro_REAC_dotplot</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span>,file_loc<span class="op">=</span><span class="cn">NULL</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span>,return_plot <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<p>Illustrated below is the dotplot for the REAC results
<img src="TS_analysis_murine_files/figure-html/show_dotplot%20-1.png" width="100%"></p>
</div>
<div class="section level3">
<h3 id="ancestor-query-plots">Ancestor query plots<a class="anchor" aria-label="anchor" href="#ancestor-query-plots"></a>
</h3>
<p>Ancestor query plots were designed to query results relating to
specific GOs.</p>
<p>This plot searches for children of the queried term IDs. Pathways,
such as GO pathways are hierarchically organized, so a single ID will be
associated to other IDs. For example, the ‘immune system process
(GO:0002376)’ contains several ‘children’, each of these children is
likely to have children of their own and so on. If we use the ID for
‘immune system process’ this plot will find all children grand-children
great-grand children etc… of ‘immune system process’. It will then split
the children based on the PART cluster in which they were found, these
results are illustrated in dotplot format. This is accomplished via the
GO.db package<span class="citation">(Carlson 2021)</span></p>
<p>Several websites exist to search for GO ancestors of interest,
‘AMIGO2’ is one and ‘quickGO’ is another.</p>
<p>A word of caution with this type of plot. It will always link terms
to the most upstream ancestor, so if the query contains two term IDs
which are related to each other, the plot will illustrate all results as
belonging to the term ID which is the most upstream (oldest/ancestor).
Each ancestor queried for will be illustrated in different colors on the
dotplot.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#Dotplot for terms relating to specific ancestors</span></span>
<span><span class="co">#Check if analysis is required, if not, set to null or empty</span></span>
<span><span class="kw">if</span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="va">target_ancestors</span><span class="op">)</span><span class="op">==</span><span class="fl">0</span><span class="op">)</span><span class="op">{</span></span>
<span>  <span class="va">ancestor_plots</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span></span>
<span>  <span class="va">ancestor_plots</span><span class="op">[[</span><span class="st">'MDS'</span><span class="op">]</span><span class="op">]</span><span class="op">&lt;-</span><span class="cn">NULL</span></span>
<span>  <span class="va">GOs_ancestors_clust</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="cn">NULL</span><span class="op">)</span></span>
<span><span class="op">}</span><span class="kw">else</span><span class="op">{</span></span>
<span>  <span class="va">GOs_ancestors_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_relation_to_ancestors.html">find_relation_to_ancestors</a></span><span class="op">(</span><span class="va">target_ancestors</span>,<span class="va">GO_clusters</span>,ontology <span class="op">=</span> <span class="va">ancestor_ontology</span><span class="op">)</span></span>
<span>  <span class="va">ancestor_plots</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/wrapper_ancestor_curation_plots.html">wrapper_ancestor_curation_plots</a></span><span class="op">(</span><span class="va">GOs_ancestors_clust</span>,<span class="va">sem_dta</span>,return_plot<span class="op">=</span><span class="cn">TRUE</span>,target_dir<span class="op">=</span><span class="cn">NULL</span><span class="op">)</span></span>
<span><span class="op">}</span></span></code></pre></div>
<p>Two formats are generated for the ancestor plot. The first is a MDS
plot. The second is a dotplot version. If more than one term was found
to be associated to ancestors, the MDS plot will be created and can be
seen below, otherwise only the dotplot will be created. The dotplot will
not be created if no terms were found. The files for both of these plots
can be found in ‘TS_results/ancestor_plots’. In the MDS and dotplot
alike, the color of each term represents the ancestor to which they have
been associated and the size is proportional to the -log transformed
adjusted p-value. In the MDS plot, the PART clusters in which these
terms are found can be seen by hovering over each dot, in the dotplot
version, this information is made available on the x axis while the y
axis represents the terms.</p>
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-8df1198cf9e000bf6952" style="width:700px;height:432.632880098888px;"></div>
<script type="application/json" data-for="htmlwidget-8df1198cf9e000bf6952">{"x":{"visdat":{"1e8d4623ade9ed":["function () ","plotlyVisDat"],"1e8d467f0b25":["function () ","data"]},"cur_data":"1e8d467f0b25","attrs":{"1e8d467f0b25":{"alpha_stroke":1,"sizes":[20,50],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","color":{},"text":{},"showlegend":true,"colors":["#A6CEE3","#FDBF6F","#FB9A99","#E31A1C","#33A02C","#1F78B4","#B2DF8A"],"size":{},"fill":{},"marker":{"sizemode":"diameter","opacity":0.4,"showlegend":true},"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":true},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"fillcolor":"rgba(166,206,227,0.5)","x":[-0.143360387430951,-0.218098560909421,-0.19667104541351,-0.19667104541351,-0.18948444230507,-0.104288583388616,-0.104288583388616,-0.142690867345511,-0.296240113416668,-0.328428940086497,-0.327892502772432,-0.233133170750654,-0.24624012652644,-0.285114665546447,-0.296806164506272,-0.309299435110811,-0.286694506367264,-0.315645697202036,-0.291801502705956,-0.290635718181557,-0.261795609296485,-0.339632370304831,-0.300074356875768,0.0569913143022759,-0.367944739411267,-0.330574182363325],"y":[-0.0799079865003289,0.0885427646187825,-0.0663881688267762,-0.0663881688267762,-0.200761504566628,-0.36103829902667,-0.36103829902667,-0.293642929846876,0.167148866652505,0.123766375462018,0.129589331183648,-0.0120678380777358,0.0225425423566559,0.121388974442393,0.193023286423717,0.149354846226635,0.183513129437038,0.165850482672666,0.195759397847334,0.241252324340905,0.258369971312039,0.275928112339952,0.156316745697152,0.0851610981525464,0.237637986194845,0.245526219108928],"type":"scatter","mode":"markers","text":["Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0001909 <br>GO.name: leukocyte mediated cytotoxicity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0001910 <br>GO.name: regulation of leukocyte mediated cytotoxicity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C17/C22 <br>GO.ID: GO:0002252 <br>GO.name: immune effector process <br>-log10(padj): 0.56414846995825","Ancestor: immune effector process <br>cluster(s): C17/C22 <br>GO.ID: GO:0002252 <br>GO.name: immune effector process <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C17 <br>GO.ID: GO:0002263 <br>GO.name: cell activation involved in immune response <br>-log10(padj): 0.419838008647064","Ancestor: immune effector process <br>cluster(s): C22/C17 <br>GO.ID: GO:0002285 <br>GO.name: lymphocyte activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22/C17 <br>GO.ID: GO:0002285 <br>GO.name: lymphocyte activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C17 <br>GO.ID: GO:0002366 <br>GO.name: leukocyte activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C1 <br>GO.ID: GO:0002369 <br>GO.name: T cell cytokine production <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002381 <br>GO.name: immunoglobulin production involved in immunoglobulin-mediated immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002426 <br>GO.name: immunoglobulin production in mucosal tissue <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002443 <br>GO.name: leukocyte mediated immunity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002449 <br>GO.name: lymphocyte mediated immunity <br>-log10(padj): 0.733507106613947","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002697 <br>GO.name: regulation of immune effector process <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C1 <br>GO.ID: GO:0002700 <br>GO.name: regulation of production of molecular mediator of immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002703 <br>GO.name: regulation of leukocyte mediated immunity <br>-log10(padj): 0.18456802374418","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002705 <br>GO.name: positive regulation of leukocyte mediated immunity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002706 <br>GO.name: regulation of lymphocyte mediated immunity <br>-log10(padj): 1.27864627357142","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002708 <br>GO.name: positive regulation of lymphocyte mediated immunity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C1 <br>GO.ID: GO:0002718 <br>GO.name: regulation of cytokine production involved in immune response <br>-log10(padj): 0.758181906669941","Ancestor: immune effector process <br>cluster(s): C1 <br>GO.ID: GO:0002720 <br>GO.name: positive regulation of cytokine production involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C1 <br>GO.ID: GO:0002724 <br>GO.name: regulation of T cell cytokine production <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C1 <br>GO.ID: GO:0035743 <br>GO.name: CD4-positive, alpha-beta T cell cytokine production <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C1 <br>GO.ID: GO:2000319 <br>GO.name: regulation of T-helper 17 cell differentiation <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:2000557 <br>GO.name: regulation of immunoglobulin production in mucosal tissue <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:2000558 <br>GO.name: positive regulation of immunoglobulin production in mucosal tissue <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(166,206,227,1)","size":[20,20,23.3785685216955,20,22.5143230120455,20,20,20,20,20,20,20,24.3928223735663,20,20,21.10533972587,20,27.6575481106797,20,24.5405946483976,20,20,20,20,20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(166,206,227,1)"}},"name":"immune effector process","textfont":{"color":"rgba(166,206,227,1)","size":[20,20,23.3785685216955,20,22.5143230120455,20,20,20,20,20,20,20,24.3928223735663,20,20,21.10533972587,20,27.6575481106797,20,24.5405946483976,20,20,20,20,20,20]},"error_y":{"color":"rgba(166,206,227,1)","width":[]},"error_x":{"color":"rgba(166,206,227,1)","width":[]},"line":{"color":"rgba(166,206,227,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(253,191,111,0.5)","x":[-0.27381247239614,-0.27381247239614,-0.282076889911385,-0.249557795659147,-0.249557795659147,-0.249557795659147],"y":[-0.000712214270234198,-0.000712214270234198,0.0263088940337967,-0.0386666556286132,-0.0386666556286132,-0.0386666556286132],"type":"scatter","mode":"markers","text":["Ancestor: immune response <br>cluster(s): C17/C22 <br>GO.ID: GO:0002250 <br>GO.name: adaptive immune response <br>-log10(padj): 0","Ancestor: immune response <br>cluster(s): C17/C22 <br>GO.ID: GO:0002250 <br>GO.name: adaptive immune response <br>-log10(padj): 0","Ancestor: immune response <br>cluster(s): C22 <br>GO.ID: GO:0002460 <br>GO.name: adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains <br>-log10(padj): 0","Ancestor: immune response <br>cluster(s): C22/C20/C17 <br>GO.ID: GO:0006955 <br>GO.name: immune response <br>-log10(padj): 0.0836850621926858","Ancestor: immune response <br>cluster(s): C22/C20/C17 <br>GO.ID: GO:0006955 <br>GO.name: immune response <br>-log10(padj): 0","Ancestor: immune response <br>cluster(s): C22/C20/C17 <br>GO.ID: GO:0006955 <br>GO.name: immune response <br>-log10(padj): 2.55235115668381"],"showlegend":true,"fill":"","marker":{"color":"rgba(253,191,111,1)","size":[20,20,20,20.5011725315523,20,35.2855032557709],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(253,191,111,1)"}},"name":"immune response","textfont":{"color":"rgba(253,191,111,1)","size":[20,20,20,20.5011725315523,20,35.2855032557709]},"error_y":{"color":"rgba(253,191,111,1)","width":[]},"error_x":{"color":"rgba(253,191,111,1)","width":[]},"line":{"color":"rgba(253,191,111,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(251,154,153,0.5)","x":[0.419097849887924,0.411035446668675,0.28740130029029,0.28740130029029,0.469290635687052,0.469290635687052,0.469290635687052,0.439765084495395,0.405386730999233,0.405386730999233,0.441071386103216,0.441071386103216,0.437401074509687,0.427116420124821,0.432161366495825,0.415998115660925,0.331955877141394,0.359235966536287,0.324778361517483,0.29620819061685,0.328546571902316,0.379794522293055,0.379794522293055,0.337959090261738,0.306124026252488,0.306124026252488,0.468698230720142,0.468698230720142,0.241794717795405,0.241794717795405],"y":[-0.111703234097375,-0.106706870660515,0.148930561145468,0.148930561145468,0.172264388506502,0.172264388506502,0.172264388506502,0.185316772146004,0.222005590462064,0.222005590462064,-0.134596150269385,-0.134596150269385,0.169360049710618,-0.116270901165826,-0.170868054141804,-0.164536524030555,-0.0117982491981211,0.0460733951304473,0.280020587272439,0.264971648266599,-0.0140377905842139,0.205778180584049,0.205778180584049,0.283399925371628,0.300015943442097,0.300015943442097,0.168571402923125,0.168571402923125,0.348383159283699,0.348383159283699],"type":"scatter","mode":"markers","text":["Ancestor: immune system development <br>cluster(s): C2 <br>GO.ID: GO:0002327 <br>GO.name: immature B cell differentiation <br>-log10(padj): 1.11188144638402","Ancestor: immune system development <br>cluster(s): C2 <br>GO.ID: GO:0002329 <br>GO.name: pre-B cell differentiation <br>-log10(padj): 2.0644413865074","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:0002520 <br>GO.name: immune system development <br>-log10(padj): 5.00935647451489","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:0002520 <br>GO.name: immune system development <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C17/C8/C2 <br>GO.ID: GO:0002521 <br>GO.name: leukocyte differentiation <br>-log10(padj): 3.23598555462033","Ancestor: immune system development <br>cluster(s): C17/C8/C2 <br>GO.ID: GO:0002521 <br>GO.name: leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C17/C8/C2 <br>GO.ID: GO:0002521 <br>GO.name: leukocyte differentiation <br>-log10(padj): 1.96467038873318","Ancestor: immune system development <br>cluster(s): C17 <br>GO.ID: GO:0002573 <br>GO.name: myeloid leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:0030097 <br>GO.name: hemopoiesis <br>-log10(padj): 2.2511759022974","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:0030097 <br>GO.name: hemopoiesis <br>-log10(padj): 0.576860332268869","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:0030098 <br>GO.name: lymphocyte differentiation <br>-log10(padj): 0.264309711350752","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:0030098 <br>GO.name: lymphocyte differentiation <br>-log10(padj): 1.52058937556621","Ancestor: immune system development <br>cluster(s): C17 <br>GO.ID: GO:0030099 <br>GO.name: myeloid cell differentiation <br>-log10(padj): 0.289171429242216","Ancestor: immune system development <br>cluster(s): C2 <br>GO.ID: GO:0030183 <br>GO.name: B cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C17 <br>GO.ID: GO:0030217 <br>GO.name: T cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C22 <br>GO.ID: GO:0042492 <br>GO.name: gamma-delta T cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C22 <br>GO.ID: GO:0045586 <br>GO.name: regulation of gamma-delta T cell differentiation <br>-log10(padj): 0.170783128307236","Ancestor: immune system development <br>cluster(s): C1 <br>GO.ID: GO:0045619 <br>GO.name: regulation of lymphocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C17 <br>GO.ID: GO:0045655 <br>GO.name: regulation of monocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C17 <br>GO.ID: GO:0045656 <br>GO.name: negative regulation of monocyte differentiation <br>-log10(padj): 0.181780804860502","Ancestor: immune system development <br>cluster(s): C2 <br>GO.ID: GO:0046637 <br>GO.name: regulation of alpha-beta T cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:0048534 <br>GO.name: hematopoietic or lymphoid organ development <br>-log10(padj): 4.56204932315134","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:0048534 <br>GO.name: hematopoietic or lymphoid organ development <br>-log10(padj): 0.308490214909116","Ancestor: immune system development <br>cluster(s): C2 <br>GO.ID: GO:1902105 <br>GO.name: regulation of leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C2 <br>GO.ID: GO:1902107 <br>GO.name: positive regulation of leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C2 <br>GO.ID: GO:1902107 <br>GO.name: positive regulation of leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:1903131 <br>GO.name: mononuclear cell differentiation <br>-log10(padj): 2.60324612492202","Ancestor: immune system development <br>cluster(s): C17/C2 <br>GO.ID: GO:1903131 <br>GO.name: mononuclear cell differentiation <br>-log10(padj): 1.79545457880392","Ancestor: immune system development <br>cluster(s): C1/C2 <br>GO.ID: GO:1903708 <br>GO.name: positive regulation of hemopoiesis <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C2 <br>GO.ID: GO:1903708 <br>GO.name: positive regulation of hemopoiesis <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(251,154,153,1)","size":[26.6588280473194,32.3635125410434,50,20,39.3796482906541,20,31.7660046678357,20,33.4818269397492,23.4546972362836,21.5828962025088,29.1064953151301,21.7317878896024,20,20,20,21.0227848377896,20,20,21.0886476483675,20,47.3211699727945,21.8474841018715,20,20,20,35.590302695562,30.7526061756932,20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(251,154,153,1)"}},"name":"immune system development","textfont":{"color":"rgba(251,154,153,1)","size":[26.6588280473194,32.3635125410434,50,20,39.3796482906541,20,31.7660046678357,20,33.4818269397492,23.4546972362836,21.5828962025088,29.1064953151301,21.7317878896024,20,20,20,21.0227848377896,20,20,21.0886476483675,20,47.3211699727945,21.8474841018715,20,20,20,35.590302695562,30.7526061756932,20,20]},"error_y":{"color":"rgba(251,154,153,1)","width":[]},"error_x":{"color":"rgba(251,154,153,1)","width":[]},"line":{"color":"rgba(251,154,153,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(227,26,28,0.5)","x":[0.025377473876904,0.0510136830151034,0.0510136830151034,0.0253331465157601,0.0253331465157601,0.0253331465157601,0.0428876299153232,0.0386164764105148,0.0519197697361397,0.0519197697361397,0.0519197697361397],"y":[-0.458681335782862,-0.494815748126015,-0.494815748126015,-0.404901604614496,-0.404901604614496,-0.404901604614496,-0.479980888007823,-0.482703661603288,-0.469933543095842,-0.469933543095842,-0.469933543095842],"type":"scatter","mode":"markers","text":["Ancestor: leukocyte activation <br>cluster(s): C17 <br>GO.ID: GO:0035710 <br>GO.name: CD4-positive, alpha-beta T cell activation <br>-log10(padj): 0","Ancestor: leukocyte activation <br>cluster(s): C22/C17 <br>GO.ID: GO:0042110 <br>GO.name: T cell activation <br>-log10(padj): 0","Ancestor: leukocyte activation <br>cluster(s): C22/C17 <br>GO.ID: GO:0042110 <br>GO.name: T cell activation <br>-log10(padj): 1.09266131887098","Ancestor: leukocyte activation <br>cluster(s): C17/C2/C22 <br>GO.ID: GO:0045321 <br>GO.name: leukocyte activation <br>-log10(padj): 3.42759892209512","Ancestor: leukocyte activation <br>cluster(s): C17/C2/C22 <br>GO.ID: GO:0045321 <br>GO.name: leukocyte activation <br>-log10(padj): 1.30175650737806","Ancestor: leukocyte activation <br>cluster(s): C17/C2/C22 <br>GO.ID: GO:0045321 <br>GO.name: leukocyte activation <br>-log10(padj): 0","Ancestor: leukocyte activation <br>cluster(s): C22 <br>GO.ID: GO:0046629 <br>GO.name: gamma-delta T cell activation <br>-log10(padj): 0","Ancestor: leukocyte activation <br>cluster(s): C17 <br>GO.ID: GO:0046631 <br>GO.name: alpha-beta T cell activation <br>-log10(padj): 0","Ancestor: leukocyte activation <br>cluster(s): C2/C17/C22 <br>GO.ID: GO:0046649 <br>GO.name: lymphocyte activation <br>-log10(padj): 1.18074058825601","Ancestor: leukocyte activation <br>cluster(s): C2/C17/C22 <br>GO.ID: GO:0046649 <br>GO.name: lymphocyte activation <br>-log10(padj): 1.22428476986445","Ancestor: leukocyte activation <br>cluster(s): C2/C17/C22 <br>GO.ID: GO:0046649 <br>GO.name: lymphocyte activation <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(227,26,28,1)","size":[20,20,26.5437226783314,40.5271811231624,27.795950521793,20,20,20,27.0712112080446,27.3319883068394,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(227,26,28,1)"}},"name":"leukocyte activation","textfont":{"color":"rgba(227,26,28,1)","size":[20,20,26.5437226783314,40.5271811231624,27.795950521793,20,20,20,27.0712112080446,27.3319883068394,20]},"error_y":{"color":"rgba(227,26,28,1)","width":[]},"error_x":{"color":"rgba(227,26,28,1)","width":[]},"line":{"color":"rgba(227,26,28,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(51,160,44,0.5)","x":[-0.108620483496455,-0.0824861115986901,-0.0876628915557497],"y":[-0.0320915069732988,-0.0812405070847248,-0.0727393977579903],"type":"scatter","mode":"markers","text":["Ancestor: leukocyte migration <br>cluster(s): C1 <br>GO.ID: GO:0030595 <br>GO.name: leukocyte chemotaxis <br>-log10(padj): 0.0153983427445967","Ancestor: leukocyte migration <br>cluster(s): C1 <br>GO.ID: GO:0050900 <br>GO.name: leukocyte migration <br>-log10(padj): 0","Ancestor: leukocyte migration <br>cluster(s): C1 <br>GO.ID: GO:0071674 <br>GO.name: mononuclear cell migration <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(51,160,44,1)","size":[20.0922174903479,20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(51,160,44,1)"}},"name":"leukocyte migration","textfont":{"color":"rgba(51,160,44,1)","size":[20.0922174903479,20,20]},"error_y":{"color":"rgba(51,160,44,1)","width":[]},"error_x":{"color":"rgba(51,160,44,1)","width":[]},"line":{"color":"rgba(51,160,44,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(31,120,180,0.5)","x":[-0.214604951116338],"y":[0.122989714209377],"type":"scatter","mode":"markers","text":"Ancestor: production of molecular mediator of immune response <br>cluster(s): C1 <br>GO.ID: GO:0002367 <br>GO.name: cytokine production involved in immune response <br>-log10(padj): 0.758181906669941","showlegend":true,"fill":"","marker":{"color":"rgba(31,120,180,1)","size":[24.5405946483976],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(31,120,180,1)"}},"name":"production of molecular mediator of immune response","textfont":{"color":"rgba(31,120,180,1)","size":24.5405946483976},"error_y":{"color":"rgba(31,120,180,1)","width":24.5405946483976},"error_x":{"color":"rgba(31,120,180,1)","width":24.5405946483976},"line":{"color":"rgba(31,120,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(178,223,138,0.5)","x":[-0.21319594668778,-0.21319594668778,-0.21319594668778,-0.19863510266397,-0.19863510266397,-0.143901146569509,-0.150350612118865,-0.0750910419219245,-0.0750910419219245,-0.215457543026448,-0.333522036347927,-0.333522036347927,-0.336810461868404,-0.0359370783042782,-0.0323511364256117,-0.319679479841038,-0.0321651049386968,-0.0295802134892707,-0.158244324468257,-0.143513881629445],"y":[0.0861759800747439,0.0861759800747439,0.0861759800747439,0.124722408879029,0.124722408879029,0.111499535081598,0.113393099172525,-0.193378604414151,-0.193378604414151,0.0629809151025784,0.147138935886244,0.147138935886244,0.157995322379528,-0.318581214626918,-0.314368892383003,0.132574926433465,-0.31852659606114,-0.196111359930553,0.0709967303349871,0.104723582894259],"type":"scatter","mode":"markers","text":["Ancestor: regulation of immune system process <br>cluster(s): C1/C17/C22 <br>GO.ID: GO:0002682 <br>GO.name: regulation of immune system process <br>-log10(padj): 0.0836850621926858","Ancestor: regulation of immune system process <br>cluster(s): C1/C17/C22 <br>GO.ID: GO:0002682 <br>GO.name: regulation of immune system process <br>-log10(padj): 2.39979107497312","Ancestor: regulation of immune system process <br>cluster(s): C1/C17/C22 <br>GO.ID: GO:0002682 <br>GO.name: regulation of immune system process <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1/C17 <br>GO.ID: GO:0002684 <br>GO.name: positive regulation of immune system process <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1/C17 <br>GO.ID: GO:0002684 <br>GO.name: positive regulation of immune system process <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1 <br>GO.ID: GO:0002687 <br>GO.name: positive regulation of leukocyte migration <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1 <br>GO.ID: GO:0002690 <br>GO.name: positive regulation of leukocyte chemotaxis <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C17/C1 <br>GO.ID: GO:0002694 <br>GO.name: regulation of leukocyte activation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C17/C1 <br>GO.ID: GO:0002694 <br>GO.name: regulation of leukocyte activation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C17 <br>GO.ID: GO:0002764 <br>GO.name: immune response-regulating signaling pathway <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C22/C1 <br>GO.ID: GO:0002819 <br>GO.name: regulation of adaptive immune response <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C22/C1 <br>GO.ID: GO:0002819 <br>GO.name: regulation of adaptive immune response <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1 <br>GO.ID: GO:0002822 <br>GO.name: regulation of adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1 <br>GO.ID: GO:0046634 <br>GO.name: regulation of alpha-beta T cell activation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C22 <br>GO.ID: GO:0046643 <br>GO.name: regulation of gamma-delta T cell activation <br>-log10(padj): 0.170783128307236","Ancestor: regulation of immune system process <br>cluster(s): C17 <br>GO.ID: GO:0050776 <br>GO.name: regulation of immune response <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C22 <br>GO.ID: GO:0050863 <br>GO.name: regulation of T cell activation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1 <br>GO.ID: GO:0050870 <br>GO.name: positive regulation of T cell activation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1 <br>GO.ID: GO:0071675 <br>GO.name: regulation of mononuclear cell migration <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C1 <br>GO.ID: GO:0071677 <br>GO.name: positive regulation of mononuclear cell migration <br>-log10(padj): 0.373777484991828"],"showlegend":true,"fill":"","marker":{"color":"rgba(178,223,138,1)","size":[20.5011725315523,34.3718524755549,20,20,20,20,20,20,20,20,20,20,20,20,21.0227848377896,20,20,20,20,22.2384760611074],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(178,223,138,1)"}},"name":"regulation of immune system process","textfont":{"color":"rgba(178,223,138,1)","size":[20.5011725315523,34.3718524755549,20,20,20,20,20,20,20,20,20,20,20,20,21.0227848377896,20,20,20,20,22.2384760611074]},"error_y":{"color":"rgba(178,223,138,1)","width":[]},"error_x":{"color":"rgba(178,223,138,1)","width":[]},"line":{"color":"rgba(178,223,138,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 class="tabset unlisted unnumbered" id="section"></h3>
<p>A interactive table is also generated in order to better navigate the
results in the event of a large number of terms found for many different
ancestors. The csv version of this data can be found in
‘TS_results/ancestor_plots’.</p>
<div class="section level4">
<h4 id="go0002252">GO:0002252<a class="anchor" aria-label="anchor" href="#go0002252"></a>
</h4>
<p>Ancestor name: immune effector process</p>
<table class="table">
<colgroup>
<col width="11%">
<col width="75%">
<col width="12%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0001909</td>
<td align="left">leukocyte mediated cytotoxicity</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0001910</td>
<td align="left">regulation of leukocyte mediated cytotoxicity</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002252</td>
<td align="left">immune effector process</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0002252</td>
<td align="left">immune effector process</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002263</td>
<td align="left">cell activation involved in immune response</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0002285</td>
<td align="left">lymphocyte activation involved in immune response</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002285</td>
<td align="left">lymphocyte activation involved in immune response</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0002366</td>
<td align="left">leukocyte activation involved in immune response</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0002369</td>
<td align="left">T cell cytokine production</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0002381</td>
<td align="left">immunoglobulin production involved in
immunoglobulin-mediated immune response</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002426</td>
<td align="left">immunoglobulin production in mucosal tissue</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002443</td>
<td align="left">leukocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002449</td>
<td align="left">lymphocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002697</td>
<td align="left">regulation of immune effector process</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002700</td>
<td align="left">regulation of production of molecular mediator of
immune response</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0002703</td>
<td align="left">regulation of leukocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002705</td>
<td align="left">positive regulation of leukocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002706</td>
<td align="left">regulation of lymphocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002708</td>
<td align="left">positive regulation of lymphocyte mediated
immunity</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002718</td>
<td align="left">regulation of cytokine production involved in immune
response</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0002720</td>
<td align="left">positive regulation of cytokine production involved in
immune response</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0002724</td>
<td align="left">regulation of T cell cytokine production</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0035743</td>
<td align="left">CD4-positive, alpha-beta T cell cytokine
production</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:2000319</td>
<td align="left">regulation of T-helper 17 cell differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:2000557</td>
<td align="left">regulation of immunoglobulin production in mucosal
tissue</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:2000558</td>
<td align="left">positive regulation of immunoglobulin production in
mucosal tissue</td>
<td align="left">C22</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002440">GO:0002440<a class="anchor" aria-label="anchor" href="#go0002440"></a>
</h4>
<p>Ancestor name: production of molecular mediator of immune
response</p>
<table class="table">
<colgroup>
<col width="16%">
<col width="65%">
<col width="17%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody><tr class="odd">
<td align="left">GO:0002367</td>
<td align="left">cytokine production involved in immune response</td>
<td align="left">C1</td>
</tr></tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002520">GO:0002520<a class="anchor" aria-label="anchor" href="#go0002520"></a>
</h4>
<p>Ancestor name: immune system development</p>
<table class="table">
<colgroup>
<col width="16%">
<col width="66%">
<col width="17%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002327</td>
<td align="left">immature B cell differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0002329</td>
<td align="left">pre-B cell differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0002520</td>
<td align="left">immune system development</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0002520</td>
<td align="left">immune system development</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0002521</td>
<td align="left">leukocyte differentiation</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0002521</td>
<td align="left">leukocyte differentiation</td>
<td align="left">C8</td>
</tr>
<tr class="odd">
<td align="left">GO:0002521</td>
<td align="left">leukocyte differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0002573</td>
<td align="left">myeloid leukocyte differentiation</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0030097</td>
<td align="left">hemopoiesis</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0030097</td>
<td align="left">hemopoiesis</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0030098</td>
<td align="left">lymphocyte differentiation</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0030098</td>
<td align="left">lymphocyte differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0030099</td>
<td align="left">myeloid cell differentiation</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0030183</td>
<td align="left">B cell differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0030217</td>
<td align="left">T cell differentiation</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0042492</td>
<td align="left">gamma-delta T cell differentiation</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0045586</td>
<td align="left">regulation of gamma-delta T cell differentiation</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0045619</td>
<td align="left">regulation of lymphocyte differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0045655</td>
<td align="left">regulation of monocyte differentiation</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0045656</td>
<td align="left">negative regulation of monocyte differentiation</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0046637</td>
<td align="left">regulation of alpha-beta T cell differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0048534</td>
<td align="left">hematopoietic or lymphoid organ development</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0048534</td>
<td align="left">hematopoietic or lymphoid organ development</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:1902105</td>
<td align="left">regulation of leukocyte differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:1902107</td>
<td align="left">positive regulation of leukocyte differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:1902107</td>
<td align="left">positive regulation of leukocyte differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:1903131</td>
<td align="left">mononuclear cell differentiation</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:1903131</td>
<td align="left">mononuclear cell differentiation</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:1903708</td>
<td align="left">positive regulation of hemopoiesis</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:1903708</td>
<td align="left">positive regulation of hemopoiesis</td>
<td align="left">C2</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002682">GO:0002682<a class="anchor" aria-label="anchor" href="#go0002682"></a>
</h4>
<p>Ancestor name: regulation of immune system process</p>
<table class="table">
<colgroup>
<col width="7%">
<col width="84%">
<col width="8%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002682</td>
<td align="left">regulation of immune system process</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0002682</td>
<td align="left">regulation of immune system process</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0002682</td>
<td align="left">regulation of immune system process</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002684</td>
<td align="left">positive regulation of immune system process</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0002684</td>
<td align="left">positive regulation of immune system process</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0002687</td>
<td align="left">positive regulation of leukocyte migration</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0002690</td>
<td align="left">positive regulation of leukocyte chemotaxis</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0002694</td>
<td align="left">regulation of leukocyte activation</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0002694</td>
<td align="left">regulation of leukocyte activation</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0002764</td>
<td align="left">immune response-regulating signaling pathway</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0002819</td>
<td align="left">regulation of adaptive immune response</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002819</td>
<td align="left">regulation of adaptive immune response</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0002822</td>
<td align="left">regulation of adaptive immune response based on somatic
recombination of immune receptors built from immunoglobulin superfamily
domains</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0046634</td>
<td align="left">regulation of alpha-beta T cell activation</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0046643</td>
<td align="left">regulation of gamma-delta T cell activation</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0050776</td>
<td align="left">regulation of immune response</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0050863</td>
<td align="left">regulation of T cell activation</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0050870</td>
<td align="left">positive regulation of T cell activation</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0071675</td>
<td align="left">regulation of mononuclear cell migration</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0071677</td>
<td align="left">positive regulation of mononuclear cell migration</td>
<td align="left">C1</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0006955">GO:0006955<a class="anchor" aria-label="anchor" href="#go0006955"></a>
</h4>
<p>Ancestor name: immune response</p>
<table class="table">
<colgroup>
<col width="8%">
<col width="82%">
<col width="8%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002250</td>
<td align="left">adaptive immune response</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0002250</td>
<td align="left">adaptive immune response</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002460</td>
<td align="left">adaptive immune response based on somatic recombination
of immune receptors built from immunoglobulin superfamily domains</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0006955</td>
<td align="left">immune response</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0006955</td>
<td align="left">immune response</td>
<td align="left">C20</td>
</tr>
<tr class="even">
<td align="left">GO:0006955</td>
<td align="left">immune response</td>
<td align="left">C17</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0045321">GO:0045321<a class="anchor" aria-label="anchor" href="#go0045321"></a>
</h4>
<p>Ancestor name: leukocyte activation</p>
<table class="table">
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0035710</td>
<td align="left">CD4-positive, alpha-beta T cell activation</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0042110</td>
<td align="left">T cell activation</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0042110</td>
<td align="left">T cell activation</td>
<td align="left">C17</td>
</tr>
<tr class="even">
<td align="left">GO:0045321</td>
<td align="left">leukocyte activation</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0045321</td>
<td align="left">leukocyte activation</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0045321</td>
<td align="left">leukocyte activation</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0046629</td>
<td align="left">gamma-delta T cell activation</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0046631</td>
<td align="left">alpha-beta T cell activation</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0046649</td>
<td align="left">lymphocyte activation</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0046649</td>
<td align="left">lymphocyte activation</td>
<td align="left">C17</td>
</tr>
<tr class="odd">
<td align="left">GO:0046649</td>
<td align="left">lymphocyte activation</td>
<td align="left">C22</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0050900">GO:0050900<a class="anchor" aria-label="anchor" href="#go0050900"></a>
</h4>
<p>Ancestor name: leukocyte migration</p>
<table class="table">
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0030595</td>
<td align="left">leukocyte chemotaxis</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0050900</td>
<td align="left">leukocyte migration</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0071674</td>
<td align="left">mononuclear cell migration</td>
<td align="left">C1</td>
</tr>
</tbody>
</table>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## R version 4.2.2 Patched (2022-11-10 r83330)</span></span>
<span><span class="co">## Platform: x86_64-pc-linux-gnu (64-bit)</span></span>
<span><span class="co">## Running under: Pop!_OS 22.04 LTS</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Matrix products: default</span></span>
<span><span class="co">## BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0</span></span>
<span><span class="co">## LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## locale:</span></span>
<span><span class="co">##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span></span>
<span><span class="co">##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></span>
<span><span class="co">##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></span>
<span><span class="co">##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></span>
<span><span class="co">##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span></span>
<span><span class="co">## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## attached base packages:</span></span>
<span><span class="co">## [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">## [8] base     </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## other attached packages:</span></span>
<span><span class="co">##  [1] org.Mm.eg.db_3.16.0         AnnotationDbi_1.60.0       </span></span>
<span><span class="co">##  [3] ggplot2_3.4.1               TimeSeriesAnalysis_0.99.0  </span></span>
<span><span class="co">##  [5] BiocFileCache_2.6.1         dbplyr_2.3.1               </span></span>
<span><span class="co">##  [7] SummarizedExperiment_1.28.0 Biobase_2.58.0             </span></span>
<span><span class="co">##  [9] GenomicRanges_1.50.2        GenomeInfoDb_1.34.9        </span></span>
<span><span class="co">## [11] IRanges_2.32.0              S4Vectors_0.36.2           </span></span>
<span><span class="co">## [13] BiocGenerics_0.44.0         MatrixGenerics_1.10.0      </span></span>
<span><span class="co">## [15] matrixStats_0.63.0          knitr_1.42                 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## loaded via a namespace (and not attached):</span></span>
<span><span class="co">##   [1] colorspace_2.1-0       rjson_0.2.21           ellipsis_0.3.2        </span></span>
<span><span class="co">##   [4] dynamicTreeCut_1.63-1  rprojroot_2.0.3        circlize_0.4.15       </span></span>
<span><span class="co">##   [7] XVector_0.38.0         GlobalOptions_0.1.2    fs_1.6.1              </span></span>
<span><span class="co">##  [10] clue_0.3-64            rstudioapi_0.14        farver_2.1.1          </span></span>
<span><span class="co">##  [13] ggrepel_0.9.3          bit64_4.0.5            fansi_1.0.4           </span></span>
<span><span class="co">##  [16] splines_4.2.2          codetools_0.2-19       doParallel_1.0.17     </span></span>
<span><span class="co">##  [19] cachem_1.0.7           GOSemSim_2.24.0        geneplotter_1.76.0    </span></span>
<span><span class="co">##  [22] jsonlite_1.8.4         annotate_1.76.0        cluster_2.1.4         </span></span>
<span><span class="co">##  [25] GO.db_3.16.0           png_0.1-8              shiny_1.7.4           </span></span>
<span><span class="co">##  [28] BiocManager_1.30.20    compiler_4.2.2         httr_1.4.5            </span></span>
<span><span class="co">##  [31] tictoc_1.1             Matrix_1.5-1           fastmap_1.1.1         </span></span>
<span><span class="co">##  [34] lazyeval_0.2.2         limma_3.54.2           cli_3.6.0             </span></span>
<span><span class="co">##  [37] later_1.3.0            htmltools_0.5.4        tools_4.2.2           </span></span>
<span><span class="co">##  [40] gtable_0.3.1           glue_1.6.2             GenomeInfoDbData_1.2.9</span></span>
<span><span class="co">##  [43] reshape2_1.4.4         dplyr_1.1.0            rappdirs_0.3.3        </span></span>
<span><span class="co">##  [46] Rcpp_1.0.10            jquerylib_0.1.4        pkgdown_2.0.7         </span></span>
<span><span class="co">##  [49] vctrs_0.5.2            Biostrings_2.66.0      nlme_3.1-162          </span></span>
<span><span class="co">##  [52] crosstalk_1.2.0        iterators_1.0.14       xfun_0.37             </span></span>
<span><span class="co">##  [55] stringr_1.5.0          mime_0.12              lifecycle_1.0.3       </span></span>
<span><span class="co">##  [58] XML_3.99-0.13          org.Hs.eg.db_3.16.0    zlibbioc_1.44.0       </span></span>
<span><span class="co">##  [61] scales_1.2.1           promises_1.2.0.1       ragg_1.2.5            </span></span>
<span><span class="co">##  [64] parallel_4.2.2         gprofiler2_0.2.1       RColorBrewer_1.1-3    </span></span>
<span><span class="co">##  [67] ComplexHeatmap_2.14.0  yaml_2.3.7             curl_5.0.0            </span></span>
<span><span class="co">##  [70] memoise_2.0.1          sass_0.4.5             stringi_1.7.12        </span></span>
<span><span class="co">##  [73] RSQLite_2.3.0          highr_0.10             desc_1.4.2            </span></span>
<span><span class="co">##  [76] foreach_1.5.2          filelock_1.0.2         BiocParallel_1.32.5   </span></span>
<span><span class="co">##  [79] shape_1.4.6            rlang_1.0.6            pkgconfig_2.0.3       </span></span>
<span><span class="co">##  [82] systemfonts_1.0.4      bitops_1.0-7           evaluate_0.20         </span></span>
<span><span class="co">##  [85] lattice_0.20-45        purrr_1.0.1            htmlwidgets_1.6.1     </span></span>
<span><span class="co">##  [88] labeling_0.4.2         bit_4.0.5              tidyselect_1.2.0      </span></span>
<span><span class="co">##  [91] plyr_1.8.8             magrittr_2.0.3         DESeq2_1.38.3         </span></span>
<span><span class="co">##  [94] R6_2.5.1               generics_0.1.3         DelayedArray_0.24.0   </span></span>
<span><span class="co">##  [97] DBI_1.1.3              mgcv_1.8-41            pillar_1.8.1          </span></span>
<span><span class="co">## [100] withr_2.5.0            KEGGREST_1.38.0        RCurl_1.98-1.10       </span></span>
<span><span class="co">## [103] tibble_3.1.8           crayon_1.5.2           utf8_1.2.3            </span></span>
<span><span class="co">## [106] plotly_4.10.1          rmarkdown_2.20         org.Ce.eg.db_3.16.0   </span></span>
<span><span class="co">## [109] GetoptLong_1.0.5       locfit_1.5-9.7         grid_4.2.2            </span></span>
<span><span class="co">## [112] data.table_1.14.8      blob_1.2.3             digest_0.6.31         </span></span>
<span><span class="co">## [115] xtable_1.8-4           httpuv_1.6.9           tidyr_1.3.0           </span></span>
<span><span class="co">## [118] textshaping_0.3.6      munsell_0.5.0          viridisLite_0.4.1     </span></span>
<span><span class="co">## [121] bslib_0.4.2</span></span></code></pre>
</div>
</div>
<div class="section level3">
<h3 class="unlisted unnumbered" id="section-1"></h3>
</div>
</div>
<div class="section level2">
<h2 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-brionne2019viseago" class="csl-entry">
Brionne, Aurélien, Amélie Juanchich, and Christelle Hennequet-Antier.
2019. <span>“ViSEAGO: A Bioconductor Package for Clustering Biological
Functions Using Gene Ontology and Semantic Similarity.”</span>
<em>BioData Mining</em> 12 (1): 1–13.
</div>
<div id="ref-GOdb_ref" class="csl-entry">
Carlson, Marc. 2021. <em>GO.db: A Set of Annotation Maps Describing the
Entire Gene Ontology</em>.
</div>
<div id="ref-gu2016complex" class="csl-entry">
Gu, Zuguang, Roland Eils, and Matthias Schlesner. 2016. <span>“Complex
Heatmaps Reveal Patterns and Correlations in Multidimensional Genomic
Data.”</span> <em>Bioinformatics</em> 32 (18): 2847–49.
</div>
<div id="ref-jassal2020reactome" class="csl-entry">
Jassal, Bijay, Lisa Matthews, Guilherme Viteri, Chuqiao Gong, Pascual
Lorente, Antonio Fabregat, Konstantinos Sidiropoulos, et al. 2020.
<span>“The Reactome Pathway Knowledgebase.”</span> <em>Nucleic Acids
Research</em> 48 (D1): D498–503.
</div>
<div id="ref-kanehisa2017kegg" class="csl-entry">
Kanehisa, Minoru, Miho Furumichi, Mao Tanabe, Yoko Sato, and Kanae
Morishima. 2017. <span>“KEGG: New Perspectives on Genomes, Pathways,
Diseases and Drugs.”</span> <em>Nucleic Acids Research</em> 45 (D1):
D353–61.
</div>
<div id="ref-kolberg2020gprofiler2" class="csl-entry">
Kolberg, Liis, Uku Raudvere, Ivan Kuzmin, Jaak Vilo, and Hedi Peterson.
2020. <span>“Gprofiler2–an r Package for Gene List Functional Enrichment
Analysis and Namespace Conversion Toolset g: Profiler.”</span>
<em>F1000Research</em> 9.
</div>
<div id="ref-love2014moderated" class="csl-entry">
Love, Michael I, Wolfgang Huber, and Simon Anders. 2014.
<span>“Moderated Estimation of Fold Change and Dispersion for RNA-Seq
Data with DESeq2.”</span> <em>Genome Biology</em> 15 (12): 1–21.
</div>
<div id="ref-timeseriesexperiment" class="csl-entry">
Nguyen, Lan Huong. 2021. <em>TimeSeriesExperiment: Analysis for Short
Time-Series Data</em>. <a href="https://github.com/nlhuong/TimeSeriesExperiment" class="external-link">https://github.com/nlhuong/TimeSeriesExperiment</a>.
</div>
<div id="ref-nilsen2013identifying" class="csl-entry">
Nilsen, Gro, Ørnulf Borgan, Knut LiestØl, and Ole Christian Lingjærde.
2013. <span>“Identifying Clusters in Genomics Data by Recursive
Partitioning.”</span> <em>Statistical Applications in Genetics and
Molecular Biology</em> 12 (5): 637–52.
</div>
<div id="ref-raudvere2019g" class="csl-entry">
Raudvere, Uku, Liis Kolberg, Ivan Kuzmin, Tambet Arak, Priit Adler, Hedi
Peterson, and Jaak Vilo. 2019. <span>“G: Profiler: A Web Server for
Functional Enrichment Analysis and Conversions of Gene Lists (2019
Update).”</span> <em>Nucleic Acids Research</em> 47 (W1): W191–98.
</div>
<div id="ref-smyth2005limma" class="csl-entry">
Smyth, Gordon K. 2005. <span>“Limma: Linear Models for Microarray
Data.”</span> In <em>Bioinformatics and Computational Biology Solutions
Using r and Bioconductor</em>, 397–420. Springer.
</div>
<div id="ref-wang2007new" class="csl-entry">
Wang, James Z, Zhidian Du, Rapeeporn Payattakool, Philip S Yu, and
Chin-Fu Chen. 2007. <span>“A New Method to Measure the Semantic
Similarity of GO Terms.”</span> <em>Bioinformatics</em> 23 (10):
1274–81.
</div>
<div id="ref-yu2010gosemsim" class="csl-entry">
Yu, Guangchuang, Fei Li, Yide Qin, Xiaochen Bo, Yibo Wu, and Shengqi
Wang. 2010. <span>“GOSemSim: An r Package for Measuring Semantic
Similarity Among GO Terms and Gene Products.”</span>
<em>Bioinformatics</em> 26 (7): 976–78.
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Yohan Lefol.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
