<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>TimeSeries Analysis – Murine example run • TimeSeriesAnalysis</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/spacelab/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="TimeSeries Analysis – Murine example run">
<meta property="og:description" content="TimeSeriesAnalysis">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TimeSeriesAnalysis</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Examples
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/TS_analysis_PBMC.html">PBMC analysis</a>
    </li>
    <li>
      <a href="../articles/TS_analysis_murine.html">Murine analysis</a>
    </li>
    <li>
      <a href="../articles/TS_analysis_celegans.html">Celegans analysis</a>
    </li>
  </ul>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/Ylefol/TimeSeriesAnalysis/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="TS_analysis_murine_files/htmlwidgets-1.5.4/htmlwidgets.js"></script><script src="TS_analysis_murine_files/plotly-binding-4.10.0/plotly.js"></script><script src="TS_analysis_murine_files/typedarray-0.1/typedarray.min.js"></script><link href="TS_analysis_murine_files/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="TS_analysis_murine_files/crosstalk-1.2.0/js/crosstalk.min.js"></script><link href="TS_analysis_murine_files/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet">
<script src="TS_analysis_murine_files/plotly-main-2.5.1/plotly-latest.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>TimeSeries Analysis – Murine example run</h1>
                        <h4 data-toc-skip class="author">Yohan
Lefol</h4>
            
            <h4 data-toc-skip class="date">02. September 2022</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/Ylefol/TimeSeriesAnalysis/blob/HEAD/vignettes/TS_analysis_murine.Rmd" class="external-link"><code>vignettes/TS_analysis_murine.Rmd</code></a></small>
      <div class="hidden name"><code>TS_analysis_murine.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="timeseries-analysis">TimeSeries analysis<a class="anchor" aria-label="anchor" href="#timeseries-analysis"></a>
</h2>
<div class="section level3">
<h3 id="parameter-set-up">Parameter set up<a class="anchor" aria-label="anchor" href="#parameter-set-up"></a>
</h3>
<p>Set the below parameters for the downstream analysis</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Give names to saved object and name of results repository</span>
<span class="va">name_result_folder</span><span class="op">&lt;-</span><span class="st">'TS_results_murine_example/'</span>
<span class="va">obj_name</span><span class="op">&lt;-</span><span class="st">'timeSeries_obj_murine_example.Rdata'</span>

<span class="co">#Path to count data and sample data respectively</span>
<span class="va">my_path_data</span><span class="op">&lt;-</span><span class="st">'data/murine/raw_counts_TS'</span>
<span class="va">my_path_sample_dta</span><span class="op">&lt;-</span><span class="st">'data/murine/sample_file.csv'</span>

<span class="co">#Set-up time series object parameters</span>
<span class="va">diff_exp_type</span><span class="op">&lt;-</span><span class="st">'DESeq2'</span> <span class="co">#package used for DE – can also be 'limma'</span>
<span class="va">p_val_filter_type</span><span class="op">&lt;-</span><span class="st">'padj'</span> <span class="co">#Either padj or pvalue, used to filter for significance</span>
<span class="va">p_thresh</span><span class="op">&lt;-</span><span class="fl">0.05</span> <span class="co">#pvalue or padj value threshold for significance</span>
<span class="va">l2fc_thresh</span><span class="op">&lt;-</span><span class="fl">1</span> <span class="co">#log(2)foldChange threshold for significance</span>
<span class="va">name_control</span><span class="op">&lt;-</span><span class="st">'Control'</span> <span class="co">#Name of experiment as seen in the sample file</span>
<span class="va">name_experiment</span><span class="op">&lt;-</span><span class="st">'Treated'</span> <span class="co">#Name of control as seen in the sample file</span>
<span class="va">graphic_vector</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"#e31a1c"</span>,<span class="st">"#1f78b4"</span><span class="op">)</span> <span class="co">#Pre-set colors for the groups</span>

<span class="co">#Declare organism and load library</span>
<span class="va">org_sem_sim</span><span class="op">=</span><span class="st">'org.Mm.eg.db'</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st">'org.Mm.eg.db'</span><span class="op">)</span></code></pre></div>
<pre><code><span class="co">## </span></code></pre>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#mmusculus</span>
<span class="co">#hsapiens</span>
<span class="co">#celegans</span>
<span class="va">my_org_gpro</span><span class="op">=</span><span class="st">'mmusculus'</span> <span class="co">#Set the species for the gprofiler analysis</span>

<span class="va">PART_l2fc</span><span class="op">&lt;-</span><span class="fl">1</span> <span class="co">#log(2)foldChange threshold for PART clustering</span>
<span class="va">PART_min_clust</span><span class="op">&lt;-</span><span class="fl">50</span> <span class="co">#Minimum cluster size for PART</span>
<span class="va">PART_recursion</span><span class="op">&lt;-</span><span class="fl">100</span> <span class="co">#Number of recursions, default is 100, using 10 for example</span>

<span class="va">log_tp_traj</span><span class="op">&lt;-</span><span class="cn">T</span> <span class="co">#Defines if timepoints should be log transformed for illustration purposes</span>

<span class="co">#Used to highlight specific genes regardless of differential gene expression significance</span>
<span class="va">genes_of_interest</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="op">)</span>

<span class="co">#The ancestors that will be queried, the ontology must be specified (BP,MF,or CC)</span>
<span class="co">#Set to an empty vector c() if not required by the analysis</span>
<span class="va">target_ancestors</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'GO:0002253'</span>,<span class="st">'GO:0019882'</span>,<span class="st">'GO:0002404'</span>,<span class="st">'GO:0002339'</span>,<span class="st">'GO:0042386'</span>,
                    <span class="st">'GO:0035172'</span>,<span class="st">'GO:0002252'</span>,<span class="st">'GO:0006955'</span>,<span class="st">'GO:0002520'</span>,<span class="st">'GO:0090713'</span>,
                    <span class="st">'GO:0045321'</span>,<span class="st">'GO:0001776'</span>,<span class="st">'GO:0050900'</span>,<span class="st">'GO:0031294'</span>,<span class="st">'GO:0002262'</span>,
                    <span class="st">'GO:0002683'</span>,<span class="st">'GO:0002684'</span>,<span class="st">'GO:0002440'</span>,<span class="st">'GO:0002682'</span>,<span class="st">'GO:0002200'</span>,
                    <span class="st">'GO:0045058'</span>,<span class="st">'GO:0002507'</span><span class="op">)</span>
<span class="va">ancestor_ontology</span><span class="op">&lt;-</span><span class="st">'BP'</span>

<span class="co">#Some extra set-up</span>
<span class="va">name_save_obj</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="va">obj_name</span><span class="op">)</span><span class="co">#The object will be saved in result folder</span>
<span class="co">#Create main directory for results</span>
<span class="fu"><a href="https://rdrr.io/r/base/files2.html" class="external-link">dir.create</a></span><span class="op">(</span><span class="va">name_result_folder</span><span class="op">)</span></code></pre></div>
<pre><code><span class="co">## Warning in dir.create(name_result_folder): 'TS_results_murine_example' already</span>
<span class="co">## exists</span></code></pre>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">my_group_names</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">name_experiment</span>,<span class="va">name_control</span><span class="op">)</span> 
<span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">graphic_vector</span><span class="op">)</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">name_experiment</span>,<span class="va">name_control</span><span class="op">)</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="create-timeseries-object">Create TimeSeries object<a class="anchor" aria-label="anchor" href="#create-timeseries-object"></a>
</h3>
<p>This code chunk first checks if a timeseries object has been saved,
if that is the case, the object is loaded instead of being created
again. To reset this analysis, the saved timeseries object should be
deleted or moved from the repository.<br>
If no timeseries object was found, the code chunk creates the timeseries
object using the parameters defined above. The raw matrix is also
created using the second function</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw">if</span><span class="op">(</span><span class="va">obj_name</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/list.files.html" class="external-link">list.files</a></span><span class="op">(</span><span class="va">name_result_folder</span><span class="op">)</span><span class="op">==</span><span class="cn">F</span><span class="op">)</span><span class="op">{</span>
  <span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/methods/new.html" class="external-link">new</a></span><span class="op">(</span><span class="st">'TimeSeries_Object'</span>,sample_data<span class="op">=</span><span class="fu"><a href="../reference/prep_sample_data.html">prep_sample_data</a></span><span class="op">(</span><span class="va">my_path_sample_dta</span>,<span class="va">my_group_names</span><span class="op">)</span>,
                 group_names<span class="op">=</span><span class="va">my_group_names</span>,group_colors<span class="op">=</span><span class="va">graphic_vector</span>,DE_method<span class="op">=</span><span class="va">diff_exp_type</span>,
                 DE_p_filter<span class="op">=</span><span class="va">p_val_filter_type</span>,DE_p_thresh<span class="op">=</span><span class="va">p_thresh</span>,DE_l2fc_thresh<span class="op">=</span><span class="va">l2fc_thresh</span>,
                 PART_l2fc_thresh<span class="op">=</span><span class="va">PART_l2fc</span>,sem_sim_org<span class="op">=</span><span class="va">org_sem_sim</span>,Gpro_org<span class="op">=</span><span class="va">my_org_gpro</span><span class="op">)</span>
<span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/create_raw_count_matrix.html">create_raw_count_matrix</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">my_path_data</span><span class="op">)</span>
<span class="op">}</span><span class="kw">else</span><span class="op">{</span>
  <span class="fu"><a href="https://rdrr.io/r/base/load.html" class="external-link">load</a></span><span class="op">(</span><span class="va">name_save_obj</span><span class="op">)</span>
<span class="op">}</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="differental-gene-expression-analysis">Differental gene expression analysis<a class="anchor" aria-label="anchor" href="#differental-gene-expression-analysis"></a>
</h3>
<p>Loads the necessary functions to perform differential gene expression
analysis If the tool used is DESeq2<span class="citation">(Love, Huber,
and Anders 2014)</span>, the data needs to be normalized using it’s
method. If the tool used is limma<span class="citation">(Smyth
2005)</span>, the normalized matrix should have been inputed, and thus
no normalization is needed.</p>
<p>This code chunk performs both the conditional and temporal
differential gene expression and saves the results within the Time
Series object.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Perform normalization if the DESeq2 tool is being used and if normalized matrix doesn't exist</span>
<span class="kw">if</span> <span class="op">(</span><span class="va">TS_object</span><span class="op">@</span><span class="va">DE_method</span><span class="op">==</span><span class="st">'DESeq2'</span> <span class="op">&amp;</span> <span class="st">'norm'</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">@</span><span class="va">count_matrix</span><span class="op">)</span><span class="op">!=</span><span class="cn">T</span><span class="op">)</span><span class="op">{</span>
  <span class="va">TS_object</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/normalize_timeSeries_with_deseq2.html">normalize_timeSeries_with_deseq2</a></span><span class="op">(</span>time_object<span class="op">=</span><span class="va">TS_object</span><span class="op">)</span>
<span class="op">}</span>

<span class="co">#Perform conditional differential gene expression analysis</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/conditional_DE_wrapper.html">conditional_DE_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span>
<span class="co">#Perform temporal differential gene expression analysis</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/temporal_DE_wrapper.html">temporal_DE_wrapper</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span>

<span class="co">#save the timeseries object</span>
<span class="fu"><a href="https://rdrr.io/r/base/save.html" class="external-link">save</a></span><span class="op">(</span><span class="va">TS_object</span>,file<span class="op">=</span><span class="va">name_save_obj</span><span class="op">)</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="part-clustering">PART clustering<a class="anchor" aria-label="anchor" href="#part-clustering"></a>
</h3>
<p>The code chunk below prepares and initiates PART clustering<span class="citation">(Nilsen et al. 2013)</span>. It first retrieves the
number of significant genes for PART clustering in the ‘signi_genes’
variable This chunk can be quite lengthy depending on the number of
genes included for clustering as well as the number of recursions set
for the clustering</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Extract genes for PART clustering based on defined log(2)foldChange threshold</span>
<span class="va">signi_genes</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/select_genes_with_l2fc.html">select_genes_with_l2fc</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span>

<span class="co">#Use all samples, but implement a custom order. In this case it is reversed</span>
<span class="va">samps_2</span><span class="op">&lt;-</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sample_data</span><span class="op">$</span><span class="va">sample</span><span class="op">[</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sample_data</span><span class="op">$</span><span class="va">group</span><span class="op">==</span><span class="va">TS_object</span><span class="op">@</span><span class="va">group_names</span><span class="op">[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span>
<span class="va">samps_1</span><span class="op">&lt;-</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sample_data</span><span class="op">$</span><span class="va">sample</span><span class="op">[</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sample_data</span><span class="op">$</span><span class="va">group</span><span class="op">==</span><span class="va">TS_object</span><span class="op">@</span><span class="va">group_names</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span>

<span class="co">#Create the matrix that will be used for PART clustering</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/prep_counts_for_PART.html">prep_counts_for_PART</a></span><span class="op">(</span>object<span class="op">=</span><span class="va">TS_object</span>,target_genes<span class="op">=</span><span class="va">signi_genes</span>,
                                     scale<span class="op">=</span><span class="cn">T</span>,target_samples<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">samps_2</span>,<span class="va">samps_1</span><span class="op">)</span><span class="op">)</span>

<span class="co">#Initiate PART clustering with a pre-defined seed – 123456</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/compute_PART.html">compute_PART</a></span><span class="op">(</span><span class="va">TS_object</span>,part_recursion<span class="op">=</span><span class="va">PART_recursion</span>,part_min_clust<span class="op">=</span><span class="va">PART_min_clust</span>,
                        dist_param<span class="op">=</span><span class="st">"euclidean"</span>, hclust_param<span class="op">=</span><span class="st">"average"</span>,
                        custom_seed<span class="op">=</span><span class="fl">123456</span><span class="op">)</span>
<span class="co">#Save the TimeSeries object to the directory to prevent loss of data in the event</span>
<span class="co">#of a downstream error</span>
<span class="fu"><a href="https://rdrr.io/r/base/save.html" class="external-link">save</a></span><span class="op">(</span><span class="va">TS_object</span>,file<span class="op">=</span><span class="va">name_save_obj</span><span class="op">)</span></code></pre></div>
</div>
<div class="section level3">
<h3 id="gprofiler-analysis">Gprofiler analysis<a class="anchor" aria-label="anchor" href="#gprofiler-analysis"></a>
</h3>
<p>The code chunk below runs a gprofiler analysis<span class="citation">(Kolberg et al. 2020; Raudvere et al. 2019)</span>.
IMPORTANT: This function requires that there be a stable internet
connection lack of connection or intermittent drops in the connection
will result in an error and the termination (stop) of the code chunk</p>
<p>If an error has occured, this code chunk can be re-run separately
from the above chunks by uncommenting (removing the ‘#’ in front) the
“load(‘timeseries_obj_res.Rdata’)” line.</p>
<p>This will load the results saved after the PART clustering code
chunk</p>
<p>This code chunk will overwrite the saved object if it is completed.
The overwritten object will then contain the gprofiler analysis results
and can be used to generate plots with the downstream code chunks.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># load('timeseries_obj_res.Rdata')</span>
<span class="va">TS_object</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/run_gprofiler_PART_clusters.html">run_gprofiler_PART_clusters</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">)</span> <span class="co">#Run the gprofiler analysis</span>
<span class="co">#Save the results in a Rdata obbject</span>
<span class="fu"><a href="https://rdrr.io/r/base/save.html" class="external-link">save</a></span><span class="op">(</span><span class="va">TS_object</span>,file<span class="op">=</span><span class="va">name_save_obj</span><span class="op">)</span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="timeseries-analysis-results">TimeSeries analysis results<a class="anchor" aria-label="anchor" href="#timeseries-analysis-results"></a>
</h2>
<p>Most plots are created in SVG format for ease of editing with SVG
editing software such as InkScape (open source software). Some plots are
created in html format as they are interactive plots. These require a
web browser to open them</p>
<p>To convert SVG files to PNG/JPG/PDF, this website is available: <a href="https://svgtopng.com/" class="external-link uri">https://svgtopng.com/</a><br>
HTML files can be opened and then saved as PNG (or other) using the
camera icon in the top right of each respective interactive plot</p>
<div class="section level3">
<h3 id="timeseries-summary-of-experiment">TimeSeries summary of experiment<a class="anchor" aria-label="anchor" href="#timeseries-summary-of-experiment"></a>
</h3>
<p><strong>Name of groups beings compared:</strong> Treated vs
Control<br><strong>Number of genes analyzed:</strong> 41115</p>
<p><strong>Differential expression parameters used:</strong><br><strong>Method:</strong> DESeq2<br><strong>P statistic filter used:</strong> padj with a 0.05
threshold<br><strong>Log(2)FoldChange significance threshold used:</strong> 1</p>
<p><strong>PART parameters used:</strong><br><strong>PART log(2)FoldChange significance threshold:</strong> 1<br><strong>number of recursions:</strong> 100<br><strong>minimum cluster size:</strong> 50<br><strong>distance parameter:</strong> euclidean<br><strong>hclust parameter:</strong> average<br><strong>custom seed used:</strong> 123456<br><strong>PART computation time:</strong> 3239.6 sec elapsed</p>
<p><strong>PCA of sample data:</strong><br><img src="TS_results_murine_example/PCA_plot.png" width="2480"></p>
</div>
<div class="section level3">
<h3 id="differential-gene-expression-results">Differential Gene Expression results<a class="anchor" aria-label="anchor" href="#differential-gene-expression-results"></a>
</h3>
<p><strong>DEGs found per analysis:</strong></p>
<table class="table">
<thead><tr class="header">
<th align="left">experiment.name</th>
<th align="left">number.of.DEGs</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">Treated_vs_Control_TP_7.5 (conditional)</td>
<td align="left">7</td>
</tr>
<tr class="even">
<td align="left">Treated_vs_Control_TP_45 (conditional)</td>
<td align="left">37</td>
</tr>
<tr class="odd">
<td align="left">Treated_vs_Control_TP_150 (conditional)</td>
<td align="left">136</td>
</tr>
<tr class="even">
<td align="left">Treated_vs_Control_TP_540 (conditional)</td>
<td align="left">429</td>
</tr>
<tr class="odd">
<td align="left">Treated_vs_Control_TP_2160 (conditional)</td>
<td align="left">702</td>
</tr>
<tr class="even">
<td align="left">TP_45_vs_TP_7.5 (temporal)</td>
<td align="left">38</td>
</tr>
<tr class="odd">
<td align="left">TP_150_vs_TP_45 (temporal)</td>
<td align="left">1297</td>
</tr>
<tr class="even">
<td align="left">TP_540_vs_TP_150 (temporal)</td>
<td align="left">1074</td>
</tr>
<tr class="odd">
<td align="left">TP_2160_vs_TP_540 (temporal)</td>
<td align="left">59</td>
</tr>
<tr class="even">
<td align="left"><strong>total unique genes</strong></td>
<td align="left"><strong>2851</strong></td>
</tr>
</tbody>
</table>
<p>Each experiment listed in the above table has separate results which
can be viewed in the ‘TS_results’ folder. In the main results folder,
the differential gene expression results are located in the
‘DE_results_conditional’ and ‘DE_results_temporal’ plot.<br>
Each individual experiment creates the following:<br><strong>DE_raw_data.csv</strong>: A csv file containing all the genes
and their associated differential expression results, regardless of
significance.<br><strong>DE_sig_data</strong>: A csv file containing only the significant
differential expression results.<br><strong>MA_plot.png</strong>: A MA plot, these are often used to
evaluate the quality of the normalization.<br><strong>volcano_plot.png</strong>: A volcano plot which splits
significantly up-regulated and down-regulated genes as well as
non-significant genes.<br><strong>PCA_plot</strong>: A PCA plot using only the samples implicated
in the differential gene expression analysis.</p>
<p>The pipeline also creates two large heatmaps which summarize
conditional and temporal differential expression results. The heatmaps
were created using the ComplexHeatmaps package<span class="citation">(Gu, Eils, and Schlesner 2016)</span>.<br>
These heatmaps illustrate each experiment as different columns
(distinguished by color) while the rows distinguish the groups. Plotted
in the heatmaps is the log transformed counts, while the log transformed
log(2)fold change is seen in a histogram at the bottom of the heatmap.
The number of genes in each experiment (colored columns) is indicated in
the legend at the bottom of the heatmap.<br>
In the case of the temporal heatmap, the rows are labelled ‘experiment’
and ‘control’. The ‘experiment’ represents the time points being
compared while the control represents the timepoint which the
‘experiment’ is compared against. For example, TP2_vs_TP1 would have the
‘experiment’ be TP2 and the control be TP1, while TP3_vs_TP2 would have
TP3 as the ‘experiment’ and TP2 as the ‘control’. Note that TP=time
point.</p>
<p><strong>The conditional heatmap</strong><br><img src="TS_results_murine_example/DE_results_conditional/custom_heat_conditional.svg"><!-- --></p>
<p><strong>The temporal heatmap</strong><br><img src="TS_results_murine_example/DE_results_temporal/custom_heat_temporal.svg"><!-- --></p>
</div>
<div class="section level3">
<h3 id="genes-of-interest-results">Genes of interest results<a class="anchor" aria-label="anchor" href="#genes-of-interest-results"></a>
</h3>
<p><img src="TS_analysis_murine_files/figure-html/genes_interest-3.png" width="65%" style="float:right; padding:10px">
If genes of interest have been specified, the pipeline will extract the
differential gene expression results, both conditional and temporal, for
each gene requested, provided that they are found in the count matrix.
These data files, along with individual trajectory plots are stored in
the ‘TS_results/genes_of_interest’ folder. Below the trajectory of the
most variable gene is shown. If no genes of interest were submitted, a
random gene is taken from the differential expression results.
Illustrated in the trajectory plots is the timepoints on the x axis and
the number of reads (RNAseq) or expression (microarray) on the y
axis.<br>
Note that data files and trajectory plots are created for all genes of
interest found, regardless of their significance based on the
differential gene expression analyses.</p>
</div>
<div class="section level3">
<h3 id="cluster-results">Cluster results<a class="anchor" aria-label="anchor" href="#cluster-results"></a>
</h3>
<p>The code chunk below create the trajectory plots for the clusters
found by PART Trajectories are first calculated by calculating the mean
of each gene for each time point using the available replicates. The
mean value of each gene is calculated once for each group
(control/experiment). Afterwards, the genes are scaled in order to be
able to represent them on the same axis of values. Each gene is plotted
as an individual colored line’, a mean line (gray color) is also plotted
to show the overall trajectory of the cluster. These plots were inspired
by <span class="citation">(Nguyen 2021)</span></p>
<p>The log_tp_traj parameter defines if the time points should be log
transformed. A log transformation of the time points is useful if the
time points are not evenly sampled through time. For example, two time
points at 0 and 15 minutes, with another time point at 24 hours (1440
minutes) will result in the first two time points being very compressed
and possibly indistinguishable from each other.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/files2.html" class="external-link">dir.create</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="st">'PART_results'</span><span class="op">)</span><span class="op">)</span> <span class="co">#create the directory to store results</span>
<span class="fu"><a href="../reference/PART_heat_map.html">PART_heat_map</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="st">'PART_results/PART_heat'</span><span class="op">)</span><span class="op">)</span> <span class="co">#Create a summary heatmap</span>
<span class="va">ts_data</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/calculate_cluster_traj_data.html">calculate_cluster_traj_data</a></span><span class="op">(</span><span class="va">TS_object</span>,scale_feat<span class="op">=</span><span class="cn">T</span><span class="op">)</span> <span class="co">#Calculate scaled gene values for genes of clusters</span>
<span class="va">mean_ts_data</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/calculate_mean_cluster_traj.html">calculate_mean_cluster_traj</a></span><span class="op">(</span><span class="va">ts_data</span><span class="op">)</span> <span class="co">#Calculate the mean scaled values for each cluster</span>

<span class="co">#Function which determines the number of SVG files to plot all cluster trajectories</span>
<span class="fu"><a href="../reference/wrapper_cluster_trajectory.html">wrapper_cluster_trajectory</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">ts_data</span>,<span class="va">mean_ts_data</span>,log_TP<span class="op">=</span><span class="va">log_tp_traj</span>,plot_name<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="st">'/PART_results/Ctraj'</span><span class="op">)</span><span class="op">)</span>

<span class="co">#find the most variable cluster, subset necessary data and plot it on a single column</span>
<span class="co">#The function will also filter for a cluster which has gprofiler results</span>
<span class="va">target_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_most_variable_cluster.html">find_most_variable_cluster</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">mean_ts_data</span><span class="op">)</span>
<span class="va">sub_ts</span><span class="op">&lt;-</span><span class="va">ts_data</span><span class="op">[</span><span class="va">ts_data</span><span class="op">$</span><span class="va">cluster</span><span class="op">==</span><span class="va">target_clust</span>,<span class="op">]</span>
<span class="va">sub_mean</span><span class="op">&lt;-</span><span class="va">mean_ts_data</span><span class="op">[</span><span class="va">mean_ts_data</span><span class="op">$</span><span class="va">cluster</span><span class="op">==</span><span class="va">target_clust</span>,<span class="op">]</span>
<span class="va">plt_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/plot_cluster_traj.html">plot_cluster_traj</a></span><span class="op">(</span>object <span class="op">=</span> <span class="va">TS_object</span>,ts_data <span class="op">=</span> <span class="va">sub_ts</span>,ts_mean_data <span class="op">=</span> <span class="va">sub_mean</span>, num_col<span class="op">=</span><span class="fl">1</span>,rem_legend_axis<span class="op">=</span><span class="cn">T</span>,log_TP<span class="op">=</span><span class="va">log_tp_traj</span><span class="op">)</span>
<span class="va">cluster_map_sum</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/t.html" class="external-link">t</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="va">TS_object</span><span class="op">@</span><span class="va">PART_results</span><span class="op">$</span><span class="va">cluster_map</span><span class="op">[</span>,<span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">1</span><span class="op">)</span><span class="op">]</span><span class="op">)</span><span class="op">)</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/base/row.names.html" class="external-link">row.names</a></span><span class="op">(</span><span class="va">cluster_map_sum</span><span class="op">)</span><span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'Cluster name'</span>,<span class="st">'Number of genes'</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/pkg/AnnotationDbi/man/Bimap-toTable.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">cluster_map_sum</span><span class="op">)</span><span class="op">=</span><span class="cn">NULL</span></code></pre></div>
<p><strong>PART result summary:</strong></p>
<table class="table">
<colgroup>
<col width="15%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="3%">
<col width="2%">
<col width="2%">
<col width="2%">
<col width="2%">
<col width="3%">
<col width="3%">
</colgroup>
<tbody>
<tr class="odd">
<td align="left">Cluster name</td>
<td align="left">C1</td>
<td align="left">C10</td>
<td align="left">C11</td>
<td align="left">C12</td>
<td align="left">C13</td>
<td align="left">C14</td>
<td align="left">C15</td>
<td align="left">C16</td>
<td align="left">C17</td>
<td align="left">C18</td>
<td align="left">C19</td>
<td align="left">C2</td>
<td align="left">C20</td>
<td align="left">C21</td>
<td align="left">C22</td>
<td align="left">C3</td>
<td align="left">C4</td>
<td align="left">C5</td>
<td align="left">C6</td>
<td align="left">C7</td>
<td align="left">C8</td>
<td align="left">C9</td>
</tr>
<tr class="even">
<td align="left">Number of genes</td>
<td align="left">418</td>
<td align="left">72</td>
<td align="left">92</td>
<td align="left">103</td>
<td align="left">53</td>
<td align="left">59</td>
<td align="left">58</td>
<td align="left">104</td>
<td align="left">99</td>
<td align="left">86</td>
<td align="left">129</td>
<td align="left">181</td>
<td align="left">67</td>
<td align="left">67</td>
<td align="left">183</td>
<td align="left">121</td>
<td align="left">57</td>
<td align="left">63</td>
<td align="left">74</td>
<td align="left">51</td>
<td align="left">567</td>
<td align="left">147</td>
</tr>
</tbody>
</table>
<p>All cluster results are found in the ‘PART_results’ folder, within
this folder you will find the following:</p>
<p><strong>PART_heat.svg</strong>: A svg heatmap of the PART result –
detailed below.<br><strong>PART_heat_with_names.svg</strong>: The same plot as above but
with sample and gene names included. These may not be visible depending
on the number of genes included as well as the length of the sample
names.<br><strong>PART_heat_cmap.csv</strong>: A csv file representing a
‘cluster_map’ which provides the cluster in which each gene was sorted
in as well as the color (in HEX code) given to that cluster.<br><strong>PART_heat_data.csv</strong>: The z-score values used in the
‘PART_heat’ heatmaps.<br><strong>Ctraj.svg</strong>: At least one, and possibly several of these
files will be in the folder. This/these SVG plots show the trajectory of
each cluster individually while being split by group. Meaning the
trajectory of each cluster is shown twice, once per group. Information
concerning the cluster and group is shown in the title of each subplot.
One example of cluster trajectory is shown below.</p>
<p><strong>PART heatmap and cluster trajectory</strong><br>
In this heatmap we illustrate all the genes which were submitted for
PART clustering. Each gene is illustrated using a Z-score, which
measures the number of standard deviations that a gene has from it’s
mean. In essence it measures the variability.<br>
Clusters are shown as the colored bar on the left hand side of the plot
while groups and timepoints are shown as the colored bars on the top of
the heatmap. PART heatmaps were created with ComplexHeatmaps<span class="citation">(Gu, Eils, and Schlesner 2016)</span>.<br>
The cluster trajectory of the most variable cluster (C21) is
illustrated, with the trajectory of the experiment group at the top and
the control group at the bottom. The x axis represents timepoints and
the y axis represents the scaled mean expression of the genes. The
expression of each gene is obtained by calculating the mean between all
replicates for each time point in each group. The values are then scaled
in order to illustrate their trajectory. The overall mean of the cluster
is shown as a thicker gray line.</p>
<p><img src="TS_results_murine_example/PART_results/PART_heat.svg" width="49%" height="49%"><img src="TS_analysis_murine_files/figure-html/test,figures-side-2.png" width="49%" height="49%"></p>
</div>
<div class="section level3">
<h3 id="gprofiler-results-standard">Gprofiler results – standard<a class="anchor" aria-label="anchor" href="#gprofiler-results-standard"></a>
</h3>
<p>The below chunk creates the gprofiler result. Some parameters
relating to the species and ontology need to be defined The species
parameter are used to calculate the semantic distance which impacts the
Multi-Dimensional Scaling (MDS) plots that will be generated by the last
function of this code chunk. Both versions of the MDS plots seen below
were inspired by <span class="citation">(Brionne, Juanchich, and
Hennequet-Antier 2019)</span>.</p>
<p>The ontology needs to be defined for the MDS plots as well. Note that
MDS plots utilise the semantic similarity metric which currently only
supports GO ontology (MF – MolecularFunctions, CC – Cellular Components,
and BP – Biological processes).<br>
The first function plots the standard gprofiler plots (gost plots) to
the directory, and it retrieves the information of the requested
ontology into the GO_clusters dataframe. This dataframe is then used
with the semantic distance to create two versions of the MDS plot.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Define specimen and ontology parameters</span>
<span class="va">my_ont_gpro</span><span class="op">=</span><span class="st">'GO:BP'</span>
<span class="va">my_ont_sem_sim</span><span class="op">=</span><span class="st">'BP'</span>

<span class="co">#Create standard gprofiler results</span>
<span class="va">gpro_res</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/gprofiler_cluster_analysis.html">gprofiler_cluster_analysis</a></span><span class="op">(</span><span class="va">TS_object</span>,<span class="va">my_ont_gpro</span>,save_path <span class="op">=</span> <span class="va">name_result_folder</span>,return_specific_cluster<span class="op">=</span><span class="va">target_clust</span><span class="op">)</span>
<span class="va">GO_clusters</span><span class="op">&lt;-</span><span class="va">gpro_res</span><span class="op">[[</span><span class="st">'GO_df'</span><span class="op">]</span><span class="op">]</span>
<span class="co">#Create semantic data</span>
<span class="va">sem_data</span> <span class="op">&lt;-</span> <span class="fu">godata</span><span class="op">(</span><span class="va">TS_object</span><span class="op">@</span><span class="va">sem_sim_org</span>, ont<span class="op">=</span><span class="va">my_ont_sem_sim</span>, computeIC<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span>


<span class="va">found_clusters</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_clusters_from_termdist.html">find_clusters_from_termdist</a></span><span class="op">(</span><span class="va">GO_clusters</span>,<span class="va">sem_data</span><span class="op">)</span>
<span class="va">clusters</span><span class="op">&lt;-</span><span class="va">found_clusters</span><span class="op">[</span>,<span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'GO.ID'</span>,<span class="st">'GO.cluster'</span><span class="op">)</span><span class="op">]</span>
<span class="va">clusters</span><span class="op">&lt;-</span><span class="fu">as.data.table</span><span class="op">(</span><span class="va">clusters</span><span class="op">)</span>
<span class="va">clusters</span><span class="op">[</span>,<span class="st">"GO.cluster"</span><span class="op">:=</span><span class="fu"><a href="https://rdrr.io/r/base/factor.html" class="external-link">factor</a></span><span class="op">(</span><span class="va">clusters</span><span class="op">$</span><span class="va">GO.cluster</span>,levels <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">clusters</span><span class="op">$</span><span class="va">GO.cluster</span><span class="op">)</span><span class="op">)</span><span class="op">]</span>

<span class="co">#Plot and save MDS and clustered MDS</span>
<span class="va">MDS_plots</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/wrapper_MDS_and_MDS_clusters.html">wrapper_MDS_and_MDS_clusters</a></span><span class="op">(</span><span class="va">GO_clusters</span>,<span class="va">sem_data</span>,<span class="va">my_ont_sem_sim</span>,target_dir<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="va">name_result_folder</span>,<span class="st">'gprofiler_results/'</span><span class="op">)</span>,return_plot<span class="op">=</span><span class="cn">T</span><span class="op">)</span></code></pre></div>
<p>Each cluster has it’s own individual gprofiler plot, these are found
in ‘TS_results/gprofiler_results/figures’. The gost plot for the most
variable cluster (C21) is shown below.<br>
These interactive plots show the results of various databases when
queried with a list of genes. More information concerning these plots
and their various datasources can be found at the following link: <a href="https://biit.cs.ut.ee/gprofiler/gost" class="external-link uri">https://biit.cs.ut.ee/gprofiler/gost</a></p>
<div id="htmlwidget-1adc832deb253656dfe6" style="width:100%;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-1adc832deb253656dfe6">{"x":{"data":[{"x":[153.062643726794,153.065087191347],"y":[1.30172864751592,1.30172864751592],"text":["CORUM:1997 (1) <br> Adiponectin homotrimer complex <br> 4.992e-02","CORUM:1998 (1) <br> Adiponectin homohexamer complex <br> 4.992e-02"],"key":["CORUM:1997","CORUM:1998"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(102,170,0,1)","opacity":0.8,"size":3.77952755905512,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(102,170,0,1)"}},"hoveron":"points","set":"SharedData85e2eb42","name":"CORUM","legendgroup":"CORUM","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[54.8916446699144,71.8687888079427,55.3357148385148],"y":[1.54167727721962,1.51862680936712,1.35422282043045],"text":["GO:0006954 (745) <br> inflammatory response <br> 2.873e-02","GO:0034330 (750) <br> cell junction organization <br> 3.030e-02","GO:0007169 (598) <br> transmembrane receptor protein tyrosine kinase signaling pathway <br> 4.424e-02"],"key":["GO:0006954","GO:0034330","GO:0007169"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.7705029049752,19.7785877529278,19.5025304198607],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData85e2eb42","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[36.4318188662931,42.1375021368657,36.5758031746609,35.7582651525729],"y":[4.41979899358261,2.29512695286282,2.21743370323175,1.35464450143364],"text":["GO:0009986 (1117) <br> cell surface <br> 3.804e-05","GO:0071944 (6988) <br> cell periphery <br> 5.068e-03","GO:0016020 (10261) <br> membrane <br> 6.061e-03","GO:0005886 (6519) <br> plasma membrane <br> 4.419e-02"],"key":["GO:0009986","GO:0071944","GO:0016020","GO:0005886"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.2528875905002,22.2799883018565,22.6771653543307,22.2072460357982],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData85e2eb42","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[5.74061801928548,5.13060203440526,13.0120085590576],"y":[2.70394002108872,2.4143341960484,1.98029424366401],"text":["GO:0005003 (16) <br> ephrin receptor activity <br> 1.977e-03","GO:0004714 (63) <br> transmembrane receptor protein tyrosine kinase activity <br> 3.852e-03","GO:0019199 (81) <br> transmembrane receptor protein kinase activity <br> 1.046e-02"],"key":["GO:0005003","GO:0004714","GO:0019199"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[14.1334713399187,16.4364350049252,16.8146555044192],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData85e2eb42","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,32.066467862776],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[34.5063337933754,45.410094637224],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[47.8499605678233,122.217074132492],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[124.656940063091,126.003746056782],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[128.443611987382,132.620662460568],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[135.060528391167,145.305525433754],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(85,116,166,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[147.745391364353,150.090102523659],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(34,170,153,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[152.529968454259,154.186637421136],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(102,170,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[156.626503351735,197.081920347003],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(153,0,153,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[199.521786277603,200.002439865931],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(0,153,198,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":61.6829876535158,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG","REAC","TF","MIRNA","CORUM","HP","WP"],"tickvals":[17.033233931388,39.9582142152997,85.0335173501577,125.330343059937,130.532137223975,140.183026912461,148.917746944006,153.358302937697,176.854211849369,199.762113071767],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG","REAC","TF","MIRNA","CORUM","HP","WP"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-37.3931091739311},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"18ed14944e86d":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"18ed15154ce10":{"x":{},"xend":{},"y":{},"yend":{}},"18ed17d6687f7":{"x":{},"xend":{},"y":{},"yend":{}},"18ed116fae1":{"x":{},"xend":{},"y":{},"yend":{}},"18ed123f5c017":{"x":{},"xend":{},"y":{},"yend":{}},"18ed1366d8395":{"x":{},"xend":{},"y":{},"yend":{}},"18ed151bde193":{"x":{},"xend":{},"y":{},"yend":{}},"18ed169943633":{"x":{},"xend":{},"y":{},"yend":{}},"18ed12eda8be5":{"x":{},"xend":{},"y":{},"yend":{}},"18ed152ba188":{"x":{},"xend":{},"y":{},"yend":{}},"18ed1fa43b69":{"x":{},"xend":{},"y":{},"yend":{}},"18ed1571dd882":{"x":{},"y":{}},"18ed13be57918":{"yintercept":{}}},"cur_data":"18ed14944e86d","visdat":{"18ed14944e86d":["function (y) ","x"],"18ed15154ce10":["function (y) ","x"],"18ed17d6687f7":["function (y) ","x"],"18ed116fae1":["function (y) ","x"],"18ed123f5c017":["function (y) ","x"],"18ed1366d8395":["function (y) ","x"],"18ed151bde193":["function (y) ","x"],"18ed169943633":["function (y) ","x"],"18ed12eda8be5":["function (y) ","x"],"18ed152ba188":["function (y) ","x"],"18ed1fa43b69":["function (y) ","x"],"18ed1571dd882":["function (y) ","x"],"18ed13be57918":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData85e2eb42"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>In addition to the standard gprofiler plot, the TimeSeries pipeline
creates Multi-Dimensional Scaling (MDS) plots. These plots illustrate a
custom selected ontology defined by the ‘my_ont_gpro’ parameter. The
first MDS plots shows all the found terms of the specified ontology and
distinguishes them based on the cluster(s) in which they are
found.<br>
In a MDS plot, the distance between each dot represents the semantic
similarity between terms<span class="citation">(Yu et al. 2010; Wang et
al. 2007)</span>. This can be loosely defined as the ‘closeness’ between
terms. Semantic similarity takes into account many factors, such as the
content of the terms (genes which belong to it), it’s associations to
other terms, the similarity or lack there-of in function etc…<br>
In an MDS plot, the dimensions will be adjusted to fill the space with
all the terms, therefor there is no set metric to measure the
similarity. The purpose of the MDS plot is moreso to help visualize
different groupings (or clusters) of relevant terms. Hovering the cursor
over the dots will provide specific information for that dot (term)</p>
<div id="htmlwidget-9ebdde91694f448ec2ad" style="width:100%;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-9ebdde91694f448ec2ad">{"x":{"visdat":{"18ed179f869d7":["function () ","plotlyVisDat"],"18ed143272a1a":["function () ","data"]},"cur_data":"18ed143272a1a","attrs":{"18ed143272a1a":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","color":{},"text":{},"showlegend":true,"colors":["#8DD3C7","#A0DAC3","#B5D9AA","#CDAD9A","#ACD5AC","#BBDDC3","#D99D8E","#C5D9A4","#DEA498","#E0AC8B","#CAE6BE","#C4DDA8","#B6BEAD","#CADFA8","#E2AF8D","#AFBCBF","#9BC2BA","#BAD9B9","#C0DAA7","#C2D8A5","#B1BD9C","#DFAC8B","#C6D3A2","#C4D0D6","#B3E1C0","#B3E1C0","#BFDCA9","#DDAE8D","#BFDCA9","#B9DD89","#B3BEAD","#D69F8F","#B8CDCD","#DFA48A","#FFFFB3","#D3D1CC","#E3D2BB","#E3CA8F","#E4C264","#B3E1C0","#D4A095","#D6B099","#CBD892","#BEBADA","#E9E8C0","#CCDEA9","#C9CDB6","#C6DBB8","#E1ECBC","#CAE6BE","#CADFA8","#C9CEC0","#EEB697","#D29092","#D3B387","#FB8072","#D2A6B7","#A9B2AD","#C1D1B3","#C5D4A2","#D29092","#DD988E","#D9F0B9","#C8E0A7","#D2E3A5","#BDC1AB","#E6B18B","#C5E791","#C1C8B5","#DEA393","#D5D1A0","#EAB895","#FDB462"],"marker":{"size":20,"opacity":0.4,"showlegend":true},"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":true},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"x":[-0.0161785417081161,-0.223270322806091,-0.243131763373523,-0.028275922115231,-0.195571645159296,-0.230244792235091,-0.242604613952785,-0.266574983267209,-0.218416546330182,0.361574452941396,-0.173032673487231,0.414647723210114,-0.225895436950611,-0.262461210498,-0.190624746078092,-0.0227576731956797,-0.223571266295368,0.382084891224153,-0.191078473257266,-0.220254589607802,-0.198953225320161,0.117987379828676,0.379610478654861,-0.201191699744645,-0.215787296696181,-0.0667493885883093,0.389988818138314,-0.162328374940106,-0.18272613625145,0.400612372704673,-0.0453911662415349,-0.202901819291908,0.377314780598281,-0.0206262922715615,-0.016130356803785,0.0428319357298962,0.421123599818818,0.00335333642290233,0.00637564933097656,0.0569002234436379,0.40742418776404,-0.000998186432859629,-0.186039711024433,0.124058987990313,0.132863511250272,-0.202282086855863,-0.199587479684491,-0.237148426390485,-0.00619509942114197,-0.191338107893973,-0.216249268543177,-0.214333587285643,0.0606149406551,-0.0653203388296526,-0.176659356379548,0.405544496961309,0.194190149725831,0.427318437753465],"y":[0.126997786138315,-0.0628841909341003,-0.0632452421170101,0.0332261022356963,-0.096533708464863,0.11960900217591,-0.0632069229406226,-0.063976351235232,0.0822712597925939,-0.169083710995788,-0.0890821856289681,-0.0585245810855303,0.0288563976082947,0.0269663768414609,-0.0672328894981083,-0.0790501218002016,-0.108702135806647,-0.048337428463097,-0.101663744456828,-0.073739710574597,0.0738769979955815,0.0923122482437842,-0.0373285080403615,-0.10054993676605,0.111594807627478,-0.0716202431158279,-0.181643146351637,-0.0905683564739561,-0.25363707601838,-0.19121768951238,-0.100784060426264,0.0786719905758131,-0.0453562037012529,0.0328155512170627,0.0436368449378889,0.163758378159738,-0.203633360184475,0.199708295135425,0.188288219228265,0.244702994527797,-0.198070982377271,0.215268151914892,-0.0931705754261084,0.104714863048948,-0.178804071431501,-0.106366096334006,-0.0282012751123436,0.00518712131660475,0.23556688470634,0.0216702401590861,0.0114216182065364,-0.00549265982328583,0.218043874457404,-0.115092361322308,-0.0974370292041101,-0.190556755430214,-0.0654690285340545,-0.205705467600846],"type":"scatter","mode":"markers","text":["modules/clusters: C1 <br>GO.ID: GO:0016310 <br>GO.name: phosphorylation","modules/clusters: C1 <br>GO.ID: GO:0042326 <br>GO.name: negative regulation of phosphorylation","modules/clusters: C1 <br>GO.ID: GO:0042325 <br>GO.name: regulation of phosphorylation","modules/clusters: C1 <br>GO.ID: GO:0006468 <br>GO.name: protein phosphorylation","modules/clusters: C1 <br>GO.ID: GO:0001932 <br>GO.name: regulation of protein phosphorylation","modules/clusters: C1 <br>GO.ID: GO:0000165 <br>GO.name: MAPK cascade","modules/clusters: C1 <br>GO.ID: GO:0045936 <br>GO.name: negative regulation of phosphate metabolic process","modules/clusters: C1 <br>GO.ID: GO:0010563 <br>GO.name: negative regulation of phosphorus metabolic process","modules/clusters: C1 <br>GO.ID: GO:0043408 <br>GO.name: regulation of MAPK cascade","modules/clusters: C1 <br>GO.ID: GO:0060412 <br>GO.name: ventricular septum morphogenesis","modules/clusters: C1 <br>GO.ID: GO:0001933 <br>GO.name: negative regulation of protein phosphorylation","modules/clusters: C1 <br>GO.ID: GO:0002521 <br>GO.name: leukocyte differentiation","modules/clusters: C1 <br>GO.ID: GO:0050804 <br>GO.name: modulation of chemical synaptic transmission","modules/clusters: C1 <br>GO.ID: GO:0099177 <br>GO.name: regulation of trans-synaptic signaling","modules/clusters: C1 <br>GO.ID: GO:0033673 <br>GO.name: negative regulation of kinase activity","modules/clusters: C1 <br>GO.ID: GO:0051348 <br>GO.name: negative regulation of transferase activity","modules/clusters: C1 <br>GO.ID: GO:0031399 <br>GO.name: regulation of protein modification process","modules/clusters: C1 <br>GO.ID: GO:0002329 <br>GO.name: pre-B cell differentiation","modules/clusters: C1 <br>GO.ID: GO:0045859 <br>GO.name: regulation of protein kinase activity","modules/clusters: C1 <br>GO.ID: GO:0043549 <br>GO.name: regulation of kinase activity","modules/clusters: C1 <br>GO.ID: GO:0043409 <br>GO.name: negative regulation of MAPK cascade","modules/clusters: C1 <br>GO.ID: GO:0009611 <br>GO.name: response to wounding","modules/clusters: C1 <br>GO.ID: GO:0030098 <br>GO.name: lymphocyte differentiation","modules/clusters: C1 <br>GO.ID: GO:0031400 <br>GO.name: negative regulation of protein modification process","modules/clusters: C1 <br>GO.ID: GO:0070371 <br>GO.name: ERK1 and ERK2 cascade","modules/clusters: C1 <br>GO.ID: GO:0045597 <br>GO.name: positive regulation of cell differentiation","modules/clusters: C1 <br>GO.ID: GO:0060411 <br>GO.name: cardiac septum morphogenesis","modules/clusters: C1 <br>GO.ID: GO:0006469 <br>GO.name: negative regulation of protein kinase activity","modules/clusters: C1 <br>GO.ID: GO:0051246 <br>GO.name: regulation of protein metabolic process","modules/clusters: C1 <br>GO.ID: GO:0003281 <br>GO.name: ventricular septum development","modules/clusters: C1 <br>GO.ID: GO:0051338 <br>GO.name: regulation of transferase activity","modules/clusters: C1 <br>GO.ID: GO:0070372 <br>GO.name: regulation of ERK1 and ERK2 cascade","modules/clusters: C1 <br>GO.ID: GO:0002327 <br>GO.name: immature B cell differentiation","modules/clusters: C1 <br>GO.ID: GO:0018108 <br>GO.name: peptidyl-tyrosine phosphorylation","modules/clusters: C1 <br>GO.ID: GO:0018212 <br>GO.name: peptidyl-tyrosine modification","modules/clusters: C1 <br>GO.ID: GO:0001783 <br>GO.name: B cell apoptotic process","modules/clusters: C1 <br>GO.ID: GO:0003007 <br>GO.name: heart morphogenesis","modules/clusters: C1 <br>GO.ID: GO:0098916 <br>GO.name: anterograde trans-synaptic signaling","modules/clusters: C1 <br>GO.ID: GO:0007268 <br>GO.name: chemical synaptic transmission","modules/clusters: C1 <br>GO.ID: GO:0071363 <br>GO.name: cellular response to growth factor stimulus","modules/clusters: C1 <br>GO.ID: GO:0048514 <br>GO.name: blood vessel morphogenesis","modules/clusters: C1 <br>GO.ID: GO:0099537 <br>GO.name: trans-synaptic signaling","modules/clusters: C1 <br>GO.ID: GO:0050730 <br>GO.name: regulation of peptidyl-tyrosine phosphorylation","modules/clusters: C1 <br>GO.ID: GO:0009628 <br>GO.name: response to abiotic stimulus","modules/clusters: C1 <br>GO.ID: GO:0002763 <br>GO.name: positive regulation of myeloid leukocyte differentiation","modules/clusters: C1 <br>GO.ID: GO:0071900 <br>GO.name: regulation of protein serine/threonine kinase activity","modules/clusters: C1 <br>GO.ID: GO:0042509 <br>GO.name: regulation of tyrosine phosphorylation of STAT protein","modules/clusters: C1 <br>GO.ID: GO:0045785 <br>GO.name: positive regulation of cell adhesion","modules/clusters: C1 <br>GO.ID: GO:0099536 <br>GO.name: synaptic signaling","modules/clusters: C1 <br>GO.ID: GO:0007260 <br>GO.name: tyrosine phosphorylation of STAT protein","modules/clusters: C1 <br>GO.ID: GO:0050806 <br>GO.name: positive regulation of synaptic transmission","modules/clusters: C1 <br>GO.ID: GO:0060291 <br>GO.name: long-term synaptic potentiation","modules/clusters: C1 <br>GO.ID: GO:0035733 <br>GO.name: hepatic stellate cell activation","modules/clusters: C1 <br>GO.ID: GO:0050790 <br>GO.name: regulation of catalytic activity","modules/clusters: C1 <br>GO.ID: GO:0071901 <br>GO.name: negative regulation of protein serine/threonine kinase activity","modules/clusters: C1 <br>GO.ID: GO:0003206 <br>GO.name: cardiac chamber morphogenesis","modules/clusters: C1 <br>GO.ID: GO:0003013 <br>GO.name: circulatory system process","modules/clusters: C1 <br>GO.ID: GO:0003231 <br>GO.name: cardiac ventricle development"],"showlegend":true,"marker":{"color":"rgba(141,211,199,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(141,211,199,1)"}},"name":"C1","textfont":{"color":"rgba(141,211,199,1)"},"error_y":{"color":"rgba(141,211,199,1)"},"error_x":{"color":"rgba(141,211,199,1)"},"line":{"color":"rgba(141,211,199,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.266630179200259,-0.296442861212508,0.4140210446583,-0.275853622937925,-0.213108185464394,-0.229810931245652],"y":[-0.0654336420152581,-0.0690751372785554,-0.0624365853151811,0.0530369901312988,-0.162902889310587,0.0867157006471449],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C15 <br>GO.ID: GO:0019220 <br>GO.name: regulation of phosphate metabolic process","modules/clusters: C1/C15 <br>GO.ID: GO:0051174 <br>GO.name: regulation of phosphorus metabolic process","modules/clusters: C1/C15 <br>GO.ID: GO:1903131 <br>GO.name: mononuclear cell differentiation","modules/clusters: C1/C15 <br>GO.ID: GO:0010648 <br>GO.name: negative regulation of cell communication","modules/clusters: C1/C15 <br>GO.ID: GO:0023057 <br>GO.name: negative regulation of signaling","modules/clusters: C1/C15 <br>GO.ID: GO:0009968 <br>GO.name: negative regulation of signal transduction"],"showlegend":true,"marker":{"color":"rgba(160,218,195,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(160,218,195,1)"}},"name":"C1/C15","textfont":{"color":"rgba(160,218,195,1)"},"error_y":{"color":"rgba(160,218,195,1)"},"error_x":{"color":"rgba(160,218,195,1)"},"line":{"color":"rgba(160,218,195,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.279524512556944],"y":[0.0268512619007968],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C15/C16 <br>GO.ID: GO:0010941 <br>GO.name: regulation of cell death","showlegend":true,"marker":{"color":"rgba(181,217,170,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(181,217,170,1)"}},"name":"C1/C15/C16","textfont":{"color":"rgba(181,217,170,1)"},"error_y":{"color":"rgba(181,217,170,1)"},"error_x":{"color":"rgba(181,217,170,1)"},"line":{"color":"rgba(181,217,170,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0626460725779088],"y":[0.298483864592333],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C15/C22 <br>GO.ID: GO:0001775 <br>GO.name: cell activation","showlegend":true,"marker":{"color":"rgba(205,173,154,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(205,173,154,1)"}},"name":"C1/C15/C22","textfont":{"color":"rgba(205,173,154,1)"},"error_y":{"color":"rgba(205,173,154,1)"},"error_x":{"color":"rgba(205,173,154,1)"},"line":{"color":"rgba(205,173,154,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.124142484555457,0.0512566391681413,-0.224697295693312],"y":[0.114282008244564,0.269611166540843,-0.00159573637039079],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C16 <br>GO.ID: GO:1901700 <br>GO.name: response to oxygen-containing compound","modules/clusters: C1/C16 <br>GO.ID: GO:1901701 <br>GO.name: cellular response to oxygen-containing compound","modules/clusters: C1/C16 <br>GO.ID: GO:0048167 <br>GO.name: regulation of synaptic plasticity"],"showlegend":true,"marker":{"color":"rgba(172,213,172,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(172,213,172,1)"}},"name":"C1/C16","textfont":{"color":"rgba(172,213,172,1)"},"error_y":{"color":"rgba(172,213,172,1)"},"error_x":{"color":"rgba(172,213,172,1)"},"line":{"color":"rgba(172,213,172,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.171132168049525],"y":[-0.119588263933234],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2 <br>GO.ID: GO:0048585 <br>GO.name: negative regulation of response to stimulus","showlegend":true,"marker":{"color":"rgba(187,221,195,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(187,221,195,1)"}},"name":"C1/C2","textfont":{"color":"rgba(187,221,195,1)"},"error_y":{"color":"rgba(187,221,195,1)"},"error_x":{"color":"rgba(187,221,195,1)"},"line":{"color":"rgba(187,221,195,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.508140304340205],"y":[-0.246964803869923],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C5/C6/C8/C15/C16/C20/C22 <br>GO.ID: GO:0048731 <br>GO.name: system development","showlegend":true,"marker":{"color":"rgba(217,157,142,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(217,157,142,1)"}},"name":"C1/C2/C5/C6/C8/C15/C16/C20/C22","textfont":{"color":"rgba(217,157,142,1)"},"error_y":{"color":"rgba(217,157,142,1)"},"error_x":{"color":"rgba(217,157,142,1)"},"line":{"color":"rgba(217,157,142,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.290350341456023],"y":[0.0111862613709993],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C6/C8/C15/C16 <br>GO.ID: GO:0048522 <br>GO.name: positive regulation of cellular process","showlegend":true,"marker":{"color":"rgba(197,217,164,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(197,217,164,1)"}},"name":"C1/C2/C6/C8/C15/C16","textfont":{"color":"rgba(197,217,164,1)"},"error_y":{"color":"rgba(197,217,164,1)"},"error_x":{"color":"rgba(197,217,164,1)"},"line":{"color":"rgba(197,217,164,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.226355468705033],"y":[-0.239218403544834],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C6/C8/C15/C16/C17/C22 <br>GO.ID: GO:0048518 <br>GO.name: positive regulation of biological process","showlegend":true,"marker":{"color":"rgba(222,164,152,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(222,164,152,1)"}},"name":"C1/C2/C6/C8/C15/C16/C17/C22","textfont":{"color":"rgba(222,164,152,1)"},"error_y":{"color":"rgba(222,164,152,1)"},"error_x":{"color":"rgba(222,164,152,1)"},"line":{"color":"rgba(222,164,152,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.320530348937678,-0.25885023414319],"y":[0.0295808286147487,-0.237925077312496],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C6/C8/C15/C16/C22 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process","modules/clusters: C1/C2/C6/C8/C15/C16/C22 <br>GO.ID: GO:0050789 <br>GO.name: regulation of biological process"],"showlegend":true,"marker":{"color":"rgba(224,172,139,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(224,172,139,1)"}},"name":"C1/C2/C6/C8/C15/C16/C22","textfont":{"color":"rgba(224,172,139,1)"},"error_y":{"color":"rgba(224,172,139,1)"},"error_x":{"color":"rgba(224,172,139,1)"},"line":{"color":"rgba(224,172,139,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.246264984470619],"y":[0.128045505301782],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C8 <br>GO.ID: GO:0035556 <br>GO.name: intracellular signal transduction","showlegend":true,"marker":{"color":"rgba(202,230,190,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(202,230,190,1)"}},"name":"C1/C2/C8","textfont":{"color":"rgba(202,230,190,1)"},"error_y":{"color":"rgba(202,230,190,1)"},"error_x":{"color":"rgba(202,230,190,1)"},"line":{"color":"rgba(202,230,190,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.00456677594826561],"y":[0.322626262540783],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C8/C15/C16 <br>GO.ID: GO:0007154 <br>GO.name: cell communication","showlegend":true,"marker":{"color":"rgba(196,221,168,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(196,221,168,1)"}},"name":"C1/C2/C8/C15/C16","textfont":{"color":"rgba(196,221,168,1)"},"error_y":{"color":"rgba(196,221,168,1)"},"error_x":{"color":"rgba(196,221,168,1)"},"line":{"color":"rgba(196,221,168,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.484465345890702],"y":[-0.231465710889899],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C8/C15/C16/C20 <br>GO.ID: GO:0007275 <br>GO.name: multicellular organism development","showlegend":true,"marker":{"color":"rgba(182,190,173,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(182,190,173,1)"}},"name":"C1/C2/C8/C15/C16/C20","textfont":{"color":"rgba(182,190,173,1)"},"error_y":{"color":"rgba(182,190,173,1)"},"error_x":{"color":"rgba(182,190,173,1)"},"line":{"color":"rgba(182,190,173,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0788329538128166,0.408110518878206,0.0356728561119201],"y":[0.0624081411979373,-0.144169344941022,0.357064690792011],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C2/C8/C16 <br>GO.ID: GO:0023052 <br>GO.name: signaling","modules/clusters: C1/C2/C8/C16 <br>GO.ID: GO:0009653 <br>GO.name: anatomical structure morphogenesis","modules/clusters: C1/C2/C8/C16 <br>GO.ID: GO:0051716 <br>GO.name: cellular response to stimulus"],"showlegend":true,"marker":{"color":"rgba(202,223,168,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(202,223,168,1)"}},"name":"C1/C2/C8/C16","textfont":{"color":"rgba(202,223,168,1)"},"error_y":{"color":"rgba(202,223,168,1)"},"error_x":{"color":"rgba(202,223,168,1)"},"line":{"color":"rgba(202,223,168,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.137963643975026],"y":[0.126922282703451],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C2/C8/C16/C22 <br>GO.ID: GO:0050896 <br>GO.name: response to stimulus","showlegend":true,"marker":{"color":"rgba(226,175,141,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(226,175,141,1)"}},"name":"C1/C2/C8/C16/C22","textfont":{"color":"rgba(226,175,141,1)"},"error_y":{"color":"rgba(226,175,141,1)"},"error_x":{"color":"rgba(226,175,141,1)"},"line":{"color":"rgba(226,175,141,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.204022230675155,0.138321510272265,0.153894825084312],"y":[-0.338369921344781,-0.191279509834788,-0.195819736267842],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C3 <br>GO.ID: GO:1903708 <br>GO.name: positive regulation of hemopoiesis","modules/clusters: C1/C3 <br>GO.ID: GO:1902107 <br>GO.name: positive regulation of leukocyte differentiation","modules/clusters: C1/C3 <br>GO.ID: GO:1902105 <br>GO.name: regulation of leukocyte differentiation"],"showlegend":true,"marker":{"color":"rgba(175,188,191,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(175,188,191,1)"}},"name":"C1/C3","textfont":{"color":"rgba(175,188,191,1)"},"error_y":{"color":"rgba(175,188,191,1)"},"error_x":{"color":"rgba(175,188,191,1)"},"line":{"color":"rgba(175,188,191,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.214105662535955],"y":[-0.279011533690546],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C5 <br>GO.ID: GO:0010605 <br>GO.name: negative regulation of macromolecule metabolic process","showlegend":true,"marker":{"color":"rgba(155,194,186,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(155,194,186,1)"}},"name":"C1/C5","textfont":{"color":"rgba(155,194,186,1)"},"error_y":{"color":"rgba(155,194,186,1)"},"error_x":{"color":"rgba(155,194,186,1)"},"line":{"color":"rgba(155,194,186,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.250082226947167,-0.296840556093559],"y":[-0.274194427374125,-0.0645365404085802],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C5/C8 <br>GO.ID: GO:0009892 <br>GO.name: negative regulation of metabolic process","modules/clusters: C1/C5/C8 <br>GO.ID: GO:0031324 <br>GO.name: negative regulation of cellular metabolic process"],"showlegend":true,"marker":{"color":"rgba(186,217,185,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(186,217,185,1)"}},"name":"C1/C5/C8","textfont":{"color":"rgba(186,217,185,1)"},"error_y":{"color":"rgba(186,217,185,1)"},"error_x":{"color":"rgba(186,217,185,1)"},"line":{"color":"rgba(186,217,185,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.294736183273545],"y":[0.0393396591872841],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C5/C8/C15/C16 <br>GO.ID: GO:0048523 <br>GO.name: negative regulation of cellular process","showlegend":true,"marker":{"color":"rgba(192,218,167,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(192,218,167,1)"}},"name":"C1/C5/C8/C15/C16","textfont":{"color":"rgba(192,218,167,1)"},"error_y":{"color":"rgba(192,218,167,1)"},"error_x":{"color":"rgba(192,218,167,1)"},"line":{"color":"rgba(192,218,167,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0536156330326999,0.126328112553287,0.268970034599503],"y":[0.276249295313897,0.121347832665604,-0.0878116057352496],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C5/C8/C16 <br>GO.ID: GO:0071310 <br>GO.name: cellular response to organic substance","modules/clusters: C1/C5/C8/C16 <br>GO.ID: GO:0010033 <br>GO.name: response to organic substance","modules/clusters: C1/C5/C8/C16 <br>GO.ID: GO:0032501 <br>GO.name: multicellular organismal process"],"showlegend":true,"marker":{"color":"rgba(194,216,165,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(194,216,165,1)"}},"name":"C1/C5/C8/C16","textfont":{"color":"rgba(194,216,165,1)"},"error_y":{"color":"rgba(194,216,165,1)"},"error_x":{"color":"rgba(194,216,165,1)"},"line":{"color":"rgba(194,216,165,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.337312652340743,0.243493852808158,0.23676981152856],"y":[-0.0773638834255852,0.147102334552938,0.172769562567132],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C5/C8/C16/C19/C20 <br>GO.ID: GO:0032502 <br>GO.name: developmental process","modules/clusters: C1/C5/C8/C16/C19/C20 <br>GO.ID: GO:0030154 <br>GO.name: cell differentiation","modules/clusters: C1/C5/C8/C16/C19/C20 <br>GO.ID: GO:0048869 <br>GO.name: cellular developmental process"],"showlegend":true,"marker":{"color":"rgba(177,189,156,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(177,189,156,1)"}},"name":"C1/C5/C8/C16/C19/C20","textfont":{"color":"rgba(177,189,156,1)"},"error_y":{"color":"rgba(177,189,156,1)"},"error_x":{"color":"rgba(177,189,156,1)"},"line":{"color":"rgba(177,189,156,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.118967687795074],"y":[-0.14574486646323],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C6/C8/C15/C16/C22 <br>GO.ID: GO:0065007 <br>GO.name: biological regulation","showlegend":true,"marker":{"color":"rgba(223,172,139,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(223,172,139,1)"}},"name":"C1/C6/C8/C15/C16/C22","textfont":{"color":"rgba(223,172,139,1)"},"error_y":{"color":"rgba(223,172,139,1)"},"error_x":{"color":"rgba(223,172,139,1)"},"line":{"color":"rgba(223,172,139,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.204142829926259],"y":[-0.198639490943728],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C6/C8/C16 <br>GO.ID: GO:0032879 <br>GO.name: regulation of localization","showlegend":true,"marker":{"color":"rgba(198,211,162,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(198,211,162,1)"}},"name":"C1/C6/C8/C16","textfont":{"color":"rgba(198,211,162,1)"},"error_y":{"color":"rgba(198,211,162,1)"},"error_x":{"color":"rgba(198,211,162,1)"},"line":{"color":"rgba(198,211,162,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0131053722331399,-0.026652130954543,0.0305940352251386],"y":[-0.0879895814431675,0.0382242292938415,-0.0916890273103629],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C7 <br>GO.ID: GO:0036211 <br>GO.name: protein modification process","modules/clusters: C1/C7 <br>GO.ID: GO:0006464 <br>GO.name: cellular protein modification process","modules/clusters: C1/C7 <br>GO.ID: GO:0043412 <br>GO.name: macromolecule modification"],"showlegend":true,"marker":{"color":"rgba(196,208,214,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(196,208,214,1)"}},"name":"C1/C7","textfont":{"color":"rgba(196,208,214,1)"},"error_y":{"color":"rgba(196,208,214,1)"},"error_x":{"color":"rgba(196,208,214,1)"},"line":{"color":"rgba(196,208,214,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.241414844933165,0.477829410589664,-0.0256502428460945,-0.0131157282509335,-0.230067151967805,-0.021255992352791,0.125366263073537,-0.204549011808811,-0.236956028820194,0.46177898760823,-0.267219774202345,0.46366870603572,0.429105719919858,0.0570400763688319,0.119829316859823,0.115601765170899],"y":[-0.3097988603919,-0.235009249276657,0.163345864724286,0.260831401737922,0.119641964968197,0.142726152658529,0.106144121368593,-0.294107517717781,0.0861916430111671,-0.222386227910247,0.0243626098760633,-0.224479926087157,-0.203827140096003,0.191907449519521,0.108370662823456,0.0889299041142643],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C8 <br>GO.ID: GO:0060255 <br>GO.name: regulation of macromolecule metabolic process","modules/clusters: C1/C8 <br>GO.ID: GO:0072359 <br>GO.name: circulatory system development","modules/clusters: C1/C8 <br>GO.ID: GO:0006793 <br>GO.name: phosphorus metabolic process","modules/clusters: C1/C8 <br>GO.ID: GO:0007267 <br>GO.name: cell-cell signaling","modules/clusters: C1/C8 <br>GO.ID: GO:0007167 <br>GO.name: enzyme-linked receptor protein signaling pathway","modules/clusters: C1/C8 <br>GO.ID: GO:0006796 <br>GO.name: phosphate-containing compound metabolic process","modules/clusters: C1/C8 <br>GO.ID: GO:0009719 <br>GO.name: response to endogenous stimulus","modules/clusters: C1/C8 <br>GO.ID: GO:0010468 <br>GO.name: regulation of gene expression","modules/clusters: C1/C8 <br>GO.ID: GO:1902531 <br>GO.name: regulation of intracellular signal transduction","modules/clusters: C1/C8 <br>GO.ID: GO:0030097 <br>GO.name: hemopoiesis","modules/clusters: C1/C8 <br>GO.ID: GO:0030155 <br>GO.name: regulation of cell adhesion","modules/clusters: C1/C8 <br>GO.ID: GO:0035295 <br>GO.name: tube development","modules/clusters: C1/C8 <br>GO.ID: GO:0035239 <br>GO.name: tube morphogenesis","modules/clusters: C1/C8 <br>GO.ID: GO:0016477 <br>GO.name: cell migration","modules/clusters: C1/C8 <br>GO.ID: GO:0070848 <br>GO.name: response to growth factor","modules/clusters: C1/C8 <br>GO.ID: GO:0071495 <br>GO.name: cellular response to endogenous stimulus"],"showlegend":true,"marker":{"color":"rgba(179,225,192,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(179,225,192,1)"}},"name":"C1/C8","textfont":{"color":"rgba(179,225,192,1)"},"error_y":{"color":"rgba(179,225,192,1)"},"error_x":{"color":"rgba(179,225,192,1)"},"line":{"color":"rgba(179,225,192,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.257377956121647],"y":[0.0885631350848305],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C8/C15 <br>GO.ID: GO:0009966 <br>GO.name: regulation of signal transduction","showlegend":true,"marker":{"color":"rgba(179,225,192,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(179,225,192,1)"}},"name":"C1/C8/C15","textfont":{"color":"rgba(179,225,192,1)"},"error_y":{"color":"rgba(179,225,192,1)"},"error_x":{"color":"rgba(179,225,192,1)"},"line":{"color":"rgba(179,225,192,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.244766405433296,-0.303129970171887,-0.0910931759169269,-0.22869772801645,0.472231981667549,0.180311529232245,-0.101564392047531,0.476835871419538,-0.044825380320676],"y":[-0.190432549816579,0.0496206196772459,-0.131185303922139,-0.212549319482494,-0.225773231860339,-0.390764155874458,-0.316252127683712,-0.221406127693213,-0.305987340966134],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:0023051 <br>GO.name: regulation of signaling","modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:0010646 <br>GO.name: regulation of cell communication","modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:0065008 <br>GO.name: regulation of biological quality","modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:0048519 <br>GO.name: negative regulation of biological process","modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:0048534 <br>GO.name: hematopoietic or lymphoid organ development","modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:2000026 <br>GO.name: regulation of multicellular organismal development","modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:0051239 <br>GO.name: regulation of multicellular organismal process","modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:0002520 <br>GO.name: immune system development","modules/clusters: C1/C8/C15/C16 <br>GO.ID: GO:0050793 <br>GO.name: regulation of developmental process"],"showlegend":true,"marker":{"color":"rgba(191,220,169,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(191,220,169,1)"}},"name":"C1/C8/C15/C16","textfont":{"color":"rgba(191,220,169,1)"},"error_y":{"color":"rgba(191,220,169,1)"},"error_x":{"color":"rgba(191,220,169,1)"},"line":{"color":"rgba(191,220,169,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.24653771466036,-0.198450978118125],"y":[0.128151535036071,-0.142293321083654],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C8/C15/C16/C22 <br>GO.ID: GO:0007166 <br>GO.name: cell surface receptor signaling pathway","modules/clusters: C1/C8/C15/C16/C22 <br>GO.ID: GO:0048583 <br>GO.name: regulation of response to stimulus"],"showlegend":true,"marker":{"color":"rgba(221,174,141,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(221,174,141,1)"}},"name":"C1/C8/C15/C16/C22","textfont":{"color":"rgba(221,174,141,1)"},"error_y":{"color":"rgba(221,174,141,1)"},"error_x":{"color":"rgba(221,174,141,1)"},"line":{"color":"rgba(221,174,141,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0473187704071406,-0.280753830517897,-0.212981020828673,-0.272718441218752,-0.242099529922549,0.133848671399756,0.035177551303102,-0.0270156418758351],"y":[0.31479673144754,-0.304467243053492,-0.187073012654175,0.0285581609035092,-0.280242819630419,0.130462989304582,0.271070751388516,-0.297667033052845],"type":"scatter","mode":"markers","text":["modules/clusters: C1/C8/C16 <br>GO.ID: GO:0070887 <br>GO.name: cellular response to chemical stimulus","modules/clusters: C1/C8/C16 <br>GO.ID: GO:0019222 <br>GO.name: regulation of metabolic process","modules/clusters: C1/C8/C16 <br>GO.ID: GO:0023056 <br>GO.name: positive regulation of signaling","modules/clusters: C1/C8/C16 <br>GO.ID: GO:0010647 <br>GO.name: positive regulation of cell communication","modules/clusters: C1/C8/C16 <br>GO.ID: GO:0051171 <br>GO.name: regulation of nitrogen compound metabolic process","modules/clusters: C1/C8/C16 <br>GO.ID: GO:0042221 <br>GO.name: response to chemical","modules/clusters: C1/C8/C16 <br>GO.ID: GO:0007155 <br>GO.name: cell adhesion","modules/clusters: C1/C8/C16 <br>GO.ID: GO:0051094 <br>GO.name: positive regulation of developmental process"],"showlegend":true,"marker":{"color":"rgba(191,220,169,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(191,220,169,1)"}},"name":"C1/C8/C16","textfont":{"color":"rgba(191,220,169,1)"},"error_y":{"color":"rgba(191,220,169,1)"},"error_x":{"color":"rgba(191,220,169,1)"},"line":{"color":"rgba(191,220,169,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.499648903437183],"y":[-0.241971403015566],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C8/C16/C19 <br>GO.ID: GO:0048513 <br>GO.name: animal organ development","showlegend":true,"marker":{"color":"rgba(185,221,137,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(185,221,137,1)"}},"name":"C1/C8/C16/C19","textfont":{"color":"rgba(185,221,137,1)"},"error_y":{"color":"rgba(185,221,137,1)"},"error_x":{"color":"rgba(185,221,137,1)"},"line":{"color":"rgba(185,221,137,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.432470569248686],"y":[-0.151913922169408],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C8/C16/C20 <br>GO.ID: GO:0048856 <br>GO.name: anatomical structure development","showlegend":true,"marker":{"color":"rgba(179,190,173,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(179,190,173,1)"}},"name":"C1/C8/C16/C20","textfont":{"color":"rgba(179,190,173,1)"},"error_y":{"color":"rgba(179,190,173,1)"},"error_x":{"color":"rgba(179,190,173,1)"},"line":{"color":"rgba(179,190,173,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.12984458042319],"y":[0.109965087432169],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C8/C16/C20/C22 <br>GO.ID: GO:0009605 <br>GO.name: response to external stimulus","showlegend":true,"marker":{"color":"rgba(214,159,143,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(214,159,143,1)"}},"name":"C1/C8/C16/C20/C22","textfont":{"color":"rgba(214,159,143,1)"},"error_y":{"color":"rgba(214,159,143,1)"},"error_x":{"color":"rgba(214,159,143,1)"},"line":{"color":"rgba(214,159,143,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.238389531231542],"y":[-0.275022701403868],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C8/C17 <br>GO.ID: GO:0080090 <br>GO.name: regulation of primary metabolic process","showlegend":true,"marker":{"color":"rgba(184,205,205,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(184,205,205,1)"}},"name":"C1/C8/C17","textfont":{"color":"rgba(184,205,205,1)"},"error_y":{"color":"rgba(184,205,205,1)"},"error_x":{"color":"rgba(184,205,205,1)"},"line":{"color":"rgba(184,205,205,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.113105158534471],"y":[0.081256892556606],"type":"scatter","mode":"markers","text":"modules/clusters: C1/C8/C21/C22 <br>GO.ID: GO:0006954 <br>GO.name: inflammatory response","showlegend":true,"marker":{"color":"rgba(223,164,138,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(223,164,138,1)"}},"name":"C1/C8/C21/C22","textfont":{"color":"rgba(223,164,138,1)"},"error_y":{"color":"rgba(223,164,138,1)"},"error_x":{"color":"rgba(223,164,138,1)"},"line":{"color":"rgba(223,164,138,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0353534976882142],"y":[0.254440625160269],"type":"scatter","mode":"markers","text":"modules/clusters: C11 <br>GO.ID: GO:0051310 <br>GO.name: metaphase plate congression","showlegend":true,"marker":{"color":"rgba(255,255,179,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(255,255,179,1)"}},"name":"C11","textfont":{"color":"rgba(255,255,179,1)"},"error_y":{"color":"rgba(255,255,179,1)"},"error_x":{"color":"rgba(255,255,179,1)"},"line":{"color":"rgba(255,255,179,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0478365557777343,0.047394534077842,0.0403049237597885,0.046355993561694,0.0406296231713,0.0511543867119958,-0.0119954177024064,0.0435861584291522,-0.0251912032479853,0.0365163355449799,-0.0133093663680706,0.0467315998470658,0.0330895647377344,0.0347161327503611,0.0506580090038724,0.0379730621237315,0.0407511648781821,0.046476300538944,-0.0122212176627963,-0.00310092826262037,0.0593986106054899,-0.0032417387560509,0.00673896411060611,-0.000499265902189303],"y":[-0.0472979208367367,-0.0383260807709672,-0.0495817548962774,-0.048799272565863,-0.0592899838722915,-0.0546429861961215,0.0665318895482788,-0.0537197017432663,0.107616168035909,-0.054799324429448,0.101705970028346,-0.0623941023965545,-0.0671502772734681,-0.0746763636699041,-0.0465004438108133,-0.0628151379491128,-0.0654574304240836,-0.0722102758874905,0.0793181945647922,0.0537420445698467,-0.046400377990039,0.0643190848212523,0.0309396777879155,0.0508198748145883],"type":"scatter","mode":"markers","text":["modules/clusters: C12 <br>GO.ID: GO:1902653 <br>GO.name: secondary alcohol biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0006695 <br>GO.name: cholesterol biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0016126 <br>GO.name: sterol biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0008203 <br>GO.name: cholesterol metabolic process","modules/clusters: C12 <br>GO.ID: GO:0016125 <br>GO.name: sterol metabolic process","modules/clusters: C12 <br>GO.ID: GO:1902652 <br>GO.name: secondary alcohol metabolic process","modules/clusters: C12 <br>GO.ID: GO:0008299 <br>GO.name: isoprenoid biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0046165 <br>GO.name: alcohol biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0044255 <br>GO.name: cellular lipid metabolic process","modules/clusters: C12 <br>GO.ID: GO:0006694 <br>GO.name: steroid biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0006720 <br>GO.name: isoprenoid metabolic process","modules/clusters: C12 <br>GO.ID: GO:0006066 <br>GO.name: alcohol metabolic process","modules/clusters: C12 <br>GO.ID: GO:0008610 <br>GO.name: lipid biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0006629 <br>GO.name: lipid metabolic process","modules/clusters: C12 <br>GO.ID: GO:0044283 <br>GO.name: small molecule biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0008202 <br>GO.name: steroid metabolic process","modules/clusters: C12 <br>GO.ID: GO:1901617 <br>GO.name: organic hydroxy compound biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:1901615 <br>GO.name: organic hydroxy compound metabolic process","modules/clusters: C12 <br>GO.ID: GO:0046490 <br>GO.name: isopentenyl diphosphate metabolic process","modules/clusters: C12 <br>GO.ID: GO:0009240 <br>GO.name: isopentenyl diphosphate biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0044281 <br>GO.name: small molecule metabolic process","modules/clusters: C12 <br>GO.ID: GO:0016114 <br>GO.name: terpenoid biosynthetic process","modules/clusters: C12 <br>GO.ID: GO:0019287 <br>GO.name: isopentenyl diphosphate biosynthetic process, mevalonate pathway","modules/clusters: C12 <br>GO.ID: GO:0045337 <br>GO.name: farnesyl diphosphate biosynthetic process"],"showlegend":true,"marker":{"color":"rgba(211,209,204,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(211,209,204,1)"}},"name":"C12","textfont":{"color":"rgba(211,209,204,1)"},"error_y":{"color":"rgba(211,209,204,1)"},"error_x":{"color":"rgba(211,209,204,1)"},"line":{"color":"rgba(211,209,204,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0492272845622385,0.0454114717667907,0.0408264319176594,0.0393015172247869],"y":[0.284941081961889,0.27860201025437,0.299743060726646,0.335722979770252],"type":"scatter","mode":"markers","text":["modules/clusters: C13 <br>GO.ID: GO:0090307 <br>GO.name: mitotic spindle assembly","modules/clusters: C13 <br>GO.ID: GO:0051256 <br>GO.name: mitotic spindle midzone assembly","modules/clusters: C13 <br>GO.ID: GO:0000022 <br>GO.name: mitotic spindle elongation","modules/clusters: C13 <br>GO.ID: GO:0051231 <br>GO.name: spindle elongation"],"showlegend":true,"marker":{"color":"rgba(227,210,187,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(227,210,187,1)"}},"name":"C13","textfont":{"color":"rgba(227,210,187,1)"},"error_y":{"color":"rgba(227,210,187,1)"},"error_x":{"color":"rgba(227,210,187,1)"},"line":{"color":"rgba(227,210,187,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0392529232056434,0.0448044932593125,0.0347879540028935,0.0374636068030503,0.0566086911628806,0.0332523612379003,0.0495845087197054,0.0452996038771927,0.0575419049864956,0.0313927483422753,0.0396214527576609,0.04251311279592,0.0526899009058236,0.0431093343940874],"y":[0.344151545689311,0.335711377030518,0.326953638850265,0.345316189727592,0.293178593286416,0.339101398767781,0.305185022218598,0.323384049541269,0.313653192349891,0.33395342640134,0.326372213247291,0.316790574428695,0.316176507110763,0.350047938887647],"type":"scatter","mode":"markers","text":["modules/clusters: C13/C14 <br>GO.ID: GO:0140014 <br>GO.name: mitotic nuclear division","modules/clusters: C13/C14 <br>GO.ID: GO:1902850 <br>GO.name: microtubule cytoskeleton organization involved in mitosis","modules/clusters: C13/C14 <br>GO.ID: GO:0000070 <br>GO.name: mitotic sister chromatid segregation","modules/clusters: C13/C14 <br>GO.ID: GO:0000819 <br>GO.name: sister chromatid segregation","modules/clusters: C13/C14 <br>GO.ID: GO:0000280 <br>GO.name: nuclear division","modules/clusters: C13/C14 <br>GO.ID: GO:1903047 <br>GO.name: mitotic cell cycle process","modules/clusters: C13/C14 <br>GO.ID: GO:0051225 <br>GO.name: spindle assembly","modules/clusters: C13/C14 <br>GO.ID: GO:0007052 <br>GO.name: mitotic spindle organization","modules/clusters: C13/C14 <br>GO.ID: GO:0048285 <br>GO.name: organelle fission","modules/clusters: C13/C14 <br>GO.ID: GO:0098813 <br>GO.name: nuclear chromosome segregation","modules/clusters: C13/C14 <br>GO.ID: GO:0000278 <br>GO.name: mitotic cell cycle","modules/clusters: C13/C14 <br>GO.ID: GO:0007059 <br>GO.name: chromosome segregation","modules/clusters: C13/C14 <br>GO.ID: GO:0000226 <br>GO.name: microtubule cytoskeleton organization","modules/clusters: C13/C14 <br>GO.ID: GO:0007051 <br>GO.name: spindle organization"],"showlegend":true,"marker":{"color":"rgba(227,202,143,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(227,202,143,1)"}},"name":"C13/C14","textfont":{"color":"rgba(227,202,143,1)"},"error_y":{"color":"rgba(227,202,143,1)"},"error_x":{"color":"rgba(227,202,143,1)"},"line":{"color":"rgba(227,202,143,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0463822908013892,0.0333710237459035,-0.276226070493544,-0.230053425270939,-0.0945428118333085,-0.109447393450542,-0.14128070005683,-0.142664722985458,-0.150627929191401,-0.177294745552915,0.0338580624905904,0.0371891718146972,-0.166163166371885,-0.208640784981955,0.0394436583277636,0.035533359346875,0.0404162964291739,-0.203349248739554,0.0350194394141098,0.0315315437117665,-0.138600186640326,-0.159172461822694,-0.195085898939547,-0.0675138219629308,-0.203991373204456,-0.0624032538533067,-0.117010886994485,-0.136771520338046,-0.106337246649294,-0.280340019152653,-0.141393697654784,-0.178734330961766,-0.248070059830834,-0.135551578944202,-0.116850365344756,-0.16001402785262,0.0500190505469888,-0.270715992959426,-0.168410951363302,-0.184324689991326,-0.187319955335507,0.0608590840944263,-0.215728290114519,0.0586436193768024,0.0320137172352669,-0.236174443915144,-0.167531928460799,0.0484043525857957,0.048135817751181,0.0412059194364561,0.0557537197195463,-0.167533904877206],"y":[0.298922567377406,0.371165328933,0.0482969663892127,0.134388644296252,0.162605984595344,0.181656781061125,0.183220013127016,0.154313152991676,0.174669196215481,0.121219026058914,0.316794656501,0.281857297632512,0.166426469061915,0.11054708671269,0.355115344449583,0.313587230557087,0.284062492196547,0.0991992417242351,0.301112632669492,0.324320174159915,0.185386188779431,0.124361391415187,0.0795795403243929,0.14538341366927,0.0967773340301608,0.141765031891515,0.169155041085778,0.15141411846952,0.159211646884997,0.0788981966643325,0.144925949998188,0.102945295459209,0.0508102583758275,0.186207664068166,0.16555593433224,0.110878617563147,0.308670213267669,0.0293721724844316,0.105598704734219,0.118132524015856,0.116089502749212,0.314252825911348,0.120300760294901,0.341206433427324,0.316477397598691,0.0882229900171383,0.14229771667788,0.25546189332335,0.281584738450128,0.283077979489907,0.312147197229217,0.113685718684707],"type":"scatter","mode":"markers","text":["modules/clusters: C14 <br>GO.ID: GO:0051301 <br>GO.name: cell division","modules/clusters: C14 <br>GO.ID: GO:0022402 <br>GO.name: cell cycle process","modules/clusters: C14 <br>GO.ID: GO:0051983 <br>GO.name: regulation of chromosome segregation","modules/clusters: C14 <br>GO.ID: GO:0010564 <br>GO.name: regulation of cell cycle process","modules/clusters: C14 <br>GO.ID: GO:0030071 <br>GO.name: regulation of mitotic metaphase/anaphase transition","modules/clusters: C14 <br>GO.ID: GO:0007091 <br>GO.name: metaphase/anaphase transition of mitotic cell cycle","modules/clusters: C14 <br>GO.ID: GO:0010965 <br>GO.name: regulation of mitotic sister chromatid separation","modules/clusters: C14 <br>GO.ID: GO:1902099 <br>GO.name: regulation of metaphase/anaphase transition of cell cycle","modules/clusters: C14 <br>GO.ID: GO:0044784 <br>GO.name: metaphase/anaphase transition of cell cycle","modules/clusters: C14 <br>GO.ID: GO:0051783 <br>GO.name: regulation of nuclear division","modules/clusters: C14 <br>GO.ID: GO:0051306 <br>GO.name: mitotic sister chromatid separation","modules/clusters: C14 <br>GO.ID: GO:0000281 <br>GO.name: mitotic cytokinesis","modules/clusters: C14 <br>GO.ID: GO:0033045 <br>GO.name: regulation of sister chromatid segregation","modules/clusters: C14 <br>GO.ID: GO:1905818 <br>GO.name: regulation of chromosome separation","modules/clusters: C14 <br>GO.ID: GO:0007049 <br>GO.name: cell cycle","modules/clusters: C14 <br>GO.ID: GO:0000910 <br>GO.name: cytokinesis","modules/clusters: C14 <br>GO.ID: GO:0061640 <br>GO.name: cytoskeleton-dependent cytokinesis","modules/clusters: C14 <br>GO.ID: GO:0032465 <br>GO.name: regulation of cytokinesis","modules/clusters: C14 <br>GO.ID: GO:0032506 <br>GO.name: cytokinetic process","modules/clusters: C14 <br>GO.ID: GO:0051304 <br>GO.name: chromosome separation","modules/clusters: C14 <br>GO.ID: GO:0007088 <br>GO.name: regulation of mitotic nuclear division","modules/clusters: C14 <br>GO.ID: GO:0071174 <br>GO.name: mitotic spindle checkpoint signaling","modules/clusters: C14 <br>GO.ID: GO:0071173 <br>GO.name: spindle assembly checkpoint signaling","modules/clusters: C14 <br>GO.ID: GO:0007094 <br>GO.name: mitotic spindle assembly checkpoint signaling","modules/clusters: C14 <br>GO.ID: GO:0031577 <br>GO.name: spindle checkpoint signaling","modules/clusters: C14 <br>GO.ID: GO:0045841 <br>GO.name: negative regulation of mitotic metaphase/anaphase transition","modules/clusters: C14 <br>GO.ID: GO:0033048 <br>GO.name: negative regulation of mitotic sister chromatid segregation","modules/clusters: C14 <br>GO.ID: GO:0033046 <br>GO.name: negative regulation of sister chromatid segregation","modules/clusters: C14 <br>GO.ID: GO:2000816 <br>GO.name: negative regulation of mitotic sister chromatid separation","modules/clusters: C14 <br>GO.ID: GO:0051726 <br>GO.name: regulation of cell cycle","modules/clusters: C14 <br>GO.ID: GO:1902100 <br>GO.name: negative regulation of metaphase/anaphase transition of cell cycle","modules/clusters: C14 <br>GO.ID: GO:1905819 <br>GO.name: negative regulation of chromosome separation","modules/clusters: C14 <br>GO.ID: GO:0051985 <br>GO.name: negative regulation of chromosome segregation","modules/clusters: C14 <br>GO.ID: GO:0033047 <br>GO.name: regulation of mitotic sister chromatid segregation","modules/clusters: C14 <br>GO.ID: GO:0045839 <br>GO.name: negative regulation of mitotic nuclear division","modules/clusters: C14 <br>GO.ID: GO:0051784 <br>GO.name: negative regulation of nuclear division","modules/clusters: C14 <br>GO.ID: GO:0007017 <br>GO.name: microtubule-based process","modules/clusters: C14 <br>GO.ID: GO:0051302 <br>GO.name: regulation of cell division","modules/clusters: C14 <br>GO.ID: GO:2001251 <br>GO.name: negative regulation of chromosome organization","modules/clusters: C14 <br>GO.ID: GO:0032954 <br>GO.name: regulation of cytokinetic process","modules/clusters: C14 <br>GO.ID: GO:0033044 <br>GO.name: regulation of chromosome organization","modules/clusters: C14 <br>GO.ID: GO:0007010 <br>GO.name: cytoskeleton organization","modules/clusters: C14 <br>GO.ID: GO:0033043 <br>GO.name: regulation of organelle organization","modules/clusters: C14 <br>GO.ID: GO:0006996 <br>GO.name: organelle organization","modules/clusters: C14 <br>GO.ID: GO:0008608 <br>GO.name: attachment of spindle microtubules to kinetochore","modules/clusters: C14 <br>GO.ID: GO:0007346 <br>GO.name: regulation of mitotic cell cycle","modules/clusters: C14 <br>GO.ID: GO:0007093 <br>GO.name: mitotic cell cycle checkpoint signaling","modules/clusters: C14 <br>GO.ID: GO:0000915 <br>GO.name: actomyosin contractile ring assembly","modules/clusters: C14 <br>GO.ID: GO:0000912 <br>GO.name: assembly of actomyosin apparatus involved in cytokinesis","modules/clusters: C14 <br>GO.ID: GO:0044837 <br>GO.name: actomyosin contractile ring organization","modules/clusters: C14 <br>GO.ID: GO:0051276 <br>GO.name: chromosome organization","modules/clusters: C14 <br>GO.ID: GO:1901990 <br>GO.name: regulation of mitotic cell cycle phase transition"],"showlegend":true,"marker":{"color":"rgba(228,194,100,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(228,194,100,1)"}},"name":"C14","textfont":{"color":"rgba(228,194,100,1)"},"error_y":{"color":"rgba(228,194,100,1)"},"error_x":{"color":"rgba(228,194,100,1)"},"line":{"color":"rgba(228,194,100,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.1155445494146,0.105561308656312,-0.204332311585561,0.118972788303869,0.355806108147315,0.336163056924056,0.337080748902134,0.117425604790419,-0.198658123033335,0.353447293191572,-0.117115766578724,-0.135746095682905,0.364304287169582,-0.172084578382965,0.335487438297533,0.341389644801723,-0.173960454402488,-0.175962635585427,0.011925576416428],"y":[0.190202188863893,0.224515068722122,0.00181228455710257,0.161314665692437,-0.0199354263030519,-0.0182589428036413,-0.0164461483733137,0.151678028261755,0.0818629902206206,-0.0141873266214965,-0.0721828051227653,-0.0777593916029707,-0.0459460360841216,0.0693920087415681,-0.0172099685883395,-0.0109464637731917,-0.00225672587446879,0.0727681136510411,0.0384811177901458],"type":"scatter","mode":"markers","text":["modules/clusters: C15 <br>GO.ID: GO:0002366 <br>GO.name: leukocyte activation involved in immune response","modules/clusters: C15 <br>GO.ID: GO:0002263 <br>GO.name: cell activation involved in immune response","modules/clusters: C15 <br>GO.ID: GO:0050679 <br>GO.name: positive regulation of epithelial cell proliferation","modules/clusters: C15 <br>GO.ID: GO:0002286 <br>GO.name: T cell activation involved in immune response","modules/clusters: C15 <br>GO.ID: GO:0042093 <br>GO.name: T-helper cell differentiation","modules/clusters: C15 <br>GO.ID: GO:0002294 <br>GO.name: CD4-positive, alpha-beta T cell differentiation involved in immune response","modules/clusters: C15 <br>GO.ID: GO:0002293 <br>GO.name: alpha-beta T cell differentiation involved in immune response","modules/clusters: C15 <br>GO.ID: GO:0002287 <br>GO.name: alpha-beta T cell activation involved in immune response","modules/clusters: C15 <br>GO.ID: GO:2001236 <br>GO.name: regulation of extrinsic apoptotic signaling pathway","modules/clusters: C15 <br>GO.ID: GO:0002292 <br>GO.name: T cell differentiation involved in immune response","modules/clusters: C15 <br>GO.ID: GO:1900373 <br>GO.name: positive regulation of purine nucleotide biosynthetic process","modules/clusters: C15 <br>GO.ID: GO:0030810 <br>GO.name: positive regulation of nucleotide biosynthetic process","modules/clusters: C15 <br>GO.ID: GO:0043367 <br>GO.name: CD4-positive, alpha-beta T cell differentiation","modules/clusters: C15 <br>GO.ID: GO:1902042 <br>GO.name: negative regulation of extrinsic apoptotic signaling pathway via death domain receptors","modules/clusters: C15 <br>GO.ID: GO:0072539 <br>GO.name: T-helper 17 cell differentiation","modules/clusters: C15 <br>GO.ID: GO:0002313 <br>GO.name: mature B cell differentiation involved in immune response","modules/clusters: C15 <br>GO.ID: GO:0001938 <br>GO.name: positive regulation of endothelial cell proliferation","modules/clusters: C15 <br>GO.ID: GO:2001237 <br>GO.name: negative regulation of extrinsic apoptotic signaling pathway","modules/clusters: C15 <br>GO.ID: GO:0010706 <br>GO.name: ganglioside biosynthetic process via lactosylceramide"],"showlegend":true,"marker":{"color":"rgba(179,225,192,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(179,225,192,1)"}},"name":"C15","textfont":{"color":"rgba(179,225,192,1)"},"error_y":{"color":"rgba(179,225,192,1)"},"error_x":{"color":"rgba(179,225,192,1)"},"line":{"color":"rgba(179,225,192,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.165749169468933],"y":[0.0765412974448541],"type":"scatter","mode":"markers","text":"modules/clusters: C15/C20/C22 <br>GO.ID: GO:0006955 <br>GO.name: immune response","showlegend":true,"marker":{"color":"rgba(212,160,149,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(212,160,149,1)"}},"name":"C15/C20/C22","textfont":{"color":"rgba(212,160,149,1)"},"error_y":{"color":"rgba(212,160,149,1)"},"error_x":{"color":"rgba(212,160,149,1)"},"line":{"color":"rgba(212,160,149,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.118974777996229,0.167045169001436],"y":[0.173742405881461,0.00481793290276407],"type":"scatter","mode":"markers","text":["modules/clusters: C15/C22 <br>GO.ID: GO:0002285 <br>GO.name: lymphocyte activation involved in immune response","modules/clusters: C15/C22 <br>GO.ID: GO:0002252 <br>GO.name: immune effector process"],"showlegend":true,"marker":{"color":"rgba(214,176,153,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(214,176,153,1)"}},"name":"C15/C22","textfont":{"color":"rgba(214,176,153,1)"},"error_y":{"color":"rgba(214,176,153,1)"},"error_x":{"color":"rgba(214,176,153,1)"},"line":{"color":"rgba(214,176,153,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0338275604103571,0.0361275519564813,-0.205105762651629,0.0336642782005009,-0.238103367230821,0.371721618885029,0.294914198742033,-0.252282026447885,0.319991446983079,-0.23955681030042,0.260797457038854,0.265301419558701,0.283300127713228,-0.0801911785609591,-0.211912258018663,0.30098438636034,-0.0704295431786343,-0.210051594248381,0.0709459729069992,0.340120525603928,0.0674886622498601,-0.18443055543211,-0.201197621443185,-0.213524024616556,0.334125451030143,-0.175523878670832,0.336702284999705,0.119427932595127,-0.14891588740375,-0.230157591523098,0.288281520912692,0.393402337835816,-0.114885577702432,0.306595303236721,0.352163842321672,0.0631821427933102,0.0411658755815349,0.368452026300829,0.275492303552395,0.274336140748745,-0.234754281532721],"y":[0.220172228923195,0.295110404119458,0.0232148549343465,0.25240821315605,0.0247656009408312,-0.128198785793218,0.130977632841943,0.0307794860705873,0.0117632465102261,0.0290315383367999,0.0909820973585147,0.0933701192477738,0.119799610968189,-0.279679383517219,0.115912339146282,0.00688405892220775,-0.0462232535224654,0.0206505529143226,0.264403534302966,0.0211370856776691,0.30243619067278,0.021132627102301,-0.201968085848787,0.0251070410793093,-0.01148211323358,0.107174760955106,0.0256183603456548,0.107032942367103,-0.118404185581206,0.124742686160566,-0.0684367929617648,-0.0408734239225984,-0.101977620831838,0.0431702190959076,-0.0337009047166763,0.262374850944,0.294611908364788,-0.024202189932987,0.0704285528576924,0.0953833840555502,0.0201074860286872],"type":"scatter","mode":"markers","text":["modules/clusters: C16 <br>GO.ID: GO:0006915 <br>GO.name: apoptotic process","modules/clusters: C16 <br>GO.ID: GO:0008219 <br>GO.name: cell death","modules/clusters: C16 <br>GO.ID: GO:0042981 <br>GO.name: regulation of apoptotic process","modules/clusters: C16 <br>GO.ID: GO:0012501 <br>GO.name: programmed cell death","modules/clusters: C16 <br>GO.ID: GO:0043067 <br>GO.name: regulation of programmed cell death","modules/clusters: C16 <br>GO.ID: GO:0000902 <br>GO.name: cell morphogenesis","modules/clusters: C16 <br>GO.ID: GO:0032989 <br>GO.name: cellular component morphogenesis","modules/clusters: C16 <br>GO.ID: GO:0060548 <br>GO.name: negative regulation of cell death","modules/clusters: C16 <br>GO.ID: GO:0048812 <br>GO.name: neuron projection morphogenesis","modules/clusters: C16 <br>GO.ID: GO:0007162 <br>GO.name: negative regulation of cell adhesion","modules/clusters: C16 <br>GO.ID: GO:0120039 <br>GO.name: plasma membrane bounded cell projection morphogenesis","modules/clusters: C16 <br>GO.ID: GO:0048858 <br>GO.name: cell projection morphogenesis","modules/clusters: C16 <br>GO.ID: GO:0032990 <br>GO.name: cell part morphogenesis","modules/clusters: C16 <br>GO.ID: GO:0051241 <br>GO.name: negative regulation of multicellular organismal process","modules/clusters: C16 <br>GO.ID: GO:0097191 <br>GO.name: extrinsic apoptotic signaling pathway","modules/clusters: C16 <br>GO.ID: GO:0007409 <br>GO.name: axonogenesis","modules/clusters: C16 <br>GO.ID: GO:0045596 <br>GO.name: negative regulation of cell differentiation","modules/clusters: C16 <br>GO.ID: GO:1901215 <br>GO.name: negative regulation of neuron death","modules/clusters: C16 <br>GO.ID: GO:0120036 <br>GO.name: plasma membrane bounded cell projection organization","modules/clusters: C16 <br>GO.ID: GO:0061564 <br>GO.name: axon development","modules/clusters: C16 <br>GO.ID: GO:0030030 <br>GO.name: cell projection organization","modules/clusters: C16 <br>GO.ID: GO:0043066 <br>GO.name: negative regulation of apoptotic process","modules/clusters: C16 <br>GO.ID: GO:0040012 <br>GO.name: regulation of locomotion","modules/clusters: C16 <br>GO.ID: GO:0043069 <br>GO.name: negative regulation of programmed cell death","modules/clusters: C16 <br>GO.ID: GO:0007416 <br>GO.name: synapse assembly","modules/clusters: C16 <br>GO.ID: GO:0008630 <br>GO.name: intrinsic apoptotic signaling pathway in response to DNA damage","modules/clusters: C16 <br>GO.ID: GO:0031175 <br>GO.name: neuron projection development","modules/clusters: C16 <br>GO.ID: GO:0033993 <br>GO.name: response to lipid","modules/clusters: C16 <br>GO.ID: GO:0032101 <br>GO.name: regulation of response to external stimulus","modules/clusters: C16 <br>GO.ID: GO:0097190 <br>GO.name: apoptotic signaling pathway","modules/clusters: C16 <br>GO.ID: GO:0021700 <br>GO.name: developmental maturation","modules/clusters: C16 <br>GO.ID: GO:0048699 <br>GO.name: generation of neurons","modules/clusters: C16 <br>GO.ID: GO:0031347 <br>GO.name: regulation of defense response","modules/clusters: C16 <br>GO.ID: GO:0000904 <br>GO.name: cell morphogenesis involved in differentiation","modules/clusters: C16 <br>GO.ID: GO:0048667 <br>GO.name: cell morphogenesis involved in neuron differentiation","modules/clusters: C16 <br>GO.ID: GO:0050808 <br>GO.name: synapse organization","modules/clusters: C16 <br>GO.ID: GO:0071216 <br>GO.name: cellular response to biotic stimulus","modules/clusters: C16 <br>GO.ID: GO:0048666 <br>GO.name: neuron development","modules/clusters: C16 <br>GO.ID: GO:0007411 <br>GO.name: axon guidance","modules/clusters: C16 <br>GO.ID: GO:0097485 <br>GO.name: neuron projection guidance","modules/clusters: C16 <br>GO.ID: GO:1901214 <br>GO.name: regulation of neuron death"],"showlegend":true,"marker":{"color":"rgba(203,216,146,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(203,216,146,1)"}},"name":"C16","textfont":{"color":"rgba(203,216,146,1)"},"error_y":{"color":"rgba(203,216,146,1)"},"error_x":{"color":"rgba(203,216,146,1)"},"line":{"color":"rgba(203,216,146,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0183724247128165],"y":[0.0519290767906285],"type":"scatter","mode":"markers","text":"modules/clusters: C17 <br>GO.ID: GO:0006260 <br>GO.name: DNA replication","showlegend":true,"marker":{"color":"rgba(190,186,218,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(190,186,218,1)"}},"name":"C17","textfont":{"color":"rgba(190,186,218,1)"},"error_y":{"color":"rgba(190,186,218,1)"},"error_x":{"color":"rgba(190,186,218,1)"},"line":{"color":"rgba(190,186,218,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.245202322821936,-0.172011644086788,-0.213559933421348,0.128338586366715,-0.0709306293110192,-0.0845925123815687,-0.174920745892544,-0.248913472610118,-0.238647540633028,-0.200003316214663,-0.205542477726703,-0.0806211569816747,-0.0191119861683237,-0.204927086419799,-0.0606144606913684,0.1287960268846,-0.22956228657807,-0.0378405532290553,-0.193193310008374,-0.239626393454968,-0.0967613817227339,-0.101096987871456,-0.0283600476066226,0.129536691990879,0.0178425281623764],"y":[0.00731948829846528,-0.00507613573729,-0.103963997707968,-0.0811921411538241,-0.0676890998294016,-0.0178039170845644,-0.00354664101649897,0.0533087665656118,-0.0902836556445135,-0.00153052685232566,-0.267087835009331,-0.018176329025034,-0.185358527156932,0.000541247995701605,-0.009143921716299,0.0337255825111307,0.00891348812993618,-0.231825202105135,0.0144792227010913,-0.0955016580330027,-0.201268824164254,-0.243293041054153,-0.198964084860412,-0.0820897227009921,0.162314466345336],"type":"scatter","mode":"markers","text":["modules/clusters: C2 <br>GO.ID: GO:0008284 <br>GO.name: positive regulation of cell population proliferation","modules/clusters: C2 <br>GO.ID: GO:1903039 <br>GO.name: positive regulation of leukocyte cell-cell adhesion","modules/clusters: C2 <br>GO.ID: GO:0045935 <br>GO.name: positive regulation of nucleobase-containing compound metabolic process","modules/clusters: C2 <br>GO.ID: GO:0035743 <br>GO.name: CD4-positive, alpha-beta T cell cytokine production","modules/clusters: C2 <br>GO.ID: GO:0050728 <br>GO.name: negative regulation of inflammatory response","modules/clusters: C2 <br>GO.ID: GO:0050671 <br>GO.name: positive regulation of lymphocyte proliferation","modules/clusters: C2 <br>GO.ID: GO:0032946 <br>GO.name: positive regulation of mononuclear cell proliferation","modules/clusters: C2 <br>GO.ID: GO:0045787 <br>GO.name: positive regulation of cell cycle","modules/clusters: C2 <br>GO.ID: GO:0031328 <br>GO.name: positive regulation of cellular biosynthetic process","modules/clusters: C2 <br>GO.ID: GO:0022409 <br>GO.name: positive regulation of cell-cell adhesion","modules/clusters: C2 <br>GO.ID: GO:0009891 <br>GO.name: positive regulation of biosynthetic process","modules/clusters: C2 <br>GO.ID: GO:0050870 <br>GO.name: positive regulation of T cell activation","modules/clusters: C2 <br>GO.ID: GO:0002726 <br>GO.name: positive regulation of T cell cytokine production","modules/clusters: C2 <br>GO.ID: GO:0070665 <br>GO.name: positive regulation of leukocyte proliferation","modules/clusters: C2 <br>GO.ID: GO:0042102 <br>GO.name: positive regulation of T cell proliferation","modules/clusters: C2 <br>GO.ID: GO:0002456 <br>GO.name: T cell mediated immunity","modules/clusters: C2 <br>GO.ID: GO:0050867 <br>GO.name: positive regulation of cell activation","modules/clusters: C2 <br>GO.ID: GO:0002720 <br>GO.name: positive regulation of cytokine production involved in immune response","modules/clusters: C2 <br>GO.ID: GO:1903037 <br>GO.name: regulation of leukocyte cell-cell adhesion","modules/clusters: C2 <br>GO.ID: GO:0019219 <br>GO.name: regulation of nucleobase-containing compound metabolic process","modules/clusters: C2 <br>GO.ID: GO:0002699 <br>GO.name: positive regulation of immune effector process","modules/clusters: C2 <br>GO.ID: GO:0002702 <br>GO.name: positive regulation of production of molecular mediator of immune response","modules/clusters: C2 <br>GO.ID: GO:0002724 <br>GO.name: regulation of T cell cytokine production","modules/clusters: C2 <br>GO.ID: GO:0002369 <br>GO.name: T cell cytokine production","modules/clusters: C2 <br>GO.ID: GO:0000707 <br>GO.name: meiotic DNA recombinase assembly"],"showlegend":true,"marker":{"color":"rgba(233,232,192,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(233,232,192,1)"}},"name":"C2","textfont":{"color":"rgba(233,232,192,1)"},"error_y":{"color":"rgba(233,232,192,1)"},"error_x":{"color":"rgba(233,232,192,1)"},"line":{"color":"rgba(233,232,192,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0909736603473671],"y":[-0.131757867642756],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C15/C16 <br>GO.ID: GO:0065009 <br>GO.name: regulation of molecular function","showlegend":true,"marker":{"color":"rgba(204,222,169,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(204,222,169,1)"}},"name":"C2/C15/C16","textfont":{"color":"rgba(204,222,169,1)"},"error_y":{"color":"rgba(204,222,169,1)"},"error_x":{"color":"rgba(204,222,169,1)"},"line":{"color":"rgba(204,222,169,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.172196375924146],"y":[-0.123441962296346],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C5 <br>GO.ID: GO:0006357 <br>GO.name: regulation of transcription by RNA polymerase II","showlegend":true,"marker":{"color":"rgba(201,205,182,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(201,205,182,1)"}},"name":"C2/C5","textfont":{"color":"rgba(201,205,182,1)"},"error_y":{"color":"rgba(201,205,182,1)"},"error_x":{"color":"rgba(201,205,182,1)"},"line":{"color":"rgba(201,205,182,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0411187942134471],"y":[0.293619319254919],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C6/C8/C15 <br>GO.ID: GO:0008283 <br>GO.name: cell population proliferation","showlegend":true,"marker":{"color":"rgba(198,219,184,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(198,219,184,1)"}},"name":"C2/C6/C8/C15","textfont":{"color":"rgba(198,219,184,1)"},"error_y":{"color":"rgba(198,219,184,1)"},"error_x":{"color":"rgba(198,219,184,1)"},"line":{"color":"rgba(198,219,184,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.212625799208334,-0.248878626496601,-0.293488360607865,-0.212843983425982,0.374784077831175],"y":[-0.301538301011068,-0.30130493510618,-0.0893729428572905,-0.272749909879319,-0.131046990280962],"type":"scatter","mode":"markers","text":["modules/clusters: C2/C8 <br>GO.ID: GO:0010604 <br>GO.name: positive regulation of macromolecule metabolic process","modules/clusters: C2/C8 <br>GO.ID: GO:0009893 <br>GO.name: positive regulation of metabolic process","modules/clusters: C2/C8 <br>GO.ID: GO:0031325 <br>GO.name: positive regulation of cellular metabolic process","modules/clusters: C2/C8 <br>GO.ID: GO:0051173 <br>GO.name: positive regulation of nitrogen compound metabolic process","modules/clusters: C2/C8 <br>GO.ID: GO:0048646 <br>GO.name: anatomical structure formation involved in morphogenesis"],"showlegend":true,"marker":{"color":"rgba(225,236,188,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(225,236,188,1)"}},"name":"C2/C8","textfont":{"color":"rgba(225,236,188,1)"},"error_y":{"color":"rgba(225,236,188,1)"},"error_x":{"color":"rgba(225,236,188,1)"},"line":{"color":"rgba(225,236,188,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.275202560695804],"y":[0.0277485407884326],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C8/C15 <br>GO.ID: GO:0042127 <br>GO.name: regulation of cell population proliferation","showlegend":true,"marker":{"color":"rgba(202,230,190,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(202,230,190,1)"}},"name":"C2/C8/C15","textfont":{"color":"rgba(202,230,190,1)"},"error_y":{"color":"rgba(202,230,190,1)"},"error_x":{"color":"rgba(202,230,190,1)"},"line":{"color":"rgba(202,230,190,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.262597066148897],"y":[0.136072111255479],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C8/C15/C16 <br>GO.ID: GO:0007165 <br>GO.name: signal transduction","showlegend":true,"marker":{"color":"rgba(202,223,168,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(202,223,168,1)"}},"name":"C2/C8/C15/C16","textfont":{"color":"rgba(202,223,168,1)"},"error_y":{"color":"rgba(202,223,168,1)"},"error_x":{"color":"rgba(202,223,168,1)"},"line":{"color":"rgba(202,223,168,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.3320173767323],"y":[-0.0730958762845034],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C8/C16/C17 <br>GO.ID: GO:0031323 <br>GO.name: regulation of cellular metabolic process","showlegend":true,"marker":{"color":"rgba(201,206,192,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(201,206,192,1)"}},"name":"C2/C8/C16/C17","textfont":{"color":"rgba(201,206,192,1)"},"error_y":{"color":"rgba(201,206,192,1)"},"error_x":{"color":"rgba(201,206,192,1)"},"line":{"color":"rgba(201,206,192,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.127753216173339],"y":[0.109745716402344],"type":"scatter","mode":"markers","text":"modules/clusters: C2/C8/C22 <br>GO.ID: GO:0006950 <br>GO.name: response to stress","showlegend":true,"marker":{"color":"rgba(238,182,151,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(238,182,151,1)"}},"name":"C2/C8/C22","textfont":{"color":"rgba(238,182,151,1)"},"error_y":{"color":"rgba(238,182,151,1)"},"error_x":{"color":"rgba(238,182,151,1)"},"line":{"color":"rgba(238,182,151,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.113009541479987,0.116667161419924,0.127190179207534,0.108433106872925],"y":[0.0775901468957696,0.0913295688908335,0.109472841768002,0.0753529963269621],"type":"scatter","mode":"markers","text":["modules/clusters: C20/C22 <br>GO.ID: GO:0051707 <br>GO.name: response to other organism","modules/clusters: C20/C22 <br>GO.ID: GO:0043207 <br>GO.name: response to external biotic stimulus","modules/clusters: C20/C22 <br>GO.ID: GO:0009607 <br>GO.name: response to biotic stimulus","modules/clusters: C20/C22 <br>GO.ID: GO:0098542 <br>GO.name: defense response to other organism"],"showlegend":true,"marker":{"color":"rgba(210,144,146,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(210,144,146,1)"}},"name":"C20/C22","textfont":{"color":"rgba(210,144,146,1)"},"error_y":{"color":"rgba(210,144,146,1)"},"error_x":{"color":"rgba(210,144,146,1)"},"line":{"color":"rgba(210,144,146,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0624725347742246],"y":[0.300946354642431],"type":"scatter","mode":"markers","text":"modules/clusters: C21 <br>GO.ID: GO:0034330 <br>GO.name: cell junction organization","showlegend":true,"marker":{"color":"rgba(211,179,135,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(211,179,135,1)"}},"name":"C21","textfont":{"color":"rgba(211,179,135,1)"},"error_y":{"color":"rgba(211,179,135,1)"},"error_x":{"color":"rgba(211,179,135,1)"},"line":{"color":"rgba(211,179,135,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.069003710298024,0.136169103345907,0.165726340275163,-0.10568268084425,-0.089645448250559,0.130711209167149,0.144861443495115,0.366383623172057,0.111742141077868,0.148355606272865,0.115081303082835,-0.0572062669605104,0.108271531387158,0.113760342193733,-0.118358847210163,0.0764568093738559,-0.160736391799381,0.153819112890867,-0.0785878411854934,0.118932349542597,-0.119702170596364,-0.073504329519806,0.0465274572235801,0.0841766501812099,-0.0795940158730519,0.127715748171863,-0.270706877668409,-0.183401665968757,0.0592384710043325,-0.0500823744111244,-0.0399080567593482,0.0823221261042082,0.0615365551567929],"y":[-0.164410750664106,-0.00478150628793296,-0.153508892067698,-0.00838688469760453,-0.18185777384803,0.0419367418276403,0.0548334772006217,-0.0442124928815413,0.208291030829893,-0.0015898540733557,0.188315965837093,-0.161025821484666,0.1594617088229,0.172219937430804,-0.205626606755028,0.184590165739217,-0.0134741658559588,0.0640204335624798,-0.123839231909859,0.107516110871191,-0.0106851053421029,-0.177540112541517,0.28930182719267,-0.0432715788178814,-0.0180574216626599,-0.134727470758615,0.0219736446020869,0.129492648995869,0.244066095617474,-0.150437659219665,-0.14614697587042,-0.030615373885923,0.301005806453388],"type":"scatter","mode":"markers","text":["modules/clusters: C22 <br>GO.ID: GO:0002706 <br>GO.name: regulation of lymphocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0002449 <br>GO.name: lymphocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0045586 <br>GO.name: regulation of gamma-delta T cell differentiation","modules/clusters: C22 <br>GO.ID: GO:0046643 <br>GO.name: regulation of gamma-delta T cell activation","modules/clusters: C22 <br>GO.ID: GO:0002703 <br>GO.name: regulation of leukocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0044419 <br>GO.name: biological process involved in interspecies interaction between organisms","modules/clusters: C22 <br>GO.ID: GO:0002460 <br>GO.name: adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains","modules/clusters: C22 <br>GO.ID: GO:0042492 <br>GO.name: gamma-delta T cell differentiation","modules/clusters: C22 <br>GO.ID: GO:0045321 <br>GO.name: leukocyte activation","modules/clusters: C22 <br>GO.ID: GO:0002443 <br>GO.name: leukocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0046649 <br>GO.name: lymphocyte activation","modules/clusters: C22 <br>GO.ID: GO:0002708 <br>GO.name: positive regulation of lymphocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0046629 <br>GO.name: gamma-delta T cell activation","modules/clusters: C22 <br>GO.ID: GO:0042110 <br>GO.name: T cell activation","modules/clusters: C22 <br>GO.ID: GO:0002697 <br>GO.name: regulation of immune effector process","modules/clusters: C22 <br>GO.ID: GO:0001909 <br>GO.name: leukocyte mediated cytotoxicity","modules/clusters: C22 <br>GO.ID: GO:0001910 <br>GO.name: regulation of leukocyte mediated cytotoxicity","modules/clusters: C22 <br>GO.ID: GO:0002250 <br>GO.name: adaptive immune response","modules/clusters: C22 <br>GO.ID: GO:0002819 <br>GO.name: regulation of adaptive immune response","modules/clusters: C22 <br>GO.ID: GO:0034097 <br>GO.name: response to cytokine","modules/clusters: C22 <br>GO.ID: GO:0050863 <br>GO.name: regulation of T cell activation","modules/clusters: C22 <br>GO.ID: GO:0002705 <br>GO.name: positive regulation of leukocyte mediated immunity","modules/clusters: C22 <br>GO.ID: GO:0001906 <br>GO.name: cell killing","modules/clusters: C22 <br>GO.ID: GO:0002381 <br>GO.name: immunoglobulin production involved in immunoglobulin-mediated immune response","modules/clusters: C22 <br>GO.ID: GO:0046645 <br>GO.name: positive regulation of gamma-delta T cell activation","modules/clusters: C22 <br>GO.ID: GO:0045588 <br>GO.name: positive regulation of gamma-delta T cell differentiation","modules/clusters: C22 <br>GO.ID: GO:0031341 <br>GO.name: regulation of cell killing","modules/clusters: C22 <br>GO.ID: GO:0019221 <br>GO.name: cytokine-mediated signaling pathway","modules/clusters: C22 <br>GO.ID: GO:0030198 <br>GO.name: extracellular matrix organization","modules/clusters: C22 <br>GO.ID: GO:2000557 <br>GO.name: regulation of immunoglobulin production in mucosal tissue","modules/clusters: C22 <br>GO.ID: GO:2000558 <br>GO.name: positive regulation of immunoglobulin production in mucosal tissue","modules/clusters: C22 <br>GO.ID: GO:0002426 <br>GO.name: immunoglobulin production in mucosal tissue","modules/clusters: C22 <br>GO.ID: GO:0045229 <br>GO.name: external encapsulating structure organization"],"showlegend":true,"marker":{"color":"rgba(251,128,114,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(251,128,114,1)"}},"name":"C22","textfont":{"color":"rgba(251,128,114,1)"},"error_y":{"color":"rgba(251,128,114,1)"},"error_x":{"color":"rgba(251,128,114,1)"},"line":{"color":"rgba(251,128,114,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.163045057129955],"y":[-0.161879045140129],"type":"scatter","mode":"markers","text":"modules/clusters: C3 <br>GO.ID: GO:0045619 <br>GO.name: regulation of lymphocyte differentiation","showlegend":true,"marker":{"color":"rgba(210,166,183,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(210,166,183,1)"}},"name":"C3","textfont":{"color":"rgba(210,166,183,1)"},"error_y":{"color":"rgba(210,166,183,1)"},"error_x":{"color":"rgba(210,166,183,1)"},"line":{"color":"rgba(210,166,183,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.147573138232562,-0.1578129047582,-0.150555306038968,-0.0203254661570997],"y":[-0.109143216941137,-0.0981482309712686,-0.093570228704782,0.00312332282343726],"type":"scatter","mode":"markers","text":["modules/clusters: C5 <br>GO.ID: GO:0045892 <br>GO.name: negative regulation of transcription, DNA-templated","modules/clusters: C5 <br>GO.ID: GO:1903507 <br>GO.name: negative regulation of nucleic acid-templated transcription","modules/clusters: C5 <br>GO.ID: GO:1902679 <br>GO.name: negative regulation of RNA biosynthetic process","modules/clusters: C5 <br>GO.ID: GO:0006366 <br>GO.name: transcription by RNA polymerase II"],"showlegend":true,"marker":{"color":"rgba(169,178,173,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(169,178,173,1)"}},"name":"C5","textfont":{"color":"rgba(169,178,173,1)"},"error_y":{"color":"rgba(169,178,173,1)"},"error_x":{"color":"rgba(169,178,173,1)"},"line":{"color":"rgba(169,178,173,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.213977011968806,-0.155951428401298,-0.159336351584981,-0.242312433294067,-0.20638459289089],"y":[-0.250375072882947,-0.113427342899746,-0.228397604228619,-0.0714766847818247,-0.244452237502327],"type":"scatter","mode":"markers","text":["modules/clusters: C5/C8 <br>GO.ID: GO:0051172 <br>GO.name: negative regulation of nitrogen compound metabolic process","modules/clusters: C5/C8 <br>GO.ID: GO:0000122 <br>GO.name: negative regulation of transcription by RNA polymerase II","modules/clusters: C5/C8 <br>GO.ID: GO:0010558 <br>GO.name: negative regulation of macromolecule biosynthetic process","modules/clusters: C5/C8 <br>GO.ID: GO:0031327 <br>GO.name: negative regulation of cellular biosynthetic process","modules/clusters: C5/C8 <br>GO.ID: GO:0009890 <br>GO.name: negative regulation of biosynthetic process"],"showlegend":true,"marker":{"color":"rgba(193,209,179,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(193,209,179,1)"}},"name":"C5/C8","textfont":{"color":"rgba(193,209,179,1)"},"error_y":{"color":"rgba(193,209,179,1)"},"error_x":{"color":"rgba(193,209,179,1)"},"line":{"color":"rgba(193,209,179,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.32855878576429],"y":[0.0632321257728434],"type":"scatter","mode":"markers","text":"modules/clusters: C5/C8/C16 <br>GO.ID: GO:0048468 <br>GO.name: cell development","showlegend":true,"marker":{"color":"rgba(197,212,162,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(197,212,162,1)"}},"name":"C5/C8/C16","textfont":{"color":"rgba(197,212,162,1)"},"error_y":{"color":"rgba(197,212,162,1)"},"error_x":{"color":"rgba(197,212,162,1)"},"line":{"color":"rgba(197,212,162,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0903339622407092,0.172609734791863],"y":[-0.293689413433858,-0.280726912502859],"type":"scatter","mode":"markers","text":["modules/clusters: C6 <br>GO.ID: GO:0032755 <br>GO.name: positive regulation of interleukin-6 production","modules/clusters: C6 <br>GO.ID: GO:1903672 <br>GO.name: positive regulation of sprouting angiogenesis"],"showlegend":true,"marker":{"color":"rgba(210,144,146,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(210,144,146,1)"}},"name":"C6","textfont":{"color":"rgba(210,144,146,1)"},"error_y":{"color":"rgba(210,144,146,1)"},"error_x":{"color":"rgba(210,144,146,1)"},"line":{"color":"rgba(210,144,146,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.180591648338251],"y":[-0.285098235004144],"type":"scatter","mode":"markers","text":"modules/clusters: C6/C8/C20/C22 <br>GO.ID: GO:0010628 <br>GO.name: positive regulation of gene expression","showlegend":true,"marker":{"color":"rgba(221,152,142,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(221,152,142,1)"}},"name":"C6/C8/C20/C22","textfont":{"color":"rgba(221,152,142,1)"},"error_y":{"color":"rgba(221,152,142,1)"},"error_x":{"color":"rgba(221,152,142,1)"},"line":{"color":"rgba(221,152,142,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.103278988232025,0.116514679234077,-0.228758068068123,-0.0630360837190362,0.107856756194963,0.478337590046251,0.094997119396988,0.0996202130449052,-0.0766835686958535,0.0455251470895982,0.05489707808188,0.00383536583249235,0.459305274616224,0.0504420945749947,-0.213059268451519,0.0459240226912966,0.0473058371221714,0.0451055961112884,0.447572485080736,0.0177570635361965,0.095922719892223,-0.146204048147827,0.0575209381452853,-0.0409999093338378,-0.160892036497424,0.0539889581815516,0.0421698223945573,-0.212403740482722,0.097182265559411,-0.154867744770091,-0.153514469436138,-0.214193840954264,-0.14726427539638,0.418967691163688,0.212359024473919,0.101031492438935,0.262132839784658,0.471160906959254,0.0540429697231113,-0.152326926629686,-0.0629392322572786,0.123291022851874,0.0507562582097669,0.13222310007749],"y":[0.0429066721231445,0.0483212940328822,0.0671990740778546,-0.11296463189148,0.0458754873595748,-0.229648054153659,0.0334397474095417,0.0391023374310498,-0.269635333904345,0.27444962084081,0.209128266097183,0.200174327520754,-0.226850198091732,0.215843382016838,0.0624312534056383,0.259224939788934,0.212663027504772,0.258156481072059,-0.214368793407491,0.0786525943093377,0.0336846678364041,-0.151279771760925,0.142471293686037,-0.0971606166781312,-0.127660064608278,0.184920682718046,0.248826941481535,0.0230750886443617,0.0361856658692417,-0.110465989363996,-0.123836321014874,0.110709234101735,-0.105786840898439,-0.196155172993782,-0.359749053352421,0.0408309000490861,-0.0640516777793324,-0.230113485496353,0.196650460556391,-0.170424543434577,-0.112744935630772,0.0328352673246808,-0.0552151217833395,0.0391046092267189],"type":"scatter","mode":"markers","text":["modules/clusters: C8 <br>GO.ID: GO:0006811 <br>GO.name: ion transport","modules/clusters: C8 <br>GO.ID: GO:0051234 <br>GO.name: establishment of localization","modules/clusters: C8 <br>GO.ID: GO:0009967 <br>GO.name: positive regulation of signal transduction","modules/clusters: C8 <br>GO.ID: GO:0042592 <br>GO.name: homeostatic process","modules/clusters: C8 <br>GO.ID: GO:0006810 <br>GO.name: transport","modules/clusters: C8 <br>GO.ID: GO:0007399 <br>GO.name: nervous system development","modules/clusters: C8 <br>GO.ID: GO:0006816 <br>GO.name: calcium ion transport","modules/clusters: C8 <br>GO.ID: GO:0006812 <br>GO.name: cation transport","modules/clusters: C8 <br>GO.ID: GO:0044057 <br>GO.name: regulation of system process","modules/clusters: C8 <br>GO.ID: GO:0051674 <br>GO.name: localization of cell","modules/clusters: C8 <br>GO.ID: GO:0048870 <br>GO.name: cell motility","modules/clusters: C8 <br>GO.ID: GO:0023061 <br>GO.name: signal release","modules/clusters: C8 <br>GO.ID: GO:0001944 <br>GO.name: vasculature development","modules/clusters: C8 <br>GO.ID: GO:0034220 <br>GO.name: ion transmembrane transport","modules/clusters: C8 <br>GO.ID: GO:1902533 <br>GO.name: positive regulation of intracellular signal transduction","modules/clusters: C8 <br>GO.ID: GO:0055085 <br>GO.name: transmembrane transport","modules/clusters: C8 <br>GO.ID: GO:0032940 <br>GO.name: secretion by cell","modules/clusters: C8 <br>GO.ID: GO:0140352 <br>GO.name: export from cell","modules/clusters: C8 <br>GO.ID: GO:0001568 <br>GO.name: blood vessel development","modules/clusters: C8 <br>GO.ID: GO:0097553 <br>GO.name: calcium ion transmembrane import into cytosol","modules/clusters: C8 <br>GO.ID: GO:0071702 <br>GO.name: organic substance transport","modules/clusters: C8 <br>GO.ID: GO:0051049 <br>GO.name: regulation of transport","modules/clusters: C8 <br>GO.ID: GO:0070588 <br>GO.name: calcium ion transmembrane transport","modules/clusters: C8 <br>GO.ID: GO:0048878 <br>GO.name: chemical homeostasis","modules/clusters: C8 <br>GO.ID: GO:0045944 <br>GO.name: positive regulation of transcription by RNA polymerase II","modules/clusters: C8 <br>GO.ID: GO:0098655 <br>GO.name: cation transmembrane transport","modules/clusters: C8 <br>GO.ID: GO:0050673 <br>GO.name: epithelial cell proliferation","modules/clusters: C8 <br>GO.ID: GO:1903530 <br>GO.name: regulation of secretion by cell","modules/clusters: C8 <br>GO.ID: GO:0030001 <br>GO.name: metal ion transport","modules/clusters: C8 <br>GO.ID: GO:1903508 <br>GO.name: positive regulation of nucleic acid-templated transcription","modules/clusters: C8 <br>GO.ID: GO:0045893 <br>GO.name: positive regulation of transcription, DNA-templated","modules/clusters: C8 <br>GO.ID: GO:0019722 <br>GO.name: calcium-mediated signaling","modules/clusters: C8 <br>GO.ID: GO:1902680 <br>GO.name: positive regulation of RNA biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0048608 <br>GO.name: reproductive structure development","modules/clusters: C8 <br>GO.ID: GO:1903706 <br>GO.name: regulation of hemopoiesis","modules/clusters: C8 <br>GO.ID: GO:0046903 <br>GO.name: secretion","modules/clusters: C8 <br>GO.ID: GO:0003006 <br>GO.name: developmental process involved in reproduction","modules/clusters: C8 <br>GO.ID: GO:0061458 <br>GO.name: reproductive system development","modules/clusters: C8 <br>GO.ID: GO:0098660 <br>GO.name: inorganic ion transmembrane transport","modules/clusters: C8 <br>GO.ID: GO:0051050 <br>GO.name: positive regulation of transport","modules/clusters: C8 <br>GO.ID: GO:0001505 <br>GO.name: regulation of neurotransmitter levels","modules/clusters: C8 <br>GO.ID: GO:0022414 <br>GO.name: reproductive process","modules/clusters: C8 <br>GO.ID: GO:0009058 <br>GO.name: biosynthetic process","modules/clusters: C8 <br>GO.ID: GO:0000003 <br>GO.name: reproduction"],"showlegend":true,"marker":{"color":"rgba(217,240,185,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(217,240,185,1)"}},"name":"C8","textfont":{"color":"rgba(217,240,185,1)"},"error_y":{"color":"rgba(217,240,185,1)"},"error_x":{"color":"rgba(217,240,185,1)"},"line":{"color":"rgba(217,240,185,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0792084967574061,0.0892929855204291],"y":[-0.30869800443111,-0.313956695084867],"type":"scatter","mode":"markers","text":["modules/clusters: C8/C15/C16 <br>GO.ID: GO:0051240 <br>GO.name: positive regulation of multicellular organismal process","modules/clusters: C8/C15/C16 <br>GO.ID: GO:0022603 <br>GO.name: regulation of anatomical structure morphogenesis"],"showlegend":true,"marker":{"color":"rgba(200,224,167,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(200,224,167,1)"}},"name":"C8/C15/C16","textfont":{"color":"rgba(200,224,167,1)"},"error_y":{"color":"rgba(200,224,167,1)"},"error_x":{"color":"rgba(200,224,167,1)"},"line":{"color":"rgba(200,224,167,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.132824911141335,-0.170500059554873,0.118573333353124,0.120286709295484,-0.0876452852100369,-0.0832839996213921,0.406427304352203,-0.253300886182446,-0.0283920825947017],"y":[0.0461957168149383,-0.144258502664902,0.103841586510993,0.0910085755705956,-0.0820614805886005,-0.060523860431459,-0.042339816618652,0.100810648773749,-0.270852311265135],"type":"scatter","mode":"markers","text":["modules/clusters: C8/C16 <br>GO.ID: GO:0040011 <br>GO.name: locomotion","modules/clusters: C8/C16 <br>GO.ID: GO:0048584 <br>GO.name: positive regulation of response to stimulus","modules/clusters: C8/C16 <br>GO.ID: GO:0006935 <br>GO.name: chemotaxis","modules/clusters: C8/C16 <br>GO.ID: GO:0042330 <br>GO.name: taxis","modules/clusters: C8/C16 <br>GO.ID: GO:0050727 <br>GO.name: regulation of inflammatory response","modules/clusters: C8/C16 <br>GO.ID: GO:0045595 <br>GO.name: regulation of cell differentiation","modules/clusters: C8/C16 <br>GO.ID: GO:0022008 <br>GO.name: neurogenesis","modules/clusters: C8/C16 <br>GO.ID: GO:0051128 <br>GO.name: regulation of cellular component organization","modules/clusters: C8/C16 <br>GO.ID: GO:0051093 <br>GO.name: negative regulation of developmental process"],"showlegend":true,"marker":{"color":"rgba(210,227,165,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(210,227,165,1)"}},"name":"C8/C16","textfont":{"color":"rgba(210,227,165,1)"},"error_y":{"color":"rgba(210,227,165,1)"},"error_x":{"color":"rgba(210,227,165,1)"},"line":{"color":"rgba(210,227,165,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.0503461607810053],"y":[0.294453248362341],"type":"scatter","mode":"markers","text":"modules/clusters: C8/C16/C20 <br>GO.ID: GO:0006928 <br>GO.name: movement of cell or subcellular component","showlegend":true,"marker":{"color":"rgba(189,193,171,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(189,193,171,1)"}},"name":"C8/C16/C20","textfont":{"color":"rgba(189,193,171,1)"},"error_y":{"color":"rgba(189,193,171,1)"},"error_x":{"color":"rgba(189,193,171,1)"},"line":{"color":"rgba(189,193,171,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.184223560452992],"y":[0.0135750985886947],"type":"scatter","mode":"markers","text":"modules/clusters: C8/C16/C22 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","showlegend":true,"marker":{"color":"rgba(230,177,139,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(230,177,139,1)"}},"name":"C8/C16/C22","textfont":{"color":"rgba(230,177,139,1)"},"error_y":{"color":"rgba(230,177,139,1)"},"error_x":{"color":"rgba(230,177,139,1)"},"line":{"color":"rgba(230,177,139,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.129551984350097],"y":[0.0515364446381537],"type":"scatter","mode":"markers","text":"modules/clusters: C8/C19 <br>GO.ID: GO:0051179 <br>GO.name: localization","showlegend":true,"marker":{"color":"rgba(197,231,145,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(197,231,145,1)"}},"name":"C8/C19","textfont":{"color":"rgba(197,231,145,1)"},"error_y":{"color":"rgba(197,231,145,1)"},"error_x":{"color":"rgba(197,231,145,1)"},"line":{"color":"rgba(197,231,145,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.100192664058176],"y":[-0.314902496284349],"type":"scatter","mode":"markers","text":"modules/clusters: C8/C20 <br>GO.ID: GO:0001819 <br>GO.name: positive regulation of cytokine production","showlegend":true,"marker":{"color":"rgba(193,200,181,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(193,200,181,1)"}},"name":"C8/C20","textfont":{"color":"rgba(193,200,181,1)"},"error_y":{"color":"rgba(193,200,181,1)"},"error_x":{"color":"rgba(193,200,181,1)"},"line":{"color":"rgba(193,200,181,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.122539628834414,0.11957503985214],"y":[-0.335531700454454,0.0935939258459763],"type":"scatter","mode":"markers","text":["modules/clusters: C8/C20/C22 <br>GO.ID: GO:0001817 <br>GO.name: regulation of cytokine production","modules/clusters: C8/C20/C22 <br>GO.ID: GO:0006952 <br>GO.name: defense response"],"showlegend":true,"marker":{"color":"rgba(222,163,147,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(222,163,147,1)"}},"name":"C8/C20/C22","textfont":{"color":"rgba(222,163,147,1)"},"error_y":{"color":"rgba(222,163,147,1)"},"error_x":{"color":"rgba(222,163,147,1)"},"line":{"color":"rgba(222,163,147,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.215661519261532],"y":[0.111782340679299],"type":"scatter","mode":"markers","text":"modules/clusters: C8/C21 <br>GO.ID: GO:0007169 <br>GO.name: transmembrane receptor protein tyrosine kinase signaling pathway","showlegend":true,"marker":{"color":"rgba(213,209,160,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(213,209,160,1)"}},"name":"C8/C21","textfont":{"color":"rgba(213,209,160,1)"},"error_y":{"color":"rgba(213,209,160,1)"},"error_x":{"color":"rgba(213,209,160,1)"},"line":{"color":"rgba(213,209,160,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[0.13953154441155,-0.163317834814451,0.0478988342852049],"y":[-0.169207419299678,-0.236331366851597,0.34332672348818],"type":"scatter","mode":"markers","text":["modules/clusters: C8/C22 <br>GO.ID: GO:0001816 <br>GO.name: cytokine production","modules/clusters: C8/C22 <br>GO.ID: GO:0002682 <br>GO.name: regulation of immune system process","modules/clusters: C8/C22 <br>GO.ID: GO:0009987 <br>GO.name: cellular process"],"showlegend":true,"marker":{"color":"rgba(234,184,149,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(234,184,149,1)"}},"name":"C8/C22","textfont":{"color":"rgba(234,184,149,1)"},"error_y":{"color":"rgba(234,184,149,1)"},"error_x":{"color":"rgba(234,184,149,1)"},"line":{"color":"rgba(234,184,149,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[-0.0629392322572786],"y":[-0.112744935630772],"type":"scatter","mode":"markers","text":"modules/clusters: C9 <br>GO.ID: GO:0090066 <br>GO.name: regulation of anatomical structure size","showlegend":true,"marker":{"color":"rgba(253,180,98,1)","size":20,"opacity":0.4,"showlegend":true,"line":{"color":"rgba(253,180,98,1)"}},"name":"C9","textfont":{"color":"rgba(253,180,98,1)"},"error_y":{"color":"rgba(253,180,98,1)"},"error_x":{"color":"rgba(253,180,98,1)"},"line":{"color":"rgba(253,180,98,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script><p>A second version of the MDS plot provides a nearest ancestor
clustering approach. All terms found are brought up to their nearest
common ancestor. Since these databases, GO ontology in particular, have
a tree-like hierarchy, we are able to find common ancestors for each
term which may provide a broader view if too many individual terms
exist.<br>
This approach does not distinguish based on clusters in which terms were
found, meaning that terms from different clusters may be paired
together.<br>
The size of each dot in this approach is relative to the number of terms
sorted in that common ancestor (dot) while the color is representative
of the dominant cluster in that common ancestor. For example, if an
ancestor has 4 terms within it, and 3 of those terms are found in
cluster 1 (C1), the dot will carry the color associated to C1.<br>
Hovering the cursor over the dots will provide more information on those
specific common ancestors.</p>
<div id="htmlwidget-810457af6bb5544841d2" style="width:100%;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-810457af6bb5544841d2">{"x":{"visdat":{"18ed1131f240":["function () ","plotlyVisDat"],"18ed14357c52f":["function () ","data"]},"cur_data":"18ed14357c52f","attrs":{"18ed14357c52f":{"alpha_stroke":1,"sizes":[20,50],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","text":{},"showlegend":false,"size":{},"fill":{},"marker":{"sizemode":"diameter","opacity":0.4,"line":{"color":["#D9F0B9","#E3D2BB","#E4C264","#A9B2AD","#CBD892","#E4C264","#E4C264","#E4C264","#CBD892","#CBD892","#E4C264","#D9F0B9","#D9F0B9","#E4C264","#CBD892","#D9F0B9","#FB8072","#8DD3C7","#E9E8C0","#FB8072","#FB8072","#B3E1C0","#8DD3C7","#E9E8C0","#D2A6B7","#8DD3C7","#8DD3C7","#D3D1CC","#D3D1CC","#8DD3C7","#D3D1CC","#8DD3C7","#D9F0B9","#CBD892","#FB8072","#E4C264","#E4C264","#E4C264","#E9E8C0","#8DD3C7","#CBD892","#8DD3C7","#A9B2AD","#E9E8C0","#8DD3C7","#CBD892","#8DD3C7","#CBD892","#CBD892","#D9F0B9","#8DD3C7","#CBD892","#CBD892","#8DD3C7","#D9F0B9","#CBD892","#E4C264","#CBD892","#8DD3C7","#D9F0B9","#B3E1C0","#FB8072","#A9A0B2","#CBD892","#CBD892","#8DD3C7","#8DD3C7","#8DD3C7"]},"color":["#D9F0B9","#E3D2BB","#E4C264","#A9B2AD","#CBD892","#E4C264","#E4C264","#E4C264","#CBD892","#CBD892","#E4C264","#D9F0B9","#D9F0B9","#E4C264","#CBD892","#D9F0B9","#FB8072","#8DD3C7","#E9E8C0","#FB8072","#FB8072","#B3E1C0","#8DD3C7","#E9E8C0","#D2A6B7","#8DD3C7","#8DD3C7","#D3D1CC","#D3D1CC","#8DD3C7","#D3D1CC","#8DD3C7","#D9F0B9","#CBD892","#FB8072","#E4C264","#E4C264","#E4C264","#E9E8C0","#8DD3C7","#CBD892","#8DD3C7","#A9B2AD","#E9E8C0","#8DD3C7","#CBD892","#8DD3C7","#CBD892","#CBD892","#D9F0B9","#8DD3C7","#CBD892","#CBD892","#8DD3C7","#D9F0B9","#CBD892","#E4C264","#CBD892","#8DD3C7","#D9F0B9","#B3E1C0","#FB8072","#A9A0B2","#CBD892","#CBD892","#8DD3C7","#8DD3C7","#8DD3C7"]},"visible":true,"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"fillcolor":"rgba(31,119,180,0.5)","x":[0.286789178326006,0.104526370532248,0.0937767546682622,-0.22081892651097,-0.201986756066996,0.105132095113934,0.197708017462432,0.096139148282214,0.371376727940432,0.277638286999661,0.0976038609025613,-0.0939580851577052,0.377792927183267,0.108167068148993,0.088227421962737,-0.186862874689644,-0.11044833232797,-0.293299662337802,-0.266311956854813,0.129850683046444,0.205963047208247,0.256781448719858,0.357276334636364,-0.00164210862877502,0.0507196924427458,0.339142270760648,0.232139799291009,0.0472917128361047,-0.0192125576347392,-0.143685104568347,0.0278490106100034,-0.0152088851702079,0.10517854309885,0.209345395970778,0.20236537771955,-0.19450687958103,-0.132632877648093,-0.23384978527797,-0.196504241997444,0.0439772166423698,0.305533398271703,0.22873850183446,-0.298506719537745,-0.316615023055144,-0.236203107719792,0.215384504039766,-0.373840065382059,-0.312881144034335,-0.272033815279799,0.111089094591887,-0.31618112774814,0.308393591014397,0.332231787607095,-0.0444766325519532,-0.258276955216356,0.163329300910179,-0.314355512137411,-0.0737666392448559,-0.172334118681862,-0.211120848241536,-0.271973569217302,0.0240137130454008,0.194759585170939,-0.171578556679385,-0.13715904668858,-0.259868748091539,-0.0893794281699119,0.14524822513867],"y":[0.114768572525301,-0.426092646367572,-0.437695473867821,0.00208584227727457,-0.048554505212745,-0.382141678735926,-0.319685629155026,-0.414322790816547,0.105425866856077,-0.163779259585862,-0.359014110354456,0.162120454376265,0.129477017971796,-0.261966329351819,-0.202447366614392,0.241765310264693,0.129849116644511,0.0637463494476838,-0.129496923143906,0.260632804885979,0.157988075463754,0.0254794542343288,0.0364754343988806,0.263874275386169,0.0647557940347619,0.107055619407106,0.148211662704399,0.151520070138374,-0.0117159057532262,0.0212269671983463,0.127467939246076,-0.109889549745759,-0.0558259591254321,0.199301883240008,0.231137902874338,-0.295197760029078,-0.356697702382667,-0.269890584615791,-0.135475224086189,-0.148999673546173,-0.160313608340954,0.234205245429218,0.214368626155979,0.228992041528435,-0.0121771675900625,0.200322263855947,-0.0303214917097066,-0.106877856901209,-0.108620178037078,-0.173474547074054,-0.00671848802073742,0.119933079600192,-0.114780771959358,0.196559290992402,0.171220336936322,0.121670354240277,-0.144103669028188,0.258764735861401,0.28830419215068,0.0937995282379243,0.202143259691583,-0.073317819492397,0.209267138429989,0.216670108978773,-0.0776375904942733,-0.0599873737125597,0.155050604027442,-0.0584175848417148],"type":"scatter","mode":"markers","text":["GO.cluster: 11 <br>GO.count: 4  –  C8: 4 <br>GO.ID: GO:0000003 <br>GO.name: reproduction","GO.cluster: 48 <br>GO.count: 9  –  C13: 8, C14: 1 <br>GO.ID: GO:0000226 <br>GO.name: microtubule cytoskeleton organization","GO.cluster: 51 <br>GO.count: 3  –  C14: 2, C13: 1 <br>GO.ID: GO:0140014 <br>GO.name: mitotic nuclear division","GO.cluster: 19 <br>GO.count: 15  –  C5: 6, C8: 4, C2: 3, C15: 2 <br>GO.ID: GO:0006139 <br>GO.name: nucleobase-containing compound metabolic process","GO.cluster: 44 <br>GO.count: 12  –  C16: 4, C1: 2, C22: 2, C8: 2, C2: 1, C21: 1 <br>GO.ID: GO:0007165 <br>GO.name: signal transduction","GO.cluster: 50 <br>GO.count: 5  –  C14: 3, C13: 2 <br>GO.ID: GO:0007049 <br>GO.name: cell cycle","GO.cluster: 54 <br>GO.count: 12  –  C14: 5, C16: 4, C22: 2, C21: 1 <br>GO.ID: GO:0016043 <br>GO.name: cellular component organization","GO.cluster: 52 <br>GO.count: 5  –  C14: 4, C11: 1 <br>GO.ID: GO:0007059 <br>GO.name: chromosome segregation","GO.cluster: 10 <br>GO.count: 4  –  C16: 2, C2: 1, C8: 1 <br>GO.ID: GO:0048856 <br>GO.name: anatomical structure development","GO.cluster: 6 <br>GO.count: 4  –  C16: 2, C1: 1, C8: 1 <br>GO.ID: GO:0048869 <br>GO.name: cellular developmental process","GO.cluster: 49 <br>GO.count: 6  –  C14: 6 <br>GO.ID: GO:0000910 <br>GO.name: cytokinesis","GO.cluster: 26 <br>GO.count: 6  –  C8: 4, C1: 1, C9: 1 <br>GO.ID: GO:0065007 <br>GO.name: biological regulation","GO.cluster: 2 <br>GO.count: 12  –  C8: 6, C1: 4, C22: 1, C6: 1 <br>GO.ID: GO:0007275 <br>GO.name: multicellular organism development","GO.cluster: 56 <br>GO.count: 11  –  C14: 3, C8: 3, C1: 1, C15: 1, C16: 1, C22: 1, C6: 1 <br>GO.ID: GO:0009987 <br>GO.name: cellular process","GO.cluster: 35 <br>GO.count: 4  –  C16: 3, C1: 1 <br>GO.ID: GO:0008219 <br>GO.name: cell death","GO.cluster: 16 <br>GO.count: 9  –  C8: 4, C1: 1, C20: 1, C22: 1, C6: 1, C7: 1 <br>GO.ID: GO:0043170 <br>GO.name: macromolecule metabolic process","GO.cluster: 66 <br>GO.count: 9  –  C22: 7, C2: 1, C8: 1 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","GO.cluster: 20 <br>GO.count: 10  –  C1: 10 <br>GO.ID: GO:0051246 <br>GO.name: regulation of protein metabolic process","GO.cluster: 39 <br>GO.count: 9  –  C2: 6, C15: 2, C16: 1 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process","GO.cluster: 67 <br>GO.count: 4  –  C22: 3, C20: 1 <br>GO.ID: GO:0006955 <br>GO.name: immune response","GO.cluster: 68 <br>GO.count: 4  –  C22: 3, C8: 1 <br>GO.ID: GO:0002376 <br>GO.name: immune system process","GO.cluster: 63 <br>GO.count: 10  –  C15: 9, C22: 1 <br>GO.ID: GO:0002263 <br>GO.name: cell activation involved in immune response","GO.cluster: 4 <br>GO.count: 11  –  C1: 7, C15: 3, C22: 1 <br>GO.ID: GO:0002520 <br>GO.name: immune system development","GO.cluster: 65 <br>GO.count: 11  –  C2: 7, C22: 4 <br>GO.ID: GO:0002252 <br>GO.name: immune effector process","GO.cluster: 3 <br>GO.count: 8  –  C3: 3, C1: 2, C22: 2, C8: 1 <br>GO.ID: GO:1903706 <br>GO.name: regulation of hemopoiesis","GO.cluster: 1 <br>GO.count: 6  –  C1: 6 <br>GO.ID: GO:0007507 <br>GO.name: heart development","GO.cluster: 33 <br>GO.count: 2  –  C1: 1, C8: 1 <br>GO.ID: GO:0032501 <br>GO.name: multicellular organismal process","GO.cluster: 15 <br>GO.count: 9  –  C12: 8, C8: 1 <br>GO.ID: GO:0008152 <br>GO.name: metabolic process","GO.cluster: 13 <br>GO.count: 12  –  C12: 10, C15: 1, C17: 1 <br>GO.ID: GO:0071704 <br>GO.name: organic substance metabolic process","GO.cluster: 21 <br>GO.count: 7  –  C1: 6, C7: 1 <br>GO.ID: GO:0036211 <br>GO.name: protein modification process","GO.cluster: 14 <br>GO.count: 6  –  C12: 6 <br>GO.ID: GO:0008202 <br>GO.name: steroid metabolic process","GO.cluster: 22 <br>GO.count: 3  –  C1: 3 <br>GO.ID: GO:0006793 <br>GO.name: phosphorus metabolic process","GO.cluster: 53 <br>GO.count: 18  –  C8: 17, C19: 1 <br>GO.ID: GO:0051179 <br>GO.name: localization","GO.cluster: 61 <br>GO.count: 2  –  C16: 2 <br>GO.ID: GO:0042330 <br>GO.name: taxis","GO.cluster: 59 <br>GO.count: 4  –  C22: 2, C1: 1, C8: 1 <br>GO.ID: GO:0006950 <br>GO.name: response to stress","GO.cluster: 47 <br>GO.count: 8  –  C14: 7, C16: 1 <br>GO.ID: GO:0051128 <br>GO.name: regulation of cellular component organization","GO.cluster: 46 <br>GO.count: 13  –  C14: 13 <br>GO.ID: GO:0051983 <br>GO.name: regulation of chromosome segregation","GO.cluster: 38 <br>GO.count: 12  –  C14: 12 <br>GO.ID: GO:0051726 <br>GO.name: regulation of cell cycle","GO.cluster: 37 <br>GO.count: 7  –  C2: 3, C1: 2, C16: 1, C8: 1 <br>GO.ID: GO:0007155 <br>GO.name: cell adhesion","GO.cluster: 43 <br>GO.count: 5  –  C1: 5 <br>GO.ID: GO:0007267 <br>GO.name: cell-cell signaling","GO.cluster: 8 <br>GO.count: 8  –  C16: 8 <br>GO.ID: GO:0032989 <br>GO.name: cellular component morphogenesis","GO.cluster: 62 <br>GO.count: 6  –  C1: 3, C16: 1, C20: 1, C8: 1 <br>GO.ID: GO:0050896 <br>GO.name: response to stimulus","GO.cluster: 18 <br>GO.count: 5  –  C5: 4, C1: 1 <br>GO.ID: GO:0009892 <br>GO.name: negative regulation of metabolic process","GO.cluster: 17 <br>GO.count: 6  –  C2: 3, C8: 2, C1: 1 <br>GO.ID: GO:0019222 <br>GO.name: regulation of metabolic process","GO.cluster: 45 <br>GO.count: 11  –  C1: 6, C15: 3, C8: 2 <br>GO.ID: GO:0009966 <br>GO.name: regulation of signal transduction","GO.cluster: 57 <br>GO.count: 6  –  C16: 4, C1: 1, C22: 1 <br>GO.ID: GO:0042221 <br>GO.name: response to chemical","GO.cluster: 24 <br>GO.count: 7  –  C1: 3, C2: 2, C8: 2 <br>GO.ID: GO:0031323 <br>GO.name: regulation of cellular metabolic process","GO.cluster: 40 <br>GO.count: 3  –  C16: 2, C1: 1 <br>GO.ID: GO:0010646 <br>GO.name: regulation of cell communication","GO.cluster: 36 <br>GO.count: 8  –  C16: 8 <br>GO.ID: GO:0010941 <br>GO.name: regulation of cell death","GO.cluster: 55 <br>GO.count: 3  –  C8: 2, C1: 1 <br>GO.ID: GO:0051674 <br>GO.name: localization of cell","GO.cluster: 23 <br>GO.count: 6  –  C1: 6 <br>GO.ID: GO:0019220 <br>GO.name: regulation of phosphate metabolic process","GO.cluster: 9 <br>GO.count: 2  –  C16: 1, C5: 1 <br>GO.ID: GO:0032502 <br>GO.name: developmental process","GO.cluster: 7 <br>GO.count: 6  –  C16: 5, C8: 1 <br>GO.ID: GO:0022008 <br>GO.name: neurogenesis","GO.cluster: 32 <br>GO.count: 5  –  C1: 3, C16: 1, C8: 1 <br>GO.ID: GO:0050793 <br>GO.name: regulation of developmental process","GO.cluster: 29 <br>GO.count: 3  –  C8: 2, C15: 1 <br>GO.ID: GO:0023051 <br>GO.name: regulation of signaling","GO.cluster: 12 <br>GO.count: 3  –  C16: 1, C22: 1, C8: 1 <br>GO.ID: GO:0008150 <br>GO.name: biological <br>GO.name: process","GO.cluster: 41 <br>GO.count: 6  –  C14: 3, C16: 1, C22: 1, C8: 1 <br>GO.ID: GO:0050794 <br>GO.name: regulation of cellular process","GO.cluster: 27 <br>GO.count: 3  –  C16: 1, C2: 1, C8: 1 <br>GO.ID: GO:0031347 <br>GO.name: regulation of defense response","GO.cluster: 28 <br>GO.count: 4  –  C1: 2, C16: 1, C8: 1 <br>GO.ID: GO:0048583 <br>GO.name: regulation of response to stimulus","GO.cluster: 30 <br>GO.count: 4  –  C8: 3, C1: 1 <br>GO.ID: GO:0032879 <br>GO.name: regulation of localization","GO.cluster: 31 <br>GO.count: 4  –  C15: 1, C16: 1, C22: 1, C8: 1 <br>GO.ID: GO:0050789 <br>GO.name: regulation of biological process","GO.cluster: 64 <br>GO.count: 10  –  C22: 7, C2: 3 <br>GO.ID: GO:0045321 <br>GO.name: leukocyte activation","GO.cluster: 60 <br>GO.count: 3  –  C20: 2, C22: 1 <br>GO.ID: GO:0043207 <br>GO.name: response to external biotic stimulus","GO.cluster: 34 <br>GO.count: 4  –  C16: 3, C8: 1 <br>GO.ID: GO:0051239 <br>GO.name: regulation of multicellular organismal process","GO.cluster: 5 <br>GO.count: 3  –  C16: 2, C1: 1 <br>GO.ID: GO:0045595 <br>GO.name: regulation of cell differentiation","GO.cluster: 42 <br>GO.count: 5  –  C1: 5 <br>GO.ID: GO:0099177 <br>GO.name: regulation of trans-synaptic signaling","GO.cluster: 25 <br>GO.count: 4  –  C1: 3, C15: 1 <br>GO.ID: GO:0065009 <br>GO.name: regulation of molecular function","GO.cluster: 58 <br>GO.count: 6  –  C1: 3, C16: 3 <br>GO.ID: GO:0051716 <br>GO.name: cellular response to stimulus"],"showlegend":false,"fill":"","marker":{"color":["#D9F0B9","#E3D2BB","#E4C264","#A9B2AD","#CBD892","#E4C264","#E4C264","#E4C264","#CBD892","#CBD892","#E4C264","#D9F0B9","#D9F0B9","#E4C264","#CBD892","#D9F0B9","#FB8072","#8DD3C7","#E9E8C0","#FB8072","#FB8072","#B3E1C0","#8DD3C7","#E9E8C0","#D2A6B7","#8DD3C7","#8DD3C7","#D3D1CC","#D3D1CC","#8DD3C7","#D3D1CC","#8DD3C7","#D9F0B9","#CBD892","#FB8072","#E4C264","#E4C264","#E4C264","#E9E8C0","#8DD3C7","#CBD892","#8DD3C7","#A9B2AD","#E9E8C0","#8DD3C7","#CBD892","#8DD3C7","#CBD892","#CBD892","#D9F0B9","#8DD3C7","#CBD892","#CBD892","#8DD3C7","#D9F0B9","#CBD892","#E4C264","#CBD892","#8DD3C7","#D9F0B9","#B3E1C0","#FB8072","#A9A0B2","#CBD892","#CBD892","#8DD3C7","#8DD3C7","#8DD3C7"],"size":[23.75,33.125,21.875,44.375,38.75,25.625,38.75,25.625,23.75,23.75,27.5,27.5,38.75,36.875,23.75,33.125,33.125,35,33.125,23.75,23.75,35,36.875,36.875,31.25,27.5,20,33.125,38.75,29.375,27.5,21.875,50,20,23.75,31.25,40.625,38.75,29.375,25.625,31.25,27.5,25.625,27.5,36.875,27.5,29.375,21.875,31.25,21.875,27.5,20,27.5,25.625,21.875,21.875,27.5,21.875,23.75,23.75,23.75,35,21.875,23.75,21.875,25.625,23.75,27.5],"sizemode":"diameter","opacity":0.4,"line":{"color":["#D9F0B9","#E3D2BB","#E4C264","#A9B2AD","#CBD892","#E4C264","#E4C264","#E4C264","#CBD892","#CBD892","#E4C264","#D9F0B9","#D9F0B9","#E4C264","#CBD892","#D9F0B9","#FB8072","#8DD3C7","#E9E8C0","#FB8072","#FB8072","#B3E1C0","#8DD3C7","#E9E8C0","#D2A6B7","#8DD3C7","#8DD3C7","#D3D1CC","#D3D1CC","#8DD3C7","#D3D1CC","#8DD3C7","#D9F0B9","#CBD892","#FB8072","#E4C264","#E4C264","#E4C264","#E9E8C0","#8DD3C7","#CBD892","#8DD3C7","#A9B2AD","#E9E8C0","#8DD3C7","#CBD892","#8DD3C7","#CBD892","#CBD892","#D9F0B9","#8DD3C7","#CBD892","#CBD892","#8DD3C7","#D9F0B9","#CBD892","#E4C264","#CBD892","#8DD3C7","#D9F0B9","#B3E1C0","#FB8072","#A9A0B2","#CBD892","#CBD892","#8DD3C7","#8DD3C7","#8DD3C7"]}},"visible":true,"textfont":{"size":[23.75,33.125,21.875,44.375,38.75,25.625,38.75,25.625,23.75,23.75,27.5,27.5,38.75,36.875,23.75,33.125,33.125,35,33.125,23.75,23.75,35,36.875,36.875,31.25,27.5,20,33.125,38.75,29.375,27.5,21.875,50,20,23.75,31.25,40.625,38.75,29.375,25.625,31.25,27.5,25.625,27.5,36.875,27.5,29.375,21.875,31.25,21.875,27.5,20,27.5,25.625,21.875,21.875,27.5,21.875,23.75,23.75,23.75,35,21.875,23.75,21.875,25.625,23.75,27.5]},"error_y":{"color":"rgba(31,119,180,1)","width":[]},"error_x":{"color":"rgba(31,119,180,1)","width":[]},"line":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 id="gprofiler-results-custom">Gprofiler results – custom<a class="anchor" aria-label="anchor" href="#gprofiler-results-custom"></a>
</h3>
<p>Dotplots were designed to illustrate the top n term results for any
single ontology. In the below code chunk three different ontologies are
demonstrated, GO:BP (biological processes), REAC (REACTOME) <span class="citation">(Jassal et al. 2020)</span> and KEGG<span class="citation">(Kanehisa et al. 2017)</span>. The ‘target_top’
parameter defines the number of terms (from most significant to least)
per cluster that will be shown on the dotplot.</p>
<p>All terms shown are considered to be significant, the ‘top’ aspect
reflects significance based on the adjusted p value. The adjusted p
value is negatively log transformed in order to offer a better
visualization, with most significant being in red and less significant
(but still statistically significant) in blue.</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">target_top</span><span class="op">=</span><span class="fl">10</span>

<span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'GO:BP'</span>
<span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span>gpro_file_location<span class="op">=</span><span class="va">name_result_folder</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span><span class="op">)</span>

<span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'REAC'</span>
<span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span>gpro_file_location<span class="op">=</span><span class="va">name_result_folder</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span><span class="op">)</span>

<span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'KEGG'</span>
<span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span>gpro_file_location<span class="op">=</span><span class="va">name_result_folder</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span><span class="op">)</span>

<span class="va">select_ontology</span><span class="op">&lt;-</span><span class="st">'HP'</span>
<span class="fu"><a href="../reference/GO_dotplot_wrapper.html">GO_dotplot_wrapper</a></span><span class="op">(</span>gpro_file_location<span class="op">=</span><span class="va">name_result_folder</span>,target_ontology<span class="op">=</span><span class="va">select_ontology</span>,top_n<span class="op">=</span><span class="va">target_top</span><span class="op">)</span></code></pre></div>
<p>Illustrated below is the dotplot for the REACTOME pathway if it
exists, otherwise it will show <a href="GO:BP" class="uri">GO:BP</a> or
KEGG dotplots if those have been created. All dotplots created above can
be found in the ‘TS_results/gprofiler_results’ folder.<br><img src="TS_results_murine_example/gprofiler_results/REAC_dotplot.svg"><!-- --></p>
</div>
<div class="section level3">
<h3 id="ancestor-query-plots">Ancestor query plots<a class="anchor" aria-label="anchor" href="#ancestor-query-plots"></a>
</h3>
<p>Ancestor query plots were designed to query results relating to
specific GOs.</p>
<p>This plot searches for children of the queried term IDs. Pathways,
such as GO pathways are hierarchically organized, so a single ID will be
associated to other IDs. For example, the ‘immune system process
(GO:0002376)’ contains several ‘children’, each of these children is
likely to have children of their own and so on. If we use the ID for
‘immune system process’ this plot will find all children grand-children
great-grand children etc… of ‘immune system process’. It will then split
the children based on the PART cluster in which they were found, these
results are illustrated in dotplot format. This is accomplished via the
GO.db package<span class="citation">(Carlson 2021)</span></p>
<p>Several websites exist to search for GO ancestors of interest,
‘AMIGO2’ is one and ‘quickGO’ is another.</p>
<p>A word of caution with this type of plot. It will always link terms
to the most upstream ancestor, so if the query contains two term IDs
which are related to each other, the plot will illustrate all results as
belonging to the term ID which is the most upstream (oldest/ancestor).
Each ancestor queried for will be illustrated in different colors on the
dotplot.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">#Dotplot for terms relating to specific ancestors</span>
<span class="co">#Check if analysis is required, if not, set to null or empty</span>
<span class="kw">if</span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="va">target_ancestors</span><span class="op">)</span><span class="op">==</span><span class="fl">0</span><span class="op">)</span><span class="op">{</span>
  <span class="va">ancestor_plots</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span>
  <span class="va">ancestor_plots</span><span class="op">[[</span><span class="st">'MDS'</span><span class="op">]</span><span class="op">]</span><span class="op">&lt;-</span><span class="cn">NULL</span>
  <span class="va">GOs_ancestors_clust</span><span class="op">&lt;-</span><span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="cn">NULL</span><span class="op">)</span>
<span class="op">}</span><span class="kw">else</span><span class="op">{</span>
  <span class="va">GOs_ancestors_clust</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/find_relation_to_ancestors.html">find_relation_to_ancestors</a></span><span class="op">(</span><span class="va">target_ancestors</span>,<span class="va">GO_clusters</span>,ontology <span class="op">=</span> <span class="va">ancestor_ontology</span><span class="op">)</span>
  <span class="va">ancestor_plots</span><span class="op">&lt;-</span><span class="fu"><a href="../reference/wrapper_ancestor_curation_plots.html">wrapper_ancestor_curation_plots</a></span><span class="op">(</span><span class="va">GOs_ancestors_clust</span>,<span class="va">sem_data</span>,return_plot<span class="op">=</span><span class="cn">T</span>,target_dir<span class="op">=</span><span class="va">name_result_folder</span><span class="op">)</span>
<span class="op">}</span></code></pre></div>
<p>Two formats are generated for the ancestor plot. The first is a MDS
plot. The second is a dotplot version. If more than one term was found
to be associated to ancestors, the MDS plot will be created and can be
seen below, otherwise only the dotplot will be created. The dotplot will
not be created if no terms were found. The files for both of these plots
can be found in ‘TS_results/ancestor_plots’. In the MDS and dotplot
alike, the color of each term represents the ancestor to which they have
been associated and the size is proportional to the -log transformed
adjusted p-value. In the MDS plot, the PART clusters in which these
terms are found can be seen by hovering over each dot, in the dotplot
version, this information is made available on the x axis while the y
axis represents the terms.</p>
<div id="htmlwidget-4b3bfa672f162a3f818c" style="width:700px;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4b3bfa672f162a3f818c">{"x":{"visdat":{"18ed161eda0ce":["function () ","plotlyVisDat"],"18ed145cc9787":["function () ","data"]},"cur_data":"18ed145cc9787","attrs":{"18ed145cc9787":{"alpha_stroke":1,"sizes":[20,50],"spans":[1,20],"x":{},"y":{},"type":"scatter","mode":"markers","color":{},"text":{},"showlegend":true,"colors":["#1F78B4","#33A02C","#A6CEE3","#FB9A99","#B2DF8A"],"size":{},"fill":{},"marker":{"sizemode":"diameter","opacity":0.4,"showlegend":true},"inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"xaxis":{"domain":[0,1],"automargin":true,"title":"Dim.1"},"yaxis":{"domain":[0,1],"automargin":true,"title":"Dim.2"},"hovermode":"closest","showlegend":true},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"fillcolor":"rgba(31,120,180,0.5)","x":[-0.237732907131186,-0.273201555437032,-0.213036475644209,-0.130853225840381,-0.130853225840381,-0.114980224315059,-0.10800699977343,0.236188928664277,0.224305230293497,0.220725709722651,0.200194124326162,-0.1668531167422,-0.375336086437744,-0.407804343893811,-0.401385995964133,-0.316854230714925,-0.35119841844471,-0.420579684681082,-0.291327208954473,-0.288952829971552,-0.335043199837359,-0.355143195965872,-0.349424228058913,-0.386502668236394,-0.378332477407601,-0.304025408633232,-0.400494736180438,-0.380535624267534,-0.374772794273295,0.225026671024029,0.127884024742626,-0.430745274394557,-0.413500055851108],"y":[-0.0491573150846418,0.141764233835405,-0.306563102932348,-0.453431567077781,-0.453431567077781,-0.494060300908794,-0.483913593239404,-0.264092889687882,-0.260473348414009,-0.251012710637617,-0.184047317486313,-0.380685552129503,0.0812069235689273,0.0696415328488505,0.0587107583828201,0.00405343807877199,0.0502719195453358,0.0122078411655152,0.150434435379494,0.197684323097888,0.246100878910983,0.195613414360579,0.239152210619181,0.220220790447301,0.259683342908825,0.236513505812403,0.238705376929255,0.27016791438774,0.0816282760722716,-0.250095351263614,-0.214779169792358,0.200148550682744,0.231873629536926],"type":"scatter","mode":"markers","text":["Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0001909 <br>GO.name: leukocyte mediated cytotoxicity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0001910 <br>GO.name: regulation of leukocyte mediated cytotoxicity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0002263 <br>GO.name: cell activation involved in immune response <br>-log10(padj): 1.2210220240036","Ancestor: immune effector process <br>cluster(s): C22/C15 <br>GO.ID: GO:0002285 <br>GO.name: lymphocyte activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22/C15 <br>GO.ID: GO:0002285 <br>GO.name: lymphocyte activation involved in immune response <br>-log10(padj): 1.10424916818194","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0002286 <br>GO.name: T cell activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0002287 <br>GO.name: alpha-beta T cell activation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0002292 <br>GO.name: T cell differentiation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0002293 <br>GO.name: alpha-beta T cell differentiation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0002294 <br>GO.name: CD4-positive, alpha-beta T cell differentiation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0002313 <br>GO.name: mature B cell differentiation involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0002366 <br>GO.name: leukocyte activation involved in immune response <br>-log10(padj): 1.25128933735145","Ancestor: immune effector process <br>cluster(s): C2 <br>GO.ID: GO:0002369 <br>GO.name: T cell cytokine production <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002381 <br>GO.name: immunoglobulin production involved in immunoglobulin-mediated immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002426 <br>GO.name: immunoglobulin production in mucosal tissue <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002443 <br>GO.name: leukocyte mediated immunity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002449 <br>GO.name: lymphocyte mediated immunity <br>-log10(padj): 0.68916363175821","Ancestor: immune effector process <br>cluster(s): C2 <br>GO.ID: GO:0002456 <br>GO.name: T cell mediated immunity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002697 <br>GO.name: regulation of immune effector process <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C2 <br>GO.ID: GO:0002699 <br>GO.name: positive regulation of immune effector process <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C2 <br>GO.ID: GO:0002702 <br>GO.name: positive regulation of production of molecular mediator of immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002703 <br>GO.name: regulation of leukocyte mediated immunity <br>-log10(padj): 0.205543600930617","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002705 <br>GO.name: positive regulation of leukocyte mediated immunity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002706 <br>GO.name: regulation of lymphocyte mediated immunity <br>-log10(padj): 1.32841219442135","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:0002708 <br>GO.name: positive regulation of lymphocyte mediated immunity <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C2 <br>GO.ID: GO:0002720 <br>GO.name: positive regulation of cytokine production involved in immune response <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C2 <br>GO.ID: GO:0002724 <br>GO.name: regulation of T cell cytokine production <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C2 <br>GO.ID: GO:0002726 <br>GO.name: positive regulation of T cell cytokine production <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C2 <br>GO.ID: GO:0035743 <br>GO.name: CD4-positive, alpha-beta T cell cytokine production <br>-log10(padj): 0.118529762148053","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0042093 <br>GO.name: T-helper cell differentiation <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C15 <br>GO.ID: GO:0072539 <br>GO.name: T-helper 17 cell differentiation <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:2000557 <br>GO.name: regulation of immunoglobulin production in mucosal tissue <br>-log10(padj): 0","Ancestor: immune effector process <br>cluster(s): C22 <br>GO.ID: GO:2000558 <br>GO.name: positive regulation of immunoglobulin production in mucosal tissue <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(31,120,180,1)","size":[20,20,47.5665589448836,20,44.9302217209118,20,20,20,20,20,20,48.2498927923527,20,20,20,20,35.5589903409283,20,20,20,20,24.6404812357265,20,49.9910667790792,20,20,20,20,22.6760022429929,20,20,20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(31,120,180,1)"}},"name":"immune effector process","textfont":{"color":"rgba(31,120,180,1)","size":[20,20,47.5665589448836,20,44.9302217209118,20,20,20,20,20,20,48.2498927923527,20,20,20,20,35.5589903409283,20,20,20,20,24.6404812357265,20,49.9910667790792,20,20,20,20,22.6760022429929,20,20,20,20]},"error_y":{"color":"rgba(31,120,180,1)","width":[]},"error_x":{"color":"rgba(31,120,180,1)","width":[]},"line":{"color":"rgba(31,120,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(51,160,44,0.5)","x":[-0.260853545556054,-0.283367881504091],"y":[-0.113335763373606,-0.075767073550755],"type":"scatter","mode":"markers","text":["Ancestor: immune response <br>cluster(s): C22 <br>GO.ID: GO:0002250 <br>GO.name: adaptive immune response <br>-log10(padj): 0","Ancestor: immune response <br>cluster(s): C22 <br>GO.ID: GO:0002460 <br>GO.name: adaptive immune response based on somatic recombination of immune receptors built from immunoglobulin superfamily domains <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(51,160,44,1)","size":[20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(51,160,44,1)"}},"name":"immune response","textfont":{"color":"rgba(51,160,44,1)","size":20},"error_y":{"color":"rgba(51,160,44,1)","width":20},"error_x":{"color":"rgba(51,160,44,1)","width":20},"line":{"color":"rgba(51,160,44,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(166,206,227,0.5)","x":[0.348578351115982,0.336991575462149,0.428897895998253,0.251446700920168,0.378525273000995,0.378525273000995,0.379128007647404,0.353466399687841,0.344209169047645,0.300365402345158,0.240258929949446,0.325912623484794,0.354683515045192,0.354683515045192,0.354683515045192,0.354683515045192,0.320922629503057,0.320922629503057,0.284071965955011,0.284071965955011,0.424269773921601,0.424269773921601,0.265185881922138,0.237324650834534,0.237324650834534],"y":[-0.090272022302955,-0.0691180085944213,0.121176463259689,0.306067196900644,0.178517389593013,0.178517389593013,-0.145269090174502,-0.168096491339402,-0.16570060449373,0.0105873804929328,0.0585545742017286,0.059247846689662,0.160685437797322,0.160685437797322,0.160685437797322,0.160685437797322,0.266202126218805,0.266202126218805,0.297682517474515,0.297682517474515,0.111925127626639,0.111925127626639,0.325971421079593,0.354887554561896,0.354887554561896],"type":"scatter","mode":"markers","text":["Ancestor: immune system development <br>cluster(s): C1 <br>GO.ID: GO:0002327 <br>GO.name: immature B cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1 <br>GO.ID: GO:0002329 <br>GO.name: pre-B cell differentiation <br>-log10(padj): 0.443723849642603","Ancestor: immune system development <br>cluster(s): C1 <br>GO.ID: GO:0002521 <br>GO.name: leukocyte differentiation <br>-log10(padj): 1.32880787889947","Ancestor: immune system development <br>cluster(s): C1 <br>GO.ID: GO:0002763 <br>GO.name: positive regulation of myeloid leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C8 <br>GO.ID: GO:0030097 <br>GO.name: hemopoiesis <br>-log10(padj): 0.252223475907468","Ancestor: immune system development <br>cluster(s): C1/C8 <br>GO.ID: GO:0030097 <br>GO.name: hemopoiesis <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1 <br>GO.ID: GO:0030098 <br>GO.name: lymphocyte differentiation <br>-log10(padj): 0.232254005356791","Ancestor: immune system development <br>cluster(s): C22 <br>GO.ID: GO:0042492 <br>GO.name: gamma-delta T cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C15 <br>GO.ID: GO:0043367 <br>GO.name: CD4-positive, alpha-beta T cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C22 <br>GO.ID: GO:0045586 <br>GO.name: regulation of gamma-delta T cell differentiation <br>-log10(padj): 0.25217431546233","Ancestor: immune system development <br>cluster(s): C22 <br>GO.ID: GO:0045588 <br>GO.name: positive regulation of gamma-delta T cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C3 <br>GO.ID: GO:0045619 <br>GO.name: regulation of lymphocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C15/C16/C8 <br>GO.ID: GO:0048534 <br>GO.name: hematopoietic or lymphoid organ development <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C15/C16/C8 <br>GO.ID: GO:0048534 <br>GO.name: hematopoietic or lymphoid organ development <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C15/C16/C8 <br>GO.ID: GO:0048534 <br>GO.name: hematopoietic or lymphoid organ development <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C15/C16/C8 <br>GO.ID: GO:0048534 <br>GO.name: hematopoietic or lymphoid organ development <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C3/C1 <br>GO.ID: GO:1902105 <br>GO.name: regulation of leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C3/C1 <br>GO.ID: GO:1902105 <br>GO.name: regulation of leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C3/C1 <br>GO.ID: GO:1902107 <br>GO.name: positive regulation of leukocyte differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C3/C1 <br>GO.ID: GO:1902107 <br>GO.name: positive regulation of leukocyte differentiation <br>-log10(padj): 0.193406863372282","Ancestor: immune system development <br>cluster(s): C1/C15 <br>GO.ID: GO:1903131 <br>GO.name: mononuclear cell differentiation <br>-log10(padj): 0.740442666096355","Ancestor: immune system development <br>cluster(s): C1/C15 <br>GO.ID: GO:1903131 <br>GO.name: mononuclear cell differentiation <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C8 <br>GO.ID: GO:1903706 <br>GO.name: regulation of hemopoiesis <br>-log10(padj): 0","Ancestor: immune system development <br>cluster(s): C1/C3 <br>GO.ID: GO:1903708 <br>GO.name: positive regulation of hemopoiesis <br>-log10(padj): 0.193406863372282","Ancestor: immune system development <br>cluster(s): C1/C3 <br>GO.ID: GO:1903708 <br>GO.name: positive regulation of hemopoiesis <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(166,206,227,1)","size":[20,30.0177878989572,50,20,25.6943553672265,20,25.2435120767604,20,20,25.6932454901874,20,20,20,20,20,20,20,20,20,24.3664746373824,36.7166979784075,20,20,24.3664746373824,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(166,206,227,1)"}},"name":"immune system development","textfont":{"color":"rgba(166,206,227,1)","size":[20,30.0177878989572,50,20,25.6943553672265,20,25.2435120767604,20,20,25.6932454901874,20,20,20,20,20,20,20,20,20,24.3664746373824,36.7166979784075,20,20,24.3664746373824,20]},"error_y":{"color":"rgba(166,206,227,1)","width":[]},"error_x":{"color":"rgba(166,206,227,1)","width":[]},"line":{"color":"rgba(166,206,227,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(251,154,153,0.5)","x":[0.00346231856319203,-0.00605767528980874,0.0011114768836942],"y":[-0.457343031884783,-0.424310868856624,-0.426151359845177],"type":"scatter","mode":"markers","text":["Ancestor: leukocyte activation <br>cluster(s): C22 <br>GO.ID: GO:0042110 <br>GO.name: T cell activation <br>-log10(padj): 0","Ancestor: leukocyte activation <br>cluster(s): C22 <br>GO.ID: GO:0046629 <br>GO.name: gamma-delta T cell activation <br>-log10(padj): 0","Ancestor: leukocyte activation <br>cluster(s): C22 <br>GO.ID: GO:0046649 <br>GO.name: lymphocyte activation <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(251,154,153,1)","size":[20,20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(251,154,153,1)"}},"name":"leukocyte activation","textfont":{"color":"rgba(251,154,153,1)","size":20},"error_y":{"color":"rgba(251,154,153,1)","width":20},"error_x":{"color":"rgba(251,154,153,1)","width":20},"line":{"color":"rgba(251,154,153,1)"},"xaxis":"x","yaxis":"y","frame":null},{"fillcolor":"rgba(178,223,138,0.5)","x":[-0.288489047406079,-0.069140420193871,-0.0464032982210367,-0.0548334115129307,-0.0737520892939308,-0.0464511306106198,-0.0554773759312363],"y":[0.0795805446066341,-0.122550329135354,-0.241125520271476,-0.160378715727132,-0.0948905712773527,-0.250351726089821,-0.157734943291986],"type":"scatter","mode":"markers","text":["Ancestor: regulation of immune system process <br>cluster(s): C22 <br>GO.ID: GO:0002819 <br>GO.name: regulation of adaptive immune response <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C2 <br>GO.ID: GO:0042102 <br>GO.name: positive regulation of T cell proliferation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C22 <br>GO.ID: GO:0046643 <br>GO.name: regulation of gamma-delta T cell activation <br>-log10(padj): 0.25217431546233","Ancestor: regulation of immune system process <br>cluster(s): C22 <br>GO.ID: GO:0046645 <br>GO.name: positive regulation of gamma-delta T cell activation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C2 <br>GO.ID: GO:0050671 <br>GO.name: positive regulation of lymphocyte proliferation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C22 <br>GO.ID: GO:0050863 <br>GO.name: regulation of T cell activation <br>-log10(padj): 0","Ancestor: regulation of immune system process <br>cluster(s): C2 <br>GO.ID: GO:0050870 <br>GO.name: positive regulation of T cell activation <br>-log10(padj): 0"],"showlegend":true,"fill":"","marker":{"color":"rgba(178,223,138,1)","size":[20,20,25.6932454901874,20,20,20,20],"sizemode":"diameter","opacity":0.4,"showlegend":true,"line":{"color":"rgba(178,223,138,1)"}},"name":"regulation of immune system process","textfont":{"color":"rgba(178,223,138,1)","size":[20,20,25.6932454901874,20,20,20,20]},"error_y":{"color":"rgba(178,223,138,1)","width":[]},"error_x":{"color":"rgba(178,223,138,1)","width":[]},"line":{"color":"rgba(178,223,138,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 class="tabset unlisted unnumbered" id="section"></h3>
<p>A interactive table is also generated in order to better navigate the
results in the event of a large number of terms found for many different
ancestors. The csv version of this data can be found in
‘TS_results/ancestor_plots’.</p>
<div class="section level4">
<h4 id="go0002252">GO:0002252<a class="anchor" aria-label="anchor" href="#go0002252"></a>
</h4>
<p>Ancestor name: immune effector process</p>
<table class="table">
<colgroup>
<col width="11%">
<col width="75%">
<col width="12%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0001909</td>
<td align="left">leukocyte mediated cytotoxicity</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0001910</td>
<td align="left">regulation of leukocyte mediated cytotoxicity</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002263</td>
<td align="left">cell activation involved in immune response</td>
<td align="left">C15</td>
</tr>
<tr class="even">
<td align="left">GO:0002285</td>
<td align="left">lymphocyte activation involved in immune response</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002285</td>
<td align="left">lymphocyte activation involved in immune response</td>
<td align="left">C15</td>
</tr>
<tr class="even">
<td align="left">GO:0002286</td>
<td align="left">T cell activation involved in immune response</td>
<td align="left">C15</td>
</tr>
<tr class="odd">
<td align="left">GO:0002287</td>
<td align="left">alpha-beta T cell activation involved in immune
response</td>
<td align="left">C15</td>
</tr>
<tr class="even">
<td align="left">GO:0002292</td>
<td align="left">T cell differentiation involved in immune response</td>
<td align="left">C15</td>
</tr>
<tr class="odd">
<td align="left">GO:0002293</td>
<td align="left">alpha-beta T cell differentiation involved in immune
response</td>
<td align="left">C15</td>
</tr>
<tr class="even">
<td align="left">GO:0002294</td>
<td align="left">CD4-positive, alpha-beta T cell differentiation
involved in immune response</td>
<td align="left">C15</td>
</tr>
<tr class="odd">
<td align="left">GO:0002313</td>
<td align="left">mature B cell differentiation involved in immune
response</td>
<td align="left">C15</td>
</tr>
<tr class="even">
<td align="left">GO:0002366</td>
<td align="left">leukocyte activation involved in immune response</td>
<td align="left">C15</td>
</tr>
<tr class="odd">
<td align="left">GO:0002369</td>
<td align="left">T cell cytokine production</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0002381</td>
<td align="left">immunoglobulin production involved in
immunoglobulin-mediated immune response</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002426</td>
<td align="left">immunoglobulin production in mucosal tissue</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002443</td>
<td align="left">leukocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002449</td>
<td align="left">lymphocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002456</td>
<td align="left">T cell mediated immunity</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0002697</td>
<td align="left">regulation of immune effector process</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002699</td>
<td align="left">positive regulation of immune effector process</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0002702</td>
<td align="left">positive regulation of production of molecular mediator
of immune response</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0002703</td>
<td align="left">regulation of leukocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002705</td>
<td align="left">positive regulation of leukocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002706</td>
<td align="left">regulation of lymphocyte mediated immunity</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0002708</td>
<td align="left">positive regulation of lymphocyte mediated
immunity</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002720</td>
<td align="left">positive regulation of cytokine production involved in
immune response</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0002724</td>
<td align="left">regulation of T cell cytokine production</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0002726</td>
<td align="left">positive regulation of T cell cytokine production</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0035743</td>
<td align="left">CD4-positive, alpha-beta T cell cytokine
production</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0042093</td>
<td align="left">T-helper cell differentiation</td>
<td align="left">C15</td>
</tr>
<tr class="odd">
<td align="left">GO:0072539</td>
<td align="left">T-helper 17 cell differentiation</td>
<td align="left">C15</td>
</tr>
<tr class="even">
<td align="left">GO:2000557</td>
<td align="left">regulation of immunoglobulin production in mucosal
tissue</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:2000558</td>
<td align="left">positive regulation of immunoglobulin production in
mucosal tissue</td>
<td align="left">C22</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002520">GO:0002520<a class="anchor" aria-label="anchor" href="#go0002520"></a>
</h4>
<p>Ancestor name: immune system development</p>
<table class="table">
<colgroup>
<col width="14%">
<col width="69%">
<col width="15%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002327</td>
<td align="left">immature B cell differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0002329</td>
<td align="left">pre-B cell differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0002521</td>
<td align="left">leukocyte differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0002763</td>
<td align="left">positive regulation of myeloid leukocyte
differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:0030097</td>
<td align="left">hemopoiesis</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0030097</td>
<td align="left">hemopoiesis</td>
<td align="left">C8</td>
</tr>
<tr class="odd">
<td align="left">GO:0030098</td>
<td align="left">lymphocyte differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0042492</td>
<td align="left">gamma-delta T cell differentiation</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0043367</td>
<td align="left">CD4-positive, alpha-beta T cell differentiation</td>
<td align="left">C15</td>
</tr>
<tr class="even">
<td align="left">GO:0045586</td>
<td align="left">regulation of gamma-delta T cell differentiation</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0045588</td>
<td align="left">positive regulation of gamma-delta T cell
differentiation</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0045619</td>
<td align="left">regulation of lymphocyte differentiation</td>
<td align="left">C3</td>
</tr>
<tr class="odd">
<td align="left">GO:0048534</td>
<td align="left">hematopoietic or lymphoid organ development</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:0048534</td>
<td align="left">hematopoietic or lymphoid organ development</td>
<td align="left">C15</td>
</tr>
<tr class="odd">
<td align="left">GO:0048534</td>
<td align="left">hematopoietic or lymphoid organ development</td>
<td align="left">C16</td>
</tr>
<tr class="even">
<td align="left">GO:0048534</td>
<td align="left">hematopoietic or lymphoid organ development</td>
<td align="left">C8</td>
</tr>
<tr class="odd">
<td align="left">GO:1902105</td>
<td align="left">regulation of leukocyte differentiation</td>
<td align="left">C3</td>
</tr>
<tr class="even">
<td align="left">GO:1902105</td>
<td align="left">regulation of leukocyte differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:1902107</td>
<td align="left">positive regulation of leukocyte differentiation</td>
<td align="left">C3</td>
</tr>
<tr class="even">
<td align="left">GO:1902107</td>
<td align="left">positive regulation of leukocyte differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:1903131</td>
<td align="left">mononuclear cell differentiation</td>
<td align="left">C1</td>
</tr>
<tr class="even">
<td align="left">GO:1903131</td>
<td align="left">mononuclear cell differentiation</td>
<td align="left">C15</td>
</tr>
<tr class="odd">
<td align="left">GO:1903706</td>
<td align="left">regulation of hemopoiesis</td>
<td align="left">C8</td>
</tr>
<tr class="even">
<td align="left">GO:1903708</td>
<td align="left">positive regulation of hemopoiesis</td>
<td align="left">C1</td>
</tr>
<tr class="odd">
<td align="left">GO:1903708</td>
<td align="left">positive regulation of hemopoiesis</td>
<td align="left">C3</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0002682">GO:0002682<a class="anchor" aria-label="anchor" href="#go0002682"></a>
</h4>
<p>Ancestor name: regulation of immune system process</p>
<table class="table">
<colgroup>
<col width="15%">
<col width="67%">
<col width="16%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002819</td>
<td align="left">regulation of adaptive immune response</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0042102</td>
<td align="left">positive regulation of T cell proliferation</td>
<td align="left">C2</td>
</tr>
<tr class="odd">
<td align="left">GO:0046643</td>
<td align="left">regulation of gamma-delta T cell activation</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0046645</td>
<td align="left">positive regulation of gamma-delta T cell
activation</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0050671</td>
<td align="left">positive regulation of lymphocyte proliferation</td>
<td align="left">C2</td>
</tr>
<tr class="even">
<td align="left">GO:0050863</td>
<td align="left">regulation of T cell activation</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0050870</td>
<td align="left">positive regulation of T cell activation</td>
<td align="left">C2</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0006955">GO:0006955<a class="anchor" aria-label="anchor" href="#go0006955"></a>
</h4>
<p>Ancestor name: immune response</p>
<table class="table">
<colgroup>
<col width="8%">
<col width="82%">
<col width="8%">
</colgroup>
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0002250</td>
<td align="left">adaptive immune response</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0002460</td>
<td align="left">adaptive immune response based on somatic recombination
of immune receptors built from immunoglobulin superfamily domains</td>
<td align="left">C22</td>
</tr>
</tbody>
</table>
</div>
<div class="section level4">
<h4 id="go0045321">GO:0045321<a class="anchor" aria-label="anchor" href="#go0045321"></a>
</h4>
<p>Ancestor name: leukocyte activation</p>
<table class="table">
<thead><tr class="header">
<th align="left">term ID</th>
<th align="left">term name</th>
<th align="left">PART cluster</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">GO:0042110</td>
<td align="left">T cell activation</td>
<td align="left">C22</td>
</tr>
<tr class="even">
<td align="left">GO:0046629</td>
<td align="left">gamma-delta T cell activation</td>
<td align="left">C22</td>
</tr>
<tr class="odd">
<td align="left">GO:0046649</td>
<td align="left">lymphocyte activation</td>
<td align="left">C22</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level3">
<h3 class="unlisted unnumbered" id="section-1"></h3>
</div>
</div>
<div class="section level2">
<h2 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-brionne2019viseago" class="csl-entry">
Brionne, Aurélien, Amélie Juanchich, and Christelle Hennequet-Antier.
2019. <span>“ViSEAGO: A Bioconductor Package for Clustering Biological
Functions Using Gene Ontology and Semantic Similarity.”</span>
<em>BioData Mining</em> 12 (1): 1–13.
</div>
<div id="ref-GOdb_ref" class="csl-entry">
Carlson, Marc. 2021. <em>GO.db: A Set of Annotation Maps Describing the
Entire Gene Ontology</em>.
</div>
<div id="ref-gu2016complex" class="csl-entry">
Gu, Zuguang, Roland Eils, and Matthias Schlesner. 2016. <span>“Complex
Heatmaps Reveal Patterns and Correlations in Multidimensional Genomic
Data.”</span> <em>Bioinformatics</em> 32 (18): 2847–49.
</div>
<div id="ref-jassal2020reactome" class="csl-entry">
Jassal, Bijay, Lisa Matthews, Guilherme Viteri, Chuqiao Gong, Pascual
Lorente, Antonio Fabregat, Konstantinos Sidiropoulos, et al. 2020.
<span>“The Reactome Pathway Knowledgebase.”</span> <em>Nucleic Acids
Research</em> 48 (D1): D498–503.
</div>
<div id="ref-kanehisa2017kegg" class="csl-entry">
Kanehisa, Minoru, Miho Furumichi, Mao Tanabe, Yoko Sato, and Kanae
Morishima. 2017. <span>“KEGG: New Perspectives on Genomes, Pathways,
Diseases and Drugs.”</span> <em>Nucleic Acids Research</em> 45 (D1):
D353–61.
</div>
<div id="ref-kolberg2020gprofiler2" class="csl-entry">
Kolberg, Liis, Uku Raudvere, Ivan Kuzmin, Jaak Vilo, and Hedi Peterson.
2020. <span>“Gprofiler2–an r Package for Gene List Functional Enrichment
Analysis and Namespace Conversion Toolset g: Profiler.”</span>
<em>F1000Research</em> 9.
</div>
<div id="ref-love2014moderated" class="csl-entry">
Love, Michael I, Wolfgang Huber, and Simon Anders. 2014.
<span>“Moderated Estimation of Fold Change and Dispersion for RNA-Seq
Data with DESeq2.”</span> <em>Genome Biology</em> 15 (12): 1–21.
</div>
<div id="ref-timeseriesexperiment" class="csl-entry">
Nguyen, Lan Huong. 2021. <em>TimeSeriesExperiment: Analysis for Short
Time-Series Data</em>. <a href="https://github.com/nlhuong/TimeSeriesExperiment" class="external-link">https://github.com/nlhuong/TimeSeriesExperiment</a>.
</div>
<div id="ref-nilsen2013identifying" class="csl-entry">
Nilsen, Gro, Ørnulf Borgan, Knut LiestØl, and Ole Christian Lingjærde.
2013. <span>“Identifying Clusters in Genomics Data by Recursive
Partitioning.”</span> <em>Statistical Applications in Genetics and
Molecular Biology</em> 12 (5): 637–52.
</div>
<div id="ref-raudvere2019g" class="csl-entry">
Raudvere, Uku, Liis Kolberg, Ivan Kuzmin, Tambet Arak, Priit Adler, Hedi
Peterson, and Jaak Vilo. 2019. <span>“G: Profiler: A Web Server for
Functional Enrichment Analysis and Conversions of Gene Lists (2019
Update).”</span> <em>Nucleic Acids Research</em> 47 (W1): W191–98.
</div>
<div id="ref-smyth2005limma" class="csl-entry">
Smyth, Gordon K. 2005. <span>“Limma: Linear Models for Microarray
Data.”</span> In <em>Bioinformatics and Computational Biology Solutions
Using r and Bioconductor</em>, 397–420. Springer.
</div>
<div id="ref-wang2007new" class="csl-entry">
Wang, James Z, Zhidian Du, Rapeeporn Payattakool, Philip S Yu, and
Chin-Fu Chen. 2007. <span>“A New Method to Measure the Semantic
Similarity of GO Terms.”</span> <em>Bioinformatics</em> 23 (10):
1274–81.
</div>
<div id="ref-yu2010gosemsim" class="csl-entry">
Yu, Guangchuang, Fei Li, Yide Qin, Xiaochen Bo, Yibo Wu, and Shengqi
Wang. 2010. <span>“GOSemSim: An r Package for Measuring Semantic
Similarity Among GO Terms and Gene Products.”</span>
<em>Bioinformatics</em> 26 (7): 976–78.
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Yohan Lefol.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.4.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
